  <script type="text/javascript" src="https://cdn.jsdelivr.net/npm/vega@5"></script>
  <script type="text/javascript" src="https://cdn.jsdelivr.net/npm/vega-lite@5.8.0"></script>
  <script type="text/javascript" src="https://cdn.jsdelivr.net/npm/vega-embed@6"></script>
  <div id="vis"></div>
  <script>
    (function(vegaEmbed) {
      var spec = {"config": {"view": {"continuousWidth": 300, "continuousHeight": 300}}, "hconcat": [{"data": {"name": "data-c34668d3738df7555619c5d5c00ce7fe"}, "mark": {"type": "circle", "opacity": 0.8, "size": 12}, "encoding": {"color": {"field": "tool", "legend": null, "scale": {"domain": ["hostile", "ncbi-scrubber", "kraken2-standard", "kraken2-standard8"]}, "type": "nominal"}, "tooltip": [{"field": "tool", "type": "ordinal"}, {"field": "sample", "type": "nominal"}, {"field": "name", "type": "nominal"}, {"field": "reads_removed", "type": "quantitative"}, {"field": "reads_removed_percent", "type": "quantitative"}], "x": {"axis": {"labelAngle": -45}, "field": "tool", "sort": ["hostile", "kraken2-standard", "kraken2-standard8", "ncbi-scrubber"], "title": null, "type": "nominal"}, "xOffset": {"field": "jitter", "type": "quantitative"}, "y": {"field": "reads_removed_percent", "scale": {"domain": [0, 0.22]}, "title": "Reads removed (%)", "type": "quantitative"}}, "height": 250, "title": "Removed simulated bacterial reads (lower is better)", "transform": [{"calculate": "sqrt(-2*log(random()))*cos(2*PI*random())", "as": "jitter"}], "width": 280}, {"data": {"name": "data-a50967b9c09b98afdf78ec5230ac7f78"}, "mark": {"type": "circle", "opacity": 0.8, "size": 18}, "encoding": {"color": {"field": "tool", "legend": null, "scale": {"domain": ["hostile", "ncbi-scrubber", "kraken2-standard", "kraken2-standard8"]}, "sort": ["hostile", "ncbi-scrubber", "kraken2-standard", "kraken2-standard8"], "type": "nominal"}, "tooltip": [{"field": "tool", "type": "nominal"}, {"field": "sample", "type": "nominal"}, {"field": "reads_removed", "type": "quantitative"}, {"field": "reads_removed_percent", "type": "quantitative"}], "x": {"axis": {"labelAngle": -45}, "field": "tool", "sort": ["kraken2-standard", "hostile", "ncbi-scrubber", "kraken2-standard8"], "title": null, "type": "nominal"}, "xOffset": {"field": "jitter", "type": "quantitative"}, "y": {"field": "reads_removed_percent", "scale": {"domain": [97, 100]}, "title": "Reads removed (%)", "type": "quantitative"}}, "height": 250, "title": "Removed real human reads (higher is better)", "transform": [{"calculate": "sqrt(-2*log(random()))*cos(2*PI*random())", "as": "jitter"}], "width": 280}], "$schema": "https://vega.github.io/schema/vega-lite/v5.8.0.json", "datasets": {"data-c34668d3738df7555619c5d5c00ce7fe": [{"sample": "GCA_016127955.1", "platform": "illumina", "tool": "hostile", "reads_removed": 56, "name": "Actinomyces oris strain FDAARGOS_1051 chromosome, complete genome", "reads_in": 212692, "reads_removed_percent": 0.0263291520132398, "reads_retained_percent": 99.97367084798677}, {"sample": "GCA_016889225.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Corynebacterium glucuronolyticum strain FDAARGOS_1190 chromosome, complete genome", "reads_in": 188214, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889885.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Streptococcus constellatus strain FDAARGOS_1208 chromosome, complete genome", "reads_in": 131912, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889925.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Bacteroides fragilis strain FDAARGOS_1225 chromosome, complete genome", "reads_in": 349930, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889945.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus aureus strain FDAARGOS_1245 chromosome, complete genome", "reads_in": 192584, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889965.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus condimenti strain FDAARGOS_1205 chromosome, complete genome", "reads_in": 180552, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889985.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Corynebacterium sp. FDAARGOS 1242 strain FDAARGOS_1242 chromosome, complete genome", "reads_in": 185000, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016890005.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1250 chromosome, complete genome", "reads_in": 337234, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016890025.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus carnosus strain FDAARGOS_1204 chromosome, complete genome", "reads_in": 176972, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016890045.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1265 chromosome, complete genome", "reads_in": 361686, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016890065.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Acinetobacter pittii strain FDAARGOS_1216 chromosome, complete genome", "reads_in": 261924, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016890085.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Chromobacterium violaceum strain FDAARGOS_1273 chromosome, complete genome", "reads_in": 320536, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016890105.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1266 chromosome, complete genome", "reads_in": 347734, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889465.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Corynebacterium macginleyi strain FDAARGOS_1195 chromosome, complete genome", "reads_in": 166236, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903095.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Bordetella pertussis strain FDAARGOS_1368 chromosome, complete genome", "reads_in": 273672, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903135.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Acinetobacter baumannii strain FDAARGOS_1360 chromosome, complete genome", "reads_in": 271340, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903155.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1326 chromosome, complete genome", "reads_in": 368106, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016728625.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Corynebacterium propinquum strain FDAARGOS_1113 chromosome, complete genome", "reads_in": 164086, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016728665.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Corynebacterium propinquum strain FDAARGOS_1112 chromosome, complete genome", "reads_in": 165342, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903195.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1305 chromosome, complete genome", "reads_in": 384390, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016728685.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Corynebacterium aurimucosum strain FDAARGOS_1110 chromosome, complete genome", "reads_in": 188632, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016727905.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Yersinia enterocolitica strain FDAARGOS_1090 chromosome, complete genome", "reads_in": 306216, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016727765.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Yersinia enterocolitica strain FDAARGOS_1082 chromosome, complete genome", "reads_in": 302752, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889005.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Clostridium] scindens strain FDAARGOS_1227 chromosome, complete genome", "reads_in": 241274, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889025.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1251 chromosome, complete genome", "reads_in": 349158, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889045.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1255 chromosome, complete genome", "reads_in": 337566, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889065.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Butyricimonas virosa strain FDAARGOS_1229 chromosome, complete genome", "reads_in": 320876, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903115.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1327 chromosome, complete genome", "reads_in": 366464, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016728205.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Corynebacterium striatum strain FDAARGOS_1116 chromosome, complete genome", "reads_in": 177712, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016728765.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Chryseobacterium sp. FDAARGOS 1104 strain FDAARGOS_1104 chromosome, complete genome", "reads_in": 246632, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016728365.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Corynebacterium tuberculostearicum strain FDAARGOS_1117 chromosome, complete genome", "reads_in": 163456, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016725325.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Sphingobacterium spiritivorum strain FDAARGOS_1145 chromosome, complete genome", "reads_in": 342354, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016128255.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Lactococcus garvieae strain FDAARGOS_1063 chromosome, complete genome", "reads_in": 147538, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016128275.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Citrobacter koseri strain FDAARGOS_1029 chromosome, complete genome", "reads_in": 315772, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016403145.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Agrobacterium tumefaciens strain FDAARGOS_1048 chromosome 1, complete sequence", "reads_in": 395512, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016598875.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Burkholderia glumae strain FDAARGOS_921 chromosome 1, complete sequence", "reads_in": 456378, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016599755.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Corynebacterium kefirresidentii strain FDAARGOS_1055 chromosome, complete genome", "reads_in": 168214, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016599795.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus pasteuri strain FDAARGOS_1069 chromosome, complete genome", "reads_in": 165654, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016599815.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Anaerococcus obesiensis strain FDAARGOS_989 chromosome, complete genome", "reads_in": 126308, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016599835.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Elizabethkingia bruuniana strain FDAARGOS_1031 chromosome, complete genome", "reads_in": 308800, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016724565.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus condimenti strain FDAARGOS_1148 chromosome, complete genome", "reads_in": 177956, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016724665.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus carnosus strain FDAARGOS_1147 chromosome, complete genome", "reads_in": 179336, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016724785.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Fusobacterium canifelinum strain FDAARGOS_1126 chromosome, complete genome", "reads_in": 156814, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016724805.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Sutterella wadsworthensis strain FDAARGOS_1159 chromosome, complete genome", "reads_in": 201768, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016724825.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Weizmannia coagulans strain FDAARGOS_1160 chromosome, complete genome", "reads_in": 222856, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016724845.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Sphingobacterium spiritivorum strain FDAARGOS_1146 chromosome, complete genome", "reads_in": 351684, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016724865.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Peribacillus psychrosaccharolyticus strain FDAARGOS_1161 chromosome, complete genome", "reads_in": 312086, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016724885.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Streptococcus dysgalactiae strain FDAARGOS_1157 chromosome, complete genome", "reads_in": 143412, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016725005.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Streptococcus constellatus strain FDAARGOS_1156 chromosome, complete genome", "reads_in": 126884, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016128035.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Alcaligenes faecalis strain FDAARGOS_1024 chromosome, complete genome", "reads_in": 271528, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127995.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Schaalia meyeri strain FDAARGOS_985 chromosome, complete genome", "reads_in": 135536, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020099215.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Enterobacter ludwigii strain FDAARGOS_1498 chromosome, complete genome", "reads_in": 315380, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016728645.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Corynebacterium glucuronolyticum strain FDAARGOS_1111 chromosome, complete genome", "reads_in": 196126, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016888885.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Moraxella osloensis strain FDAARGOS_1202 chromosome, complete genome", "reads_in": 183946, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016888905.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Parabacteroides distasonis strain FDAARGOS_1234 chromosome, complete genome", "reads_in": 320802, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016888925.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1260 chromosome, complete genome", "reads_in": 345352, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016728505.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Corynebacterium riegelii strain FDAARGOS_1114 chromosome, complete genome", "reads_in": 167948, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016888945.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Phocaeicola coprophilus strain FDAARGOS_1220 chromosome, complete genome", "reads_in": 274240, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889085.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1256 chromosome, complete genome", "reads_in": 346170, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889125.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1249 chromosome, complete genome", "reads_in": 348824, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016728045.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Serratia ureilytica strain FDAARGOS_1089 chromosome, complete genome", "reads_in": 346696, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889165.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus epidermidis strain FDAARGOS_1243 chromosome, complete genome", "reads_in": 168996, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127695.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1059 chromosome, complete genome", "reads_in": 361050, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127715.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus saccharolyticus strain FDAARGOS_1074 chromosome, complete genome", "reads_in": 156434, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127735.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Streptococcus oralis strain FDAARGOS_1075 chromosome, complete genome", "reads_in": 132994, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127755.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Burkholderia ambifaria strain FDAARGOS_1027 chromosome 1, complete sequence", "reads_in": 501574, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127775.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Pediococcus pentosaceus strain FDAARGOS_1009 chromosome, complete genome", "reads_in": 116796, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127795.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Stutzerimonas stutzeri strain FDAARGOS_1065 chromosome, complete genome", "reads_in": 295130, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127815.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Pediococcus acidilactici strain FDAARGOS_1007 chromosome, complete genome", "reads_in": 137886, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127855.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Actinomyces naeslundii strain FDAARGOS_1037 chromosome, complete genome", "reads_in": 210142, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016727405.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Bacillus cereus strain FDAARGOS_1084 chromosome, complete genome", "reads_in": 354920, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127875.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Streptococcus constellatus strain FDAARGOS_1015 chromosome, complete genome", "reads_in": 135906, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127895.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Moraxella nonliquefaciens strain FDAARGOS_1006 chromosome, complete genome", "reads_in": 154890, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127915.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Streptococcus oralis strain FDAARGOS_1021 chromosome, complete genome", "reads_in": 127050, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127935.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus pettenkoferi strain FDAARGOS_1071 chromosome, complete genome", "reads_in": 166242, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889765.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Corynebacterium minutissimum strain FDAARGOS_1196 chromosome, complete genome", "reads_in": 181688, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889725.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1252 chromosome, complete genome", "reads_in": 362544, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903235.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1324 chromosome, complete genome", "reads_in": 376682, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889705.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Amycolatopsis sp. FDAARGOS 1241 strain FDAARGOS_1241 chromosome, complete genome", "reads_in": 653208, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903255.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1311 chromosome, complete genome", "reads_in": 370730, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903295.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1323 chromosome, complete genome", "reads_in": 366060, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903315.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1289 chromosome, complete genome", "reads_in": 368416, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903335.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1295 chromosome, complete genome", "reads_in": 342824, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903355.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Streptococcus anginosus strain FDAARGOS_1357 chromosome, complete genome", "reads_in": 132094, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889905.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1259 chromosome, complete genome", "reads_in": 348288, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889865.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1253 chromosome, complete genome", "reads_in": 313026, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889245.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Mammaliicoccus vitulinus strain FDAARGOS_1207 chromosome, complete genome", "reads_in": 176858, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889845.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1248 chromosome, complete genome", "reads_in": 337768, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889265.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1264 chromosome, complete genome", "reads_in": 336128, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127675.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Campylobacter coli strain FDAARGOS_1028 chromosome, complete genome", "reads_in": 125030, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016128235.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia fergusonii strain FDAARGOS_1032 chromosome, complete genome", "reads_in": 309820, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127635.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Enterococcus casseliflavus strain FDAARGOS_998 chromosome, complete genome", "reads_in": 238722, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127595.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Fusobacterium canifelinum strain FDAARGOS_999 chromosome, complete genome", "reads_in": 161678, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889145.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus aureus strain FDAARGOS_1246 chromosome, complete genome", "reads_in": 192792, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889105.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1254 chromosome, complete genome", "reads_in": 326142, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016728105.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Corynebacterium striatum strain FDAARGOS_1115 chromosome, complete genome", "reads_in": 193656, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016728025.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Listeria monocytogenes strain FDAARGOS_1080 chromosome, complete genome", "reads_in": 197582, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016728005.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Serratia liquefaciens strain FDAARGOS_1081 chromosome, complete genome", "reads_in": 367766, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016728085.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1077 chromosome, complete genome", "reads_in": 339426, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889185.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1257 chromosome, complete genome", "reads_in": 354122, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016888985.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Chromobacterium violaceum strain FDAARGOS_1274 chromosome, complete genome", "reads_in": 320536, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016888965.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1261 chromosome, complete genome", "reads_in": 336674, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016728785.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Thomasclavelia ramosa strain FDAARGOS_1105 chromosome, complete genome", "reads_in": 236248, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016728745.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Corynebacterium amycolatum strain FDAARGOS_1107 chromosome, complete genome", "reads_in": 180436, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016728725.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Corynebacterium amycolatum strain FDAARGOS_1108 chromosome, complete genome", "reads_in": 165074, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016728805.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Acinetobacter johnsonii strain FDAARGOS_1092 chromosome, complete genome", "reads_in": 275708, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016728705.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Corynebacterium aurimucosum strain FDAARGOS_1109 chromosome, complete genome", "reads_in": 182518, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016728825.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Achromobacter xylosoxidans strain FDAARGOS_1091 chromosome, complete genome", "reads_in": 460132, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903175.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Bordetella pertussis strain FDAARGOS_1369 chromosome, complete genome", "reads_in": 273602, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903215.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Acinetobacter baumannii strain FDAARGOS_1359 chromosome, complete genome", "reads_in": 259886, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127975.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Corynebacterium urealyticum strain FDAARGOS_994 chromosome, complete genome", "reads_in": 158948, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889745.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Bacteroides caccae strain FDAARGOS_1219 chromosome, complete genome", "reads_in": 304730, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889785.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Corynebacterium glucuronolyticum strain FDAARGOS_1191 chromosome, complete genome", "reads_in": 192446, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889805.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Rhodanobacter sp. FDAARGOS 1247 strain FDAARGOS_1247 chromosome, complete genome", "reads_in": 261508, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889825.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Acinetobacter pittii strain FDAARGOS_1214 chromosome, complete genome", "reads_in": 273260, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016727585.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus aureus strain FDAARGOS_1086 chromosome, complete genome", "reads_in": 185098, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016726625.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Acinetobacter johnsonii strain FDAARGOS_1094 chromosome, complete genome", "reads_in": 234960, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016727425.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1076 chromosome, complete genome", "reads_in": 360454, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127835.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Salmonella enterica strain FDAARGOS_1066 chromosome, complete genome", "reads_in": 315766, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127555.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Streptococcus oralis strain FDAARGOS_1020 chromosome, complete genome", "reads_in": 128770, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127615.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Corynebacterium amycolatum strain FDAARGOS_991 chromosome, complete genome", "reads_in": 188596, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889285.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Kytococcus sedentarius strain FDAARGOS_1200 chromosome, complete genome", "reads_in": 163250, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016725465.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Sphingobacterium multivorum strain FDAARGOS_1142 chromosome, complete genome", "reads_in": 436790, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016727385.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus epidermidis strain FDAARGOS_1083 chromosome, complete genome", "reads_in": 165308, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127435.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Salmonella enterica strain FDAARGOS_1067 chromosome, complete genome", "reads_in": 337820, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127455.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus equorum strain FDAARGOS_1068 chromosome, complete genome", "reads_in": 184822, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016128135.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Streptococcus dysgalactiae strain FDAARGOS_1016 chromosome, complete genome", "reads_in": 151136, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016128195.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Paraburkholderia ginsengisoli strain FDAARGOS_1049 chromosome 1, complete sequence", "reads_in": 439818, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016128215.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Pediococcus pentosaceus strain FDAARGOS_1010 chromosome, complete genome", "reads_in": 129894, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016725645.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Sphingobacterium spiritivorum strain FDAARGOS_1144 chromosome, complete genome", "reads_in": 342552, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016725865.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Sphingobacterium multivorum strain FDAARGOS_1143 chromosome, complete genome", "reads_in": 401680, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016725905.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Proteus mirabilis strain FDAARGOS_1079 chromosome, complete genome", "reads_in": 269036, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016725945.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Acinetobacter johnsonii strain FDAARGOS_1093 chromosome, complete genome", "reads_in": 257722, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016726205.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Acinetobacter ursingii strain FDAARGOS_1095 chromosome, complete genome", "reads_in": 240834, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016726245.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Brevibacterium casei strain FDAARGOS_1100 chromosome, complete genome", "reads_in": 256458, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016726265.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Campylobacter lari strain FDAARGOS_1101 chromosome, complete genome", "reads_in": 100892, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127415.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Delftia acidovorans strain FDAARGOS_997 chromosome, complete genome", "reads_in": 453112, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016726285.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Shigella boydii strain FDAARGOS_1139 chromosome, complete genome", "reads_in": 303264, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016726325.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Serratia plymuthica strain FDAARGOS_1138 chromosome, complete genome", "reads_in": 365626, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016726345.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Acinetobacter ursingii strain FDAARGOS_1096 chromosome, complete genome", "reads_in": 232482, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016726365.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Rothia aeria strain FDAARGOS_1137 chromosome, complete genome", "reads_in": 172262, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016726485.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Peptoniphilus harei strain FDAARGOS_1136 chromosome, complete genome", "reads_in": 128638, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903395.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1313 chromosome, complete genome", "reads_in": 385540, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016726765.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Pediococcus acidilactici strain FDAARGOS_1133 chromosome, complete genome", "reads_in": 130226, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016726885.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Pediococcus pentosaceus strain FDAARGOS_1134 chromosome, complete genome", "reads_in": 116092, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016726945.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Moraxella osloensis strain FDAARGOS_1130 chromosome, complete genome", "reads_in": 169780, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016726965.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Mobiluncus curtisii strain FDAARGOS_1128 chromosome, complete genome", "reads_in": 144282, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127375.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Lactococcus garvieae strain FDAARGOS_1002 chromosome, complete genome", "reads_in": 152810, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127335.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus pseudintermedius strain FDAARGOS_1072 chromosome, complete genome", "reads_in": 189290, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016727125.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Enterococcus hirae strain FDAARGOS_1124 chromosome, complete genome", "reads_in": 195226, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016726305.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Bacteroides caccae strain FDAARGOS_1097 chromosome, complete genome", "reads_in": 304730, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127395.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Corynebacterium glucuronolyticum strain FDAARGOS_1053 chromosome, complete genome", "reads_in": 194320, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127355.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Kingella denitrificans strain FDAARGOS_1060 chromosome, complete genome", "reads_in": 148004, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002588845.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Serratia fonticola strain FDAARGOS_411 chromosome, complete genome", "reads_in": 381692, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016027835.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Serratia plymuthica strain FDAARGOS_889 chromosome, complete genome", "reads_in": 362786, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016027855.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Atlantibacter hermannii strain FDAARGOS_888 chromosome, complete genome", "reads_in": 303658, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016027975.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Lactococcus lactis strain FDAARGOS_887 chromosome, complete genome", "reads_in": 172198, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016028175.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Streptococcus oralis strain FDAARGOS_886 chromosome, complete genome", "reads_in": 127586, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016028235.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus equorum strain FDAARGOS_884 chromosome, complete genome", "reads_in": 190484, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016028255.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Streptococcus oralis strain FDAARGOS_885 chromosome, complete genome", "reads_in": 134954, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016028275.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus carnosus strain FDAARGOS_883 chromosome, complete genome", "reads_in": 171424, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016028295.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus auricularis strain FDAARGOS_882 chromosome, complete genome", "reads_in": 147772, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_000783435.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Achromobacter insolitus strain FDAARGOS_88 chromosome, complete genome", "reads_in": 459536, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016027575.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Bacillus tropicus strain FDAARGOS_897 chromosome, complete genome", "reads_in": 381134, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016028495.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Salmonella enterica strain FDAARGOS_878 chromosome, complete genome", "reads_in": 331248, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016028515.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Stutzerimonas frequens strain FDAARGOS_877 chromosome, complete genome", "reads_in": 292980, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016028635.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Stutzerimonas degradans strain FDAARGOS_876 chromosome, complete genome", "reads_in": 269474, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016028655.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Stutzerimonas stutzeri strain FDAARGOS_875 chromosome, complete genome", "reads_in": 302358, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016028675.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Proteus penneri strain FDAARGOS_874 chromosome, complete genome", "reads_in": 256720, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016028695.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Pediococcus pentosaceus strain FDAARGOS_873 chromosome, complete genome", "reads_in": 116058, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016028715.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Neisseria cinerea strain FDAARGOS_871 chromosome, complete genome", "reads_in": 122194, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016028735.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Oligella ureolytica strain FDAARGOS_872 chromosome, complete genome", "reads_in": 188856, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016028755.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Moraxella osloensis strain FDAARGOS_870 chromosome, complete genome", "reads_in": 162388, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016028775.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Moraxella nonliquefaciens strain FDAARGOS_869 chromosome, complete genome", "reads_in": 156250, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016028795.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Macrococcus caseolyticus strain FDAARGOS_868 chromosome, complete genome", "reads_in": 144730, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016028815.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Lactococcus lactis strain FDAARGOS_866 chromosome, complete genome", "reads_in": 173712, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016027595.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Serratia plymuthica strain FDAARGOS_896 chromosome, complete genome", "reads_in": 368456, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016027455.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Burkholderia humptydooensis strain FDAARGOS_899 chromosome 1, complete sequence", "reads_in": 485776, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016026375.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_941 chromosome, complete genome", "reads_in": 337554, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002591095.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Yersinia frederiksenii strain FDAARGOS_417 chromosome, complete genome", "reads_in": 333378, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019047205.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Acinetobacter pittii strain FDAARGOS 1399 chromosome, complete genome", "reads_in": 265342, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127315.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Achromobacter deleyi strain FDAARGOS_1050 chromosome, complete genome", "reads_in": 465818, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016128055.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Pediococcus acidilactici strain FDAARGOS_1008 chromosome, complete genome", "reads_in": 133972, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016128075.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Corynebacterium bovis strain FDAARGOS_1052 chromosome, complete genome", "reads_in": 176814, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016128095.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Streptococcus dysgalactiae strain FDAARGOS_1017 chromosome, complete genome", "reads_in": 144956, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903715.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Klebsiella oxytoca strain FDAARGOS_1332 chromosome, complete genome", "reads_in": 403376, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903415.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Klebsiella oxytoca strain FDAARGOS_1334 chromosome, complete genome", "reads_in": 415644, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903435.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1302 chromosome, complete genome", "reads_in": 342060, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020097215.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Clostridioides difficile strain FDAARGOS_1528 chromosome, complete genome", "reads_in": 291690, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020097235.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Enterobacter bugandensis strain FDAARGOS_1520 chromosome, complete genome", "reads_in": 334858, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020097255.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Thomasclavelia ramosa strain FDAARGOS_1519 chromosome, complete genome", "reads_in": 254472, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020097275.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Bacteroides fragilis strain FDAARGOS_1512 chromosome, complete genome", "reads_in": 349070, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020097295.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Aeromonas enteropelogenes strain FDAARGOS_1510 chromosome, complete genome", "reads_in": 292826, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020097315.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Aeromonas enteropelogenes strain FDAARGOS_1509 chromosome, complete genome", "reads_in": 302706, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020097335.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Streptococcus pneumoniae strain FDAARGOS_1508 chromosome, complete genome", "reads_in": 141108, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020097355.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Proteus vulgaris strain FDAARGOS_1507 chromosome, complete genome", "reads_in": 264242, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020097375.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Porphyromonas endodontalis strain FDAARGOS_1506 chromosome, complete genome", "reads_in": 140196, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020097395.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Leclercia adecarboxylata strain FDAARGOS_1505 chromosome, complete genome", "reads_in": 337142, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020097415.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Klebsiella quasipneumoniae strain FDAARGOS_1504 chromosome, complete genome", "reads_in": 344172, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020097435.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Gemella morbillorum strain FDAARGOS_1501 chromosome, complete genome", "reads_in": 117396, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020097455.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Gemella haemolysans strain FDAARGOS_1500 chromosome, complete genome", "reads_in": 128550, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020097475.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia fergusonii strain FDAARGOS_1499 chromosome, complete genome", "reads_in": 309728, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020097495.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Corynebacterium freneyi strain FDAARGOS_1495 chromosome, complete genome", "reads_in": 192122, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020097615.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Corynebacterium falsenii strain FDAARGOS_1493 chromosome, complete genome", "reads_in": 170792, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020097655.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Corynebacterium coyleae strain FDAARGOS_1492 chromosome, complete genome", "reads_in": 172188, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016725225.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Mobiluncus curtisii strain FDAARGOS_1127 chromosome, complete genome", "reads_in": 143540, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016725185.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Streptococcus anginosus strain FDAARGOS_1155 chromosome, complete genome", "reads_in": 152270, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016727305.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Enterococcus casseliflavus strain FDAARGOS_1122 chromosome, complete genome", "reads_in": 243400, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016725165.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Streptococcus lutetiensis strain FDAARGOS_1158 chromosome, complete genome", "reads_in": 119568, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016727325.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Enterococcus casseliflavus strain FDAARGOS_1121 chromosome, complete genome", "reads_in": 243364, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016727345.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Enterococcus casseliflavus strain FDAARGOS_1120 chromosome, complete genome", "reads_in": 236566, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016727365.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Corynebacterium ulcerans strain FDAARGOS_1118 chromosome, complete genome", "reads_in": 163584, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903375.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1286 chromosome, complete genome", "reads_in": 344596, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016725825.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Enterobacter hormaechei strain FDAARGOS_1078 chromosome, complete genome", "reads_in": 338684, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016725305.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus equorum strain FDAARGOS_1149 chromosome, complete genome", "reads_in": 182404, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016725265.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus equorum strain FDAARGOS_1150 chromosome, complete genome", "reads_in": 186494, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016128115.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Providencia stuartii strain FDAARGOS_1040 chromosome, complete genome", "reads_in": 289014, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127535.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Streptococcus salivarius strain FDAARGOS_1045 chromosome, complete genome", "reads_in": 154588, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127515.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Acinetobacter baumannii strain FDAARGOS_1036 chromosome, complete genome", "reads_in": 270026, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127495.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Kytococcus sedentarius strain FDAARGOS_1061 chromosome, complete genome", "reads_in": 185572, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127475.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Anaerococcus vaginalis strain FDAARGOS_988 chromosome, complete genome", "reads_in": 126264, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016126955.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Pseudomonas aeruginosa strain FDAARGOS_1041 chromosome, complete genome", "reads_in": 456166, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016126975.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Pediococcus pentosaceus strain FDAARGOS_1011 chromosome, complete genome", "reads_in": 119050, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016126995.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1058 chromosome, complete genome", "reads_in": 360064, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127015.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Corynebacterium tuberculostearicum strain FDAARGOS_993 chromosome, complete genome", "reads_in": 160572, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127035.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Leuconostoc pseudomesenteroides strain FDAARGOS_1004 chromosome, complete genome", "reads_in": 139694, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127055.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus felis strain FDAARGOS_1014 chromosome, complete genome", "reads_in": 165302, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127075.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Macrococcus caseolyticus strain FDAARGOS_1005 chromosome, complete genome", "reads_in": 144448, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127095.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Corynebacterium urealyticum strain FDAARGOS_996 chromosome, complete genome", "reads_in": 156730, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127115.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Aggregatibacter segnis strain FDAARGOS_987 chromosome, complete genome", "reads_in": 137500, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127135.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Lactococcus lactis strain FDAARGOS_1064 chromosome, complete genome", "reads_in": 167518, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127155.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Brevibacterium casei strain FDAARGOS_990 chromosome, complete genome", "reads_in": 256754, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127175.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Veillonella parvula strain FDAARGOS_1046 chromosome, complete genome", "reads_in": 142154, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127215.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Haemophilus parainfluenzae strain FDAARGOS_1000 chromosome, complete genome", "reads_in": 142780, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127235.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Enterococcus hirae strain FDAARGOS_1057 chromosome, complete genome", "reads_in": 188792, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127255.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Leuconostoc pseudomesenteroides strain FDAARGOS_1003 chromosome, complete genome", "reads_in": 140978, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127275.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Streptococcus infantarius strain FDAARGOS_1019 chromosome, complete genome", "reads_in": 130582, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127295.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Leuconostoc mesenteroides strain FDAARGOS_1033 chromosome, complete genome", "reads_in": 138384, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016727005.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Myroides odoratus strain FDAARGOS_1132 chromosome, complete genome", "reads_in": 277848, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016727265.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Enterococcus hirae strain FDAARGOS_1125 chromosome, complete genome", "reads_in": 204300, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016128015.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Brucella anthropi strain FDAARGOS_1039 chromosome 1, complete sequence", "reads_in": 344930, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016725205.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus pasteuri strain FDAARGOS_1152 chromosome, complete genome", "reads_in": 170422, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016725245.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Mammaliicoccus vitulinus strain FDAARGOS_1153 chromosome, complete genome", "reads_in": 169568, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016725285.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus pasteuri strain FDAARGOS_1151 chromosome, complete genome", "reads_in": 175648, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889325.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1258 chromosome, complete genome", "reads_in": 348734, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889345.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Acinetobacter pittii strain FDAARGOS_1215 chromosome, complete genome", "reads_in": 259280, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889365.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Corynebacterium kroppenstedtii strain FDAARGOS_1192 chromosome, complete genome", "reads_in": 163120, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019930805.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Legionella pneumophila strain FDAARGOS_1482 chromosome, complete genome", "reads_in": 229162, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019930745.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Mycolicibacterium smegmatis strain FDAARGOS_1484 chromosome, complete genome", "reads_in": 462514, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_021183565.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Nocardia yamanashiensis strain FDAARGOS_1623 chromosome, complete genome", "reads_in": 564586, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019930725.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Capnocytophaga ochracea strain FDAARGOS_1468 chromosome, complete genome", "reads_in": 174364, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020099275.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Corynebacterium falsenii strain FDAARGOS_1494 chromosome, complete genome", "reads_in": 178492, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020099295.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Clostridium sporogenes strain FDAARGOS_1491 chromosome, complete genome", "reads_in": 271160, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020099315.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Citrobacter europaeus strain FDAARGOS_1490 chromosome, complete genome", "reads_in": 339568, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020099335.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Citrobacter amalonaticus strain FDAARGOS_1489 chromosome, complete genome", "reads_in": 338134, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020099355.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Burkholderia vietnamiensis strain FDAARGOS_1488 chromosome 1, complete sequence", "reads_in": 462034, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020099375.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Acinetobacter variabilis strain FDAARGOS_1487 chromosome, complete genome", "reads_in": 232316, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020097195.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Enterobacter hormaechei subsp. oharae strain FDAARGOS_1533 chromosome, complete genome", "reads_in": 315438, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020097175.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Aeromonas enteropelogenes strain FDAARGOS_1536 chromosome, complete genome", "reads_in": 288620, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735905.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Prevotella melaninogenica strain FDAARGOS_1567 chromosome 1, complete sequence", "reads_in": 220450, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019931025.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Clostridioides difficile strain FDAARGOS_1469 chromosome, complete genome", "reads_in": 269708, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048605.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS 1375 chromosome, complete genome", "reads_in": 349518, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048625.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Enterobacter hormaechei strain FDAARGOS 1435 chromosome, complete genome", "reads_in": 317498, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048645.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Streptococcus mutans strain FDAARGOS 1458 chromosome, complete genome", "reads_in": 134624, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048665.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Aeromonas sp. FDAARGOS 1408 chromosome, complete genome", "reads_in": 322962, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048685.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Acinetobacter nosocomialis strain FDAARGOS 1395 chromosome, complete genome", "reads_in": 260174, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048705.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Aeromonas sp. FDAARGOS 1409 chromosome, complete genome", "reads_in": 291864, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048725.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Aeromonas sp. FDAARGOS 1407 chromosome, complete genome", "reads_in": 313436, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048805.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Citrobacter braakii strain FDAARGOS 1421 chromosome, complete genome", "reads_in": 371974, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019056115.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS 1383 chromosome, complete genome", "reads_in": 337060, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019930605.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Prevotella nigrescens strain FDAARGOS_1486 chromosome, complete genome", "reads_in": 178206, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019930625.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Nocardia asteroides strain FDAARGOS_1485 chromosome, complete genome", "reads_in": 494394, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019930645.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Gordonia bronchialis strain FDAARGOS_1476 chromosome, complete genome", "reads_in": 353808, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019930665.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Bacteroides nordii strain FDAARGOS_1461 chromosome, complete genome", "reads_in": 378572, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020736225.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Campylobacter curvus strain FDAARGOS_1547 chromosome, complete genome", "reads_in": 131660, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020736185.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Campylobacter gracilis strain FDAARGOS_1548 chromosome, complete genome", "reads_in": 152116, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020736165.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Capnocytophaga ochracea strain FDAARGOS_1550 chromosome, complete genome", "reads_in": 180882, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019047085.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Aeromonas sp. FDAARGOS 1404 chromosome, complete genome", "reads_in": 315912, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019047105.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Enterobacter cloacae strain FDAARGOS 1431 chromosome, complete genome", "reads_in": 373508, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904555.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1285 chromosome, complete genome", "reads_in": 345962, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904115.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1290 chromosome, complete genome", "reads_in": 347130, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020099255.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Enterobacter bugandensis strain FDAARGOS_1496 chromosome, complete genome", "reads_in": 334858, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019930765.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Legionella pneumophila strain FDAARGOS_1483 chromosome, complete genome", "reads_in": 227276, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019930845.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Eikenella corrodens strain FDAARGOS_1474 chromosome, complete genome", "reads_in": 146172, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019930865.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Campylobacter coli strain FDAARGOS_1466 chromosome, complete genome", "reads_in": 122190, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019930885.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Legionella anisa strain FDAARGOS_1480 chromosome, complete genome", "reads_in": 296538, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019930925.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Bartonella henselae strain FDAARGOS_1462 chromosome, complete genome", "reads_in": 129782, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019930945.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Campylobacter hominis strain FDAARGOS_1467 chromosome, complete genome", "reads_in": 114540, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019930965.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Enterobacter ludwigii strain FDAARGOS_1475 chromosome, complete genome", "reads_in": 315382, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019930985.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Campylobacter coli strain FDAARGOS_1465 chromosome, complete genome", "reads_in": 128994, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020149725.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Campylobacter coli strain FDAARGOS_1464 chromosome, complete genome", "reads_in": 107916, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020221755.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Lactococcus lactis strain FDAARGOS_867 chromosome, complete genome", "reads_in": 178994, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020450285.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "UNVERIFIED_ORG: Hafnia paralvei strain FDAARGOS_1502 chromosome, complete genome", "reads_in": 322462, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020736205.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Campylobacter rectus strain FDAARGOS_1549 chromosome, complete genome", "reads_in": 171454, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735945.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Parabacteroides distasonis strain FDAARGOS_1565 chromosome, complete genome", "reads_in": 320804, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735965.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Bordetella parapertussis strain FDAARGOS_1540 chromosome, complete genome", "reads_in": 318370, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735985.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Bordetella holmesii strain FDAARGOS_1539 chromosome, complete genome", "reads_in": 245874, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020736005.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Mycobacterium intracellulare strain FDAARGOS_1564 chromosome, complete genome", "reads_in": 376386, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020736025.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Haemophilus influenzae strain FDAARGOS_1561 chromosome, complete genome", "reads_in": 122286, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020736065.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Finegoldia magna strain FDAARGOS_1556 chromosome, complete genome", "reads_in": 115922, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020736085.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Corynebacterium diphtheriae subsp. diphtheriae strain FDAARGOS_1554 chromosome, complete genome", "reads_in": 164242, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020736105.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Corynebacterium diphtheriae strain FDAARGOS_1553 chromosome, complete genome", "reads_in": 166080, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020736125.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Corynebacterium diphtheriae strain FDAARGOS_1552 chromosome, complete genome", "reads_in": 157962, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020736145.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Campylobacter jejuni strain FDAARGOS_1546 chromosome, complete genome", "reads_in": 108536, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019930685.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Legionella longbeachae strain FDAARGOS_1481 chromosome, complete genome", "reads_in": 277696, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019931005.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Granulicatella adiacens strain FDAARGOS_1477 chromosome, complete genome", "reads_in": 131028, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019930705.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Haemophilus influenzae strain FDAARGOS_1479 chromosome, complete genome", "reads_in": 126840, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020736265.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Bordetella pertussis strain FDAARGOS_1543 chromosome, complete genome", "reads_in": 273102, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735865.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Parabacteroides johnsonii strain FDAARGOS_1580 chromosome, complete genome", "reads_in": 317744, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_021183665.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Mycolicibacterium smegmatis strain FDAARGOS_1618 chromosome, complete genome", "reads_in": 459422, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_021183685.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Mycobacterium kansasii strain FDAARGOS_1615 chromosome, complete genome", "reads_in": 430348, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_021183705.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Mycobacterium kansasii strain FDAARGOS_1614 chromosome, complete genome", "reads_in": 430260, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_021183725.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Mycobacterium ostraviense strain FDAARGOS_1613 chromosome, complete genome", "reads_in": 408516, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_021183745.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Mycobacterium intracellulare strain FDAARGOS_1612 chromosome, complete genome", "reads_in": 371710, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_021183765.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Mycobacterium intracellulare strain FDAARGOS_1611 chromosome, complete genome", "reads_in": 365956, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_021183785.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Mycobacterium intracellulare subsp. intracellulare strain FDAARGOS_1610 chromosome, complete genome", "reads_in": 360844, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_021183805.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Mycobacterium avium subsp. avium strain FDAARGOS_1609 chromosome, complete genome", "reads_in": 330370, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735925.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Bordetella parapertussis strain FDAARGOS_1541 chromosome, complete genome", "reads_in": 318372, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735885.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Bordetella pertussis strain FDAARGOS_1542 chromosome, complete genome", "reads_in": 274846, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735285.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Mycobacterium avium subsp. avium strain FDAARGOS_1607 chromosome, complete genome", "reads_in": 330462, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735585.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Campylobacter fetus subsp. testudinum strain FDAARGOS_1591 chromosome, complete genome", "reads_in": 118366, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735345.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Mycobacteroides abscessus subsp. abscessus strain FDAARGOS_1605 chromosome, complete genome", "reads_in": 335566, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735365.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Blautia massiliensis strain FDAARGOS_1576 chromosome, complete genome", "reads_in": 248852, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735385.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Granulicatella elegans strain FDAARGOS_1559 chromosome, complete genome", "reads_in": 126042, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735405.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Mycobacterium avium subsp. avium strain FDAARGOS_1606 chromosome, complete genome", "reads_in": 330704, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735425.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Mycobacteroides abscessus subsp. abscessus strain FDAARGOS_1604 chromosome, complete genome", "reads_in": 339370, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735445.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Prevotella copri DSM 18205 strain FDAARGOS_1573 chromosome, complete genome", "reads_in": 236774, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735465.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Collinsella stercoris DSM 13279 strain FDAARGOS_1572 chromosome, complete genome", "reads_in": 164050, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735485.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Mycobacteroides abscessus subsp. abscessus strain FDAARGOS_1603 chromosome, complete genome", "reads_in": 322226, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735505.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Mycobacteroides abscessus strain FDAARGOS_1602 chromosome, complete genome", "reads_in": 322236, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735525.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Helicobacter pylori strain FDAARGOS_1601 chromosome, complete genome", "reads_in": 109606, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735545.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Gordonia otitidis strain FDAARGOS_1600 chromosome, complete genome", "reads_in": 340588, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735565.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Hoylesella buccalis strain FDAARGOS_1588 chromosome, complete genome", "reads_in": 220560, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735605.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Parabacteroides merdae strain FDAARGOS_1586 chromosome, complete genome", "reads_in": 291720, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016028475.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Serratia rubidaea strain FDAARGOS_880 chromosome, complete genome", "reads_in": 337078, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_021183605.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Nocardia asteroides strain FDAARGOS_1621 chromosome, complete genome", "reads_in": 477942, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019047065.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Aeromonas sp. FDAARGOS 1414 chromosome, complete genome", "reads_in": 299588, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735625.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Finegoldia magna strain FDAARGOS_1557 chromosome, complete genome", "reads_in": 132318, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904475.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1292 chromosome, complete genome", "reads_in": 341372, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_021183585.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Nocardia asteroides strain FDAARGOS_1622 chromosome, complete genome", "reads_in": 465562, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904515.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Klebsiella oxytoca strain FDAARGOS_1331 chromosome, complete genome", "reads_in": 400534, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735645.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Acidaminococcus fermentans DSM 20731 strain FDAARGOS_1571 chromosome, complete genome", "reads_in": 155546, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735685.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Lachnospiraceae bacterium GAM79 strain FDAARGOS_1582 chromosome, complete genome", "reads_in": 198604, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735705.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Coprococcus eutactus ATCC 27759 strain FDAARGOS_1581 chromosome, complete genome", "reads_in": 207556, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735725.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Alistipes senegalensis strain FDAARGOS_1578 chromosome, complete genome", "reads_in": 268220, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735745.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Lachnospira eligens strain FDAARGOS_1570 chromosome, complete genome", "reads_in": 184852, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735765.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Streptococcus anginosus subsp. anginosus strain FDAARGOS_1569 chromosome, complete genome", "reads_in": 125088, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735785.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Prevotella melaninogenica strain FDAARGOS_1566 chromosome 1, complete sequence", "reads_in": 211214, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735805.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Helicobacter pylori strain FDAARGOS_1563 chromosome, complete genome", "reads_in": 110868, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735825.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Haemophilus influenzae strain FDAARGOS_1562 chromosome, complete genome", "reads_in": 122840, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735845.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Tsukamurella paurometabola strain FDAARGOS_1558 chromosome, complete genome", "reads_in": 313538, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048585.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Aeromonas sp. FDAARGOS 1418 chromosome, complete genome", "reads_in": 296936, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048565.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Aeromonas sp. FDAARGOS 1406 chromosome, complete genome", "reads_in": 309722, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048545.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Providencia rettgeri strain FDAARGOS 1451 chromosome, complete genome", "reads_in": 292254, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904495.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1281 chromosome, complete genome", "reads_in": 372684, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904155.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Coprococcus comes strain FDAARGOS_1339 chromosome, complete genome", "reads_in": 224872, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904175.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1372 chromosome, complete genome", "reads_in": 321568, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904195.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1296 chromosome, complete genome", "reads_in": 333376, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904235.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1287 chromosome, complete genome", "reads_in": 369658, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904275.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1328 chromosome, complete genome", "reads_in": 383876, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904295.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1306 chromosome, complete genome", "reads_in": 370864, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904375.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Bordetella pertussis strain FDAARGOS_1366 chromosome, complete genome", "reads_in": 273696, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904395.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1277 chromosome, complete genome", "reads_in": 316756, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904415.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1314 chromosome, complete genome", "reads_in": 380180, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904435.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1301 chromosome, complete genome", "reads_in": 347014, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904455.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1298 chromosome, complete genome", "reads_in": 357678, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735665.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Lachnospiraceae bacterium GAM79 strain FDAARGOS_1585 chromosome, complete genome", "reads_in": 197388, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019047045.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Aeromonas sp. FDAARGOS 1419 chromosome, complete genome", "reads_in": 305918, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019047025.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Enterobacter roggenkampii strain FDAARGOS 1430 chromosome, complete genome", "reads_in": 327818, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019047005.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Enterobacter ludwigii strain FDAARGOS 1436 chromosome, complete genome", "reads_in": 329562, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048025.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus haemolyticus strain FDAARGOS 1453 chromosome, complete genome", "reads_in": 171250, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048045.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Citrobacter braakii strain FDAARGOS 1422 chromosome, complete genome", "reads_in": 321184, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048065.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Citrobacter farmeri strain FDAARGOS 1423 chromosome, complete genome", "reads_in": 330986, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048085.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Aeromonas sp. FDAARGOS 1416 chromosome, complete genome", "reads_in": 301256, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048105.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Providencia rettgeri strain FDAARGOS 1450 chromosome, complete genome", "reads_in": 309462, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048125.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Klebsiella aerogenes strain FDAARGOS 1442 chromosome, complete genome", "reads_in": 352118, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048145.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS 1386 chromosome, complete genome", "reads_in": 346162, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048165.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Corynebacterium coyleae strain FDAARGOS 1425 chromosome, complete genome", "reads_in": 167350, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903835.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1299 chromosome, complete genome", "reads_in": 344708, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903815.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1280 chromosome, complete genome", "reads_in": 366450, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048225.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Acinetobacter pittii strain FDAARGOS 1396 chromosome, complete genome", "reads_in": 261056, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903775.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1307 chromosome, complete genome", "reads_in": 378308, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904575.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1276 chromosome, complete genome", "reads_in": 338828, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904595.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1310 chromosome, complete genome", "reads_in": 372866, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904615.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1283 chromosome, complete genome", "reads_in": 347664, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904635.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1320 chromosome, complete genome", "reads_in": 351422, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904655.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1284 chromosome, complete genome", "reads_in": 337852, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016906185.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Streptomyces californicus strain FDAARGOS_1209 chromosome", "reads_in": 535448, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016906205.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Streptomyces californicus strain FDAARGOS_1212 chromosome", "reads_in": 538842, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016906225.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Streptomyces californicus strain FDAARGOS_1211 chromosome", "reads_in": 536880, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016906245.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Streptomyces californicus strain FDAARGOS_1213 chromosome", "reads_in": 538938, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016906265.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Streptomyces californicus strain FDAARGOS_1210 chromosome", "reads_in": 548102, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904135.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus epidermidis strain FDAARGOS_1361 chromosome, complete genome", "reads_in": 168278, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904095.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1319 chromosome, complete genome", "reads_in": 368102, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016939475.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Acinetobacter calcoaceticus strain FDAARGOS_1218 chromosome, complete genome", "reads_in": 261016, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903795.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1308 chromosome, complete genome", "reads_in": 370052, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903455.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1318 chromosome, complete genome", "reads_in": 346980, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048005.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1385 chromosome, complete genome", "reads_in": 348010, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903475.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1315 chromosome, complete genome", "reads_in": 368824, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019047985.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Klebsiella quasipneumoniae strain FDAARGOS 1443 chromosome, complete genome", "reads_in": 359656, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048525.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Acinetobacter lwoffii strain FDAARGOS 1394 chromosome, complete genome", "reads_in": 231122, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889385.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Haemophilus parahaemolyticus strain FDAARGOS_1199 chromosome, complete genome", "reads_in": 139564, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889405.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Corynebacterium kroppenstedtii strain FDAARGOS_1194 chromosome, complete genome", "reads_in": 167458, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889425.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Corynebacterium amycolatum strain FDAARGOS_1189 chromosome, complete genome", "reads_in": 164996, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889445.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Corynebacterium striatum strain FDAARGOS_1197 chromosome, complete genome", "reads_in": 200318, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016727445.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Morganella morganii strain FDAARGOS_1085 chromosome, complete genome", "reads_in": 265002, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889605.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1268 chromosome, complete genome", "reads_in": 333912, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889625.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1262 chromosome, complete genome", "reads_in": 348356, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889645.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Salmonella enterica strain FDAARGOS_1271 chromosome, complete genome", "reads_in": 320272, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889665.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Enterocloster bolteae strain FDAARGOS_1231 chromosome, complete genome", "reads_in": 440932, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889685.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1267 chromosome, complete genome", "reads_in": 362318, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020099175.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Klebsiella quasipneumoniae strain FDAARGOS_1503 chromosome, complete genome", "reads_in": 366590, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020099235.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Enterobacter bugandensis strain FDAARGOS_1497 chromosome, complete genome", "reads_in": 334858, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002587885.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_446 chromosome, complete genome", "reads_in": 384644, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904075.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1303 chromosome, complete genome", "reads_in": 352828, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903855.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1291 chromosome, complete genome", "reads_in": 348814, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903875.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1329 chromosome, complete genome", "reads_in": 370504, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903895.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1321 chromosome, complete genome", "reads_in": 362858, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903915.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1300 chromosome, complete genome", "reads_in": 363838, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903935.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1288 chromosome, complete genome", "reads_in": 378792, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903955.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1309 chromosome, complete genome", "reads_in": 373694, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903975.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Klebsiella oxytoca strain FDAARGOS_1333 chromosome, complete genome", "reads_in": 404192, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903995.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1337 chromosome, complete genome", "reads_in": 331158, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904015.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Ruminococcus] gnavus strain FDAARGOS_1342 chromosome, complete genome", "reads_in": 213130, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904035.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1322 chromosome, complete genome", "reads_in": 363140, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904055.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1293 chromosome, complete genome", "reads_in": 342718, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016939455.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Leuconostoc falkenbergense strain FDAARGOS_1201 chromosome, complete genome", "reads_in": 126594, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019046905.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Enterobacter bugandensis strain FDAARGOS 1427 chromosome, complete genome", "reads_in": 335846, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048245.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Enterobacter hormaechei strain FDAARGOS 1433 chromosome, complete genome", "reads_in": 327728, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016026855.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Bacillus tropicus strain FDAARGOS_920 chromosome, complete genome", "reads_in": 380104, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903495.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1312 chromosome, complete genome", "reads_in": 371120, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903535.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1373 chromosome, complete genome", "reads_in": 321672, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002407525.3", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Burkholderia cepacia strain FDAARGOS_388 chromosome 1, complete sequence", "reads_in": 571322, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002587105.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Campylobacter jejuni strain FDAARGOS_422 chromosome, complete genome", "reads_in": 112690, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002587165.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_444 chromosome, complete genome", "reads_in": 392066, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002587225.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Campylobacter jejuni strain FDAARGOS_421 chromosome, complete genome", "reads_in": 118520, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019047905.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1388 chromosome, complete genome", "reads_in": 345864, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019047145.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1379 chromosome, complete genome", "reads_in": 335034, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019047945.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Enterobacter hormaechei strain FDAARGOS 1434 chromosome, complete genome", "reads_in": 318836, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019047125.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Aeromonas sp. FDAARGOS 1411 chromosome, complete genome", "reads_in": 302630, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048285.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS 1389 chromosome, complete genome", "reads_in": 345514, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048305.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Acinetobacter lwoffii strain FDAARGOS 1393 chromosome, complete genome", "reads_in": 233634, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048325.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Enterococcus durans strain FDAARGOS 1437 chromosome, complete genome", "reads_in": 206380, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048345.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS 1377 chromosome, complete genome", "reads_in": 356918, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048365.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Aeromonas sp. FDAARGOS 1415 chromosome, complete genome", "reads_in": 289762, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048385.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Pantoea agglomerans strain FDAARGOS 1447 chromosome, complete genome", "reads_in": 312802, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048405.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Acinetobacter seifertii strain FDAARGOS 1400 chromosome, complete genome", "reads_in": 266604, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048425.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS 1380 chromosome, complete genome", "reads_in": 349396, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048445.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS 1382 chromosome, complete genome", "reads_in": 337060, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048465.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Jonesia denitrificans strain FDAARGOS 1440 chromosome, complete genome", "reads_in": 183310, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048485.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Enterobacter asburiae strain FDAARGOS 1432 chromosome, complete genome", "reads_in": 326758, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048505.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS 1387 chromosome, complete genome", "reads_in": 348454, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019047965.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1378 chromosome, complete genome", "reads_in": 333822, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019047925.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Klebsiella aerogenes strain FDAARGOS 1441 chromosome, complete genome", "reads_in": 346180, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019046925.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Aeromonas sp. FDAARGOS 1402 chromosome, complete genome", "reads_in": 328168, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019047185.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Acinetobacter pittii strain FDAARGOS 1397 chromosome, complete genome", "reads_in": 278826, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019046945.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Streptococcus gordonii strain FDAARGOS 1454 chromosome, complete genome", "reads_in": 145842, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019046965.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Acinetobacter pittii strain FDAARGOS 1398 chromosome, complete genome", "reads_in": 255868, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019046985.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Aeromonas sp. FDAARGOS 1417 chromosome, complete genome", "reads_in": 319550, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002407165.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus lugdunensis strain FDAARGOS_377 chromosome, complete genome", "reads_in": 171442, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903515.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1330 chromosome, complete genome", "reads_in": 365880, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002393505.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Providencia alcalifaciens strain FDAARGOS_408 chromosome, complete genome", "reads_in": 266008, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002393465.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_401 chromosome, complete genome", "reads_in": 359158, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903555.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus epidermidis strain FDAARGOS_1363 chromosome, complete genome", "reads_in": 167308, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903575.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus epidermidis strain FDAARGOS_1364 chromosome, complete genome", "reads_in": 163244, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903615.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1304 chromosome, complete genome", "reads_in": 351638, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016126915.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Corynebacterium minutissimum strain FDAARGOS_992 chromosome, complete genome", "reads_in": 179734, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903735.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1325 chromosome, complete genome", "reads_in": 366224, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903755.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1317 chromosome, complete genome", "reads_in": 351878, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048185.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Lelliottia amnigena strain FDAARGOS 1446 chromosome, complete genome", "reads_in": 327628, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048265.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Aeromonas sp. FDAARGOS 1405 chromosome, complete genome", "reads_in": 307614, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904535.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1297 chromosome, complete genome", "reads_in": 345586, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019047165.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1384 chromosome, complete genome", "reads_in": 347978, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019047325.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Aeromonas sp. FDAARGOS 1403 chromosome, complete genome", "reads_in": 321318, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019047465.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Lelliottia amnigena strain FDAARGOS 1444 chromosome, complete genome", "reads_in": 300368, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019047505.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Streptococcus gordonii strain FDAARGOS 1455 chromosome, complete genome", "reads_in": 143654, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019047545.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia fergusonii strain FDAARGOS 1438 chromosome, complete genome", "reads_in": 302878, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019047645.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1376 chromosome, complete genome", "reads_in": 356636, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019047725.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS 1374 chromosome, complete genome", "reads_in": 349518, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019047745.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1381 chromosome, complete genome", "reads_in": 349546, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019047765.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Citrobacter pasteurii strain FDAARGOS 1424 chromosome, complete genome", "reads_in": 334754, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019047785.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Enterobacter cancerogenus strain FDAARGOS 1428 chromosome, complete genome", "reads_in": 321448, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019047805.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Corynebacterium freneyi strain FDAARGOS 1426 chromosome, complete genome", "reads_in": 192120, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019047825.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Streptococcus mitis strain FDAARGOS 1456 chromosome, complete genome", "reads_in": 124590, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019047885.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Providencia rettgeri strain FDAARGOS 1449 chromosome, complete genome", "reads_in": 318484, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016126935.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Stenotrophomonas maltophilia strain FDAARGOS_1044 chromosome, complete genome", "reads_in": 333618, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_000626615.3", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus aureus strain FDAARGOS_5 chromosome, complete genome", "reads_in": 187346, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016028855.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Rothia kristinae strain FDAARGOS_864 chromosome, complete genome", "reads_in": 158516, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002591075.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_439 chromosome, complete genome", "reads_in": 383526, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002393445.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Cedecea neteri strain FDAARGOS_392 chromosome, complete genome", "reads_in": 364620, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002393485.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Ralstonia pickettii strain FDAARGOS_410 chromosome 1, complete sequence", "reads_in": 321204, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016027775.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Delftia acidovorans strain FDAARGOS_891 chromosome, complete genome", "reads_in": 476700, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889305.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Acinetobacter pittii strain FDAARGOS_1217 chromosome, complete genome", "reads_in": 263804, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016026555.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Brevibacterium casei strain FDAARGOS_937 chromosome, complete genome", "reads_in": 258770, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002208805.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus pettenkoferi strain FDAARGOS_288 chromosome, complete genome", "reads_in": 166824, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002208825.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Brevundimonas vesicularis strain FDAARGOS_289 chromosome, complete genome", "reads_in": 223036, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002208845.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Citrobacter braakii strain FDAARGOS_290 chromosome, complete genome", "reads_in": 322318, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002208865.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli O157 strain FDAARGOS_293 chromosome, complete genome", "reads_in": 379110, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002208885.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Stenotrophomonas maltophilia strain FDAARGOS_325 chromosome, complete genome", "reads_in": 323434, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002208905.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus saprophyticus strain FDAARGOS_336 chromosome, complete genome", "reads_in": 175718, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002208925.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Chryseobacterium indologenes strain FDAARGOS_337 chromosome, complete genome", "reads_in": 341292, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002208945.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Enterococcus faecalis strain FDAARGOS_338 chromosome, complete genome", "reads_in": 190734, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002208965.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Salmonella enterica subsp. enterica serovar Typhimurium var. 5- strain FDAARGOS_312 chromosome, complete genome", "reads_in": 347590, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002073635.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Paracoccus yeei strain FDAARGOS_252 chromosome, complete genome", "reads_in": 321974, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002073595.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Neisseria meningitidis strain FDAARGOS_210 chromosome, complete genome", "reads_in": 151552, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002073575.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Neisseria meningitidis strain FDAARGOS_209 chromosome, complete genome", "reads_in": 145416, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002208785.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Bacillus anthracis strain FDAARGOS_341 chromosome, complete genome", "reads_in": 360696, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002073555.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Neisseria meningitidis strain FDAARGOS_211 chromosome, complete genome", "reads_in": 153720, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001559115.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus simulans strain FDAARGOS_124 chromosome, complete genome", "reads_in": 176504, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001188125.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Salmonella enterica subsp. enterica serovar Heidelberg strain FDAARGOS_54 chromosome, complete genome", "reads_in": 316752, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002953395.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Vibrio diabolicus strain FDAARGOS_105 chromosome 1, complete sequence", "reads_in": 361742, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001188025.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Listeria monocytogenes strain FDAARGOS_58 chromosome, complete genome", "reads_in": 193778, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002209265.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus saprophyticus strain FDAARGOS_355 chromosome, complete genome", "reads_in": 175946, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002386165.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Salmonella enterica subsp. enterica serovar Paratyphi A strain FDAARGOS_368 chromosome, complete genome", "reads_in": 305866, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002393365.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_403 chromosome, complete genome", "reads_in": 365028, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002386185.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus simulans strain FDAARGOS_383 chromosome, complete genome", "reads_in": 191464, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002386205.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Burkholderia thailandensis strain FDAARGOS_426 chromosome 1, complete sequence", "reads_in": 448658, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002386245.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus aureus strain FDAARGOS_412 chromosome, complete genome", "reads_in": 184640, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002386265.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Weissella paramesenteroides strain FDAARGOS_414 chromosome, complete genome", "reads_in": 129860, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002386285.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Streptococcus gordonii strain FDAARGOS_371 chromosome, complete genome", "reads_in": 141420, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001559075.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Citrobacter amalonaticus strain FDAARGOS_122 chromosome, complete genome", "reads_in": 326982, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002208765.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Psychrobacter cryohalolentis strain FDAARGOS_308 chromosome, complete genome", "reads_in": 207242, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002208745.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Pseudomonas protegens strain FDAARGOS_307 chromosome, complete genome", "reads_in": 471722, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001559055.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Bordetella bronchiseptica strain FDAARGOS_176 chromosome, complete genome", "reads_in": 355948, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002073315.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Yersinia sp. FDAARGOS_228 chromosome, complete genome", "reads_in": 330870, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002073335.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Vibrio cholerae strain FDAARGOS_223 chromosome 1, complete sequence", "reads_in": 269486, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002073355.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Streptococcus intermedius strain FDAARGOS_233 chromosome, complete genome", "reads_in": 127600, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002073375.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Corynebacterium diphtheriae strain FDAARGOS_197 chromosome, complete genome", "reads_in": 165936, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002073395.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus lugdunensis strain FDAARGOS_222 chromosome, complete genome", "reads_in": 169354, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002073415.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Bacillus sp. FDAARGOS_235 chromosome, complete genome", "reads_in": 373892, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002073435.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Streptococcus gordonii strain FDAARGOS_257 chromosome, complete genome", "reads_in": 148180, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002073455.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Legionella longbeachae strain FDAARGOS_201 chromosome, complete genome", "reads_in": 277516, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002073495.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Leptospira interrogans serovar Copenhageni strain FDAARGOS_203 chromosome, complete genome", "reads_in": 308718, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002073515.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Streptococcus equinus strain FDAARGOS_251 chromosome, complete genome", "reads_in": 154102, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002073535.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Neisseria gonorrhoeae strain FDAARGOS_204 chromosome, complete genome", "reads_in": 147496, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812125.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Bacillus sp. FDAARGOS_527 chromosome, complete genome", "reads_in": 380076, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812165.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Pseudomonas aeruginosa strain FDAARGOS_532 chromosome, complete genome", "reads_in": 462084, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016028835.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Lactococcus lactis strain FDAARGOS_865 chromosome, complete genome", "reads_in": 171814, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013331815.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Bacillus tropicus strain FDAARGOS_782 chromosome, complete genome", "reads_in": 353154, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013267595.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Prevotella melaninogenica strain FDAARGOS_760 chromosome 1, complete sequence", "reads_in": 223532, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013267615.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Parabacteroides distasonis strain FDAARGOS_759 chromosome, complete genome", "reads_in": 328402, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001559215.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Klebsiella aerogenes strain FDAARGOS_152 chromosome, complete genome", "reads_in": 354476, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001188045.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Listeria monocytogenes strain FDAARGOS_57 chromosome unnamed, complete genome", "reads_in": 195692, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002073775.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Vibrio parahaemolyticus strain FDAARGOS_191 chromosome 1, complete sequence", "reads_in": 344886, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001559135.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Serratia liquefaciens strain FDAARGOS_125 chromosome, complete genome", "reads_in": 352316, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002073795.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Streptococcus sp. 'group B' strain FDAARGOS_229 chromosome, complete genome", "reads_in": 145218, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002073815.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Streptococcus sp. FDAARGOS_192 chromosome, complete genome", "reads_in": 162368, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002073835.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Streptococcus salivarius strain FDAARGOS_259 chromosome, complete genome", "reads_in": 150622, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002208665.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Haemophilus] ducreyi strain FDAARGOS_297 chromosome, complete genome", "reads_in": 113420, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002208685.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Kocuria rhizophila strain FDAARGOS_302 chromosome, complete genome", "reads_in": 179858, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002208725.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Prevotella melaninogenica strain FDAARGOS_306 chromosome 1, complete sequence", "reads_in": 211214, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002386305.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Morganella morganii strain FDAARGOS_365 chromosome, complete genome", "reads_in": 271714, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002386325.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Salmonella enterica subsp. enterica serovar Saintpaul strain FDAARGOS_373 chromosome, complete genome", "reads_in": 322540, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002386345.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Streptococcus oralis strain FDAARGOS_367 chromosome, complete genome", "reads_in": 136880, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002386365.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Rothia mucilaginosa strain FDAARGOS_369 chromosome, complete genome", "reads_in": 160584, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002393425.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Burkholderia gladioli pv. gladioli strain FDAARGOS_389 chromosome 1, complete sequence", "reads_in": 552478, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002393385.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Pasteurella multocida strain FDAARGOS_384 chromosome, complete genome", "reads_in": 156448, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002951115.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Stenotrophomonas maltophilia strain FDAARGOS_92 chromosome, complete genome", "reads_in": 321348, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002278015.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Enterococcus hirae strain FDAARGOS_234 chromosome, complete genome", "reads_in": 189710, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002209065.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Campylobacter jejuni strain FDAARGOS_266 chromosome, complete genome", "reads_in": 108030, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002209085.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Burkholderia cepacia strain FDAARGOS_345 chromosome 1, complete sequence", "reads_in": 571324, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002209105.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli O104:H4 strain FDAARGOS_348 chromosome, complete genome", "reads_in": 362832, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002209125.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Nocardia brasiliensis strain FDAARGOS_352 chromosome, complete genome", "reads_in": 595700, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002209145.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Jonesia denitrificans strain FDAARGOS_301 chromosome, complete genome", "reads_in": 185920, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002209165.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Mammaliicoccus sciuri strain FDAARGOS_285 chromosome, complete genome", "reads_in": 190256, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002209185.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Corynebacterium jeikeium strain FDAARGOS_328 chromosome, complete genome", "reads_in": 176726, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002209205.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Salmonella enterica subsp. enterica serovar Hadar strain FDAARGOS_313 chromosome, complete genome", "reads_in": 329616, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003613595.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Burkholderia gladioli pv. gladioli strain FDAARGOS_188 chromosome 1, complete sequence", "reads_in": 592836, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003614835.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Burkholderia thailandensis strain FDAARGOS_242 chromosome 1, complete sequence", "reads_in": 443554, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001019575.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus aureus strain FDAARGOS_47 chromosome, complete genome", "reads_in": 193428, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013267655.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Mycobacterium tuberculosis strain FDAARGOS_756 chromosome, complete genome", "reads_in": 294306, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002953535.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Vibrio alginolyticus strain FDAARGOS_110 chromosome 1, complete sequence", "reads_in": 351246, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002953975.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Vibrio alginolyticus strain FDAARGOS_114 chromosome 1, complete sequence", "reads_in": 345218, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002983625.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Yersinia intermedia strain FDAARGOS_358 chromosome, complete genome", "reads_in": 330710, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002984395.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Klebsiella oxytoca strain FDAARGOS_335 chromosome, complete genome", "reads_in": 403320, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002983645.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli O104:H4 strain FDAARGOS_349 chromosome, complete genome", "reads_in": 366744, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002983665.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Providencia stuartii strain FDAARGOS_291 chromosome, complete genome", "reads_in": 292966, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002983685.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Streptobacillus moniliformis strain FDAARGOS_310 chromosome, complete genome", "reads_in": 112396, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002983705.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Helicobacter pylori strain FDAARGOS_300 chromosome, complete genome", "reads_in": 109716, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002983725.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Campylobacter jejuni subsp. doylei strain FDAARGOS_295 chromosome, complete genome", "reads_in": 123004, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002983785.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Enterococcus faecium strain FDAARGOS_323 chromosome, complete genome", "reads_in": 180694, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002983865.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Roseobacter denitrificans strain FDAARGOS_309 chromosome, complete genome", "reads_in": 287254, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002944935.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_144 chromosome, complete genome", "reads_in": 334812, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002073295.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Pasteurella multocida strain FDAARGOS_216 chromosome, complete genome", "reads_in": 153012, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002943995.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Pseudomonas monteilii strain FDAARGOS_171 chromosome, complete genome", "reads_in": 393152, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002591295.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Pasteurella multocida strain FDAARGOS_385 chromosome, complete genome", "reads_in": 156448, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002386385.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Citrobacter werkmanii strain FDAARGOS_364 chromosome, complete genome", "reads_in": 325674, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002393245.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Citrobacter koseri strain FDAARGOS_393 chromosome, complete genome", "reads_in": 325632, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002393265.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Enterococcus sp. FDAARGOS_375 chromosome, complete genome", "reads_in": 236622, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002944995.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus epidermidis strain FDAARGOS_153 chromosome, complete genome", "reads_in": 170132, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002951195.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Pandoraea apista strain FDAARGOS_126 chromosome, complete genome", "reads_in": 355100, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812145.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Enterobacter roggenkampii strain FDAARGOS_523 chromosome, complete genome", "reads_in": 346032, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003798225.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Yersinia pestis strain FDAARGOS_601 chromosome, complete genome", "reads_in": 328732, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003019945.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Burkholderia vietnamiensis strain FDAARGOS_239 chromosome 1, complete sequence", "reads_in": 462028, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003020025.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Burkholderia thailandensis strain FDAARGOS_237 chromosome 1, complete sequence", "reads_in": 444854, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003020185.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Burkholderia thailandensis strain FDAARGOS_238 chromosome 1, complete sequence", "reads_in": 451244, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003030425.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Enterococcus faecalis strain FDAARGOS_324 chromosome, complete genome", "reads_in": 210658, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003050665.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Helicobacter pylori strain FDAARGOS_298 chromosome, complete genome", "reads_in": 112060, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003546885.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Klebsiella aerogenes strain FDAARGOS_327 chromosome, complete genome", "reads_in": 360012, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003546905.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Yersinia pseudotuberculosis strain FDAARGOS_342 chromosome, complete genome", "reads_in": 327036, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002277995.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Elizabethkingia anophelis strain FDAARGOS_198 chromosome, complete genome", "reads_in": 271614, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002209245.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Shewanella sp. FDAARGOS_354 chromosome, complete genome", "reads_in": 338000, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003614815.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Burkholderia thailandensis strain FDAARGOS_241 chromosome 1, complete sequence", "reads_in": 445608, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002209225.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Salmonella enterica subsp. enterica serovar Typhimurium strain FDAARGOS_321 chromosome, complete genome", "reads_in": 334714, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002393405.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Lelliottia amnigena strain FDAARGOS_395 chromosome, complete genome", "reads_in": 297974, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002591115.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Klebsiella aerogenes strain FDAARGOS_363 chromosome, complete genome", "reads_in": 349698, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002591155.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Proteus vulgaris strain FDAARGOS_366 chromosome, complete genome", "reads_in": 264132, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002591175.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus capitis strain FDAARGOS_378 chromosome, complete genome", "reads_in": 165676, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002591195.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Yersinia frederiksenii strain FDAARGOS_418 chromosome, complete genome", "reads_in": 328936, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002591215.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus lugdunensis strain FDAARGOS_381 chromosome, complete genome", "reads_in": 171380, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002591235.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Pseudomonas sp. FDAARGOS_380 chromosome, complete genome", "reads_in": 433290, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002591255.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Brucella canis strain FDAARGOS_420 chromosome 1, complete sequence", "reads_in": 220852, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002591275.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Chryseobacterium indologenes strain FDAARGOS_379 chromosome, complete genome", "reads_in": 325922, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002754815.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Roseomonas sp. FDAARGOS_362 chromosome, complete genome", "reads_in": 344946, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002073275.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Neisseria meningitidis strain FDAARGOS_215 chromosome, complete genome", "reads_in": 153722, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002073255.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Pasteurella multocida strain FDAARGOS_218 chromosome, complete genome", "reads_in": 155702, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002073235.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Neisseria meningitidis strain FDAARGOS_214 chromosome, complete genome", "reads_in": 159830, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_022213445.1", "platform": "illumina", "tool": "hostile", "reads_removed": 2, "name": "Burkholderia vietnamiensis strain FDAARGOS_721 chromosome 1, complete sequence", "reads_in": 467904, "reads_removed_percent": 0.0004274381069621119, "reads_retained_percent": 99.99957256189303}, {"sample": "GCA_013267275.1", "platform": "illumina", "tool": "hostile", "reads_removed": 2, "name": "Bacillus cereus strain FDAARGOS_798 chromosome, complete genome", "reads_in": 412640, "reads_removed_percent": 0.00048468398604110117, "reads_retained_percent": 99.99951531601396}, {"sample": "GCA_003813085.1", "platform": "illumina", "tool": "hostile", "reads_removed": 2, "name": "Alcaligenes faecalis strain FDAARGOS_491 chromosome, complete genome", "reads_in": 271526, "reads_removed_percent": 0.0007365777126315712, "reads_retained_percent": 99.99926342228737}, {"sample": "GCA_016728065.1", "platform": "illumina", "tool": "hostile", "reads_removed": 2, "name": "Pseudomonas fluorescens strain FDAARGOS_1088 chromosome, complete genome", "reads_in": 408974, "reads_removed_percent": 0.0004890286424075858, "reads_retained_percent": 99.9995109713576}, {"sample": "GCA_021183625.1", "platform": "illumina", "tool": "hostile", "reads_removed": 2, "name": "Nocardia asteroides strain FDAARGOS_1620 chromosome, complete genome", "reads_in": 465792, "reads_removed_percent": 0.00042937620225336627, "reads_retained_percent": 99.99957062379775}, {"sample": "GCA_016027715.1", "platform": "illumina", "tool": "hostile", "reads_removed": 2, "name": "Lactococcus garvieae strain FDAARGOS_893 chromosome, complete genome", "reads_in": 140684, "reads_removed_percent": 0.001421625771231981, "reads_retained_percent": 99.99857837422877}, {"sample": "GCA_020736045.1", "platform": "illumina", "tool": "hostile", "reads_removed": 2, "name": "Haemophilus influenzae strain FDAARGOS_1560 chromosome, complete genome", "reads_in": 126044, "reads_removed_percent": 0.0015867474850052362, "reads_retained_percent": 99.998413252515}, {"sample": "GCA_016903275.1", "platform": "illumina", "tool": "hostile", "reads_removed": 2, "name": "Escherichia coli strain FDAARGOS_1282 chromosome, complete genome", "reads_in": 352200, "reads_removed_percent": 0.0005678591709256105, "reads_retained_percent": 99.99943214082907}, {"sample": "GCA_016027815.1", "platform": "illumina", "tool": "hostile", "reads_removed": 2, "name": "Delftia lacustris strain FDAARGOS_890 chromosome, complete genome", "reads_in": 484968, "reads_removed_percent": 0.0004123983438082513, "reads_retained_percent": 99.99958760165619}, {"sample": "GCA_001188185.2", "platform": "illumina", "tool": "hostile", "reads_removed": 2, "name": "Vibrio parahaemolyticus strain FDAARGOS_51 chromosome 1, complete sequence", "reads_in": 343280, "reads_removed_percent": 0.0005826147751106967, "reads_retained_percent": 99.99941738522489}, {"sample": "GCA_008693825.1", "platform": "illumina", "tool": "hostile", "reads_removed": 2, "name": "Pseudomonas oryzihabitans strain FDAARGOS_657 chromosome, complete genome", "reads_in": 350992, "reads_removed_percent": 0.0005698135570041483, "reads_retained_percent": 99.99943018644299}, {"sample": "GCA_016904215.1", "platform": "illumina", "tool": "hostile", "reads_removed": 2, "name": "Bordetella pertussis strain FDAARGOS_1367 chromosome, complete genome", "reads_in": 273706, "reads_removed_percent": 0.00073071105492755, "reads_retained_percent": 99.99926928894507}, {"sample": "GCA_013267255.1", "platform": "illumina", "tool": "hostile", "reads_removed": 2, "name": "Bacillus cereus strain FDAARGOS_799 chromosome, complete genome", "reads_in": 412650, "reads_removed_percent": 0.0004846722403974312, "reads_retained_percent": 99.9995153277596}, {"sample": "GCA_016904355.1", "platform": "illumina", "tool": "hostile", "reads_removed": 2, "name": "Bordetella pertussis strain FDAARGOS_1365 chromosome, complete genome", "reads_in": 273672, "reads_removed_percent": 0.0007308018357742114, "reads_retained_percent": 99.99926919816423}, {"sample": "GCA_019930825.1", "platform": "illumina", "tool": "hostile", "reads_removed": 2, "name": "Aeromonas enteropelogenes strain FDAARGOS_1459 chromosome, complete genome", "reads_in": 294464, "reads_removed_percent": 0.0006792001738752445, "reads_retained_percent": 99.99932079982612}, {"sample": "GCA_016128155.1", "platform": "illumina", "tool": "hostile", "reads_removed": 2, "name": "Lactococcus garvieae strain FDAARGOS_1062 chromosome, complete genome", "reads_in": 135644, "reads_removed_percent": 0.0014744478192916751, "reads_retained_percent": 99.99852555218071}, {"sample": "GCA_016027035.1", "platform": "illumina", "tool": "hostile", "reads_removed": 2, "name": "Achromobacter xylosoxidans strain FDAARGOS_915 chromosome, complete genome", "reads_in": 454194, "reads_removed_percent": 0.0004403404712523723, "reads_retained_percent": 99.99955965952874}, {"sample": "GCA_008693485.1", "platform": "illumina", "tool": "hostile", "reads_removed": 2, "name": "Burkholderia multivorans strain FDAARGOS_623 chromosome 1, complete sequence", "reads_in": 431910, "reads_removed_percent": 0.0004630594336783126, "reads_retained_percent": 99.99953694056632}, {"sample": "GCA_002073475.2", "platform": "illumina", "tool": "hostile", "reads_removed": 2, "name": "Haemophilus influenzae strain FDAARGOS_199 chromosome, complete genome", "reads_in": 122046, "reads_removed_percent": 0.0016387263818560217, "reads_retained_percent": 99.99836127361814}, {"sample": "GCA_016726045.1", "platform": "illumina", "tool": "hostile", "reads_removed": 2, "name": "Sphingobacterium multivorum strain FDAARGOS_1140 chromosome, complete genome", "reads_in": 377044, "reads_removed_percent": 0.0005304420704214893, "reads_retained_percent": 99.99946955792957}, {"sample": "GCA_016599775.1", "platform": "illumina", "tool": "hostile", "reads_removed": 2, "name": "Stutzerimonas balearica strain FDAARGOS_1013 chromosome, complete genome", "reads_in": 294082, "reads_removed_percent": 0.00068008242599003, "reads_retained_percent": 99.99931991757401}, {"sample": "GCA_003019965.1", "platform": "illumina", "tool": "hostile", "reads_removed": 2, "name": "Burkholderia multivorans strain FDAARGOS_246 chromosome 1, complete sequence", "reads_in": 421526, "reads_removed_percent": 0.000474466580946371, "reads_retained_percent": 99.99952553341906}, {"sample": "GCA_016904255.1", "platform": "illumina", "tool": "hostile", "reads_removed": 2, "name": "Escherichia coli strain FDAARGOS_1371 chromosome, complete genome", "reads_in": 342696, "reads_removed_percent": 0.0005836076289189253, "reads_retained_percent": 99.99941639237107}, {"sample": "GCA_019930905.1", "platform": "illumina", "tool": "hostile", "reads_removed": 2, "name": "Haemophilus aegyptius strain FDAARGOS_1478 chromosome, complete genome", "reads_in": 132884, "reads_removed_percent": 0.0015050720929532524, "reads_retained_percent": 99.99849492790705}, {"sample": "GCA_016128175.1", "platform": "illumina", "tool": "hostile", "reads_removed": 2, "name": "Rothia kristinae strain FDAARGOS_1001 chromosome, complete genome", "reads_in": 167590, "reads_removed_percent": 0.0011933886270063846, "reads_retained_percent": 99.998806611373}, {"sample": "GCA_016725925.1", "platform": "illumina", "tool": "hostile", "reads_removed": 2, "name": "Sphingobacterium multivorum strain FDAARGOS_1141 chromosome, complete genome", "reads_in": 419822, "reads_removed_percent": 0.00047639237581641744, "reads_retained_percent": 99.99952360762418}, {"sample": "GCA_001018645.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus aureus strain FDAARGOS_2 chromosome, complete genome", "reads_in": 187334, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_021183645.1", "platform": "illumina", "tool": "hostile", "reads_removed": 2, "name": "Nocardia gipuzkoensis strain FDAARGOS_1619 chromosome, complete genome", "reads_in": 541242, "reads_removed_percent": 0.0003695204732818222, "reads_retained_percent": 99.99963047952672}, {"sample": "GCA_001018725.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus aureus strain FDAARGOS_10 chromosome, complete genome", "reads_in": 191592, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002073615.2", "platform": "illumina", "tool": "hostile", "reads_removed": 4, "name": "Neisseria gonorrhoeae strain FDAARGOS_205 chromosome, complete genome", "reads_in": 151930, "reads_removed_percent": 0.00263279141709998, "reads_retained_percent": 99.9973672085829}, {"sample": "GCA_016904315.1", "platform": "illumina", "tool": "hostile", "reads_removed": 4, "name": "Klebsiella pneumoniae strain FDAARGOS_1316 chromosome, complete genome", "reads_in": 365988, "reads_removed_percent": 0.0010929320086997386, "reads_retained_percent": 99.9989070679913}, {"sample": "GCA_016026415.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Corynebacterium amycolatum strain FDAARGOS_938 chromosome, complete genome", "reads_in": 186842, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812365.1", "platform": "illumina", "tool": "hostile", "reads_removed": 16, "name": "Burkholderia multivorans strain FDAARGOS_547 chromosome 1, complete sequence", "reads_in": 426274, "reads_removed_percent": 0.0037534543509573654, "reads_retained_percent": 99.99624654564904}, {"sample": "GCA_001558255.2", "platform": "illumina", "tool": "hostile", "reads_removed": 10, "name": "Grimontia hollisae strain FDAARGOS_111 chromosome 1, complete sequence", "reads_in": 266562, "reads_removed_percent": 0.0037514724529377776, "reads_retained_percent": 99.99624852754707}, {"sample": "GCA_003798365.1", "platform": "illumina", "tool": "hostile", "reads_removed": 10, "name": "Burkholderia pseudomallei strain FDAARGOS_594 chromosome 1, complete sequence", "reads_in": 478692, "reads_removed_percent": 0.002089025928989831, "reads_retained_percent": 99.997910974071}, {"sample": "GCA_003812865.1", "platform": "illumina", "tool": "hostile", "reads_removed": 8, "name": "Chryseobacterium indologenes strain FDAARGOS_510 chromosome, complete genome", "reads_in": 328196, "reads_removed_percent": 0.0024375677948542942, "reads_retained_percent": 99.99756243220514}, {"sample": "GCA_016127575.1", "platform": "illumina", "tool": "hostile", "reads_removed": 8, "name": "Cupriavidus necator strain FDAARGOS_1030 chromosome 1, complete sequence", "reads_in": 494398, "reads_removed_percent": 0.0016181295231776827, "reads_retained_percent": 99.99838187047682}, {"sample": "GCA_020736245.1", "platform": "illumina", "tool": "hostile", "reads_removed": 8, "name": "Burkholderia dolosa strain FDAARGOS_1544 chromosome 1, complete sequence", "reads_in": 427432, "reads_removed_percent": 0.0018716427408336299, "reads_retained_percent": 99.99812835725916}, {"sample": "GCA_009730175.1", "platform": "illumina", "tool": "hostile", "reads_removed": 8, "name": "Burkholderia multivorans strain FDAARGOS_726 chromosome 1, complete sequence", "reads_in": 434580, "reads_removed_percent": 0.0018408578397533253, "reads_retained_percent": 99.99815914216025}, {"sample": "GCA_003854935.1", "platform": "illumina", "tool": "hostile", "reads_removed": 6, "name": "Cupriavidus pauculus strain FDAARGOS_614 chromosome 1, complete sequence", "reads_in": 411854, "reads_removed_percent": 0.0014568269338163524, "reads_retained_percent": 99.99854317306618}, {"sample": "GCA_016889205.1", "platform": "illumina", "tool": "hostile", "reads_removed": 6, "name": "Burkholderia dolosa strain FDAARGOS_1272 chromosome 1, complete sequence", "reads_in": 427274, "reads_removed_percent": 0.001404251136273211, "reads_retained_percent": 99.99859574886372}, {"sample": "GCA_001558495.2", "platform": "illumina", "tool": "hostile", "reads_removed": 6, "name": "Vibrio parahaemolyticus strain ATCC 17802 chromosome 1, complete sequence", "reads_in": 343498, "reads_removed_percent": 0.0017467350610483904, "reads_retained_percent": 99.99825326493895}, {"sample": "GCA_002591135.1", "platform": "illumina", "tool": "hostile", "reads_removed": 6, "name": "Escherichia coli strain FDAARGOS_448 chromosome, complete genome", "reads_in": 363122, "reads_removed_percent": 0.0016523372310132682, "reads_retained_percent": 99.99834766276899}, {"sample": "GCA_003813045.1", "platform": "illumina", "tool": "hostile", "reads_removed": 6, "name": "Burkholderia dolosa strain FDAARGOS_562 chromosome 1, complete sequence", "reads_in": 427274, "reads_removed_percent": 0.001404251136273211, "reads_retained_percent": 99.99859574886372}, {"sample": "GCA_003798245.1", "platform": "illumina", "tool": "hostile", "reads_removed": 6, "name": "Burkholderia pseudomallei strain FDAARGOS_593 chromosome 1, complete sequence", "reads_in": 483702, "reads_removed_percent": 0.0012404331592592132, "reads_retained_percent": 99.99875956684075}, {"sample": "GCA_016726985.1", "platform": "illumina", "tool": "hostile", "reads_removed": 6, "name": "Myroides odoratus strain FDAARGOS_1131 chromosome, complete genome", "reads_in": 282832, "reads_removed_percent": 0.0021214006901623577, "reads_retained_percent": 99.99787859930984}, {"sample": "GCA_016026395.1", "platform": "illumina", "tool": "hostile", "reads_removed": 6, "name": "Edwardsiella hoshinae strain FDAARGOS_940 chromosome, complete genome", "reads_in": 254474, "reads_removed_percent": 0.0023578047266125416, "reads_retained_percent": 99.99764219527339}, {"sample": "GCA_016903595.1", "platform": "illumina", "tool": "hostile", "reads_removed": 6, "name": "Escherichia coli strain FDAARGOS_1279 chromosome, complete genome", "reads_in": 356446, "reads_removed_percent": 0.0016832844245692196, "reads_retained_percent": 99.99831671557543}, {"sample": "GCA_016127655.1", "platform": "illumina", "tool": "hostile", "reads_removed": 4, "name": "Brevundimonas diminuta strain FDAARGOS_1026 chromosome, complete genome", "reads_in": 236606, "reads_removed_percent": 0.0016905742035282284, "reads_retained_percent": 99.99830942579648}, {"sample": "GCA_009730195.1", "platform": "illumina", "tool": "hostile", "reads_removed": 4, "name": "Burkholderia multivorans strain FDAARGOS_722 chromosome 1, complete sequence", "reads_in": 439364, "reads_removed_percent": 0.0009104068608261031, "reads_retained_percent": 99.99908959313917}, {"sample": "GCA_019931045.1", "platform": "illumina", "tool": "hostile", "reads_removed": 4, "name": "Cupriavidus pauculus strain FDAARGOS_1472 chromosome 1, complete sequence", "reads_in": 432224, "reads_removed_percent": 0.0009254460650033316, "reads_retained_percent": 99.999074553935}, {"sample": "GCA_016127195.1", "platform": "illumina", "tool": "hostile", "reads_removed": 4, "name": "Aeromonas jandaei strain FDAARGOS_986 chromosome, complete genome", "reads_in": 303406, "reads_removed_percent": 0.0013183654904649217, "reads_retained_percent": 99.99868163450954}, {"sample": "GCA_008693525.1", "platform": "illumina", "tool": "hostile", "reads_removed": 4, "name": "Burkholderia multivorans strain FDAARGOS_622 chromosome 1, complete sequence", "reads_in": 431910, "reads_removed_percent": 0.0009261188673566252, "reads_retained_percent": 99.99907388113265}, {"sample": "GCA_008693705.1", "platform": "illumina", "tool": "hostile", "reads_removed": 4, "name": "Aeromonas veronii strain FDAARGOS_632 chromosome, complete genome", "reads_in": 304126, "reads_removed_percent": 0.0013152443395171738, "reads_retained_percent": 99.99868475566048}, {"sample": "GCA_003812965.1", "platform": "illumina", "tool": "hostile", "reads_removed": 4, "name": "Burkholderia multivorans strain FDAARGOS_496 chromosome 1, complete sequence", "reads_in": 519760, "reads_removed_percent": 0.0007695859627520395, "reads_retained_percent": 99.99923041403724}, {"sample": "GCA_019930785.1", "platform": "illumina", "tool": "hostile", "reads_removed": 4, "name": "Burkholderia dolosa strain FDAARGOS_1463 chromosome 1, complete sequence", "reads_in": 413044, "reads_removed_percent": 0.000968419829364426, "reads_retained_percent": 99.99903158017064}, {"sample": "GCA_019048745.1", "platform": "illumina", "tool": "hostile", "reads_removed": 4, "name": "Aeromonas sp. FDAARGOS 1410 chromosome, complete genome", "reads_in": 300686, "reads_removed_percent": 0.0013302914003312425, "reads_retained_percent": 99.99866970859966}, {"sample": "GCA_016904335.1", "platform": "illumina", "tool": "hostile", "reads_removed": 4, "name": "Escherichia coli strain FDAARGOS_1370 chromosome, complete genome", "reads_in": 356214, "reads_removed_percent": 0.0011229204916145914, "reads_retained_percent": 99.99887707950839}, {"sample": "GCA_016026875.1", "platform": "illumina", "tool": "hostile", "reads_removed": 4, "name": "Aeromonas hydrophila strain FDAARGOS_916 chromosome, complete genome", "reads_in": 315580, "reads_removed_percent": 0.001267507446606249, "reads_retained_percent": 99.99873249255339}, {"sample": "GCA_002983965.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Salmonella enterica subsp. enterica serovar Typhimurium strain FDAARGOS_320 chromosome, complete genome", "reads_in": 335804, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001018685.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus aureus strain FDAARGOS_4 chromosome, complete genome", "reads_in": 187452, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001019255.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus aureus strain FDAARGOS_1 chromosome, complete genome", "reads_in": 190366, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001558415.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Vibrio fluvialis strain ATCC 33809 chromosome 1, complete sequence", "reads_in": 321850, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001558435.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Vibrio harveyi strain FDAARGOS_107 chromosome 1, complete sequence", "reads_in": 402594, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001558475.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Vibrio mimicus strain FDAARGOS_112 chromosome 1, complete sequence", "reads_in": 295914, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001558515.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Vibrio vulnificus strain FDAARGOS_119 chromosome 1, complete sequence", "reads_in": 331920, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001558735.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Pantoea vagans strain FDAARGOS_160 chromosome, complete genome", "reads_in": 320590, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001558755.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Achromobacter xylosoxidans strain FDAARGOS_147 chromosome, complete genome", "reads_in": 459130, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001558775.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus lugdunensis strain FDAARGOS_141 chromosome, complete genome", "reads_in": 171456, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001558795.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus aureus strain FDAARGOS_159 chromosome, complete genome", "reads_in": 189278, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001558815.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus lugdunensis strain FDAARGOS_143 chromosome, complete genome", "reads_in": 174258, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001558855.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Providencia stuartii strain FDAARGOS_145 chromosome, complete genome", "reads_in": 287008, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001558875.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Enterococcus gallinarum strain FDAARGOS_163 chromosome, complete genome", "reads_in": 254766, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001558895.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Morganella morganii strain FDAARGOS_172 chromosome, complete genome", "reads_in": 260538, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001019395.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus aureus strain FDAARGOS_20 chromosome, complete genome", "reads_in": 187888, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001019275.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus aureus strain FDAARGOS_7 chromosome, complete genome", "reads_in": 181938, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002208985.3", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Citrobacter koseri strain FDAARGOS_287 chromosome, complete genome", "reads_in": 328978, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_000784015.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Proteus mirabilis strain FDAARGOS_60 chromosome, complete genome", "reads_in": 273704, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_000783595.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Proteus mirabilis strain FDAARGOS_80 chromosome, complete genome", "reads_in": 269616, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_000783675.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Enterobacter cloacae complex sp. strain FDAARGOS_77 chromosome, complete genome", "reads_in": 325134, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_000783735.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Shigella flexneri strain FDAARGOS_74 chromosome, complete genome", "reads_in": 320390, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002073715.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Neisseria mucosa strain FDAARGOS_260 chromosome, complete genome", "reads_in": 185596, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001559175.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Streptococcus pyogenes strain FDAARGOS_149 chromosome, complete genome", "reads_in": 122642, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002073755.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Citrobacter braakii strain FDAARGOS_253 chromosome, complete genome", "reads_in": 349620, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001559195.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Achromobacter xylosoxidans strain FDAARGOS_150 chromosome, complete genome", "reads_in": 417916, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001559235.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Citrobacter sp. FDAARGOS_156 strain FDAARGOS_155 chromosome, complete genome", "reads_in": 315166, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001559255.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Hafnia paralvei strain FDAARGOS_158 chromosome, complete genome", "reads_in": 300914, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001593585.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Klebsiella aerogenes strain FDAARGOS_139 chromosome, complete genome", "reads_in": 353188, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002073215.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Moraxella catarrhalis strain FDAARGOS_213 chromosome, complete genome", "reads_in": 127200, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001019305.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus aureus strain FDAARGOS_9 chromosome, complete genome", "reads_in": 187646, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001018655.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus aureus strain FDAARGOS_6 chromosome, complete genome", "reads_in": 183952, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001558395.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Bordetella pertussis strain FDAARGOS_179 chromosome, complete genome", "reads_in": 273390, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001558295.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Shigella sonnei strain FDAARGOS_90 chromosome, complete genome", "reads_in": 332556, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_000783875.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Proteus mirabilis strain FDAARGOS_67 chromosome, complete genome", "reads_in": 263698, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_000783995.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Citrobacter freundii strain FDAARGOS_61 chromosome, complete genome", "reads_in": 350566, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_000783955.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Morganella morganii strain FDAARGOS_63 chromosome, complete genome", "reads_in": 266668, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_000783935.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Raoultella planticola strain FDAARGOS_64 chromosome, complete genome", "reads_in": 388262, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_000783915.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Serratia marcescens strain FDAARGOS_65 chromosome, complete genome", "reads_in": 349894, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001018775.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus aureus strain FDAARGOS_14 chromosome, complete genome", "reads_in": 184314, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001018735.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus aureus strain FDAARGOS_12 chromosome, complete genome", "reads_in": 181716, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001019125.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus aureus strain FDAARGOS_43 chromosome, complete genome", "reads_in": 188042, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001018805.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus aureus strain FDAARGOS_15 chromosome, complete genome", "reads_in": 186660, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001018835.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus aureus strain FDAARGOS_19 chromosome, complete genome", "reads_in": 191936, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001018845.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus aureus strain FDAARGOS_21 chromosome, complete genome", "reads_in": 192082, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001018915.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus aureus strain FDAARGOS_27 chromosome, complete genome", "reads_in": 182690, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001018975.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus aureus strain FDAARGOS_30 chromosome, complete genome", "reads_in": 190776, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001019205.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus aureus strain FDAARGOS_48 chromosome, complete genome", "reads_in": 189826, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001558915.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Achromobacter xylosoxidans strain FDAARGOS_162 chromosome, complete genome", "reads_in": 437716, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001558935.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Citrobacter amalonaticus strain FDAARGOS_165 chromosome, complete genome", "reads_in": 338936, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001559035.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Bartonella bacilliformis strain ATCC 35685D-5 chromosome, complete genome", "reads_in": 96228, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002208705.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Mycobacterium avium subsp. paratuberculosis strain FDAARGOS_305 chromosome, complete genome", "reads_in": 322166, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002073655.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Neisseria gonorrhoeae strain FDAARGOS_207 chromosome, complete genome", "reads_in": 145954, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002073675.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Neisseria meningitidis strain FDAARGOS_212 chromosome, complete genome", "reads_in": 149660, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002073695.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Streptococcus agalactiae strain FDAARGOS_254 chromosome, complete genome", "reads_in": 147914, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_000783815.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Salmonella enterica strain FDAARGOS_70 chromosome, complete genome", "reads_in": 319106, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_000783575.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Proteus mirabilis strain FDAARGOS_81 chromosome, complete genome", "reads_in": 271822, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_000783775.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Aeromonas caviae strain FDAARGOS_72 chromosome, complete genome", "reads_in": 301220, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_000783755.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Citrobacter freundii strain FDAARGOS_73 chromosome, complete genome", "reads_in": 363416, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002073735.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Clostridioides difficile strain FDAARGOS_267 chromosome, complete genome", "reads_in": 285744, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001558275.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus saprophyticus strain FDAARGOS_168 chromosome, complete genome", "reads_in": 176676, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001558355.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Salmonella enterica strain FDAARGOS_94 chromosome, complete genome", "reads_in": 330094, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002984035.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Salmonella enterica subsp. enterica serovar Heidelberg strain FDAARGOS_319 chromosome, complete genome", "reads_in": 333918, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016026695.1", "platform": "illumina", "tool": "hostile", "reads_removed": 18, "name": "Lactococcus garvieae strain FDAARGOS_929 chromosome, complete genome", "reads_in": 138956, "reads_removed_percent": 0.012953740752468407, "reads_retained_percent": 99.98704625924753}, {"sample": "GCA_009730475.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Corynebacterium xerosis strain FDAARGOS_674 chromosome, complete genome", "reads_in": 188102, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009730395.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Campylobacter coli strain FDAARGOS_735 chromosome, complete genome", "reads_in": 114436, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009730415.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Micrococcus luteus strain FDAARGOS_677 chromosome, complete genome", "reads_in": 165218, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812925.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Klebsiella oxytoca strain FDAARGOS_500 chromosome, complete genome", "reads_in": 391940, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812905.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Pseudomonas aeruginosa strain FDAARGOS_501 chromosome, complete genome", "reads_in": 457722, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003955815.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Francisella tularensis strain FDAARGOS_595 chromosome, complete genome", "reads_in": 126280, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009730575.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Shewanella putrefaciens strain FDAARGOS_681 chromosome, complete genome", "reads_in": 310642, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009734165.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Bacteroides ovatus strain FDAARGOS_733 chromosome, complete genome", "reads_in": 438092, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009806595.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Bacillus anthracis strain FDAARGOS_698 chromosome, complete genome", "reads_in": 348384, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009806615.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Bacillus anthracis strain FDAARGOS_701 chromosome, complete genome", "reads_in": 348494, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009806635.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Bacillus anthracis strain FDAARGOS_700 chromosome, complete genome", "reads_in": 348884, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009806655.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Bacillus anthracis strain FDAARGOS_706 chromosome, complete genome", "reads_in": 348660, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009806675.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Bacillus anthracis strain FDAARGOS_705 chromosome, complete genome", "reads_in": 348874, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009730375.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Capnocytophaga sp. FDAARGOS_737 chromosome, complete genome", "reads_in": 176848, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009806695.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Bacillus anthracis strain FDAARGOS_704 chromosome, complete genome", "reads_in": 348496, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013267515.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Streptococcus intermedius strain FDAARGOS_769 chromosome, complete genome", "reads_in": 128892, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013267235.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Bacillus cereus strain FDAARGOS_802 chromosome, complete genome", "reads_in": 404570, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013267295.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Bacillus thuringiensis strain FDAARGOS_796 chromosome, complete genome", "reads_in": 413816, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013267315.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Bacillus thuringiensis strain FDAARGOS_795 chromosome, complete genome", "reads_in": 381992, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013267335.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Bacillus thuringiensis strain FDAARGOS_793 chromosome, complete genome", "reads_in": 354256, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013267355.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Achromobacter pestifer strain FDAARGOS_790 chromosome, complete genome", "reads_in": 478488, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013267375.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Achromobacter denitrificans strain FDAARGOS_788 chromosome, complete genome", "reads_in": 449126, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013267395.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Achromobacter denitrificans strain FDAARGOS_786 chromosome, complete genome", "reads_in": 449124, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013267415.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Abiotrophia defectiva strain FDAARGOS_785 chromosome, complete genome", "reads_in": 136456, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013267435.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Niallia circulans strain FDAARGOS_783 chromosome, complete genome", "reads_in": 345304, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013267455.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Bacillus cereus strain FDAARGOS_781 chromosome, complete genome", "reads_in": 376156, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013267475.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Bacillus cereus strain FDAARGOS_780 chromosome, complete genome", "reads_in": 376156, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_012273615.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Chryseobacterium gallinarum strain FDAARGOS_636 chromosome, complete genome", "reads_in": 289264, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009730355.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Citrobacter portucalensis strain FDAARGOS_738 chromosome, complete genome", "reads_in": 344180, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009730335.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Schaalia odontolytica strain FDAARGOS_732 chromosome, complete genome", "reads_in": 162176, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009730315.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Gemella morbillorum strain FDAARGOS_741 chromosome, complete genome", "reads_in": 118630, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_008693845.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Pasteurella multocida strain FDAARGOS_644 chromosome, complete genome", "reads_in": 150004, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_008693865.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Paracoccus yeei strain FDAARGOS_643 chromosome, complete genome", "reads_in": 312652, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_008693885.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Klebsiella aerogenes strain FDAARGOS_641 chromosome, complete genome", "reads_in": 352422, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_008693905.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Enterobacter hormaechei strain FDAARGOS_642 chromosome, complete genome", "reads_in": 313208, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_008693925.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Dermabacter vaginalis strain FDAARGOS_640 chromosome, complete genome", "reads_in": 152620, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_008693965.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Pseudomonas luteola strain FDAARGOS_637 chromosome 1, complete sequence", "reads_in": 393426, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_008694005.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Streptococcus pyogenes strain FDAARGOS_668 chromosome, complete genome", "reads_in": 122756, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_008693985.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Stenotrophomonas maltophilia strain FDAARGOS_649 chromosome, complete genome", "reads_in": 308392, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009729895.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Salmonella enterica strain FDAARGOS_710 chromosome, complete genome", "reads_in": 307402, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009729915.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Salmonella enterica strain FDAARGOS_708 chromosome, complete genome", "reads_in": 342448, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009729935.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Salmonella enterica strain FDAARGOS_707 chromosome, complete genome", "reads_in": 303738, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009729955.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Salmonella enterica strain FDAARGOS_688 chromosome, complete genome", "reads_in": 308930, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009729975.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Salmonella enterica strain FDAARGOS_687 chromosome, complete genome", "reads_in": 337762, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009729995.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Shigella sonnei strain FDAARGOS_715 chromosome, complete genome", "reads_in": 340654, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009730015.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Salmonella enterica strain FDAARGOS_712 chromosome, complete genome", "reads_in": 311360, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009730035.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Burkholderia multivorans strain FDAARGOS_719 chromosome 1, complete sequence", "reads_in": 437710, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009730055.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Yersinia intermedia strain FDAARGOS_730 chromosome, complete genome", "reads_in": 328594, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009730075.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Yersinia intermedia strain FDAARGOS_729 chromosome, complete genome", "reads_in": 328594, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009730095.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Acinetobacter lwoffii strain FDAARGOS_552 chromosome, complete genome", "reads_in": 232364, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009730135.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus hominis strain FDAARGOS_746 chromosome, complete genome", "reads_in": 158146, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_006364815.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Enterococcus faecalis strain FDAARGOS_611 chromosome, complete genome", "reads_in": 189696, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009730155.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Enterococcus gallinarum strain FDAARGOS_728 chromosome, complete genome", "reads_in": 218328, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009730215.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Mycobacterium tuberculosis strain FDAARGOS_751 chromosome, complete genome", "reads_in": 296132, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009730235.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Mycobacterium tuberculosis strain FDAARGOS_750 chromosome, complete genome", "reads_in": 295644, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009730255.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Lactobacillus jensenii strain FDAARGOS_749 chromosome, complete genome", "reads_in": 107268, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009730275.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Lactobacillus crispatus strain FDAARGOS_743 chromosome, complete genome", "reads_in": 159130, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009730295.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Gemella sanguinis strain FDAARGOS_742 chromosome, complete genome", "reads_in": 119774, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016026215.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_945 chromosome, complete genome", "reads_in": 371304, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016026335.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_943 chromosome, complete genome", "reads_in": 362216, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812185.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Klebsiella aerogenes strain FDAARGOS_513 chromosome, complete genome", "reads_in": 351446, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016028435.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Sphingomonas paucimobilis strain FDAARGOS_881 chromosome, complete genome", "reads_in": 301070, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009730455.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Cupriavidus metallidurans strain FDAARGOS_675 chromosome, complete genome", "reads_in": 462760, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016026535.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Delftia acidovorans strain FDAARGOS_939 chromosome, complete genome", "reads_in": 423412, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016026575.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Fannyhessea vaginae strain FDAARGOS_934 chromosome, complete genome", "reads_in": 96468, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016026595.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Brevibacterium casei strain FDAARGOS_936 chromosome, complete genome", "reads_in": 257772, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016026615.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Aeromonas allosaccharophila strain FDAARGOS_933 chromosome, complete genome", "reads_in": 308694, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016026635.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus pseudintermedius strain FDAARGOS_930 chromosome, complete genome", "reads_in": 170856, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016026675.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Clostridium perfringens strain FDAARGOS_931 chromosome, complete genome", "reads_in": 225034, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016026715.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Salmonella enterica strain FDAARGOS_928 chromosome, complete genome", "reads_in": 330094, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016026735.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Serratia rubidaea strain FDAARGOS_926 chromosome, complete genome", "reads_in": 333000, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016026755.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Lysinibacillus macroides strain FDAARGOS_925 chromosome, complete genome", "reads_in": 324176, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016026835.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Bacillus licheniformis strain FDAARGOS_923 chromosome, complete genome", "reads_in": 312892, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016026895.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Neisseria meningitidis strain FDAARGOS_914 chromosome, complete genome", "reads_in": 145422, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016027435.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Burkholderia oklahomensis strain FDAARGOS_900 chromosome 1, complete sequence", "reads_in": 487968, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016026915.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus epidermidis strain FDAARGOS_913 chromosome, complete genome", "reads_in": 175120, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016026975.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Aerococcus urinae strain FDAARGOS_911 chromosome, complete genome", "reads_in": 131618, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016026995.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Clostridium perfringens strain FDAARGOS_905 chromosome, complete genome", "reads_in": 223168, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016027015.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Bacillus cereus strain FDAARGOS_918 chromosome, complete genome", "reads_in": 363314, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016027055.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Acinetobacter johnsonii strain FDAARGOS_910 chromosome, complete genome", "reads_in": 241162, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016027075.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Delftia acidovorans strain FDAARGOS_909 chromosome, complete genome", "reads_in": 447236, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016027095.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Sphingomonas paucimobilis strain FDAARGOS_908 chromosome, complete genome", "reads_in": 291138, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016027115.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Serratia plymuthica strain FDAARGOS_907 chromosome, complete genome", "reads_in": 364598, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016027135.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Thomasclavelia ramosa strain FDAARGOS_906 chromosome, complete genome", "reads_in": 216294, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016027155.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Clostridium perfringens strain FDAARGOS_904 chromosome, complete genome", "reads_in": 224056, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016027375.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Clostridium perfringens strain FDAARGOS_903 chromosome, complete genome", "reads_in": 223682, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016027415.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Brevibacterium casei strain FDAARGOS_902 chromosome, complete genome", "reads_in": 264006, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009730495.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Clostridioides difficile strain FDAARGOS_723 chromosome, complete genome", "reads_in": 278540, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002984125.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Moraxella catarrhalis strain FDAARGOS_304 chromosome, complete genome", "reads_in": 129466, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016027695.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Enterobacter asburiae strain FDAARGOS_892 chromosome, complete genome", "reads_in": 317798, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_008693805.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Providencia stuartii strain FDAARGOS_645 chromosome, complete genome", "reads_in": 301984, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016027615.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Corynebacterium minutissimum strain FDAARGOS_894 chromosome, complete genome", "reads_in": 179730, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_012273555.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Chromobacterium violaceum strain FDAARGOS_635 chromosome, complete genome", "reads_in": 320050, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009732965.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Aerococcus viridans strain FDAARGOS_672 chromosome, complete genome", "reads_in": 138814, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002589795.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_433 chromosome, complete genome", "reads_in": 341458, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016027675.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Serratia plymuthica strain FDAARGOS_895 chromosome, complete genome", "reads_in": 356612, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009730535.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Streptococcus gordonii strain FDAARGOS_683 chromosome, complete genome", "reads_in": 146440, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013267715.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus warneri strain FDAARGOS_754 chromosome, complete genome", "reads_in": 164000, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013267735.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Rothia dentocariosa strain FDAARGOS_752 chromosome, complete genome", "reads_in": 168894, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013267775.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Bacillus cereus strain FDAARGOS_797 chromosome, complete genome", "reads_in": 374962, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013267795.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Bacillus thuringiensis strain FDAARGOS_794 chromosome, complete genome", "reads_in": 356574, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013267815.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Bacillus thuringiensis strain FDAARGOS_792 chromosome, complete genome", "reads_in": 355252, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013267835.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Neisseria mucosa strain FDAARGOS_758 chromosome, complete genome", "reads_in": 148318, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013343075.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Bacillus thuringiensis strain FDAARGOS_791 chromosome, complete genome", "reads_in": 355852, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013267555.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Bacteroides fragilis strain FDAARGOS_763 chromosome, complete genome", "reads_in": 373716, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013343095.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Achromobacter denitrificans strain FDAARGOS_787 chromosome, complete genome", "reads_in": 454436, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013343115.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Streptococcus sanguinis strain FDAARGOS_770 chromosome, complete genome", "reads_in": 157256, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013343135.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Achromobacter xylosoxidans strain FDAARGOS_789 chromosome, complete genome", "reads_in": 453762, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013343155.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Pseudomonas sp. FDAARGOS_761 chromosome, complete genome", "reads_in": 425656, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013347325.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Cupriavidus gilardii strain FDAARGOS_639 chromosome 1, complete sequence", "reads_in": 384002, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013364115.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Shigella flexneri strain FDAARGOS_691 chromosome, complete genome", "reads_in": 302920, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013364135.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Pantoea ananatis strain FDAARGOS_680 chromosome, complete genome", "reads_in": 351950, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016025875.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Prescottella equi strain FDAARGOS_952 chromosome, complete genome", "reads_in": 347914, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016026035.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Aggregatibacter actinomycetemcomitans strain FDAARGOS_948 chromosome, complete genome", "reads_in": 140358, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016026055.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Burkholderia glumae strain FDAARGOS_949 chromosome 1, complete sequence", "reads_in": 454724, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016026175.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Listeria welshimeri strain FDAARGOS_947 chromosome, complete genome", "reads_in": 195648, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016026195.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_946 chromosome, complete genome", "reads_in": 359310, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013267575.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus hominis strain FDAARGOS_762 chromosome, complete genome", "reads_in": 150368, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013267535.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Finegoldia magna strain FDAARGOS_764 chromosome, complete genome", "reads_in": 139338, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016026235.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_944 chromosome, complete genome", "reads_in": 355392, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009730515.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Streptococcus mitis strain FDAARGOS_684 chromosome, complete genome", "reads_in": 124590, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_008693785.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Roseomonas mucosa strain FDAARGOS_658 chromosome 1, complete sequence", "reads_in": 333084, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013267495.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Streptococcus salivarius strain FDAARGOS_771 chromosome, complete genome", "reads_in": 133300, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_008693745.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Vibrio parahaemolyticus strain FDAARGOS_662 chromosome 1, complete sequence", "reads_in": 343608, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002951295.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Acinetobacter nosocomialis strain FDAARGOS_129 chromosome, complete genome", "reads_in": 266846, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002954055.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus epidermidis strain FDAARGOS_161 chromosome, complete genome", "reads_in": 167716, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002951395.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Elizabethkingia anophelis strain FDAARGOS_132 chromosome, complete genome", "reads_in": 266560, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002951455.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Elizabethkingia anophelis strain FDAARGOS_134 chromosome, complete genome", "reads_in": 266560, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002951475.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Pseudomonas fulva strain FDAARGOS_167 chromosome, complete genome", "reads_in": 324340, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002951635.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_156 chromosome, complete genome", "reads_in": 370044, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002953295.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Klebsiella quasipneumoniae strain FDAARGOS_93 chromosome, complete genome", "reads_in": 367536, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002953335.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Vibrio diabolicus strain FDAARGOS_96 chromosome 1, complete sequence", "reads_in": 344910, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013267635.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Mycobacterium tuberculosis strain FDAARGOS_757 chromosome, complete genome", "reads_in": 294528, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009730555.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Mammaliicoccus fleurettii strain FDAARGOS_682 chromosome, complete genome", "reads_in": 168504, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013267675.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus capitis strain FDAARGOS_753 chromosome, complete genome", "reads_in": 166064, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812805.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Streptococcus agalactiae strain FDAARGOS_512 chromosome, complete genome", "reads_in": 142276, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812225.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Acinetobacter sp. FDAARGOS_494 chromosome, complete genome", "reads_in": 263550, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812245.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Acinetobacter sp. FDAARGOS_493 chromosome, complete genome", "reads_in": 268098, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812305.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Enterococcus sp. FDAARGOS_553 chromosome, complete genome", "reads_in": 218900, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812325.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Citrobacter freundii strain FDAARGOS_550 chromosome, complete genome", "reads_in": 331276, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812345.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Citrobacter freundii strain FDAARGOS_549 chromosome, complete genome", "reads_in": 340144, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812385.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Acinetobacter baumannii strain FDAARGOS_533 chromosome, complete genome", "reads_in": 259026, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812405.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Citrobacter koseri strain FDAARGOS_530 chromosome, complete genome", "reads_in": 305400, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812425.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus epidermidis strain FDAARGOS_529 chromosome, complete genome", "reads_in": 171828, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812445.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Streptococcus sp. FDAARGOS_522 chromosome, complete genome", "reads_in": 140076, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812465.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Streptococcus sp. FDAARGOS_521 chromosome, complete genome", "reads_in": 143800, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812505.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus hominis strain FDAARGOS_575 chromosome, complete genome", "reads_in": 150494, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812545.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Listeria monocytogenes strain FDAARGOS_555 chromosome, complete genome", "reads_in": 190138, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812565.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Listeria monocytogenes strain FDAARGOS_554 chromosome, complete genome", "reads_in": 189124, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013267695.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Streptococcus gallolyticus strain FDAARGOS_755 chromosome, complete genome", "reads_in": 150534, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812585.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Burkholderia multivorans strain FDAARGOS_548 chromosome 1, complete sequence", "reads_in": 443592, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003019925.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Pluralibacter gergoviae strain FDAARGOS_186 chromosome, complete genome", "reads_in": 360538, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812605.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Chryseobacterium indologenes strain FDAARGOS_537 chromosome, complete genome", "reads_in": 324424, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812105.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_531 chromosome, complete genome", "reads_in": 384250, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812065.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Acinetobacter baumannii strain FDAARGOS_540 chromosome, complete genome", "reads_in": 260684, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_008693765.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus aureus strain FDAARGOS_660 chromosome, complete genome", "reads_in": 191386, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002984285.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Salmonella enterica subsp. enterica serovar Typhimurium strain FDAARGOS_317 chromosome, complete genome", "reads_in": 346190, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002209045.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Campylobacter jejuni strain FDAARGOS_265 chromosome, complete genome", "reads_in": 113074, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002209025.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Campylobacter jejuni strain FDAARGOS_263 chromosome, complete genome", "reads_in": 109432, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002209005.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Campylobacter jejuni strain FDAARGOS_262 chromosome, complete genome", "reads_in": 117896, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001558375.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus saprophyticus strain FDAARGOS_137 chromosome, complete genome", "reads_in": 171946, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001019415.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus aureus strain FDAARGOS_24 chromosome, complete genome", "reads_in": 179832, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001019435.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus aureus strain FDAARGOS_25 chromosome, complete genome", "reads_in": 191900, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001019485.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus aureus strain FDAARGOS_35 chromosome, complete genome", "reads_in": 197060, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001019495.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus aureus strain FDAARGOS_40 chromosome, complete genome", "reads_in": 192622, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001019535.2", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus aureus strain FDAARGOS_42 chromosome, complete genome", "reads_in": 187956, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002953515.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Vibrio alginolyticus strain FDAARGOS_108 chromosome 1, complete sequence", "reads_in": 363068, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002953375.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Vibrio fluvialis strain FDAARGOS_100 chromosome 1, complete sequence", "reads_in": 324504, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003711125.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Enterococcus avium strain FDAARGOS_184 chromosome, complete genome", "reads_in": 248226, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002953355.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Vibrio diabolicus strain FDAARGOS_99 chromosome 1, complete sequence", "reads_in": 342120, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003798205.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Yersinia pestis strain FDAARGOS_603 chromosome, complete genome", "reads_in": 328642, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003798265.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Burkholderia pseudomallei strain FDAARGOS_592 chromosome 1, complete sequence", "reads_in": 476446, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002984565.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus cohnii strain FDAARGOS_334 chromosome, complete genome", "reads_in": 177418, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003798285.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Yersinia pseudotuberculosis strain FDAARGOS_583 chromosome, complete genome", "reads_in": 310146, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003798305.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Yersinia pseudotuberculosis strain FDAARGOS_579 chromosome, complete genome", "reads_in": 312098, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003798325.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Propionibacterium acidifaciens strain FDAARGOS_576 chromosome, complete genome", "reads_in": 204852, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003798345.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Yersinia pestis strain FDAARGOS_602 chromosome, complete genome", "reads_in": 318966, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003798385.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Yersinia pseudotuberculosis strain FDAARGOS_584 chromosome, complete genome", "reads_in": 312628, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003798405.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Yersinia pseudotuberculosis strain FDAARGOS_582 chromosome, complete genome", "reads_in": 322164, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003798425.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Yersinia pseudotuberculosis strain FDAARGOS_581 chromosome, complete genome", "reads_in": 339798, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003798445.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Yersinia pseudotuberculosis strain FDAARGOS_580 chromosome, complete genome", "reads_in": 332068, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003798465.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Cutibacterium acnes strain FDAARGOS_577 chromosome, complete genome", "reads_in": 166356, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812085.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_566 chromosome 1, complete sequence", "reads_in": 377804, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812625.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_536 chromosome, complete genome", "reads_in": 340302, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002984485.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Oligella urethralis strain FDAARGOS_329 chromosome, complete genome", "reads_in": 167422, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002984195.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Providencia rettgeri strain FDAARGOS_330 chromosome, complete genome", "reads_in": 301240, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_006364635.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Legionella pneumophila strain FDAARGOS_779 chromosome, complete genome", "reads_in": 227280, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_006364655.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Listeria monocytogenes strain FDAARGOS_778 chromosome, complete genome", "reads_in": 196732, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_006364675.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus aureus strain FDAARGOS_773 chromosome, complete genome", "reads_in": 185804, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_006364695.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_772 chromosome, complete genome", "reads_in": 358738, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_006364735.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Pseudomonas aeruginosa strain FDAARGOS_767 chromosome, complete genome", "reads_in": 417578, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812885.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Pseudomonas aeruginosa strain FDAARGOS_571 chromosome, complete genome", "reads_in": 466652, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_006364775.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus aureus strain FDAARGOS_766 chromosome, complete genome", "reads_in": 199754, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_006364795.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Pseudomonas aeruginosa strain FDAARGOS_610 chromosome, complete genome", "reads_in": 452814, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009730115.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus hominis strain FDAARGOS_747 chromosome, complete genome", "reads_in": 149472, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_008693945.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Citrobacter koseri strain FDAARGOS_646 chromosome, complete genome", "reads_in": 305402, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_006365335.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Salmonella enterica strain FDAARGOS_768 chromosome, complete genome", "reads_in": 330318, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_008693365.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Yersinia pseudotuberculosis strain FDAARGOS_665 chromosome, complete genome", "reads_in": 291134, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_008693385.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Cupriavidus pauculus strain FDAARGOS_664 chromosome 1, complete sequence", "reads_in": 470942, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_008693405.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_630 chromosome, complete genome", "reads_in": 376186, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_008693425.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_631 chromosome 1, complete sequence", "reads_in": 364570, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_008693445.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_629 chromosome, complete genome", "reads_in": 381246, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_008693465.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Klebsiella variicola strain FDAARGOS_627 chromosome, complete genome", "reads_in": 397174, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_008693505.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Streptococcus agalactiae strain FDAARGOS_669 chromosome, complete genome", "reads_in": 137712, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_008693545.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Bordetella hinzii strain FDAARGOS_621 chromosome, complete genome", "reads_in": 324030, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_008693565.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Klebsiella michiganensis strain FDAARGOS_647 chromosome, complete genome", "reads_in": 443772, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_008693585.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Streptococcus agalactiae strain FDAARGOS_670 chromosome, complete genome", "reads_in": 147382, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_008693605.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Citrobacter portucalensis strain FDAARGOS_617 chromosome, complete genome", "reads_in": 328624, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_008693625.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Vibrio parahaemolyticus strain FDAARGOS_667 chromosome 1, complete sequence", "reads_in": 333544, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_008693645.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Citrobacter werkmanii strain FDAARGOS_616 chromosome, complete genome", "reads_in": 336620, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_008693665.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Bordetella bronchiseptica strain FDAARGOS_634 chromosome, complete genome", "reads_in": 341072, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_008693685.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Vibrio vulnificus strain FDAARGOS_663 chromosome 1, complete sequence", "reads_in": 331654, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812645.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Streptococcus pyogenes strain FDAARGOS_534 chromosome, complete genome", "reads_in": 122516, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_006364495.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Bacillus subtilis strain FDAARGOS_606 chromosome, complete genome", "reads_in": 269708, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_006364355.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Vibrio furnissii strain FDAARGOS_777 chromosome 1, complete sequence", "reads_in": 332890, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_008693725.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Streptococcus dysgalactiae strain FDAARGOS_654 chromosome, complete genome", "reads_in": 150236, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_006364235.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Streptococcus pyogenes strain FDAARGOS_774 chromosome, complete genome", "reads_in": 127658, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_006364295.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_775 chromosome, complete genome", "reads_in": 371594, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812685.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Streptococcus sp. FDAARGOS_520 chromosome, complete genome", "reads_in": 139974, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812705.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Chryseobacterium arthrosphaerae strain FDAARGOS_519 chromosome, complete genome", "reads_in": 357458, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812725.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Streptococcus pyogenes strain FDAARGOS_514 chromosome, complete genome", "reads_in": 126808, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812745.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Serratia sp. FDAARGOS_506 chromosome, complete genome", "reads_in": 324182, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812765.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Gardnerella vaginalis strain FDAARGOS_568 chromosome, complete genome", "reads_in": 114440, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812785.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Cutibacterium acnes strain FDAARGOS_503 chromosome, complete genome", "reads_in": 166302, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_006364755.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Salmonella enterica strain FDAARGOS_765 chromosome, complete genome", "reads_in": 298806, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812205.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Acinetobacter sp. FDAARGOS_560 chromosome, complete genome", "reads_in": 277772, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812525.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Proteus vulgaris strain FDAARGOS_556 chromosome, complete genome", "reads_in": 277230, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009730435.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Gordonia bronchialis strain FDAARGOS_676 chromosome, complete genome", "reads_in": 353264, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812825.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Fusobacterium necrophorum strain FDAARGOS_565 chromosome, complete genome", "reads_in": 178560, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812945.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_497 chromosome, complete genome", "reads_in": 367552, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812985.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Stenotrophomonas maltophilia strain FDAARGOS_507 chromosome, complete genome", "reads_in": 305172, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812665.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Enterococcus faecalis strain FDAARGOS_528 chromosome, complete genome", "reads_in": 197576, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003813025.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Pseudomonas aeruginosa strain FDAARGOS_570 chromosome, complete genome", "reads_in": 477010, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003813005.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Pseudomonas aeruginosa strain FDAARGOS_505 chromosome, complete genome", "reads_in": 468654, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_006364035.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Listeria monocytogenes strain FDAARGOS_607 chromosome, complete genome", "reads_in": 199470, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003956025.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus cohnii strain FDAARGOS_538 chromosome, complete genome", "reads_in": 180056, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003956005.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus haemolyticus strain FDAARGOS_517 chromosome, complete genome", "reads_in": 177968, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003955985.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Corynebacterium jeikeium strain FDAARGOS_574 chromosome, complete genome", "reads_in": 168544, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812845.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Klebsiella sp. FDAARGOS_511 chromosome, complete genome", "reads_in": 419348, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003955775.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Francisella tularensis strain FDAARGOS_599 chromosome, complete genome", "reads_in": 126190, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003855135.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Shigella flexneri strain FDAARGOS_535 chromosome, complete genome", "reads_in": 324836, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003855115.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Shigella sonnei strain FDAARGOS_524 chromosome, complete genome", "reads_in": 330206, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003827915.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Staphylococcus aureus strain FDAARGOS_504 chromosome, complete genome", "reads_in": 192972, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003813065.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Bifidobacterium breve strain FDAARGOS_561 chromosome, complete genome", "reads_in": 151710, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003955795.1", "platform": "illumina", "tool": "hostile", "reads_removed": 0, "name": "Francisella tularensis strain FDAARGOS_598 chromosome, complete genome", "reads_in": 126240, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_008693885.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Klebsiella aerogenes strain FDAARGOS_641 chromosome, complete genome", "reads_in": 352422, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_008693945.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Citrobacter koseri strain FDAARGOS_646 chromosome, complete genome", "reads_in": 305402, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_008693925.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Dermabacter vaginalis strain FDAARGOS_640 chromosome, complete genome", "reads_in": 152620, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_008693905.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Enterobacter hormaechei strain FDAARGOS_642 chromosome, complete genome", "reads_in": 313208, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_008693865.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Paracoccus yeei strain FDAARGOS_643 chromosome, complete genome", "reads_in": 312652, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009806595.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Bacillus anthracis strain FDAARGOS_698 chromosome, complete genome", "reads_in": 348384, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009730135.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus hominis strain FDAARGOS_746 chromosome, complete genome", "reads_in": 158146, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009730155.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Enterococcus gallinarum strain FDAARGOS_728 chromosome, complete genome", "reads_in": 218328, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009730215.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Mycobacterium tuberculosis strain FDAARGOS_751 chromosome, complete genome", "reads_in": 296132, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009806615.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Bacillus anthracis strain FDAARGOS_701 chromosome, complete genome", "reads_in": 348494, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009806675.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Bacillus anthracis strain FDAARGOS_705 chromosome, complete genome", "reads_in": 348874, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009729895.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Salmonella enterica strain FDAARGOS_710 chromosome, complete genome", "reads_in": 307402, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_008693965.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Pseudomonas luteola strain FDAARGOS_637 chromosome 1, complete sequence", "reads_in": 393426, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009729955.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Salmonella enterica strain FDAARGOS_688 chromosome, complete genome", "reads_in": 308930, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_008694005.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Streptococcus pyogenes strain FDAARGOS_668 chromosome, complete genome", "reads_in": 122756, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009729915.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Salmonella enterica strain FDAARGOS_708 chromosome, complete genome", "reads_in": 342448, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009730115.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus hominis strain FDAARGOS_747 chromosome, complete genome", "reads_in": 149472, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009729935.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Salmonella enterica strain FDAARGOS_707 chromosome, complete genome", "reads_in": 303738, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009729975.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Salmonella enterica strain FDAARGOS_687 chromosome, complete genome", "reads_in": 337762, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009729995.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Shigella sonnei strain FDAARGOS_715 chromosome, complete genome", "reads_in": 340654, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009730015.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Salmonella enterica strain FDAARGOS_712 chromosome, complete genome", "reads_in": 311360, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002951455.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Elizabethkingia anophelis strain FDAARGOS_134 chromosome, complete genome", "reads_in": 266560, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002951395.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Elizabethkingia anophelis strain FDAARGOS_132 chromosome, complete genome", "reads_in": 266560, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009730095.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Acinetobacter lwoffii strain FDAARGOS_552 chromosome, complete genome", "reads_in": 232364, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009806635.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Bacillus anthracis strain FDAARGOS_700 chromosome, complete genome", "reads_in": 348884, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009734165.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Bacteroides ovatus strain FDAARGOS_733 chromosome, complete genome", "reads_in": 438092, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002951475.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Pseudomonas fulva strain FDAARGOS_167 chromosome, complete genome", "reads_in": 324340, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009732965.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Aerococcus viridans strain FDAARGOS_672 chromosome, complete genome", "reads_in": 138814, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009730235.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Mycobacterium tuberculosis strain FDAARGOS_750 chromosome, complete genome", "reads_in": 295644, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009730555.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Mammaliicoccus fleurettii strain FDAARGOS_682 chromosome, complete genome", "reads_in": 168504, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016727305.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Enterococcus casseliflavus strain FDAARGOS_1122 chromosome, complete genome", "reads_in": 243400, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048465.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Jonesia denitrificans strain FDAARGOS 1440 chromosome, complete genome", "reads_in": 183310, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048505.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS 1387 chromosome, complete genome", "reads_in": 348454, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048525.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Acinetobacter lwoffii strain FDAARGOS 1394 chromosome, complete genome", "reads_in": 231122, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048545.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Providencia rettgeri strain FDAARGOS 1451 chromosome, complete genome", "reads_in": 292254, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048565.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Aeromonas sp. FDAARGOS 1406 chromosome, complete genome", "reads_in": 309722, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048585.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Aeromonas sp. FDAARGOS 1418 chromosome, complete genome", "reads_in": 296936, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048605.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS 1375 chromosome, complete genome", "reads_in": 349518, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048625.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Enterobacter hormaechei strain FDAARGOS 1435 chromosome, complete genome", "reads_in": 317498, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048645.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Streptococcus mutans strain FDAARGOS 1458 chromosome, complete genome", "reads_in": 134624, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001019435.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus aureus strain FDAARGOS_25 chromosome, complete genome", "reads_in": 191900, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016727365.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Corynebacterium ulcerans strain FDAARGOS_1118 chromosome, complete genome", "reads_in": 163584, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016726205.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Acinetobacter ursingii strain FDAARGOS_1095 chromosome, complete genome", "reads_in": 240834, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016727325.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Enterococcus casseliflavus strain FDAARGOS_1121 chromosome, complete genome", "reads_in": 243364, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009806655.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Bacillus anthracis strain FDAARGOS_706 chromosome, complete genome", "reads_in": 348660, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009730255.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Lactobacillus jensenii strain FDAARGOS_749 chromosome, complete genome", "reads_in": 107268, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009730275.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Lactobacillus crispatus strain FDAARGOS_743 chromosome, complete genome", "reads_in": 159130, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009730315.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Gemella morbillorum strain FDAARGOS_741 chromosome, complete genome", "reads_in": 118630, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009730335.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Schaalia odontolytica strain FDAARGOS_732 chromosome, complete genome", "reads_in": 162176, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002944995.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus epidermidis strain FDAARGOS_153 chromosome, complete genome", "reads_in": 170132, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009730395.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Campylobacter coli strain FDAARGOS_735 chromosome, complete genome", "reads_in": 114436, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009730415.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Micrococcus luteus strain FDAARGOS_677 chromosome, complete genome", "reads_in": 165218, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009730435.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Gordonia bronchialis strain FDAARGOS_676 chromosome, complete genome", "reads_in": 353264, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009730455.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Cupriavidus metallidurans strain FDAARGOS_675 chromosome, complete genome", "reads_in": 462760, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009730475.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Corynebacterium xerosis strain FDAARGOS_674 chromosome, complete genome", "reads_in": 188102, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009730515.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Streptococcus mitis strain FDAARGOS_684 chromosome, complete genome", "reads_in": 124590, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009730535.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Streptococcus gordonii strain FDAARGOS_683 chromosome, complete genome", "reads_in": 146440, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009730575.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Shewanella putrefaciens strain FDAARGOS_681 chromosome, complete genome", "reads_in": 310642, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009806695.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Bacillus anthracis strain FDAARGOS_704 chromosome, complete genome", "reads_in": 348496, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812805.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Streptococcus agalactiae strain FDAARGOS_512 chromosome, complete genome", "reads_in": 142276, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013267715.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus warneri strain FDAARGOS_754 chromosome, complete genome", "reads_in": 164000, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013267415.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Abiotrophia defectiva strain FDAARGOS_785 chromosome, complete genome", "reads_in": 136456, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002591175.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus capitis strain FDAARGOS_378 chromosome, complete genome", "reads_in": 165676, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013267475.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Bacillus cereus strain FDAARGOS_780 chromosome, complete genome", "reads_in": 376156, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013267495.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Streptococcus salivarius strain FDAARGOS_771 chromosome, complete genome", "reads_in": 133300, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013267535.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Finegoldia magna strain FDAARGOS_764 chromosome, complete genome", "reads_in": 139338, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002591115.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Klebsiella aerogenes strain FDAARGOS_363 chromosome, complete genome", "reads_in": 349698, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013267575.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus hominis strain FDAARGOS_762 chromosome, complete genome", "reads_in": 150368, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002591075.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_439 chromosome, complete genome", "reads_in": 383526, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_008693845.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Pasteurella multocida strain FDAARGOS_644 chromosome, complete genome", "reads_in": 150004, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_008693825.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Pseudomonas oryzihabitans strain FDAARGOS_657 chromosome, complete genome", "reads_in": 350992, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002951635.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_156 chromosome, complete genome", "reads_in": 370044, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_008693785.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Roseomonas mucosa strain FDAARGOS_658 chromosome 1, complete sequence", "reads_in": 333084, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812525.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Proteus vulgaris strain FDAARGOS_556 chromosome, complete genome", "reads_in": 277230, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812545.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Listeria monocytogenes strain FDAARGOS_555 chromosome, complete genome", "reads_in": 190138, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812565.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Listeria monocytogenes strain FDAARGOS_554 chromosome, complete genome", "reads_in": 189124, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002983965.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Salmonella enterica subsp. enterica serovar Typhimurium strain FDAARGOS_320 chromosome, complete genome", "reads_in": 335804, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812605.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Chryseobacterium indologenes strain FDAARGOS_537 chromosome, complete genome", "reads_in": 324424, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812625.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_536 chromosome, complete genome", "reads_in": 340302, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812645.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Streptococcus pyogenes strain FDAARGOS_534 chromosome, complete genome", "reads_in": 122516, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812665.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Enterococcus faecalis strain FDAARGOS_528 chromosome, complete genome", "reads_in": 197576, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812685.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Streptococcus sp. FDAARGOS_520 chromosome, complete genome", "reads_in": 139974, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812705.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Chryseobacterium arthrosphaerae strain FDAARGOS_519 chromosome, complete genome", "reads_in": 357458, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812725.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Streptococcus pyogenes strain FDAARGOS_514 chromosome, complete genome", "reads_in": 126808, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812745.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Serratia sp. FDAARGOS_506 chromosome, complete genome", "reads_in": 324182, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812765.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Gardnerella vaginalis strain FDAARGOS_568 chromosome, complete genome", "reads_in": 114440, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812785.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Cutibacterium acnes strain FDAARGOS_503 chromosome, complete genome", "reads_in": 166302, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048445.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS 1382 chromosome, complete genome", "reads_in": 337060, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002589795.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_433 chromosome, complete genome", "reads_in": 341458, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013267695.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Streptococcus gallolyticus strain FDAARGOS_755 chromosome, complete genome", "reads_in": 150534, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013267395.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Achromobacter denitrificans strain FDAARGOS_786 chromosome, complete genome", "reads_in": 449124, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002591235.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Pseudomonas sp. FDAARGOS_380 chromosome, complete genome", "reads_in": 433290, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013267735.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Rothia dentocariosa strain FDAARGOS_752 chromosome, complete genome", "reads_in": 168894, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002588845.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Serratia fonticola strain FDAARGOS_411 chromosome, complete genome", "reads_in": 381692, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002587885.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_446 chromosome, complete genome", "reads_in": 384644, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013267815.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Bacillus thuringiensis strain FDAARGOS_792 chromosome, complete genome", "reads_in": 355252, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002587225.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Campylobacter jejuni strain FDAARGOS_421 chromosome, complete genome", "reads_in": 118520, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013331815.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Bacillus tropicus strain FDAARGOS_782 chromosome, complete genome", "reads_in": 353154, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002587165.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_444 chromosome, complete genome", "reads_in": 392066, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002587105.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Campylobacter jejuni strain FDAARGOS_422 chromosome, complete genome", "reads_in": 112690, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013343135.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Achromobacter xylosoxidans strain FDAARGOS_789 chromosome, complete genome", "reads_in": 453762, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013343155.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Pseudomonas sp. FDAARGOS_761 chromosome, complete genome", "reads_in": 425656, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013347325.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Cupriavidus gilardii strain FDAARGOS_639 chromosome 1, complete sequence", "reads_in": 384002, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002407165.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus lugdunensis strain FDAARGOS_377 chromosome, complete genome", "reads_in": 171442, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013364135.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Pantoea ananatis strain FDAARGOS_680 chromosome, complete genome", "reads_in": 351950, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002393485.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Ralstonia pickettii strain FDAARGOS_410 chromosome 1, complete sequence", "reads_in": 321204, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016026035.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Aggregatibacter actinomycetemcomitans strain FDAARGOS_948 chromosome, complete genome", "reads_in": 140358, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002393465.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_401 chromosome, complete genome", "reads_in": 359158, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016026175.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Listeria welshimeri strain FDAARGOS_947 chromosome, complete genome", "reads_in": 195648, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013267675.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus capitis strain FDAARGOS_753 chromosome, complete genome", "reads_in": 166064, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013267655.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Mycobacterium tuberculosis strain FDAARGOS_756 chromosome, complete genome", "reads_in": 294306, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013267635.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Mycobacterium tuberculosis strain FDAARGOS_757 chromosome, complete genome", "reads_in": 294528, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013267375.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Achromobacter denitrificans strain FDAARGOS_788 chromosome, complete genome", "reads_in": 449126, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002754815.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Roseomonas sp. FDAARGOS_362 chromosome, complete genome", "reads_in": 344946, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002591295.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Pasteurella multocida strain FDAARGOS_385 chromosome, complete genome", "reads_in": 156448, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013267235.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Bacillus cereus strain FDAARGOS_802 chromosome, complete genome", "reads_in": 404570, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002591275.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Chryseobacterium indologenes strain FDAARGOS_379 chromosome, complete genome", "reads_in": 325922, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002591255.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Brucella canis strain FDAARGOS_420 chromosome 1, complete sequence", "reads_in": 220852, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013267295.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Bacillus thuringiensis strain FDAARGOS_796 chromosome, complete genome", "reads_in": 413816, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002591215.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus lugdunensis strain FDAARGOS_381 chromosome, complete genome", "reads_in": 171380, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048425.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS 1380 chromosome, complete genome", "reads_in": 349396, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020097615.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Corynebacterium falsenii strain FDAARGOS_1493 chromosome, complete genome", "reads_in": 170792, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048385.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Pantoea agglomerans strain FDAARGOS 1447 chromosome, complete genome", "reads_in": 312802, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735645.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Acidaminococcus fermentans DSM 20731 strain FDAARGOS_1571 chromosome, complete genome", "reads_in": 155546, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735665.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Lachnospiraceae bacterium GAM79 strain FDAARGOS_1585 chromosome, complete genome", "reads_in": 197388, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735685.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Lachnospiraceae bacterium GAM79 strain FDAARGOS_1582 chromosome, complete genome", "reads_in": 198604, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735705.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Coprococcus eutactus ATCC 27759 strain FDAARGOS_1581 chromosome, complete genome", "reads_in": 207556, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735745.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Lachnospira eligens strain FDAARGOS_1570 chromosome, complete genome", "reads_in": 184852, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735765.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Streptococcus anginosus subsp. anginosus strain FDAARGOS_1569 chromosome, complete genome", "reads_in": 125088, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001018685.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus aureus strain FDAARGOS_4 chromosome, complete genome", "reads_in": 187452, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020736005.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Mycobacterium intracellulare strain FDAARGOS_1564 chromosome, complete genome", "reads_in": 376386, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735845.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Tsukamurella paurometabola strain FDAARGOS_1558 chromosome, complete genome", "reads_in": 313538, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735865.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Parabacteroides johnsonii strain FDAARGOS_1580 chromosome, complete genome", "reads_in": 317744, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735885.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Bordetella pertussis strain FDAARGOS_1542 chromosome, complete genome", "reads_in": 274846, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735905.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Prevotella melaninogenica strain FDAARGOS_1567 chromosome 1, complete sequence", "reads_in": 220450, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735925.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Bordetella parapertussis strain FDAARGOS_1541 chromosome, complete genome", "reads_in": 318372, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001018655.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus aureus strain FDAARGOS_6 chromosome, complete genome", "reads_in": 183952, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735965.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Bordetella parapertussis strain FDAARGOS_1540 chromosome, complete genome", "reads_in": 318370, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735985.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Bordetella holmesii strain FDAARGOS_1539 chromosome, complete genome", "reads_in": 245874, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020097175.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Aeromonas enteropelogenes strain FDAARGOS_1536 chromosome, complete genome", "reads_in": 288620, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001019125.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus aureus strain FDAARGOS_43 chromosome, complete genome", "reads_in": 188042, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019931025.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Clostridioides difficile strain FDAARGOS_1469 chromosome, complete genome", "reads_in": 269708, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048005.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1385 chromosome, complete genome", "reads_in": 348010, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019047785.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Enterobacter cancerogenus strain FDAARGOS 1428 chromosome, complete genome", "reads_in": 321448, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019047805.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Corynebacterium freneyi strain FDAARGOS 1426 chromosome, complete genome", "reads_in": 192120, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019047885.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Providencia rettgeri strain FDAARGOS 1449 chromosome, complete genome", "reads_in": 318484, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019047905.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1388 chromosome, complete genome", "reads_in": 345864, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019047925.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Klebsiella aerogenes strain FDAARGOS 1441 chromosome, complete genome", "reads_in": 346180, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019047945.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Enterobacter hormaechei strain FDAARGOS 1434 chromosome, complete genome", "reads_in": 318836, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019047965.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1378 chromosome, complete genome", "reads_in": 333822, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735625.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Finegoldia magna strain FDAARGOS_1557 chromosome, complete genome", "reads_in": 132318, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019047985.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Klebsiella quasipneumoniae strain FDAARGOS 1443 chromosome, complete genome", "reads_in": 359656, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735605.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Parabacteroides merdae strain FDAARGOS_1586 chromosome, complete genome", "reads_in": 291720, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001018725.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus aureus strain FDAARGOS_10 chromosome, complete genome", "reads_in": 191592, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812825.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Fusobacterium necrophorum strain FDAARGOS_565 chromosome, complete genome", "reads_in": 178560, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020097655.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Corynebacterium coyleae strain FDAARGOS_1492 chromosome, complete genome", "reads_in": 172188, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020099175.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Klebsiella quasipneumoniae strain FDAARGOS_1503 chromosome, complete genome", "reads_in": 366590, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001018735.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus aureus strain FDAARGOS_12 chromosome, complete genome", "reads_in": 181716, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735565.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Hoylesella buccalis strain FDAARGOS_1588 chromosome, complete genome", "reads_in": 220560, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016939475.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Acinetobacter calcoaceticus strain FDAARGOS_1218 chromosome, complete genome", "reads_in": 261016, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020736265.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Bordetella pertussis strain FDAARGOS_1543 chromosome, complete genome", "reads_in": 273102, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020736085.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Corynebacterium diphtheriae subsp. diphtheriae strain FDAARGOS_1554 chromosome, complete genome", "reads_in": 164242, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020736105.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Corynebacterium diphtheriae strain FDAARGOS_1553 chromosome, complete genome", "reads_in": 166080, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020736125.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Corynebacterium diphtheriae strain FDAARGOS_1552 chromosome, complete genome", "reads_in": 157962, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020736145.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Campylobacter jejuni strain FDAARGOS_1546 chromosome, complete genome", "reads_in": 108536, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020736165.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Capnocytophaga ochracea strain FDAARGOS_1550 chromosome, complete genome", "reads_in": 180882, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020736185.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Campylobacter gracilis strain FDAARGOS_1548 chromosome, complete genome", "reads_in": 152116, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020736205.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Campylobacter rectus strain FDAARGOS_1549 chromosome, complete genome", "reads_in": 171454, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020736225.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Campylobacter curvus strain FDAARGOS_1547 chromosome, complete genome", "reads_in": 131660, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_000783935.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Raoultella planticola strain FDAARGOS_64 chromosome, complete genome", "reads_in": 388262, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_000783955.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Morganella morganii strain FDAARGOS_63 chromosome, complete genome", "reads_in": 266668, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_021183605.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Nocardia asteroides strain FDAARGOS_1621 chromosome, complete genome", "reads_in": 477942, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_000783815.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Salmonella enterica strain FDAARGOS_70 chromosome, complete genome", "reads_in": 319106, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_021183645.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Nocardia gipuzkoensis strain FDAARGOS_1619 chromosome, complete genome", "reads_in": 541242, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_021183725.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Mycobacterium ostraviense strain FDAARGOS_1613 chromosome, complete genome", "reads_in": 408516, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_021183745.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Mycobacterium intracellulare strain FDAARGOS_1612 chromosome, complete genome", "reads_in": 371710, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_021183765.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Mycobacterium intracellulare strain FDAARGOS_1611 chromosome, complete genome", "reads_in": 365956, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_021183785.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Mycobacterium intracellulare subsp. intracellulare strain FDAARGOS_1610 chromosome, complete genome", "reads_in": 360844, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_021183805.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Mycobacterium avium subsp. avium strain FDAARGOS_1609 chromosome, complete genome", "reads_in": 330370, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020736065.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Finegoldia magna strain FDAARGOS_1556 chromosome, complete genome", "reads_in": 115922, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001018645.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus aureus strain FDAARGOS_2 chromosome, complete genome", "reads_in": 187334, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735785.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Prevotella melaninogenica strain FDAARGOS_1566 chromosome 1, complete sequence", "reads_in": 211214, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048405.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Acinetobacter seifertii strain FDAARGOS 1400 chromosome, complete genome", "reads_in": 266604, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048025.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus haemolyticus strain FDAARGOS 1453 chromosome, complete genome", "reads_in": 171250, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048045.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Citrobacter braakii strain FDAARGOS 1422 chromosome, complete genome", "reads_in": 321184, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001019495.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus aureus strain FDAARGOS_40 chromosome, complete genome", "reads_in": 192622, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019930785.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Burkholderia dolosa strain FDAARGOS_1463 chromosome 1, complete sequence", "reads_in": 413044, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019056115.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS 1383 chromosome, complete genome", "reads_in": 337060, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019930605.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Prevotella nigrescens strain FDAARGOS_1486 chromosome, complete genome", "reads_in": 178206, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019930625.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Nocardia asteroides strain FDAARGOS_1485 chromosome, complete genome", "reads_in": 494394, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019930645.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Gordonia bronchialis strain FDAARGOS_1476 chromosome, complete genome", "reads_in": 353808, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001019305.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus aureus strain FDAARGOS_9 chromosome, complete genome", "reads_in": 187646, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019930725.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Capnocytophaga ochracea strain FDAARGOS_1468 chromosome, complete genome", "reads_in": 174364, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019930745.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Mycolicibacterium smegmatis strain FDAARGOS_1484 chromosome, complete genome", "reads_in": 462514, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019930765.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Legionella pneumophila strain FDAARGOS_1483 chromosome, complete genome", "reads_in": 227276, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019930805.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Legionella pneumophila strain FDAARGOS_1482 chromosome, complete genome", "reads_in": 229162, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048305.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Acinetobacter lwoffii strain FDAARGOS 1393 chromosome, complete genome", "reads_in": 233634, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019930845.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Eikenella corrodens strain FDAARGOS_1474 chromosome, complete genome", "reads_in": 146172, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019930865.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Campylobacter coli strain FDAARGOS_1466 chromosome, complete genome", "reads_in": 122190, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019930885.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Legionella anisa strain FDAARGOS_1480 chromosome, complete genome", "reads_in": 296538, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001019255.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus aureus strain FDAARGOS_1 chromosome, complete genome", "reads_in": 190366, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019930925.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Bartonella henselae strain FDAARGOS_1462 chromosome, complete genome", "reads_in": 129782, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001019205.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus aureus strain FDAARGOS_48 chromosome, complete genome", "reads_in": 189826, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019930965.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Enterobacter ludwigii strain FDAARGOS_1475 chromosome, complete genome", "reads_in": 315382, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019930985.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Campylobacter coli strain FDAARGOS_1465 chromosome, complete genome", "reads_in": 128994, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048805.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Citrobacter braakii strain FDAARGOS 1421 chromosome, complete genome", "reads_in": 371974, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001019395.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus aureus strain FDAARGOS_20 chromosome, complete genome", "reads_in": 187888, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001019415.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus aureus strain FDAARGOS_24 chromosome, complete genome", "reads_in": 179832, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048705.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Aeromonas sp. FDAARGOS 1409 chromosome, complete genome", "reads_in": 291864, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048325.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Enterococcus durans strain FDAARGOS 1437 chromosome, complete genome", "reads_in": 206380, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048345.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS 1377 chromosome, complete genome", "reads_in": 356918, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048365.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Aeromonas sp. FDAARGOS 1415 chromosome, complete genome", "reads_in": 289762, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048245.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Enterobacter hormaechei strain FDAARGOS 1433 chromosome, complete genome", "reads_in": 327728, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019047745.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1381 chromosome, complete genome", "reads_in": 349546, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001019535.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus aureus strain FDAARGOS_42 chromosome, complete genome", "reads_in": 187956, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019047505.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Streptococcus gordonii strain FDAARGOS 1455 chromosome, complete genome", "reads_in": 143654, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048065.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Citrobacter farmeri strain FDAARGOS 1423 chromosome, complete genome", "reads_in": 330986, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001188045.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Listeria monocytogenes strain FDAARGOS_57 chromosome unnamed, complete genome", "reads_in": 195692, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001188025.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Listeria monocytogenes strain FDAARGOS_58 chromosome, complete genome", "reads_in": 193778, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048125.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Klebsiella aerogenes strain FDAARGOS 1442 chromosome, complete genome", "reads_in": 352118, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048145.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS 1386 chromosome, complete genome", "reads_in": 346162, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048165.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Corynebacterium coyleae strain FDAARGOS 1425 chromosome, complete genome", "reads_in": 167350, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001019575.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus aureus strain FDAARGOS_47 chromosome, complete genome", "reads_in": 193428, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019047765.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Citrobacter pasteurii strain FDAARGOS 1424 chromosome, complete genome", "reads_in": 334754, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019047725.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS 1374 chromosome, complete genome", "reads_in": 349518, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019931005.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Granulicatella adiacens strain FDAARGOS_1477 chromosome, complete genome", "reads_in": 131028, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001558355.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Salmonella enterica strain FDAARGOS_94 chromosome, complete genome", "reads_in": 330094, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019046905.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Enterobacter bugandensis strain FDAARGOS 1427 chromosome, complete genome", "reads_in": 335846, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001558435.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Vibrio harveyi strain FDAARGOS_107 chromosome 1, complete sequence", "reads_in": 402594, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019046945.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Streptococcus gordonii strain FDAARGOS 1454 chromosome, complete genome", "reads_in": 145842, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019046965.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Acinetobacter pittii strain FDAARGOS 1398 chromosome, complete genome", "reads_in": 255868, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001558395.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Bordetella pertussis strain FDAARGOS_179 chromosome, complete genome", "reads_in": 273390, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019047005.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Enterobacter ludwigii strain FDAARGOS 1436 chromosome, complete genome", "reads_in": 329562, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019047025.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Enterobacter roggenkampii strain FDAARGOS 1430 chromosome, complete genome", "reads_in": 327818, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001558375.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus saprophyticus strain FDAARGOS_137 chromosome, complete genome", "reads_in": 171946, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001558295.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Shigella sonnei strain FDAARGOS_90 chromosome, complete genome", "reads_in": 332556, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019047645.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1376 chromosome, complete genome", "reads_in": 356636, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001558275.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus saprophyticus strain FDAARGOS_168 chromosome, complete genome", "reads_in": 176676, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001188125.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Salmonella enterica subsp. enterica serovar Heidelberg strain FDAARGOS_54 chromosome, complete genome", "reads_in": 316752, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019047165.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1384 chromosome, complete genome", "reads_in": 347978, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019047185.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Acinetobacter pittii strain FDAARGOS 1397 chromosome, complete genome", "reads_in": 278826, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019047205.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Acinetobacter pittii strain FDAARGOS 1399 chromosome, complete genome", "reads_in": 265342, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019047465.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Lelliottia amnigena strain FDAARGOS 1444 chromosome, complete genome", "reads_in": 300368, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016727345.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Enterococcus casseliflavus strain FDAARGOS_1120 chromosome, complete genome", "reads_in": 236566, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812845.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Klebsiella sp. FDAARGOS_511 chromosome, complete genome", "reads_in": 419348, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016726345.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Acinetobacter ursingii strain FDAARGOS_1096 chromosome, complete genome", "reads_in": 232482, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812905.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Pseudomonas aeruginosa strain FDAARGOS_501 chromosome, complete genome", "reads_in": 457722, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016026375.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_941 chromosome, complete genome", "reads_in": 337554, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127455.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus equorum strain FDAARGOS_1068 chromosome, complete genome", "reads_in": 184822, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002386305.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Morganella morganii strain FDAARGOS_365 chromosome, complete genome", "reads_in": 271714, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016027095.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Sphingomonas paucimobilis strain FDAARGOS_908 chromosome, complete genome", "reads_in": 291138, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002386265.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Weissella paramesenteroides strain FDAARGOS_414 chromosome, complete genome", "reads_in": 129860, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016027135.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Thomasclavelia ramosa strain FDAARGOS_906 chromosome, complete genome", "reads_in": 216294, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016027155.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Clostridium perfringens strain FDAARGOS_904 chromosome, complete genome", "reads_in": 224056, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016027375.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Clostridium perfringens strain FDAARGOS_903 chromosome, complete genome", "reads_in": 223682, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016027415.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Brevibacterium casei strain FDAARGOS_902 chromosome, complete genome", "reads_in": 264006, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002386245.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus aureus strain FDAARGOS_412 chromosome, complete genome", "reads_in": 184640, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016027455.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Burkholderia humptydooensis strain FDAARGOS_899 chromosome 1, complete sequence", "reads_in": 485776, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002386205.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Burkholderia thailandensis strain FDAARGOS_426 chromosome 1, complete sequence", "reads_in": 448658, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127475.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Anaerococcus vaginalis strain FDAARGOS_988 chromosome, complete genome", "reads_in": 126264, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016027615.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Corynebacterium minutissimum strain FDAARGOS_894 chromosome, complete genome", "reads_in": 179730, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016027695.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Enterobacter asburiae strain FDAARGOS_892 chromosome, complete genome", "reads_in": 317798, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002386165.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Salmonella enterica subsp. enterica serovar Paratyphi A strain FDAARGOS_368 chromosome, complete genome", "reads_in": 305866, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002278015.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Enterococcus hirae strain FDAARGOS_234 chromosome, complete genome", "reads_in": 189710, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002277995.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Elizabethkingia anophelis strain FDAARGOS_198 chromosome, complete genome", "reads_in": 271614, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016027835.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Serratia plymuthica strain FDAARGOS_889 chromosome, complete genome", "reads_in": 362786, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002209265.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus saprophyticus strain FDAARGOS_355 chromosome, complete genome", "reads_in": 175946, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016027975.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Lactococcus lactis strain FDAARGOS_887 chromosome, complete genome", "reads_in": 172198, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016027035.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Achromobacter xylosoxidans strain FDAARGOS_915 chromosome, complete genome", "reads_in": 454194, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002386325.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Salmonella enterica subsp. enterica serovar Saintpaul strain FDAARGOS_373 chromosome, complete genome", "reads_in": 322540, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016026995.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Clostridium perfringens strain FDAARGOS_905 chromosome, complete genome", "reads_in": 223168, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016026675.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Clostridium perfringens strain FDAARGOS_931 chromosome, complete genome", "reads_in": 225034, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002393385.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Pasteurella multocida strain FDAARGOS_384 chromosome, complete genome", "reads_in": 156448, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002386185.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus simulans strain FDAARGOS_383 chromosome, complete genome", "reads_in": 191464, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127535.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Streptococcus salivarius strain FDAARGOS_1045 chromosome, complete genome", "reads_in": 154588, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002208745.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Pseudomonas protegens strain FDAARGOS_307 chromosome, complete genome", "reads_in": 471722, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016725825.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Enterobacter hormaechei strain FDAARGOS_1078 chromosome, complete genome", "reads_in": 338684, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016726885.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Pediococcus pentosaceus strain FDAARGOS_1134 chromosome, complete genome", "reads_in": 116092, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002073815.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Streptococcus sp. FDAARGOS_192 chromosome, complete genome", "reads_in": 162368, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016726965.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Mobiluncus curtisii strain FDAARGOS_1128 chromosome, complete genome", "reads_in": 144282, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002073795.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Streptococcus sp. 'group B' strain FDAARGOS_229 chromosome, complete genome", "reads_in": 145218, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016727005.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Myroides odoratus strain FDAARGOS_1132 chromosome, complete genome", "reads_in": 277848, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016727125.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Enterococcus hirae strain FDAARGOS_1124 chromosome, complete genome", "reads_in": 195226, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016727265.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Enterococcus hirae strain FDAARGOS_1125 chromosome, complete genome", "reads_in": 204300, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002073835.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Streptococcus salivarius strain FDAARGOS_259 chromosome, complete genome", "reads_in": 150622, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002208705.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Mycobacterium avium subsp. paratuberculosis strain FDAARGOS_305 chromosome, complete genome", "reads_in": 322166, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016724805.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Sutterella wadsworthensis strain FDAARGOS_1159 chromosome, complete genome", "reads_in": 201768, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016725225.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Mobiluncus curtisii strain FDAARGOS_1127 chromosome, complete genome", "reads_in": 143540, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016724825.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Weizmannia coagulans strain FDAARGOS_1160 chromosome, complete genome", "reads_in": 222856, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016724845.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Sphingobacterium spiritivorum strain FDAARGOS_1146 chromosome, complete genome", "reads_in": 351684, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002208805.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus pettenkoferi strain FDAARGOS_288 chromosome, complete genome", "reads_in": 166824, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016724885.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Streptococcus dysgalactiae strain FDAARGOS_1157 chromosome, complete genome", "reads_in": 143412, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016725005.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Streptococcus constellatus strain FDAARGOS_1156 chromosome, complete genome", "reads_in": 126884, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016725165.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Streptococcus lutetiensis strain FDAARGOS_1158 chromosome, complete genome", "reads_in": 119568, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002208785.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Bacillus anthracis strain FDAARGOS_341 chromosome, complete genome", "reads_in": 360696, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016725205.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus pasteuri strain FDAARGOS_1152 chromosome, complete genome", "reads_in": 170422, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016725245.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Mammaliicoccus vitulinus strain FDAARGOS_1153 chromosome, complete genome", "reads_in": 169568, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002208725.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Prevotella melaninogenica strain FDAARGOS_306 chromosome 1, complete sequence", "reads_in": 211214, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016725265.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus equorum strain FDAARGOS_1150 chromosome, complete genome", "reads_in": 186494, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016725285.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus pasteuri strain FDAARGOS_1151 chromosome, complete genome", "reads_in": 175648, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016725305.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus equorum strain FDAARGOS_1149 chromosome, complete genome", "reads_in": 182404, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016725325.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Sphingobacterium spiritivorum strain FDAARGOS_1145 chromosome, complete genome", "reads_in": 342354, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002208765.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Psychrobacter cryohalolentis strain FDAARGOS_308 chromosome, complete genome", "reads_in": 207242, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016725645.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Sphingobacterium spiritivorum strain FDAARGOS_1144 chromosome, complete genome", "reads_in": 342552, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016026415.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Corynebacterium amycolatum strain FDAARGOS_938 chromosome, complete genome", "reads_in": 186842, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002393365.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_403 chromosome, complete genome", "reads_in": 365028, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016026555.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Brevibacterium casei strain FDAARGOS_937 chromosome, complete genome", "reads_in": 258770, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002393265.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Enterococcus sp. FDAARGOS_375 chromosome, complete genome", "reads_in": 236622, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002209065.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Campylobacter jejuni strain FDAARGOS_266 chromosome, complete genome", "reads_in": 108030, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127395.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Corynebacterium glucuronolyticum strain FDAARGOS_1053 chromosome, complete genome", "reads_in": 194320, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002209025.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Campylobacter jejuni strain FDAARGOS_263 chromosome, complete genome", "reads_in": 109432, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127435.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Salmonella enterica strain FDAARGOS_1067 chromosome, complete genome", "reads_in": 337820, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127015.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Corynebacterium tuberculostearicum strain FDAARGOS_993 chromosome, complete genome", "reads_in": 160572, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016126975.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Pediococcus pentosaceus strain FDAARGOS_1011 chromosome, complete genome", "reads_in": 119050, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016028275.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus carnosus strain FDAARGOS_883 chromosome, complete genome", "reads_in": 171424, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002209245.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Shewanella sp. FDAARGOS_354 chromosome, complete genome", "reads_in": 338000, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016028295.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus auricularis strain FDAARGOS_882 chromosome, complete genome", "reads_in": 147772, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016028435.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Sphingomonas paucimobilis strain FDAARGOS_881 chromosome, complete genome", "reads_in": 301070, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_000783435.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Achromobacter insolitus strain FDAARGOS_88 chromosome, complete genome", "reads_in": 459536, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016028515.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Stutzerimonas frequens strain FDAARGOS_877 chromosome, complete genome", "reads_in": 292980, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016028635.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Stutzerimonas degradans strain FDAARGOS_876 chromosome, complete genome", "reads_in": 269474, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016028655.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Stutzerimonas stutzeri strain FDAARGOS_875 chromosome, complete genome", "reads_in": 302358, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016028675.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Proteus penneri strain FDAARGOS_874 chromosome, complete genome", "reads_in": 256720, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016028695.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Pediococcus pentosaceus strain FDAARGOS_873 chromosome, complete genome", "reads_in": 116058, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016028735.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Oligella ureolytica strain FDAARGOS_872 chromosome, complete genome", "reads_in": 188856, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016126955.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Pseudomonas aeruginosa strain FDAARGOS_1041 chromosome, complete genome", "reads_in": 456166, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002209225.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Salmonella enterica subsp. enterica serovar Typhimurium strain FDAARGOS_321 chromosome, complete genome", "reads_in": 334714, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002209205.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Salmonella enterica subsp. enterica serovar Hadar strain FDAARGOS_313 chromosome, complete genome", "reads_in": 329616, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016028795.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Macrococcus caseolyticus strain FDAARGOS_868 chromosome, complete genome", "reads_in": 144730, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016028815.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Lactococcus lactis strain FDAARGOS_866 chromosome, complete genome", "reads_in": 173712, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016028835.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Lactococcus lactis strain FDAARGOS_865 chromosome, complete genome", "reads_in": 171814, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016028855.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Rothia kristinae strain FDAARGOS_864 chromosome, complete genome", "reads_in": 158516, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016126915.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Corynebacterium minutissimum strain FDAARGOS_992 chromosome, complete genome", "reads_in": 179734, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002209185.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Corynebacterium jeikeium strain FDAARGOS_328 chromosome, complete genome", "reads_in": 176726, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_000626615.3", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus aureus strain FDAARGOS_5 chromosome, complete genome", "reads_in": 187346, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127335.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus pseudintermedius strain FDAARGOS_1072 chromosome, complete genome", "reads_in": 189290, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016726765.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Pediococcus acidilactici strain FDAARGOS_1133 chromosome, complete genome", "reads_in": 130226, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127315.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Achromobacter deleyi strain FDAARGOS_1050 chromosome, complete genome", "reads_in": 465818, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127255.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Leuconostoc pseudomesenteroides strain FDAARGOS_1003 chromosome, complete genome", "reads_in": 140978, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016026595.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Brevibacterium casei strain FDAARGOS_936 chromosome, complete genome", "reads_in": 257772, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002393245.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Citrobacter koseri strain FDAARGOS_393 chromosome, complete genome", "reads_in": 325632, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016026635.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus pseudintermedius strain FDAARGOS_930 chromosome, complete genome", "reads_in": 170856, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002386385.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Citrobacter werkmanii strain FDAARGOS_364 chromosome, complete genome", "reads_in": 325674, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016026975.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Aerococcus urinae strain FDAARGOS_911 chromosome, complete genome", "reads_in": 131618, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016026715.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Salmonella enterica strain FDAARGOS_928 chromosome, complete genome", "reads_in": 330094, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016026735.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Serratia rubidaea strain FDAARGOS_926 chromosome, complete genome", "reads_in": 333000, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016026755.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Lysinibacillus macroides strain FDAARGOS_925 chromosome, complete genome", "reads_in": 324176, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016026835.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Bacillus licheniformis strain FDAARGOS_923 chromosome, complete genome", "reads_in": 312892, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016026855.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Bacillus tropicus strain FDAARGOS_920 chromosome, complete genome", "reads_in": 380104, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002386365.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Rothia mucilaginosa strain FDAARGOS_369 chromosome, complete genome", "reads_in": 160584, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002386345.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Streptococcus oralis strain FDAARGOS_367 chromosome, complete genome", "reads_in": 136880, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016026915.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus epidermidis strain FDAARGOS_913 chromosome, complete genome", "reads_in": 175120, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016028175.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Streptococcus oralis strain FDAARGOS_886 chromosome, complete genome", "reads_in": 127586, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016028235.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus equorum strain FDAARGOS_884 chromosome, complete genome", "reads_in": 190484, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016028255.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Streptococcus oralis strain FDAARGOS_885 chromosome, complete genome", "reads_in": 134954, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002209105.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Escherichia coli O104:H4 strain FDAARGOS_348 chromosome, complete genome", "reads_in": 362832, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127035.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Leuconostoc pseudomesenteroides strain FDAARGOS_1004 chromosome, complete genome", "reads_in": 139694, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127055.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus felis strain FDAARGOS_1014 chromosome, complete genome", "reads_in": 165302, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127075.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Macrococcus caseolyticus strain FDAARGOS_1005 chromosome, complete genome", "reads_in": 144448, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127095.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Corynebacterium urealyticum strain FDAARGOS_996 chromosome, complete genome", "reads_in": 156730, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127115.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Aggregatibacter segnis strain FDAARGOS_987 chromosome, complete genome", "reads_in": 137500, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127135.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Lactococcus lactis strain FDAARGOS_1064 chromosome, complete genome", "reads_in": 167518, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002209165.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Mammaliicoccus sciuri strain FDAARGOS_285 chromosome, complete genome", "reads_in": 190256, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002209145.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Jonesia denitrificans strain FDAARGOS_301 chromosome, complete genome", "reads_in": 185920, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127235.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Enterococcus hirae strain FDAARGOS_1057 chromosome, complete genome", "reads_in": 188792, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016126995.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1058 chromosome, complete genome", "reads_in": 360064, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127295.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Leuconostoc mesenteroides strain FDAARGOS_1033 chromosome, complete genome", "reads_in": 138384, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002208685.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Kocuria rhizophila strain FDAARGOS_302 chromosome, complete genome", "reads_in": 179858, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016726625.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Acinetobacter johnsonii strain FDAARGOS_1094 chromosome, complete genome", "reads_in": 234960, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016726365.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Rothia aeria strain FDAARGOS_1137 chromosome, complete genome", "reads_in": 172262, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_008693445.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_629 chromosome, complete genome", "reads_in": 381246, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_008693505.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Streptococcus agalactiae strain FDAARGOS_669 chromosome, complete genome", "reads_in": 137712, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002953375.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Vibrio fluvialis strain FDAARGOS_100 chromosome 1, complete sequence", "reads_in": 324504, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_006364755.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Salmonella enterica strain FDAARGOS_765 chromosome, complete genome", "reads_in": 298806, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_008693585.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Streptococcus agalactiae strain FDAARGOS_670 chromosome, complete genome", "reads_in": 147382, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_008693605.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Citrobacter portucalensis strain FDAARGOS_617 chromosome, complete genome", "reads_in": 328624, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_008693645.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Citrobacter werkmanii strain FDAARGOS_616 chromosome, complete genome", "reads_in": 336620, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_008693665.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Bordetella bronchiseptica strain FDAARGOS_634 chromosome, complete genome", "reads_in": 341072, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_008693685.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Vibrio vulnificus strain FDAARGOS_663 chromosome 1, complete sequence", "reads_in": 331654, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002953295.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Klebsiella quasipneumoniae strain FDAARGOS_93 chromosome, complete genome", "reads_in": 367536, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_008693725.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Streptococcus dysgalactiae strain FDAARGOS_654 chromosome, complete genome", "reads_in": 150236, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_008693765.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus aureus strain FDAARGOS_660 chromosome, complete genome", "reads_in": 191386, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_006364775.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus aureus strain FDAARGOS_766 chromosome, complete genome", "reads_in": 199754, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_006364695.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_772 chromosome, complete genome", "reads_in": 358738, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002983785.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Enterococcus faecium strain FDAARGOS_323 chromosome, complete genome", "reads_in": 180694, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003955985.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Corynebacterium jeikeium strain FDAARGOS_574 chromosome, complete genome", "reads_in": 168544, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003813005.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Pseudomonas aeruginosa strain FDAARGOS_505 chromosome, complete genome", "reads_in": 468654, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003813025.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Pseudomonas aeruginosa strain FDAARGOS_570 chromosome, complete genome", "reads_in": 477010, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003813065.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Bifidobacterium breve strain FDAARGOS_561 chromosome, complete genome", "reads_in": 151710, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002983725.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Campylobacter jejuni subsp. doylei strain FDAARGOS_295 chromosome, complete genome", "reads_in": 123004, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003827915.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus aureus strain FDAARGOS_504 chromosome, complete genome", "reads_in": 192972, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003855115.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Shigella sonnei strain FDAARGOS_524 chromosome, complete genome", "reads_in": 330206, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002983685.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Streptobacillus moniliformis strain FDAARGOS_310 chromosome, complete genome", "reads_in": 112396, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002983665.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Providencia stuartii strain FDAARGOS_291 chromosome, complete genome", "reads_in": 292966, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003956005.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus haemolyticus strain FDAARGOS_517 chromosome, complete genome", "reads_in": 177968, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_006364675.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus aureus strain FDAARGOS_773 chromosome, complete genome", "reads_in": 185804, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003956025.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus cohnii strain FDAARGOS_538 chromosome, complete genome", "reads_in": 180056, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_008693425.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_631 chromosome 1, complete sequence", "reads_in": 364570, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_006364035.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Listeria monocytogenes strain FDAARGOS_607 chromosome, complete genome", "reads_in": 199470, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_008693405.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_630 chromosome, complete genome", "reads_in": 376186, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_006365335.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Salmonella enterica strain FDAARGOS_768 chromosome, complete genome", "reads_in": 330318, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812505.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus hominis strain FDAARGOS_575 chromosome, complete genome", "reads_in": 150494, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812465.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Streptococcus sp. FDAARGOS_521 chromosome, complete genome", "reads_in": 143800, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812445.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Streptococcus sp. FDAARGOS_522 chromosome, complete genome", "reads_in": 140076, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812145.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Enterobacter roggenkampii strain FDAARGOS_523 chromosome, complete genome", "reads_in": 346032, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003614835.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Burkholderia thailandensis strain FDAARGOS_242 chromosome 1, complete sequence", "reads_in": 443554, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003546885.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Klebsiella aerogenes strain FDAARGOS_327 chromosome, complete genome", "reads_in": 360012, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003050665.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Helicobacter pylori strain FDAARGOS_298 chromosome, complete genome", "reads_in": 112060, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003030425.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Enterococcus faecalis strain FDAARGOS_324 chromosome, complete genome", "reads_in": 210658, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003798465.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Cutibacterium acnes strain FDAARGOS_577 chromosome, complete genome", "reads_in": 166356, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812105.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_531 chromosome, complete genome", "reads_in": 384250, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812125.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Bacillus sp. FDAARGOS_527 chromosome, complete genome", "reads_in": 380076, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812165.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Pseudomonas aeruginosa strain FDAARGOS_532 chromosome, complete genome", "reads_in": 462084, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812425.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus epidermidis strain FDAARGOS_529 chromosome, complete genome", "reads_in": 171828, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812185.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Klebsiella aerogenes strain FDAARGOS_513 chromosome, complete genome", "reads_in": 351446, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003019925.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Pluralibacter gergoviae strain FDAARGOS_186 chromosome, complete genome", "reads_in": 360538, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002984565.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus cohnii strain FDAARGOS_334 chromosome, complete genome", "reads_in": 177418, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812305.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Enterococcus sp. FDAARGOS_553 chromosome, complete genome", "reads_in": 218900, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002984195.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Providencia rettgeri strain FDAARGOS_330 chromosome, complete genome", "reads_in": 301240, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002984125.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Moraxella catarrhalis strain FDAARGOS_304 chromosome, complete genome", "reads_in": 129466, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002984035.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Salmonella enterica subsp. enterica serovar Heidelberg strain FDAARGOS_319 chromosome, complete genome", "reads_in": 333918, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812405.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Citrobacter koseri strain FDAARGOS_530 chromosome, complete genome", "reads_in": 305400, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812925.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Klebsiella oxytoca strain FDAARGOS_500 chromosome, complete genome", "reads_in": 391940, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812945.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_497 chromosome, complete genome", "reads_in": 367552, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812965.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Burkholderia multivorans strain FDAARGOS_496 chromosome 1, complete sequence", "reads_in": 519760, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_008693525.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Burkholderia multivorans strain FDAARGOS_622 chromosome 1, complete sequence", "reads_in": 431910, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_006364795.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Pseudomonas aeruginosa strain FDAARGOS_610 chromosome, complete genome", "reads_in": 452814, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_006364815.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Enterococcus faecalis strain FDAARGOS_611 chromosome, complete genome", "reads_in": 189696, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002954055.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus epidermidis strain FDAARGOS_161 chromosome, complete genome", "reads_in": 167716, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812885.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Pseudomonas aeruginosa strain FDAARGOS_571 chromosome, complete genome", "reads_in": 466652, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_006364235.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Streptococcus pyogenes strain FDAARGOS_774 chromosome, complete genome", "reads_in": 127658, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_006364355.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Vibrio furnissii strain FDAARGOS_777 chromosome 1, complete sequence", "reads_in": 332890, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127575.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Cupriavidus necator strain FDAARGOS_1030 chromosome 1, complete sequence", "reads_in": 494398, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127595.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Fusobacterium canifelinum strain FDAARGOS_999 chromosome, complete genome", "reads_in": 161678, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127615.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Corynebacterium amycolatum strain FDAARGOS_991 chromosome, complete genome", "reads_in": 188596, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127635.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Enterococcus casseliflavus strain FDAARGOS_998 chromosome, complete genome", "reads_in": 238722, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127655.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Brevundimonas diminuta strain FDAARGOS_1026 chromosome, complete genome", "reads_in": 236606, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127675.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Campylobacter coli strain FDAARGOS_1028 chromosome, complete genome", "reads_in": 125030, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127695.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1059 chromosome, complete genome", "reads_in": 361050, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127735.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Streptococcus oralis strain FDAARGOS_1075 chromosome, complete genome", "reads_in": 132994, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127915.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Streptococcus oralis strain FDAARGOS_1021 chromosome, complete genome", "reads_in": 127050, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002208985.3", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Citrobacter koseri strain FDAARGOS_287 chromosome, complete genome", "reads_in": 328978, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127775.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Pediococcus pentosaceus strain FDAARGOS_1009 chromosome, complete genome", "reads_in": 116796, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127795.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Stutzerimonas stutzeri strain FDAARGOS_1065 chromosome, complete genome", "reads_in": 295130, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127815.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Pediococcus acidilactici strain FDAARGOS_1007 chromosome, complete genome", "reads_in": 137886, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127835.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Salmonella enterica strain FDAARGOS_1066 chromosome, complete genome", "reads_in": 315766, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127855.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Actinomyces naeslundii strain FDAARGOS_1037 chromosome, complete genome", "reads_in": 210142, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127875.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Streptococcus constellatus strain FDAARGOS_1015 chromosome, complete genome", "reads_in": 135906, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002208965.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Salmonella enterica subsp. enterica serovar Typhimurium var. 5- strain FDAARGOS_312 chromosome, complete genome", "reads_in": 347590, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016724565.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus condimenti strain FDAARGOS_1148 chromosome, complete genome", "reads_in": 177956, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016724665.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus carnosus strain FDAARGOS_1147 chromosome, complete genome", "reads_in": 179336, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016724785.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Fusobacterium canifelinum strain FDAARGOS_1126 chromosome, complete genome", "reads_in": 156814, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016726485.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Peptoniphilus harei strain FDAARGOS_1136 chromosome, complete genome", "reads_in": 128638, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003798325.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Propionibacterium acidifaciens strain FDAARGOS_576 chromosome, complete genome", "reads_in": 204852, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016726245.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Brevibacterium casei strain FDAARGOS_1100 chromosome, complete genome", "reads_in": 256458, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016726265.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Campylobacter lari strain FDAARGOS_1101 chromosome, complete genome", "reads_in": 100892, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016726285.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Shigella boydii strain FDAARGOS_1139 chromosome, complete genome", "reads_in": 303264, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016726305.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Bacteroides caccae strain FDAARGOS_1097 chromosome, complete genome", "reads_in": 304730, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016726325.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Serratia plymuthica strain FDAARGOS_1138 chromosome, complete genome", "reads_in": 365626, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127555.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Streptococcus oralis strain FDAARGOS_1020 chromosome, complete genome", "reads_in": 128770, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_006364295.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_775 chromosome, complete genome", "reads_in": 371594, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127715.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus saccharolyticus strain FDAARGOS_1074 chromosome, complete genome", "reads_in": 156434, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002208945.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Enterococcus faecalis strain FDAARGOS_338 chromosome, complete genome", "reads_in": 190734, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_006364495.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Bacillus subtilis strain FDAARGOS_606 chromosome, complete genome", "reads_in": 269708, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_006364635.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Legionella pneumophila strain FDAARGOS_779 chromosome, complete genome", "reads_in": 227280, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_006364655.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Listeria monocytogenes strain FDAARGOS_778 chromosome, complete genome", "reads_in": 196732, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016026215.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_945 chromosome, complete genome", "reads_in": 371304, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002393445.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Cedecea neteri strain FDAARGOS_392 chromosome, complete genome", "reads_in": 364620, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002393405.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Lelliottia amnigena strain FDAARGOS_395 chromosome, complete genome", "reads_in": 297974, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127495.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Kytococcus sedentarius strain FDAARGOS_1061 chromosome, complete genome", "reads_in": 185572, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127995.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Schaalia meyeri strain FDAARGOS_985 chromosome, complete genome", "reads_in": 135536, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016128015.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Brucella anthropi strain FDAARGOS_1039 chromosome 1, complete sequence", "reads_in": 344930, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002208925.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Chryseobacterium indologenes strain FDAARGOS_337 chromosome, complete genome", "reads_in": 341292, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016128055.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Pediococcus acidilactici strain FDAARGOS_1008 chromosome, complete genome", "reads_in": 133972, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016128075.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Corynebacterium bovis strain FDAARGOS_1052 chromosome, complete genome", "reads_in": 176814, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016128095.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Streptococcus dysgalactiae strain FDAARGOS_1017 chromosome, complete genome", "reads_in": 144956, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016128115.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Providencia stuartii strain FDAARGOS_1040 chromosome, complete genome", "reads_in": 289014, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016128135.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Streptococcus dysgalactiae strain FDAARGOS_1016 chromosome, complete genome", "reads_in": 151136, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002208905.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus saprophyticus strain FDAARGOS_336 chromosome, complete genome", "reads_in": 175718, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016128175.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Rothia kristinae strain FDAARGOS_1001 chromosome, complete genome", "reads_in": 167590, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016128195.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Paraburkholderia ginsengisoli strain FDAARGOS_1049 chromosome 1, complete sequence", "reads_in": 439818, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016128215.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Pediococcus pentosaceus strain FDAARGOS_1010 chromosome, complete genome", "reads_in": 129894, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016128235.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Escherichia fergusonii strain FDAARGOS_1032 chromosome, complete genome", "reads_in": 309820, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002208845.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Citrobacter braakii strain FDAARGOS_290 chromosome, complete genome", "reads_in": 322318, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016598875.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Burkholderia glumae strain FDAARGOS_921 chromosome 1, complete sequence", "reads_in": 456378, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016599755.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Corynebacterium kefirresidentii strain FDAARGOS_1055 chromosome, complete genome", "reads_in": 168214, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016599775.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Stutzerimonas balearica strain FDAARGOS_1013 chromosome, complete genome", "reads_in": 294082, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016599795.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus pasteuri strain FDAARGOS_1069 chromosome, complete genome", "reads_in": 165654, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002208825.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Brevundimonas vesicularis strain FDAARGOS_289 chromosome, complete genome", "reads_in": 223036, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127975.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Corynebacterium urealyticum strain FDAARGOS_994 chromosome, complete genome", "reads_in": 158948, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127935.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus pettenkoferi strain FDAARGOS_1071 chromosome, complete genome", "reads_in": 166242, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020097495.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Corynebacterium freneyi strain FDAARGOS_1495 chromosome, complete genome", "reads_in": 192122, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812085.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_566 chromosome 1, complete sequence", "reads_in": 377804, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020097435.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Gemella morbillorum strain FDAARGOS_1501 chromosome, complete genome", "reads_in": 117396, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013343115.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 10, "name": "Streptococcus sanguinis strain FDAARGOS_770 chromosome, complete genome", "reads_in": 157256, "reads_removed_percent": 0.006359057841990131, "reads_retained_percent": 99.99364094215801}, {"sample": "GCA_019048665.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 10, "name": "Aeromonas sp. FDAARGOS 1408 chromosome, complete genome", "reads_in": 322962, "reads_removed_percent": 0.0030963395074343116, "reads_retained_percent": 99.99690366049256}, {"sample": "GCA_003812385.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 10, "name": "Acinetobacter baumannii strain FDAARGOS_533 chromosome, complete genome", "reads_in": 259026, "reads_removed_percent": 0.0038606163087875345, "reads_retained_percent": 99.99613938369122}, {"sample": "GCA_019048725.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 10, "name": "Aeromonas sp. FDAARGOS 1407 chromosome, complete genome", "reads_in": 313436, "reads_removed_percent": 0.003190443982184561, "reads_retained_percent": 99.99680955601781}, {"sample": "GCA_003955795.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 10, "name": "Francisella tularensis strain FDAARGOS_598 chromosome, complete genome", "reads_in": 126240, "reads_removed_percent": 0.007921419518377692, "reads_retained_percent": 99.99207858048162}, {"sample": "GCA_008693465.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 10, "name": "Klebsiella variicola strain FDAARGOS_627 chromosome, complete genome", "reads_in": 397174, "reads_removed_percent": 0.0025177881734453917, "reads_retained_percent": 99.99748221182655}, {"sample": "GCA_013343075.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 10, "name": "Bacillus thuringiensis strain FDAARGOS_791 chromosome, complete genome", "reads_in": 355852, "reads_removed_percent": 0.0028101570315749243, "reads_retained_percent": 99.99718984296842}, {"sample": "GCA_016128255.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 10, "name": "Lactococcus garvieae strain FDAARGOS_1063 chromosome, complete genome", "reads_in": 147538, "reads_removed_percent": 0.006777914842277921, "reads_retained_percent": 99.99322208515773}, {"sample": "GCA_002386285.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 10, "name": "Streptococcus gordonii strain FDAARGOS_371 chromosome, complete genome", "reads_in": 141420, "reads_removed_percent": 0.007071135624381275, "reads_retained_percent": 99.99292886437561}, {"sample": "GCA_016904575.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 10, "name": "Escherichia coli strain FDAARGOS_1276 chromosome, complete genome", "reads_in": 338828, "reads_removed_percent": 0.002951349947465971, "reads_retained_percent": 99.99704865005253}, {"sample": "GCA_003798205.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 10, "name": "Yersinia pestis strain FDAARGOS_603 chromosome, complete genome", "reads_in": 328642, "reads_removed_percent": 0.0030428247150394655, "reads_retained_percent": 99.99695717528496}, {"sample": "GCA_016027675.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 10, "name": "Serratia plymuthica strain FDAARGOS_895 chromosome, complete genome", "reads_in": 356612, "reads_removed_percent": 0.0028041681154868598, "reads_retained_percent": 99.99719583188451}, {"sample": "GCA_012273555.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 10, "name": "Chromobacterium violaceum strain FDAARGOS_635 chromosome, complete genome", "reads_in": 320050, "reads_removed_percent": 0.0031245117950320263, "reads_retained_percent": 99.99687548820496}, {"sample": "GCA_016728005.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 10, "name": "Serratia liquefaciens strain FDAARGOS_1081 chromosome, complete genome", "reads_in": 367766, "reads_removed_percent": 0.002719120310197245, "reads_retained_percent": 99.9972808796898}, {"sample": "GCA_016904475.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 10, "name": "Escherichia coli strain FDAARGOS_1292 chromosome, complete genome", "reads_in": 341372, "reads_removed_percent": 0.0029293556589292617, "reads_retained_percent": 99.99707064434106}, {"sample": "GCA_016027015.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 10, "name": "Bacillus cereus strain FDAARGOS_918 chromosome, complete genome", "reads_in": 363314, "reads_removed_percent": 0.00275244003809377, "reads_retained_percent": 99.9972475599619}, {"sample": "GCA_016725945.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 10, "name": "Acinetobacter johnsonii strain FDAARGOS_1093 chromosome, complete genome", "reads_in": 257722, "reads_removed_percent": 0.0038801499289932563, "reads_retained_percent": 99.99611985007101}, {"sample": "GCA_016126935.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 8, "name": "Stenotrophomonas maltophilia strain FDAARGOS_1044 chromosome, complete genome", "reads_in": 333618, "reads_removed_percent": 0.0023979521488648693, "reads_retained_percent": 99.99760204785113}, {"sample": "GCA_003020025.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 8, "name": "Burkholderia thailandensis strain FDAARGOS_237 chromosome 1, complete sequence", "reads_in": 444854, "reads_removed_percent": 0.0017983428270848412, "reads_retained_percent": 99.99820165717291}, {"sample": "GCA_008693985.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 8, "name": "Stenotrophomonas maltophilia strain FDAARGOS_649 chromosome, complete genome", "reads_in": 308392, "reads_removed_percent": 0.0025941010142934966, "reads_retained_percent": 99.9974058989857}, {"sample": "GCA_016724865.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 8, "name": "Peribacillus psychrosaccharolyticus strain FDAARGOS_1161 chromosome, complete genome", "reads_in": 312086, "reads_removed_percent": 0.0025633959870035826, "reads_retained_percent": 99.99743660401299}, {"sample": "GCA_002983705.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 8, "name": "Helicobacter pylori strain FDAARGOS_300 chromosome, complete genome", "reads_in": 109716, "reads_removed_percent": 0.007291552736155164, "reads_retained_percent": 99.99270844726385}, {"sample": "GCA_016889065.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 8, "name": "Butyricimonas virosa strain FDAARGOS_1229 chromosome, complete genome", "reads_in": 320876, "reads_removed_percent": 0.0024931749336192174, "reads_retained_percent": 99.99750682506638}, {"sample": "GCA_016028715.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 8, "name": "Neisseria cinerea strain FDAARGOS_871 chromosome, complete genome", "reads_in": 122194, "reads_removed_percent": 0.006546966299490974, "reads_retained_percent": 99.9934530337005}, {"sample": "GCA_002208665.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 8, "name": "Haemophilus] ducreyi strain FDAARGOS_297 chromosome, complete genome", "reads_in": 113420, "reads_removed_percent": 0.007053429730206314, "reads_retained_percent": 99.9929465702698}, {"sample": "GCA_006364735.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 8, "name": "Pseudomonas aeruginosa strain FDAARGOS_767 chromosome, complete genome", "reads_in": 417578, "reads_removed_percent": 0.0019158097409346278, "reads_retained_percent": 99.99808419025906}, {"sample": "GCA_016026535.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 10, "name": "Delftia acidovorans strain FDAARGOS_939 chromosome, complete genome", "reads_in": 423412, "reads_removed_percent": 0.0023617658450870546, "reads_retained_percent": 99.99763823415492}, {"sample": "GCA_019048265.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 10, "name": "Aeromonas sp. FDAARGOS 1405 chromosome, complete genome", "reads_in": 307614, "reads_removed_percent": 0.003250827335556899, "reads_retained_percent": 99.99674917266444}, {"sample": "GCA_016127515.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 10, "name": "Acinetobacter baumannii strain FDAARGOS_1036 chromosome, complete genome", "reads_in": 270026, "reads_removed_percent": 0.0037033470850955093, "reads_retained_percent": 99.9962966529149}, {"sample": "GCA_002984485.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 10, "name": "Oligella urethralis strain FDAARGOS_329 chromosome, complete genome", "reads_in": 167422, "reads_removed_percent": 0.005972930678166549, "reads_retained_percent": 99.99402706932183}, {"sample": "GCA_003614815.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 14, "name": "Burkholderia thailandensis strain FDAARGOS_241 chromosome 1, complete sequence", "reads_in": 445608, "reads_removed_percent": 0.003141774833485934, "reads_retained_percent": 99.99685822516652}, {"sample": "GCA_013267775.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 14, "name": "Bacillus cereus strain FDAARGOS_797 chromosome, complete genome", "reads_in": 374962, "reads_removed_percent": 0.0037337116827838553, "reads_retained_percent": 99.99626628831722}, {"sample": "GCA_003019965.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 14, "name": "Burkholderia multivorans strain FDAARGOS_246 chromosome 1, complete sequence", "reads_in": 421526, "reads_removed_percent": 0.0033212660666245976, "reads_retained_percent": 99.99667873393338}, {"sample": "GCA_019930685.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 14, "name": "Legionella longbeachae strain FDAARGOS_1481 chromosome, complete genome", "reads_in": 277696, "reads_removed_percent": 0.005041484212952293, "reads_retained_percent": 99.99495851578705}, {"sample": "GCA_013267435.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 14, "name": "Niallia circulans strain FDAARGOS_783 chromosome, complete genome", "reads_in": 345304, "reads_removed_percent": 0.004054398443110998, "reads_retained_percent": 99.99594560155688}, {"sample": "GCA_019046925.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 12, "name": "Aeromonas sp. FDAARGOS 1402 chromosome, complete genome", "reads_in": 328168, "reads_removed_percent": 0.0036566636600765456, "reads_retained_percent": 99.99634333633992}, {"sample": "GCA_020735945.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 12, "name": "Parabacteroides distasonis strain FDAARGOS_1565 chromosome, complete genome", "reads_in": 320804, "reads_removed_percent": 0.0037406017381329407, "reads_retained_percent": 99.99625939826187}, {"sample": "GCA_002393505.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 12, "name": "Providencia alcalifaciens strain FDAARGOS_408 chromosome, complete genome", "reads_in": 266008, "reads_removed_percent": 0.004511142522029413, "reads_retained_percent": 99.99548885747797}, {"sample": "GCA_013267335.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 12, "name": "Bacillus thuringiensis strain FDAARGOS_793 chromosome, complete genome", "reads_in": 354256, "reads_removed_percent": 0.0033873808771058213, "reads_retained_percent": 99.9966126191229}, {"sample": "GCA_009730375.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 12, "name": "Capnocytophaga sp. FDAARGOS_737 chromosome, complete genome", "reads_in": 176848, "reads_removed_percent": 0.006785488102777526, "reads_retained_percent": 99.99321451189722}, {"sample": "GCA_009730075.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 12, "name": "Yersinia intermedia strain FDAARGOS_729 chromosome, complete genome", "reads_in": 328594, "reads_removed_percent": 0.0036519230418084326, "reads_retained_percent": 99.99634807695818}, {"sample": "GCA_020735585.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 12, "name": "Campylobacter fetus subsp. testudinum strain FDAARGOS_1591 chromosome, complete genome", "reads_in": 118366, "reads_removed_percent": 0.010138046398459017, "reads_retained_percent": 99.98986195360155}, {"sample": "GCA_016727905.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 12, "name": "Yersinia enterocolitica strain FDAARGOS_1090 chromosome, complete genome", "reads_in": 306216, "reads_removed_percent": 0.003918802413982287, "reads_retained_percent": 99.99608119758602}, {"sample": "GCA_013267595.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 12, "name": "Prevotella melaninogenica strain FDAARGOS_760 chromosome 1, complete sequence", "reads_in": 223532, "reads_removed_percent": 0.005368358892686506, "reads_retained_percent": 99.99463164110732}, {"sample": "GCA_002208865.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 12, "name": "Escherichia coli O157 strain FDAARGOS_293 chromosome, complete genome", "reads_in": 379110, "reads_removed_percent": 0.0031653082218881068, "reads_retained_percent": 99.9968346917781}, {"sample": "GCA_016888965.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 12, "name": "Escherichia coli strain FDAARGOS_1261 chromosome, complete genome", "reads_in": 336674, "reads_removed_percent": 0.003564278797887571, "reads_retained_percent": 99.99643572120212}, {"sample": "GCA_016599835.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 12, "name": "Elizabethkingia bruuniana strain FDAARGOS_1031 chromosome, complete genome", "reads_in": 308800, "reads_removed_percent": 0.0038860103626943, "reads_retained_percent": 99.99611398963731}, {"sample": "GCA_016028755.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 12, "name": "Moraxella osloensis strain FDAARGOS_870 chromosome, complete genome", "reads_in": 162388, "reads_removed_percent": 0.007389708599157573, "reads_retained_percent": 99.99261029140084}, {"sample": "GCA_016127355.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 12, "name": "Kingella denitrificans strain FDAARGOS_1060 chromosome, complete genome", "reads_in": 148004, "reads_removed_percent": 0.008107888975973622, "reads_retained_percent": 99.99189211102403}, {"sample": "GCA_016889285.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 12, "name": "Kytococcus sedentarius strain FDAARGOS_1200 chromosome, complete genome", "reads_in": 163250, "reads_removed_percent": 0.007350689127105666, "reads_retained_percent": 99.99264931087289}, {"sample": "GCA_016027855.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 12, "name": "Atlantibacter hermannii strain FDAARGOS_888 chromosome, complete genome", "reads_in": 303658, "reads_removed_percent": 0.003951814212041178, "reads_retained_percent": 99.99604818578796}, {"sample": "GCA_002209045.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 12, "name": "Campylobacter jejuni strain FDAARGOS_265 chromosome, complete genome", "reads_in": 113074, "reads_removed_percent": 0.010612519235191114, "reads_retained_percent": 99.9893874807648}, {"sample": "GCA_003812325.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 10, "name": "Citrobacter freundii strain FDAARGOS_550 chromosome, complete genome", "reads_in": 331276, "reads_removed_percent": 0.0030186309904732004, "reads_retained_percent": 99.99698136900953}, {"sample": "GCA_013267795.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 10, "name": "Bacillus thuringiensis strain FDAARGOS_794 chromosome, complete genome", "reads_in": 356574, "reads_removed_percent": 0.0028044669549658697, "reads_retained_percent": 99.99719553304503}, {"sample": "GCA_013267615.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 10, "name": "Parabacteroides distasonis strain FDAARGOS_759 chromosome, complete genome", "reads_in": 328402, "reads_removed_percent": 0.0030450484467207874, "reads_retained_percent": 99.99695495155328}, {"sample": "GCA_019930945.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 10, "name": "Campylobacter hominis strain FDAARGOS_1467 chromosome, complete genome", "reads_in": 114540, "reads_removed_percent": 0.008730574471800244, "reads_retained_percent": 99.9912694255282}, {"sample": "GCA_019048185.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 10, "name": "Lelliottia amnigena strain FDAARGOS 1446 chromosome, complete genome", "reads_in": 327628, "reads_removed_percent": 0.0030522421771033, "reads_retained_percent": 99.9969477578229}, {"sample": "GCA_016904235.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 8, "name": "Escherichia coli strain FDAARGOS_1287 chromosome, complete genome", "reads_in": 369658, "reads_removed_percent": 0.002164162550249149, "reads_retained_percent": 99.99783583744976}, {"sample": "GCA_002943995.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 8, "name": "Pseudomonas monteilii strain FDAARGOS_171 chromosome, complete genome", "reads_in": 393152, "reads_removed_percent": 0.002034836399153508, "reads_retained_percent": 99.99796516360085}, {"sample": "GCA_000783775.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 8, "name": "Aeromonas caviae strain FDAARGOS_72 chromosome, complete genome", "reads_in": 301220, "reads_removed_percent": 0.002655866144346325, "reads_retained_percent": 99.99734413385565}, {"sample": "GCA_003613595.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 8, "name": "Burkholderia gladioli pv. gladioli strain FDAARGOS_188 chromosome 1, complete sequence", "reads_in": 592836, "reads_removed_percent": 0.001349445715172493, "reads_retained_percent": 99.99865055428482}, {"sample": "GCA_016026055.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 4, "name": "Burkholderia glumae strain FDAARGOS_949 chromosome 1, complete sequence", "reads_in": 454724, "reads_removed_percent": 0.000879654471723507, "reads_retained_percent": 99.99912034552828}, {"sample": "GCA_016127155.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 4, "name": "Brevibacterium casei strain FDAARGOS_990 chromosome, complete genome", "reads_in": 256754, "reads_removed_percent": 0.0015579114638915072, "reads_retained_percent": 99.99844208853611}, {"sample": "GCA_013364115.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 4, "name": "Shigella flexneri strain FDAARGOS_691 chromosome, complete genome", "reads_in": 302920, "reads_removed_percent": 0.001320480654958405, "reads_retained_percent": 99.99867951934505}, {"sample": "GCA_016888885.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 4, "name": "Moraxella osloensis strain FDAARGOS_1202 chromosome, complete genome", "reads_in": 183946, "reads_removed_percent": 0.0021745512269905295, "reads_retained_percent": 99.99782544877301}, {"sample": "GCA_003812065.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 4, "name": "Acinetobacter baumannii strain FDAARGOS_540 chromosome, complete genome", "reads_in": 260684, "reads_removed_percent": 0.0015344248208559023, "reads_retained_percent": 99.99846557517914}, {"sample": "GCA_008693485.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 4, "name": "Burkholderia multivorans strain FDAARGOS_623 chromosome 1, complete sequence", "reads_in": 431910, "reads_removed_percent": 0.0009261188673566252, "reads_retained_percent": 99.99907388113265}, {"sample": "GCA_016127375.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 4, "name": "Lactococcus garvieae strain FDAARGOS_1002 chromosome, complete genome", "reads_in": 152810, "reads_removed_percent": 0.0026176297362738043, "reads_retained_percent": 99.99738237026372}, {"sample": "GCA_016889645.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 4, "name": "Salmonella enterica strain FDAARGOS_1271 chromosome, complete genome", "reads_in": 320272, "reads_removed_percent": 0.0012489384023579957, "reads_retained_percent": 99.99875106159764}, {"sample": "GCA_019048225.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 4, "name": "Acinetobacter pittii strain FDAARGOS 1396 chromosome, complete genome", "reads_in": 261056, "reads_removed_percent": 0.001532238293699436, "reads_retained_percent": 99.9984677617063}, {"sample": "GCA_009730355.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 4, "name": "Citrobacter portucalensis strain FDAARGOS_738 chromosome, complete genome", "reads_in": 344180, "reads_removed_percent": 0.0011621825788831425, "reads_retained_percent": 99.99883781742112}, {"sample": "GCA_009730055.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 4, "name": "Yersinia intermedia strain FDAARGOS_730 chromosome, complete genome", "reads_in": 328594, "reads_removed_percent": 0.0012173076806028108, "reads_retained_percent": 99.9987826923194}, {"sample": "GCA_008693705.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 4, "name": "Aeromonas veronii strain FDAARGOS_632 chromosome, complete genome", "reads_in": 304126, "reads_removed_percent": 0.0013152443395171738, "reads_retained_percent": 99.99868475566048}, {"sample": "GCA_016128275.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 4, "name": "Citrobacter koseri strain FDAARGOS_1029 chromosome, complete genome", "reads_in": 315772, "reads_removed_percent": 0.001266736759434022, "reads_retained_percent": 99.99873326324057}, {"sample": "GCA_020097215.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 4, "name": "Clostridioides difficile strain FDAARGOS_1528 chromosome, complete genome", "reads_in": 291690, "reads_removed_percent": 0.001371318865919298, "reads_retained_percent": 99.99862868113408}, {"sample": "GCA_016889205.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 4, "name": "Burkholderia dolosa strain FDAARGOS_1272 chromosome 1, complete sequence", "reads_in": 427274, "reads_removed_percent": 0.0009361674241821407, "reads_retained_percent": 99.99906383257581}, {"sample": "GCA_016888925.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 4, "name": "Escherichia coli strain FDAARGOS_1260 chromosome, complete genome", "reads_in": 345352, "reads_removed_percent": 0.0011582385508119252, "reads_retained_percent": 99.99884176144919}, {"sample": "GCA_016890045.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 4, "name": "Escherichia coli strain FDAARGOS_1265 chromosome, complete genome", "reads_in": 361686, "reads_removed_percent": 0.0011059316644824518, "reads_retained_percent": 99.99889406833552}, {"sample": "GCA_016028475.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 4, "name": "Serratia rubidaea strain FDAARGOS_880 chromosome, complete genome", "reads_in": 337078, "reads_removed_percent": 0.0011866689608933244, "reads_retained_percent": 99.9988133310391}, {"sample": "GCA_001558735.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 4, "name": "Pantoea vagans strain FDAARGOS_160 chromosome, complete genome", "reads_in": 320590, "reads_removed_percent": 0.0012476995539474093, "reads_retained_percent": 99.99875230044606}, {"sample": "GCA_002953355.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 4, "name": "Vibrio diabolicus strain FDAARGOS_99 chromosome 1, complete sequence", "reads_in": 342120, "reads_removed_percent": 0.00116918040453642, "reads_retained_percent": 99.99883081959547}, {"sample": "GCA_002953395.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 4, "name": "Vibrio diabolicus strain FDAARGOS_105 chromosome 1, complete sequence", "reads_in": 361742, "reads_removed_percent": 0.0011057604591117427, "reads_retained_percent": 99.99889423954089}, {"sample": "GCA_002951295.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 4, "name": "Acinetobacter nosocomialis strain FDAARGOS_129 chromosome, complete genome", "reads_in": 266846, "reads_removed_percent": 0.001498991927928468, "reads_retained_percent": 99.99850100807207}, {"sample": "GCA_003020185.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 4, "name": "Burkholderia thailandensis strain FDAARGOS_238 chromosome 1, complete sequence", "reads_in": 451244, "reads_removed_percent": 0.0008864383792360673, "reads_retained_percent": 99.99911356162076}, {"sample": "GCA_003019945.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 4, "name": "Burkholderia vietnamiensis strain FDAARGOS_239 chromosome 1, complete sequence", "reads_in": 462028, "reads_removed_percent": 0.0008657483962010961, "reads_retained_percent": 99.9991342516038}, {"sample": "GCA_000783675.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 4, "name": "Enterobacter cloacae complex sp. strain FDAARGOS_77 chromosome, complete genome", "reads_in": 325134, "reads_removed_percent": 0.0012302619842895544, "reads_retained_percent": 99.99876973801571}, {"sample": "GCA_016889345.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 2, "name": "Acinetobacter pittii strain FDAARGOS_1215 chromosome, complete genome", "reads_in": 259280, "reads_removed_percent": 0.0007713668620796051, "reads_retained_percent": 99.99922863313792}, {"sample": "GCA_002953515.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 2, "name": "Vibrio alginolyticus strain FDAARGOS_108 chromosome 1, complete sequence", "reads_in": 363068, "reads_removed_percent": 0.0005508609957363359, "reads_retained_percent": 99.99944913900427}, {"sample": "GCA_016903815.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 4, "name": "Escherichia coli strain FDAARGOS_1280 chromosome, complete genome", "reads_in": 366450, "reads_removed_percent": 0.0010915541001500885, "reads_retained_percent": 99.99890844589984}, {"sample": "GCA_016028775.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 14, "name": "Moraxella nonliquefaciens strain FDAARGOS_869 chromosome, complete genome", "reads_in": 156250, "reads_removed_percent": 0.00896, "reads_retained_percent": 99.99104}, {"sample": "GCA_016026235.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 4, "name": "Escherichia coli strain FDAARGOS_944 chromosome, complete genome", "reads_in": 355392, "reads_removed_percent": 0.0011255177381595534, "reads_retained_percent": 99.99887448226184}, {"sample": "GCA_021183625.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 4, "name": "Nocardia asteroides strain FDAARGOS_1620 chromosome, complete genome", "reads_in": 465792, "reads_removed_percent": 0.0008587524045067325, "reads_retained_percent": 99.99914124759549}, {"sample": "GCA_016127195.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 8, "name": "Aeromonas jandaei strain FDAARGOS_986 chromosome, complete genome", "reads_in": 303406, "reads_removed_percent": 0.0026367309809298433, "reads_retained_percent": 99.99736326901908}, {"sample": "GCA_016127215.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 8, "name": "Haemophilus parainfluenzae strain FDAARGOS_1000 chromosome, complete genome", "reads_in": 142780, "reads_removed_percent": 0.005603025633842275, "reads_retained_percent": 99.99439697436615}, {"sample": "GCA_016025875.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 8, "name": "Prescottella equi strain FDAARGOS_952 chromosome, complete genome", "reads_in": 347914, "reads_removed_percent": 0.0022994188218927666, "reads_retained_percent": 99.99770058117811}, {"sample": "GCA_001558515.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 8, "name": "Vibrio vulnificus strain FDAARGOS_119 chromosome 1, complete sequence", "reads_in": 331920, "reads_removed_percent": 0.002410219329959026, "reads_retained_percent": 99.99758978067004}, {"sample": "GCA_013267275.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 8, "name": "Bacillus cereus strain FDAARGOS_798 chromosome, complete genome", "reads_in": 412640, "reads_removed_percent": 0.0019387359441644047, "reads_retained_percent": 99.99806126405583}, {"sample": "GCA_016904395.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 8, "name": "Escherichia coli strain FDAARGOS_1277 chromosome, complete genome", "reads_in": 316756, "reads_removed_percent": 0.002525603303489121, "reads_retained_percent": 99.99747439669652}, {"sample": "GCA_003812245.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 6, "name": "Acinetobacter sp. FDAARGOS_493 chromosome, complete genome", "reads_in": 268098, "reads_removed_percent": 0.002237987601548687, "reads_retained_percent": 99.99776201239845}, {"sample": "GCA_020097295.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 6, "name": "Aeromonas enteropelogenes strain FDAARGOS_1510 chromosome, complete genome", "reads_in": 292826, "reads_removed_percent": 0.0020489983812912787, "reads_retained_percent": 99.9979510016187}, {"sample": "GCA_016888905.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 6, "name": "Parabacteroides distasonis strain FDAARGOS_1234 chromosome, complete genome", "reads_in": 320802, "reads_removed_percent": 0.0018703125292236333, "reads_retained_percent": 99.99812968747078}, {"sample": "GCA_000783755.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 6, "name": "Citrobacter freundii strain FDAARGOS_73 chromosome, complete genome", "reads_in": 363416, "reads_removed_percent": 0.001651000506306822, "reads_retained_percent": 99.9983489994937}, {"sample": "GCA_020097355.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 6, "name": "Proteus vulgaris strain FDAARGOS_1507 chromosome, complete genome", "reads_in": 264242, "reads_removed_percent": 0.002270645847367186, "reads_retained_percent": 99.99772935415264}, {"sample": "GCA_021183565.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 6, "name": "Nocardia yamanashiensis strain FDAARGOS_1623 chromosome, complete genome", "reads_in": 564586, "reads_removed_percent": 0.0010627256077904873, "reads_retained_percent": 99.9989372743922}, {"sample": "GCA_020097315.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 6, "name": "Aeromonas enteropelogenes strain FDAARGOS_1509 chromosome, complete genome", "reads_in": 302706, "reads_removed_percent": 0.001982121266179065, "reads_retained_percent": 99.99801787873382}, {"sample": "GCA_016728825.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 6, "name": "Achromobacter xylosoxidans strain FDAARGOS_1091 chromosome, complete genome", "reads_in": 460132, "reads_removed_percent": 0.0013039736423461094, "reads_retained_percent": 99.99869602635765}, {"sample": "GCA_003812225.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 6, "name": "Acinetobacter sp. FDAARGOS_494 chromosome, complete genome", "reads_in": 263550, "reads_removed_percent": 0.0022766078542970974, "reads_retained_percent": 99.9977233921457}, {"sample": "GCA_009730175.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 6, "name": "Burkholderia multivorans strain FDAARGOS_726 chromosome 1, complete sequence", "reads_in": 434580, "reads_removed_percent": 0.0013806433798149939, "reads_retained_percent": 99.99861935662018}, {"sample": "GCA_008693385.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 6, "name": "Cupriavidus pauculus strain FDAARGOS_664 chromosome 1, complete sequence", "reads_in": 470942, "reads_removed_percent": 0.001274042238747022, "reads_retained_percent": 99.99872595776125}, {"sample": "GCA_008693805.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 6, "name": "Providencia stuartii strain FDAARGOS_645 chromosome, complete genome", "reads_in": 301984, "reads_removed_percent": 0.001986860231005616, "reads_retained_percent": 99.998013139769}, {"sample": "GCA_013267835.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 6, "name": "Neisseria mucosa strain FDAARGOS_758 chromosome, complete genome", "reads_in": 148318, "reads_removed_percent": 0.004045361992475627, "reads_retained_percent": 99.99595463800752}, {"sample": "GCA_001558415.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 6, "name": "Vibrio fluvialis strain ATCC 33809 chromosome 1, complete sequence", "reads_in": 321850, "reads_removed_percent": 0.0018642224638806897, "reads_retained_percent": 99.99813577753612}, {"sample": "GCA_013267255.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 6, "name": "Bacillus cereus strain FDAARGOS_799 chromosome, complete genome", "reads_in": 412650, "reads_removed_percent": 0.0014540167211922936, "reads_retained_percent": 99.9985459832788}, {"sample": "GCA_016889265.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 6, "name": "Escherichia coli strain FDAARGOS_1264 chromosome, complete genome", "reads_in": 336128, "reads_removed_percent": 0.001785034272658035, "reads_retained_percent": 99.99821496572734}, {"sample": "GCA_016904555.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 6, "name": "Escherichia coli strain FDAARGOS_1285 chromosome, complete genome", "reads_in": 345962, "reads_removed_percent": 0.0017342945178950289, "reads_retained_percent": 99.99826570548211}, {"sample": "GCA_016026615.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 6, "name": "Aeromonas allosaccharophila strain FDAARGOS_933 chromosome, complete genome", "reads_in": 308694, "reads_removed_percent": 0.00194367237458454, "reads_retained_percent": 99.99805632762542}, {"sample": "GCA_016027115.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 6, "name": "Serratia plymuthica strain FDAARGOS_907 chromosome, complete genome", "reads_in": 364598, "reads_removed_percent": 0.0016456480836428066, "reads_retained_percent": 99.99835435191636}, {"sample": "GCA_016890065.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 6, "name": "Acinetobacter pittii strain FDAARGOS_1216 chromosome, complete genome", "reads_in": 261924, "reads_removed_percent": 0.0022907408255829938, "reads_retained_percent": 99.99770925917441}, {"sample": "GCA_002407525.3", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 6, "name": "Burkholderia cepacia strain FDAARGOS_388 chromosome 1, complete sequence", "reads_in": 571322, "reads_removed_percent": 0.001050195861528175, "reads_retained_percent": 99.99894980413848}, {"sample": "GCA_020735525.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 4, "name": "Helicobacter pylori strain FDAARGOS_1601 chromosome, complete genome", "reads_in": 109606, "reads_removed_percent": 0.0036494352498950785, "reads_retained_percent": 99.9963505647501}, {"sample": "GCA_021183705.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 14, "name": "Mycobacterium kansasii strain FDAARGOS_1614 chromosome, complete genome", "reads_in": 430260, "reads_removed_percent": 0.0032538465114117046, "reads_retained_percent": 99.99674615348859}, {"sample": "GCA_016890085.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 14, "name": "Chromobacterium violaceum strain FDAARGOS_1273 chromosome, complete genome", "reads_in": 320536, "reads_removed_percent": 0.004367684129083785, "reads_retained_percent": 99.99563231587092}, {"sample": "GCA_003812345.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 16, "name": "Citrobacter freundii strain FDAARGOS_549 chromosome, complete genome", "reads_in": 340144, "reads_removed_percent": 0.004703890117126863, "reads_retained_percent": 99.99529610988287}, {"sample": "GCA_016127415.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 46, "name": "Delftia acidovorans strain FDAARGOS_997 chromosome, complete genome", "reads_in": 453112, "reads_removed_percent": 0.010152015395752044, "reads_retained_percent": 99.98984798460425}, {"sample": "GCA_002073575.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 46, "name": "Neisseria meningitidis strain FDAARGOS_209 chromosome, complete genome", "reads_in": 145416, "reads_removed_percent": 0.03163338284645431, "reads_retained_percent": 99.96836661715355}, {"sample": "GCA_002073275.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 46, "name": "Neisseria meningitidis strain FDAARGOS_215 chromosome, complete genome", "reads_in": 153722, "reads_removed_percent": 0.02992414878807197, "reads_retained_percent": 99.97007585121193}, {"sample": "GCA_000784015.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 44, "name": "Proteus mirabilis strain FDAARGOS_60 chromosome, complete genome", "reads_in": 273704, "reads_removed_percent": 0.01607576067576652, "reads_retained_percent": 99.98392423932424}, {"sample": "GCA_016726985.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 44, "name": "Myroides odoratus strain FDAARGOS_1131 chromosome, complete genome", "reads_in": 282832, "reads_removed_percent": 0.015556938394523958, "reads_retained_percent": 99.98444306160548}, {"sample": "GCA_019048085.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 44, "name": "Aeromonas sp. FDAARGOS 1416 chromosome, complete genome", "reads_in": 301256, "reads_removed_percent": 0.0146055182303423, "reads_retained_percent": 99.98539448176966}, {"sample": "GCA_019048105.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 40, "name": "Providencia rettgeri strain FDAARGOS 1450 chromosome, complete genome", "reads_in": 309462, "reads_removed_percent": 0.012925658077566874, "reads_retained_percent": 99.98707434192244}, {"sample": "GCA_009730195.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 40, "name": "Burkholderia multivorans strain FDAARGOS_722 chromosome 1, complete sequence", "reads_in": 439364, "reads_removed_percent": 0.009104068608261032, "reads_retained_percent": 99.99089593139173}, {"sample": "GCA_002073555.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 40, "name": "Neisseria meningitidis strain FDAARGOS_211 chromosome, complete genome", "reads_in": 153720, "reads_removed_percent": 0.026021337496747333, "reads_retained_percent": 99.97397866250326}, {"sample": "GCA_003798285.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 40, "name": "Yersinia pseudotuberculosis strain FDAARGOS_583 chromosome, complete genome", "reads_in": 310146, "reads_removed_percent": 0.012897151664054994, "reads_retained_percent": 99.98710284833595}, {"sample": "GCA_003812365.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 38, "name": "Burkholderia multivorans strain FDAARGOS_547 chromosome 1, complete sequence", "reads_in": 426274, "reads_removed_percent": 0.008914454083523744, "reads_retained_percent": 99.99108554591648}, {"sample": "GCA_016026335.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 38, "name": "Escherichia coli strain FDAARGOS_943 chromosome, complete genome", "reads_in": 362216, "reads_removed_percent": 0.01049097775912715, "reads_retained_percent": 99.98950902224087}, {"sample": "GCA_000783875.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 38, "name": "Proteus mirabilis strain FDAARGOS_67 chromosome, complete genome", "reads_in": 263698, "reads_removed_percent": 0.01441042404568863, "reads_retained_percent": 99.98558957595431}, {"sample": "GCA_003812865.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 38, "name": "Chryseobacterium indologenes strain FDAARGOS_510 chromosome, complete genome", "reads_in": 328196, "reads_removed_percent": 0.011578447025557898, "reads_retained_percent": 99.98842155297444}, {"sample": "GCA_013267555.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 38, "name": "Bacteroides fragilis strain FDAARGOS_763 chromosome, complete genome", "reads_in": 373716, "reads_removed_percent": 0.010168149075768766, "reads_retained_percent": 99.98983185092423}, {"sample": "GCA_012273615.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 38, "name": "Chryseobacterium gallinarum strain FDAARGOS_636 chromosome, complete genome", "reads_in": 289264, "reads_removed_percent": 0.013136788539189113, "reads_retained_percent": 99.98686321146081}, {"sample": "GCA_008693625.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 36, "name": "Vibrio parahaemolyticus strain FDAARGOS_667 chromosome 1, complete sequence", "reads_in": 333544, "reads_removed_percent": 0.010793178711054615, "reads_retained_percent": 99.98920682128895}, {"sample": "GCA_019047085.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 34, "name": "Aeromonas sp. FDAARGOS 1404 chromosome, complete genome", "reads_in": 315912, "reads_removed_percent": 0.010762490820228418, "reads_retained_percent": 99.98923750917977}, {"sample": "GCA_003798345.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 34, "name": "Yersinia pestis strain FDAARGOS_602 chromosome, complete genome", "reads_in": 318966, "reads_removed_percent": 0.01065944332624794, "reads_retained_percent": 99.98934055667375}, {"sample": "GCA_016128155.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 34, "name": "Lactococcus garvieae strain FDAARGOS_1062 chromosome, complete genome", "reads_in": 135644, "reads_removed_percent": 0.02506561292795848, "reads_retained_percent": 99.97493438707204}, {"sample": "GCA_016027595.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 32, "name": "Serratia plymuthica strain FDAARGOS_896 chromosome, complete genome", "reads_in": 368456, "reads_removed_percent": 0.00868489046181905, "reads_retained_percent": 99.99131510953818}, {"sample": "GCA_016889605.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 32, "name": "Escherichia coli strain FDAARGOS_1268 chromosome, complete genome", "reads_in": 333912, "reads_removed_percent": 0.009583363281343588, "reads_retained_percent": 99.99041663671866}, {"sample": "GCA_016026875.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 32, "name": "Aeromonas hydrophila strain FDAARGOS_916 chromosome, complete genome", "reads_in": 315580, "reads_removed_percent": 0.010140059572849991, "reads_retained_percent": 99.98985994042715}, {"sample": "GCA_000783575.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 32, "name": "Proteus mirabilis strain FDAARGOS_81 chromosome, complete genome", "reads_in": 271822, "reads_removed_percent": 0.011772409885881203, "reads_retained_percent": 99.98822759011412}, {"sample": "GCA_019047065.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 30, "name": "Aeromonas sp. FDAARGOS 1414 chromosome, complete genome", "reads_in": 299588, "reads_removed_percent": 0.010013752219715075, "reads_retained_percent": 99.98998624778028}, {"sample": "GCA_016027775.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 30, "name": "Delftia acidovorans strain FDAARGOS_891 chromosome, complete genome", "reads_in": 476700, "reads_removed_percent": 0.0062932662051604785, "reads_retained_percent": 99.99370673379484}, {"sample": "GCA_016906205.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 30, "name": "Streptomyces californicus strain FDAARGOS_1212 chromosome", "reads_in": 538842, "reads_removed_percent": 0.005567494738717472, "reads_retained_percent": 99.99443250526129}, {"sample": "GCA_016906265.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 48, "name": "Streptomyces californicus strain FDAARGOS_1210 chromosome", "reads_in": 548102, "reads_removed_percent": 0.008757494043079572, "reads_retained_percent": 99.99124250595692}, {"sample": "GCA_016906185.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 30, "name": "Streptomyces californicus strain FDAARGOS_1209 chromosome", "reads_in": 535448, "reads_removed_percent": 0.005602784957642945, "reads_retained_percent": 99.99439721504235}, {"sample": "GCA_016889385.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 50, "name": "Haemophilus parahaemolyticus strain FDAARGOS_1199 chromosome, complete genome", "reads_in": 139564, "reads_removed_percent": 0.03582585767103264, "reads_retained_percent": 99.96417414232897}, {"sample": "GCA_003798385.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 54, "name": "Yersinia pseudotuberculosis strain FDAARGOS_584 chromosome, complete genome", "reads_in": 312628, "reads_removed_percent": 0.017272925009915938, "reads_retained_percent": 99.98272707499008}, {"sample": "GCA_020097475.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Escherichia fergusonii strain FDAARGOS_1499 chromosome, complete genome", "reads_in": 309728, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019930905.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 272, "name": "Haemophilus aegyptius strain FDAARGOS_1478 chromosome, complete genome", "reads_in": 132884, "reads_removed_percent": 0.20468980464164233, "reads_retained_percent": 99.79531019535835}, {"sample": "GCA_020736045.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 262, "name": "Haemophilus influenzae strain FDAARGOS_1560 chromosome, complete genome", "reads_in": 126044, "reads_removed_percent": 0.20786392053568592, "reads_retained_percent": 99.79213607946431}, {"sample": "GCA_002073615.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 172, "name": "Neisseria gonorrhoeae strain FDAARGOS_205 chromosome, complete genome", "reads_in": 151930, "reads_removed_percent": 0.11321003093529916, "reads_retained_percent": 99.88678996906471}, {"sample": "GCA_002073475.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 172, "name": "Haemophilus influenzae strain FDAARGOS_199 chromosome, complete genome", "reads_in": 122046, "reads_removed_percent": 0.14093046883961785, "reads_retained_percent": 99.85906953116039}, {"sample": "GCA_002073535.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 156, "name": "Neisseria gonorrhoeae strain FDAARGOS_204 chromosome, complete genome", "reads_in": 147496, "reads_removed_percent": 0.10576558008352768, "reads_retained_percent": 99.89423441991647}, {"sample": "GCA_016725865.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 142, "name": "Sphingobacterium multivorum strain FDAARGOS_1143 chromosome, complete genome", "reads_in": 401680, "reads_removed_percent": 0.03535152360087632, "reads_retained_percent": 99.96464847639912}, {"sample": "GCA_001558255.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 140, "name": "Grimontia hollisae strain FDAARGOS_111 chromosome 1, complete sequence", "reads_in": 266562, "reads_removed_percent": 0.05252061434112889, "reads_retained_percent": 99.94747938565887}, {"sample": "GCA_019930705.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 140, "name": "Haemophilus influenzae strain FDAARGOS_1479 chromosome, complete genome", "reads_in": 126840, "reads_removed_percent": 0.11037527593818984, "reads_retained_percent": 99.8896247240618}, {"sample": "GCA_002073655.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 140, "name": "Neisseria gonorrhoeae strain FDAARGOS_207 chromosome, complete genome", "reads_in": 145954, "reads_removed_percent": 0.09592063252805678, "reads_retained_percent": 99.90407936747194}, {"sample": "GCA_016726045.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 130, "name": "Sphingobacterium multivorum strain FDAARGOS_1140 chromosome, complete genome", "reads_in": 377044, "reads_removed_percent": 0.0344787345773968, "reads_retained_percent": 99.96552126542261}, {"sample": "GCA_020736025.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 128, "name": "Haemophilus influenzae strain FDAARGOS_1561 chromosome, complete genome", "reads_in": 122286, "reads_removed_percent": 0.10467265263398919, "reads_retained_percent": 99.895327347366}, {"sample": "GCA_016725925.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 120, "name": "Sphingobacterium multivorum strain FDAARGOS_1141 chromosome, complete genome", "reads_in": 419822, "reads_removed_percent": 0.028583542548985045, "reads_retained_percent": 99.97141645745101}, {"sample": "GCA_016027715.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 118, "name": "Lactococcus garvieae strain FDAARGOS_893 chromosome, complete genome", "reads_in": 140684, "reads_removed_percent": 0.08387592050268687, "reads_retained_percent": 99.91612407949731}, {"sample": "GCA_020735825.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 118, "name": "Haemophilus influenzae strain FDAARGOS_1562 chromosome, complete genome", "reads_in": 122840, "reads_removed_percent": 0.09605991533702378, "reads_retained_percent": 99.90394008466298}, {"sample": "GCA_016128035.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 112, "name": "Alcaligenes faecalis strain FDAARGOS_1024 chromosome, complete genome", "reads_in": 271528, "reads_removed_percent": 0.04124804808343891, "reads_retained_percent": 99.95875195191657}, {"sample": "GCA_016026695.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 92, "name": "Lactococcus garvieae strain FDAARGOS_929 chromosome, complete genome", "reads_in": 138956, "reads_removed_percent": 0.06620800829039408, "reads_retained_percent": 99.93379199170961}, {"sample": "GCA_016725465.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 86, "name": "Sphingobacterium multivorum strain FDAARGOS_1142 chromosome, complete genome", "reads_in": 436790, "reads_removed_percent": 0.01968909544632432, "reads_retained_percent": 99.98031090455368}, {"sample": "GCA_003798265.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 78, "name": "Burkholderia pseudomallei strain FDAARGOS_592 chromosome 1, complete sequence", "reads_in": 476446, "reads_removed_percent": 0.016371215205920504, "reads_retained_percent": 99.98362878479408}, {"sample": "GCA_003813085.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 74, "name": "Alcaligenes faecalis strain FDAARGOS_491 chromosome, complete genome", "reads_in": 271526, "reads_removed_percent": 0.027253375367368132, "reads_retained_percent": 99.97274662463263}, {"sample": "GCA_016026395.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 66, "name": "Edwardsiella hoshinae strain FDAARGOS_940 chromosome, complete genome", "reads_in": 254474, "reads_removed_percent": 0.025935851992737964, "reads_retained_percent": 99.97406414800726}, {"sample": "GCA_008693365.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 66, "name": "Yersinia pseudotuberculosis strain FDAARGOS_665 chromosome, complete genome", "reads_in": 291134, "reads_removed_percent": 0.022669973276910286, "reads_retained_percent": 99.9773300267231}, {"sample": "GCA_003798245.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 64, "name": "Burkholderia pseudomallei strain FDAARGOS_593 chromosome 1, complete sequence", "reads_in": 483702, "reads_removed_percent": 0.013231287032098275, "reads_retained_percent": 99.9867687129679}, {"sample": "GCA_016127955.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 62, "name": "Actinomyces oris strain FDAARGOS_1051 chromosome, complete genome", "reads_in": 212692, "reads_removed_percent": 0.02915013258608692, "reads_retained_percent": 99.97084986741392}, {"sample": "GCA_003798365.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 62, "name": "Burkholderia pseudomallei strain FDAARGOS_594 chromosome 1, complete sequence", "reads_in": 478692, "reads_removed_percent": 0.01295196075973695, "reads_retained_percent": 99.98704803924026}, {"sample": "GCA_020735445.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 60, "name": "Prevotella copri DSM 18205 strain FDAARGOS_1573 chromosome, complete genome", "reads_in": 236774, "reads_removed_percent": 0.02534062016944428, "reads_retained_percent": 99.97465937983056}, {"sample": "GCA_002073595.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 58, "name": "Neisseria meningitidis strain FDAARGOS_210 chromosome, complete genome", "reads_in": 151552, "reads_removed_percent": 0.038270692567567564, "reads_retained_percent": 99.96172930743244}, {"sample": "GCA_016026895.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 54, "name": "Neisseria meningitidis strain FDAARGOS_914 chromosome, complete genome", "reads_in": 145422, "reads_removed_percent": 0.03713330857779428, "reads_retained_percent": 99.9628666914222}, {"sample": "GCA_019048485.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 2, "name": "Enterobacter asburiae strain FDAARGOS 1432 chromosome, complete genome", "reads_in": 326758, "reads_removed_percent": 0.0006120737671304145, "reads_retained_percent": 99.99938792623287}, {"sample": "GCA_019047125.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 30, "name": "Aeromonas sp. FDAARGOS 1411 chromosome, complete genome", "reads_in": 302630, "reads_removed_percent": 0.00991309519875756, "reads_retained_percent": 99.99008690480125}, {"sample": "GCA_000783915.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 28, "name": "Serratia marcescens strain FDAARGOS_65 chromosome, complete genome", "reads_in": 349894, "reads_removed_percent": 0.008002423591144748, "reads_retained_percent": 99.99199757640885}, {"sample": "GCA_003955775.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 20, "name": "Francisella tularensis strain FDAARGOS_599 chromosome, complete genome", "reads_in": 126190, "reads_removed_percent": 0.015849116411760046, "reads_retained_percent": 99.98415088358824}, {"sample": "GCA_016888985.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 20, "name": "Chromobacterium violaceum strain FDAARGOS_1274 chromosome, complete genome", "reads_in": 320536, "reads_removed_percent": 0.006239548755833979, "reads_retained_percent": 99.99376045124417}, {"sample": "GCA_016127755.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 20, "name": "Burkholderia ambifaria strain FDAARGOS_1027 chromosome 1, complete sequence", "reads_in": 501574, "reads_removed_percent": 0.003987447515222081, "reads_retained_percent": 99.99601255248477}, {"sample": "GCA_016027075.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 20, "name": "Delftia acidovorans strain FDAARGOS_909 chromosome, complete genome", "reads_in": 447236, "reads_removed_percent": 0.0044719119212228, "reads_retained_percent": 99.99552808807877}, {"sample": "GCA_016903215.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 20, "name": "Acinetobacter baumannii strain FDAARGOS_1359 chromosome, complete genome", "reads_in": 259886, "reads_removed_percent": 0.0076956819528562525, "reads_retained_percent": 99.99230431804715}, {"sample": "GCA_002073455.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 20, "name": "Legionella longbeachae strain FDAARGOS_201 chromosome, complete genome", "reads_in": 277516, "reads_removed_percent": 0.007206791680479684, "reads_retained_percent": 99.99279320831953}, {"sample": "GCA_016725185.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 18, "name": "Streptococcus anginosus strain FDAARGOS_1155 chromosome, complete genome", "reads_in": 152270, "reads_removed_percent": 0.011821107243711828, "reads_retained_percent": 99.9881788927563}, {"sample": "GCA_016127895.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 18, "name": "Moraxella nonliquefaciens strain FDAARGOS_1006 chromosome, complete genome", "reads_in": 154890, "reads_removed_percent": 0.011621150493898896, "reads_retained_percent": 99.9883788495061}, {"sample": "GCA_013267515.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 18, "name": "Streptococcus intermedius strain FDAARGOS_769 chromosome, complete genome", "reads_in": 128892, "reads_removed_percent": 0.013965180150823946, "reads_retained_percent": 99.98603481984918}, {"sample": "GCA_016026575.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 18, "name": "Fannyhessea vaginae strain FDAARGOS_934 chromosome, complete genome", "reads_in": 96468, "reads_removed_percent": 0.018659037193680804, "reads_retained_percent": 99.98134096280631}, {"sample": "GCA_022213445.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 18, "name": "Burkholderia vietnamiensis strain FDAARGOS_721 chromosome 1, complete sequence", "reads_in": 467904, "reads_removed_percent": 0.003846942962659007, "reads_retained_percent": 99.99615305703735}, {"sample": "GCA_016903615.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 18, "name": "Escherichia coli strain FDAARGOS_1304 chromosome, complete genome", "reads_in": 351638, "reads_removed_percent": 0.005118900687638992, "reads_retained_percent": 99.99488109931237}, {"sample": "GCA_019048745.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 18, "name": "Aeromonas sp. FDAARGOS 1410 chromosome, complete genome", "reads_in": 300686, "reads_removed_percent": 0.005986311301490591, "reads_retained_percent": 99.99401368869852}, {"sample": "GCA_016726945.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 16, "name": "Moraxella osloensis strain FDAARGOS_1130 chromosome, complete genome", "reads_in": 169780, "reads_removed_percent": 0.009423960419366238, "reads_retained_percent": 99.99057603958063}, {"sample": "GCA_003812985.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 16, "name": "Stenotrophomonas maltophilia strain FDAARGOS_507 chromosome, complete genome", "reads_in": 305172, "reads_removed_percent": 0.00524294496218526, "reads_retained_percent": 99.99475705503781}, {"sample": "GCA_016903375.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 16, "name": "Escherichia coli strain FDAARGOS_1286 chromosome, complete genome", "reads_in": 344596, "reads_removed_percent": 0.004643118318262545, "reads_retained_percent": 99.99535688168174}, {"sample": "GCA_002209125.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 16, "name": "Nocardia brasiliensis strain FDAARGOS_352 chromosome, complete genome", "reads_in": 595700, "reads_removed_percent": 0.00268591572939399, "reads_retained_percent": 99.99731408427061}, {"sample": "GCA_019047045.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 16, "name": "Aeromonas sp. FDAARGOS 1419 chromosome, complete genome", "reads_in": 305918, "reads_removed_percent": 0.005230159716002328, "reads_retained_percent": 99.994769840284}, {"sample": "GCA_019048685.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 16, "name": "Acinetobacter nosocomialis strain FDAARGOS 1395 chromosome, complete genome", "reads_in": 260174, "reads_removed_percent": 0.006149730564929624, "reads_retained_percent": 99.99385026943507}, {"sample": "GCA_016027815.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 16, "name": "Delftia lacustris strain FDAARGOS_890 chromosome, complete genome", "reads_in": 484968, "reads_removed_percent": 0.0032991867504660105, "reads_retained_percent": 99.99670081324953}, {"sample": "GCA_008693545.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 16, "name": "Bordetella hinzii strain FDAARGOS_621 chromosome, complete genome", "reads_in": 324030, "reads_removed_percent": 0.004937814399901244, "reads_retained_percent": 99.9950621856001}, {"sample": "GCA_019931045.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 16, "name": "Cupriavidus pauculus strain FDAARGOS_1472 chromosome 1, complete sequence", "reads_in": 432224, "reads_removed_percent": 0.0037017842600133265, "reads_retained_percent": 99.99629821573998}, {"sample": "GCA_008693565.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 16, "name": "Klebsiella michiganensis strain FDAARGOS_647 chromosome, complete genome", "reads_in": 443772, "reads_removed_percent": 0.0036054550534959395, "reads_retained_percent": 99.9963945449465}, {"sample": "GCA_020097275.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 16, "name": "Bacteroides fragilis strain FDAARGOS_1512 chromosome, complete genome", "reads_in": 349070, "reads_removed_percent": 0.0045836078723465205, "reads_retained_percent": 99.99541639212765}, {"sample": "GCA_003798425.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 16, "name": "Yersinia pseudotuberculosis strain FDAARGOS_581 chromosome, complete genome", "reads_in": 339798, "reads_removed_percent": 0.004708679862741982, "reads_retained_percent": 99.99529132013726}, {"sample": "GCA_016728765.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 16, "name": "Chryseobacterium sp. FDAARGOS 1104 strain FDAARGOS_1104 chromosome, complete genome", "reads_in": 246632, "reads_removed_percent": 0.006487398228940284, "reads_retained_percent": 99.99351260177106}, {"sample": "GCA_001559195.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 16, "name": "Achromobacter xylosoxidans strain FDAARGOS_150 chromosome, complete genome", "reads_in": 417916, "reads_removed_percent": 0.003828520563941079, "reads_retained_percent": 99.99617147943606}, {"sample": "GCA_002983625.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 20, "name": "Yersinia intermedia strain FDAARGOS_358 chromosome, complete genome", "reads_in": 330710, "reads_removed_percent": 0.0060475945692600765, "reads_retained_percent": 99.99395240543075}, {"sample": "GCA_002591195.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 28, "name": "Yersinia frederiksenii strain FDAARGOS_418 chromosome, complete genome", "reads_in": 328936, "reads_removed_percent": 0.00851229418488703, "reads_retained_percent": 99.99148770581512}, {"sample": "GCA_002953535.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 20, "name": "Vibrio alginolyticus strain FDAARGOS_110 chromosome 1, complete sequence", "reads_in": 351246, "reads_removed_percent": 0.005694015020811625, "reads_retained_percent": 99.99430598497919}, {"sample": "GCA_016906225.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 22, "name": "Streptomyces californicus strain FDAARGOS_1211 chromosome", "reads_in": 536880, "reads_removed_percent": 0.004097749962747727, "reads_retained_percent": 99.99590225003725}, {"sample": "GCA_002073215.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 28, "name": "Moraxella catarrhalis strain FDAARGOS_213 chromosome, complete genome", "reads_in": 127200, "reads_removed_percent": 0.0220125786163522, "reads_retained_percent": 99.97798742138365}, {"sample": "GCA_002073235.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 28, "name": "Neisseria meningitidis strain FDAARGOS_214 chromosome, complete genome", "reads_in": 159830, "reads_removed_percent": 0.0175186135268723, "reads_retained_percent": 99.98248138647313}, {"sample": "GCA_002073675.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 28, "name": "Neisseria meningitidis strain FDAARGOS_212 chromosome, complete genome", "reads_in": 149660, "reads_removed_percent": 0.018709073900841908, "reads_retained_percent": 99.98129092609916}, {"sample": "GCA_019047105.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 28, "name": "Enterobacter cloacae strain FDAARGOS 1431 chromosome, complete genome", "reads_in": 373508, "reads_removed_percent": 0.007496492712338156, "reads_retained_percent": 99.99250350728767}, {"sample": "GCA_002073315.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 26, "name": "Yersinia sp. FDAARGOS_228 chromosome, complete genome", "reads_in": 330870, "reads_removed_percent": 0.007858071145767219, "reads_retained_percent": 99.99214192885424}, {"sample": "GCA_016889905.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 26, "name": "Escherichia coli strain FDAARGOS_1259 chromosome, complete genome", "reads_in": 348288, "reads_removed_percent": 0.0074650863653068725, "reads_retained_percent": 99.9925349136347}, {"sample": "GCA_016728785.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 26, "name": "Thomasclavelia ramosa strain FDAARGOS_1105 chromosome, complete genome", "reads_in": 236248, "reads_removed_percent": 0.011005384172564423, "reads_retained_percent": 99.98899461582744}, {"sample": "GCA_002073775.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 26, "name": "Vibrio parahaemolyticus strain FDAARGOS_191 chromosome 1, complete sequence", "reads_in": 344886, "reads_removed_percent": 0.007538722940333907, "reads_retained_percent": 99.99246127705966}, {"sample": "GCA_009730035.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 26, "name": "Burkholderia multivorans strain FDAARGOS_719 chromosome 1, complete sequence", "reads_in": 437710, "reads_removed_percent": 0.00594000594000594, "reads_retained_percent": 99.99405999406}, {"sample": "GCA_000783595.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 26, "name": "Proteus mirabilis strain FDAARGOS_80 chromosome, complete genome", "reads_in": 269616, "reads_removed_percent": 0.00964334460862857, "reads_retained_percent": 99.99035665539137}, {"sample": "GCA_002591155.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 26, "name": "Proteus vulgaris strain FDAARGOS_366 chromosome, complete genome", "reads_in": 264132, "reads_removed_percent": 0.009843563066951373, "reads_retained_percent": 99.99015643693305}, {"sample": "GCA_019046985.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 26, "name": "Aeromonas sp. FDAARGOS 1417 chromosome, complete genome", "reads_in": 319550, "reads_removed_percent": 0.008136441871381631, "reads_retained_percent": 99.99186355812861}, {"sample": "GCA_003798445.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 26, "name": "Yersinia pseudotuberculosis strain FDAARGOS_580 chromosome, complete genome", "reads_in": 332068, "reads_removed_percent": 0.00782972162328198, "reads_retained_percent": 99.99217027837672}, {"sample": "GCA_003798225.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 26, "name": "Yersinia pestis strain FDAARGOS_601 chromosome, complete genome", "reads_in": 328732, "reads_removed_percent": 0.007909178297214752, "reads_retained_percent": 99.99209082170279}, {"sample": "GCA_016906245.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 26, "name": "Streptomyces californicus strain FDAARGOS_1213 chromosome", "reads_in": 538938, "reads_removed_percent": 0.004824302609947712, "reads_retained_percent": 99.99517569739005}, {"sample": "GCA_016903135.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 26, "name": "Acinetobacter baumannii strain FDAARGOS_1360 chromosome, complete genome", "reads_in": 271340, "reads_removed_percent": 0.009582074150512272, "reads_retained_percent": 99.99041792584948}, {"sample": "GCA_003546905.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 24, "name": "Yersinia pseudotuberculosis strain FDAARGOS_342 chromosome, complete genome", "reads_in": 327036, "reads_removed_percent": 0.007338641617436612, "reads_retained_percent": 99.99266135838256}, {"sample": "GCA_016027435.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 24, "name": "Burkholderia oklahomensis strain FDAARGOS_900 chromosome 1, complete sequence", "reads_in": 487968, "reads_removed_percent": 0.004918355301987015, "reads_retained_percent": 99.99508164469802}, {"sample": "GCA_002393425.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 24, "name": "Burkholderia gladioli pv. gladioli strain FDAARGOS_389 chromosome 1, complete sequence", "reads_in": 552478, "reads_removed_percent": 0.004344064379034097, "reads_retained_percent": 99.99565593562096}, {"sample": "GCA_016725905.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 24, "name": "Proteus mirabilis strain FDAARGOS_1079 chromosome, complete genome", "reads_in": 269036, "reads_removed_percent": 0.008920739231924353, "reads_retained_percent": 99.99107926076807}, {"sample": "GCA_000783995.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 22, "name": "Citrobacter freundii strain FDAARGOS_61 chromosome, complete genome", "reads_in": 350566, "reads_removed_percent": 0.006275565799307405, "reads_retained_percent": 99.99372443420069}, {"sample": "GCA_021183685.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 22, "name": "Mycobacterium kansasii strain FDAARGOS_1615 chromosome, complete genome", "reads_in": 430348, "reads_removed_percent": 0.005112141801518771, "reads_retained_percent": 99.99488785819848}, {"sample": "GCA_003812205.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 22, "name": "Acinetobacter sp. FDAARGOS_560 chromosome, complete genome", "reads_in": 277772, "reads_removed_percent": 0.007920164739426581, "reads_retained_percent": 99.99207983526057}, {"sample": "GCA_003812585.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 22, "name": "Burkholderia multivorans strain FDAARGOS_548 chromosome 1, complete sequence", "reads_in": 443592, "reads_removed_percent": 0.004959512344677091, "reads_retained_percent": 99.99504048765532}, {"sample": "GCA_003798305.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 22, "name": "Yersinia pseudotuberculosis strain FDAARGOS_579 chromosome, complete genome", "reads_in": 312098, "reads_removed_percent": 0.0070490679209735405, "reads_retained_percent": 99.99295093207903}, {"sample": "GCA_001558495.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 22, "name": "Vibrio parahaemolyticus strain ATCC 17802 chromosome 1, complete sequence", "reads_in": 343498, "reads_removed_percent": 0.006404695223844097, "reads_retained_percent": 99.99359530477615}, {"sample": "GCA_001188185.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 22, "name": "Vibrio parahaemolyticus strain FDAARGOS_51 chromosome 1, complete sequence", "reads_in": 343280, "reads_removed_percent": 0.006408762526217665, "reads_retained_percent": 99.99359123747378}, {"sample": "GCA_013267315.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 20, "name": "Bacillus thuringiensis strain FDAARGOS_795 chromosome, complete genome", "reads_in": 381992, "reads_removed_percent": 0.005235711742654296, "reads_retained_percent": 99.99476428825734}, {"sample": "GCA_002953975.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 2, "name": "Vibrio alginolyticus strain FDAARGOS_114 chromosome 1, complete sequence", "reads_in": 345218, "reads_removed_percent": 0.0005793440666477414, "reads_retained_percent": 99.99942065593335}, {"sample": "GCA_002591095.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 18, "name": "Yersinia frederiksenii strain FDAARGOS_417 chromosome, complete genome", "reads_in": 333378, "reads_removed_percent": 0.0053992764969494084, "reads_retained_percent": 99.99460072350305}, {"sample": "GCA_019048285.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 2, "name": "Escherichia coli strain FDAARGOS 1389 chromosome, complete genome", "reads_in": 345514, "reads_removed_percent": 0.0005788477456774545, "reads_retained_percent": 99.99942115225433}, {"sample": "GCA_016728105.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Corynebacterium striatum strain FDAARGOS_1115 chromosome, complete genome", "reads_in": 193656, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016728205.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Corynebacterium striatum strain FDAARGOS_1116 chromosome, complete genome", "reads_in": 177712, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016728365.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Corynebacterium tuberculostearicum strain FDAARGOS_1117 chromosome, complete genome", "reads_in": 163456, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016728505.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Corynebacterium riegelii strain FDAARGOS_1114 chromosome, complete genome", "reads_in": 167948, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016728625.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Corynebacterium propinquum strain FDAARGOS_1113 chromosome, complete genome", "reads_in": 164086, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016728645.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Corynebacterium glucuronolyticum strain FDAARGOS_1111 chromosome, complete genome", "reads_in": 196126, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889165.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus epidermidis strain FDAARGOS_1243 chromosome, complete genome", "reads_in": 168996, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002073355.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Streptococcus intermedius strain FDAARGOS_233 chromosome, complete genome", "reads_in": 127600, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903295.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1323 chromosome, complete genome", "reads_in": 366060, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889225.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Corynebacterium glucuronolyticum strain FDAARGOS_1190 chromosome, complete genome", "reads_in": 188214, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889845.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1248 chromosome, complete genome", "reads_in": 337768, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889865.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1253 chromosome, complete genome", "reads_in": 313026, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001559235.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Citrobacter sp. FDAARGOS_156 strain FDAARGOS_155 chromosome, complete genome", "reads_in": 315166, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889925.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Bacteroides fragilis strain FDAARGOS_1225 chromosome, complete genome", "reads_in": 349930, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889945.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus aureus strain FDAARGOS_1245 chromosome, complete genome", "reads_in": 192584, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889965.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus condimenti strain FDAARGOS_1205 chromosome, complete genome", "reads_in": 180552, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889985.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Corynebacterium sp. FDAARGOS 1242 strain FDAARGOS_1242 chromosome, complete genome", "reads_in": 185000, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016890005.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1250 chromosome, complete genome", "reads_in": 337234, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016890025.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus carnosus strain FDAARGOS_1204 chromosome, complete genome", "reads_in": 176972, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001558475.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Vibrio mimicus strain FDAARGOS_112 chromosome 1, complete sequence", "reads_in": 295914, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016890105.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1266 chromosome, complete genome", "reads_in": 347734, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903095.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Bordetella pertussis strain FDAARGOS_1368 chromosome, complete genome", "reads_in": 273672, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903115.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1327 chromosome, complete genome", "reads_in": 366464, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001559115.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus simulans strain FDAARGOS_124 chromosome, complete genome", "reads_in": 176504, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903155.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1326 chromosome, complete genome", "reads_in": 368106, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903175.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Bordetella pertussis strain FDAARGOS_1369 chromosome, complete genome", "reads_in": 273602, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903195.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1305 chromosome, complete genome", "reads_in": 384390, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016728085.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1077 chromosome, complete genome", "reads_in": 339426, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001559075.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Citrobacter amalonaticus strain FDAARGOS_122 chromosome, complete genome", "reads_in": 326982, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016728065.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Pseudomonas fluorescens strain FDAARGOS_1088 chromosome, complete genome", "reads_in": 408974, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016728045.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Serratia ureilytica strain FDAARGOS_1089 chromosome, complete genome", "reads_in": 346696, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002073635.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Paracoccus yeei strain FDAARGOS_252 chromosome, complete genome", "reads_in": 321974, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002073515.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Streptococcus equinus strain FDAARGOS_251 chromosome, complete genome", "reads_in": 154102, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002073495.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Leptospira interrogans serovar Copenhageni strain FDAARGOS_203 chromosome, complete genome", "reads_in": 308718, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002073435.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Streptococcus gordonii strain FDAARGOS_257 chromosome, complete genome", "reads_in": 148180, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016888945.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Phocaeicola coprophilus strain FDAARGOS_1220 chromosome, complete genome", "reads_in": 274240, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002073415.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Bacillus sp. FDAARGOS_235 chromosome, complete genome", "reads_in": 373892, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002073395.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus lugdunensis strain FDAARGOS_222 chromosome, complete genome", "reads_in": 169354, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889005.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Clostridium] scindens strain FDAARGOS_1227 chromosome, complete genome", "reads_in": 241274, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889025.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1251 chromosome, complete genome", "reads_in": 349158, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889045.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1255 chromosome, complete genome", "reads_in": 337566, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002073375.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Corynebacterium diphtheriae strain FDAARGOS_197 chromosome, complete genome", "reads_in": 165936, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889085.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1256 chromosome, complete genome", "reads_in": 346170, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889105.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1254 chromosome, complete genome", "reads_in": 326142, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889125.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1249 chromosome, complete genome", "reads_in": 348824, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889145.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus aureus strain FDAARGOS_1246 chromosome, complete genome", "reads_in": 192792, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016728725.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Corynebacterium amycolatum strain FDAARGOS_1108 chromosome, complete genome", "reads_in": 165074, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016728705.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Corynebacterium aurimucosum strain FDAARGOS_1109 chromosome, complete genome", "reads_in": 182518, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016728685.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Corynebacterium aurimucosum strain FDAARGOS_1110 chromosome, complete genome", "reads_in": 188632, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016728025.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Listeria monocytogenes strain FDAARGOS_1080 chromosome, complete genome", "reads_in": 197582, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016727385.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus epidermidis strain FDAARGOS_1083 chromosome, complete genome", "reads_in": 165308, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016727405.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Bacillus cereus strain FDAARGOS_1084 chromosome, complete genome", "reads_in": 354920, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016727425.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1076 chromosome, complete genome", "reads_in": 360454, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016727445.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Morganella morganii strain FDAARGOS_1085 chromosome, complete genome", "reads_in": 265002, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016727585.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus aureus strain FDAARGOS_1086 chromosome, complete genome", "reads_in": 185098, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016727765.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Yersinia enterocolitica strain FDAARGOS_1082 chromosome, complete genome", "reads_in": 302752, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002073755.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Citrobacter braakii strain FDAARGOS_253 chromosome, complete genome", "reads_in": 349620, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002073735.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Clostridioides difficile strain FDAARGOS_267 chromosome, complete genome", "reads_in": 285744, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016728665.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Corynebacterium propinquum strain FDAARGOS_1112 chromosome, complete genome", "reads_in": 165342, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016728805.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Acinetobacter johnsonii strain FDAARGOS_1092 chromosome, complete genome", "reads_in": 275708, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903255.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1311 chromosome, complete genome", "reads_in": 370730, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889785.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Corynebacterium glucuronolyticum strain FDAARGOS_1191 chromosome, complete genome", "reads_in": 192446, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735345.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Mycobacteroides abscessus subsp. abscessus strain FDAARGOS_1605 chromosome, complete genome", "reads_in": 335566, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020099255.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Enterobacter bugandensis strain FDAARGOS_1496 chromosome, complete genome", "reads_in": 334858, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735365.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Blautia massiliensis strain FDAARGOS_1576 chromosome, complete genome", "reads_in": 248852, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735385.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Granulicatella elegans strain FDAARGOS_1559 chromosome, complete genome", "reads_in": 126042, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735405.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Mycobacterium avium subsp. avium strain FDAARGOS_1606 chromosome, complete genome", "reads_in": 330704, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735425.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Mycobacteroides abscessus subsp. abscessus strain FDAARGOS_1604 chromosome, complete genome", "reads_in": 339370, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001018775.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus aureus strain FDAARGOS_14 chromosome, complete genome", "reads_in": 184314, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735465.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Collinsella stercoris DSM 13279 strain FDAARGOS_1572 chromosome, complete genome", "reads_in": 164050, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735485.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Mycobacteroides abscessus subsp. abscessus strain FDAARGOS_1603 chromosome, complete genome", "reads_in": 322226, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735505.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Mycobacteroides abscessus strain FDAARGOS_1602 chromosome, complete genome", "reads_in": 322236, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020099275.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Corynebacterium falsenii strain FDAARGOS_1494 chromosome, complete genome", "reads_in": 178492, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020099235.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Enterobacter bugandensis strain FDAARGOS_1497 chromosome, complete genome", "reads_in": 334858, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735545.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Gordonia otitidis strain FDAARGOS_1600 chromosome, complete genome", "reads_in": 340588, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001018805.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus aureus strain FDAARGOS_15 chromosome, complete genome", "reads_in": 186660, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020097195.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Enterobacter hormaechei subsp. oharae strain FDAARGOS_1533 chromosome, complete genome", "reads_in": 315438, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001018975.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus aureus strain FDAARGOS_30 chromosome, complete genome", "reads_in": 190776, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020097235.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Enterobacter bugandensis strain FDAARGOS_1520 chromosome, complete genome", "reads_in": 334858, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020097255.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Thomasclavelia ramosa strain FDAARGOS_1519 chromosome, complete genome", "reads_in": 254472, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001018915.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus aureus strain FDAARGOS_27 chromosome, complete genome", "reads_in": 182690, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001018845.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus aureus strain FDAARGOS_21 chromosome, complete genome", "reads_in": 192082, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001018835.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus aureus strain FDAARGOS_19 chromosome, complete genome", "reads_in": 191936, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020097335.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Streptococcus pneumoniae strain FDAARGOS_1508 chromosome, complete genome", "reads_in": 141108, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020097375.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Porphyromonas endodontalis strain FDAARGOS_1506 chromosome, complete genome", "reads_in": 140196, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020099215.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Enterobacter ludwigii strain FDAARGOS_1498 chromosome, complete genome", "reads_in": 315380, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020097395.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Leclercia adecarboxylata strain FDAARGOS_1505 chromosome, complete genome", "reads_in": 337142, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002983645.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 2, "name": "Escherichia coli O104:H4 strain FDAARGOS_349 chromosome, complete genome", "reads_in": 366744, "reads_removed_percent": 0.0005453395283903759, "reads_retained_percent": 99.99945466047161}, {"sample": "GCA_020097415.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Klebsiella quasipneumoniae strain FDAARGOS_1504 chromosome, complete genome", "reads_in": 344172, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020450285.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "UNVERIFIED_ORG: Hafnia paralvei strain FDAARGOS_1502 chromosome, complete genome", "reads_in": 322462, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889825.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Acinetobacter pittii strain FDAARGOS_1214 chromosome, complete genome", "reads_in": 273260, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020221755.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Lactococcus lactis strain FDAARGOS_867 chromosome, complete genome", "reads_in": 178994, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020099375.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Acinetobacter variabilis strain FDAARGOS_1487 chromosome, complete genome", "reads_in": 232316, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889765.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Corynebacterium minutissimum strain FDAARGOS_1196 chromosome, complete genome", "reads_in": 181688, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889425.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Corynebacterium amycolatum strain FDAARGOS_1189 chromosome, complete genome", "reads_in": 164996, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889245.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Mammaliicoccus vitulinus strain FDAARGOS_1207 chromosome, complete genome", "reads_in": 176858, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002073335.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Vibrio cholerae strain FDAARGOS_223 chromosome 1, complete sequence", "reads_in": 269486, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002073295.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Pasteurella multocida strain FDAARGOS_216 chromosome, complete genome", "reads_in": 153012, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889325.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1258 chromosome, complete genome", "reads_in": 348734, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889365.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Corynebacterium kroppenstedtii strain FDAARGOS_1192 chromosome, complete genome", "reads_in": 163120, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002073255.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Pasteurella multocida strain FDAARGOS_218 chromosome, complete genome", "reads_in": 155702, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889405.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Corynebacterium kroppenstedtii strain FDAARGOS_1194 chromosome, complete genome", "reads_in": 167458, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889445.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Corynebacterium striatum strain FDAARGOS_1197 chromosome, complete genome", "reads_in": 200318, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889745.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Bacteroides caccae strain FDAARGOS_1219 chromosome, complete genome", "reads_in": 304730, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889465.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Corynebacterium macginleyi strain FDAARGOS_1195 chromosome, complete genome", "reads_in": 166236, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001593585.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Klebsiella aerogenes strain FDAARGOS_139 chromosome, complete genome", "reads_in": 353188, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889625.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1262 chromosome, complete genome", "reads_in": 348356, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001559255.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Hafnia paralvei strain FDAARGOS_158 chromosome, complete genome", "reads_in": 300914, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889665.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Enterocloster bolteae strain FDAARGOS_1231 chromosome, complete genome", "reads_in": 440932, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889685.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1267 chromosome, complete genome", "reads_in": 362318, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889705.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Amycolatopsis sp. FDAARGOS 1241 strain FDAARGOS_1241 chromosome, complete genome", "reads_in": 653208, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889725.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1252 chromosome, complete genome", "reads_in": 362544, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001559215.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Klebsiella aerogenes strain FDAARGOS_152 chromosome, complete genome", "reads_in": 354476, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019047545.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Escherichia fergusonii strain FDAARGOS 1438 chromosome, complete genome", "reads_in": 302878, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016939455.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Leuconostoc falkenbergense strain FDAARGOS_1201 chromosome, complete genome", "reads_in": 126594, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735285.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Mycobacterium avium subsp. avium strain FDAARGOS_1607 chromosome, complete genome", "reads_in": 330462, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020099295.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Clostridium sporogenes strain FDAARGOS_1491 chromosome, complete genome", "reads_in": 271160, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020099315.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Citrobacter europaeus strain FDAARGOS_1490 chromosome, complete genome", "reads_in": 339568, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020099335.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Citrobacter amalonaticus strain FDAARGOS_1489 chromosome, complete genome", "reads_in": 338134, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020099355.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Burkholderia vietnamiensis strain FDAARGOS_1488 chromosome 1, complete sequence", "reads_in": 462034, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020149725.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Campylobacter coli strain FDAARGOS_1464 chromosome, complete genome", "reads_in": 107916, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002073695.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Streptococcus agalactiae strain FDAARGOS_254 chromosome, complete genome", "reads_in": 147914, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889305.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Acinetobacter pittii strain FDAARGOS_1217 chromosome, complete genome", "reads_in": 263804, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016728745.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Corynebacterium amycolatum strain FDAARGOS_1107 chromosome, complete genome", "reads_in": 180436, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001019485.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 2, "name": "Staphylococcus aureus strain FDAARGOS_35 chromosome, complete genome", "reads_in": 197060, "reads_removed_percent": 0.0010149193139145439, "reads_retained_percent": 99.99898508068608}, {"sample": "GCA_020735725.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 2, "name": "Alistipes senegalensis strain FDAARGOS_1578 chromosome, complete genome", "reads_in": 268220, "reads_removed_percent": 0.000745656550592797, "reads_retained_percent": 99.99925434344941}, {"sample": "GCA_003813045.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 2, "name": "Burkholderia dolosa strain FDAARGOS_562 chromosome 1, complete sequence", "reads_in": 427274, "reads_removed_percent": 0.00046808371209107035, "reads_retained_percent": 99.9995319162879}, {"sample": "GCA_003955815.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 2, "name": "Francisella tularensis strain FDAARGOS_595 chromosome, complete genome", "reads_in": 126280, "reads_removed_percent": 0.0015837820715869496, "reads_retained_percent": 99.99841621792841}, {"sample": "GCA_002951195.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 2, "name": "Pandoraea apista strain FDAARGOS_126 chromosome, complete genome", "reads_in": 355100, "reads_removed_percent": 0.0005632216277105042, "reads_retained_percent": 99.99943677837229}, {"sample": "GCA_019047825.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 2, "name": "Streptococcus mitis strain FDAARGOS 1456 chromosome, complete genome", "reads_in": 124590, "reads_removed_percent": 0.0016052652700858816, "reads_retained_percent": 99.99839473472991}, {"sample": "GCA_019047325.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 2, "name": "Aeromonas sp. FDAARGOS 1403 chromosome, complete genome", "reads_in": 321318, "reads_removed_percent": 0.0006224363403232935, "reads_retained_percent": 99.99937756365968}, {"sample": "GCA_016904495.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 2, "name": "Escherichia coli strain FDAARGOS_1281 chromosome, complete genome", "reads_in": 372684, "reads_removed_percent": 0.0005366476693391721, "reads_retained_percent": 99.99946335233066}, {"sample": "GCA_016026195.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 2, "name": "Escherichia coli strain FDAARGOS_946 chromosome, complete genome", "reads_in": 359310, "reads_removed_percent": 0.0005566224151846595, "reads_retained_percent": 99.99944337758481}, {"sample": "GCA_019047145.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 2, "name": "Escherichia coli strain FDAARGOS_1379 chromosome, complete genome", "reads_in": 335034, "reads_removed_percent": 0.0005969543389626128, "reads_retained_percent": 99.99940304566104}, {"sample": "GCA_020736245.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 2, "name": "Burkholderia dolosa strain FDAARGOS_1544 chromosome 1, complete sequence", "reads_in": 427432, "reads_removed_percent": 0.00046791068520840747, "reads_retained_percent": 99.9995320893148}, {"sample": "GCA_016027575.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 2, "name": "Bacillus tropicus strain FDAARGOS_897 chromosome, complete genome", "reads_in": 381134, "reads_removed_percent": 0.0005247498255206831, "reads_retained_percent": 99.99947525017448}, {"sample": "GCA_013343095.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 2, "name": "Achromobacter denitrificans strain FDAARGOS_787 chromosome, complete genome", "reads_in": 454436, "reads_removed_percent": 0.00044010597751938666, "reads_retained_percent": 99.99955989402248}, {"sample": "GCA_002984285.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 2, "name": "Salmonella enterica subsp. enterica serovar Typhimurium strain FDAARGOS_317 chromosome, complete genome", "reads_in": 346190, "reads_removed_percent": 0.000577717438400878, "reads_retained_percent": 99.9994222825616}, {"sample": "GCA_016027055.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 2, "name": "Acinetobacter johnsonii strain FDAARGOS_910 chromosome, complete genome", "reads_in": 241162, "reads_removed_percent": 0.0008293180517660328, "reads_retained_percent": 99.99917068194823}, {"sample": "GCA_021183585.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 2, "name": "Nocardia asteroides strain FDAARGOS_1622 chromosome, complete genome", "reads_in": 465562, "reads_removed_percent": 0.000429588325507666, "reads_retained_percent": 99.9995704116745}, {"sample": "GCA_002984395.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 2, "name": "Klebsiella oxytoca strain FDAARGOS_335 chromosome, complete genome", "reads_in": 403320, "reads_removed_percent": 0.0004958841614598829, "reads_retained_percent": 99.99950411583853}, {"sample": "GCA_021183665.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 2, "name": "Mycolicibacterium smegmatis strain FDAARGOS_1618 chromosome, complete genome", "reads_in": 459422, "reads_removed_percent": 0.0004353296098140707, "reads_retained_percent": 99.99956467039019}, {"sample": "GCA_020735805.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 2, "name": "Helicobacter pylori strain FDAARGOS_1563 chromosome, complete genome", "reads_in": 110868, "reads_removed_percent": 0.0018039470361150196, "reads_retained_percent": 99.99819605296389}, {"sample": "GCA_002983865.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 2, "name": "Roseobacter denitrificans strain FDAARGOS_309 chromosome, complete genome", "reads_in": 287254, "reads_removed_percent": 0.0006962479199593391, "reads_retained_percent": 99.99930375208004}, {"sample": "GCA_003798405.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 2, "name": "Yersinia pseudotuberculosis strain FDAARGOS_582 chromosome, complete genome", "reads_in": 322164, "reads_removed_percent": 0.0006208018276405806, "reads_retained_percent": 99.99937919817236}, {"sample": "GCA_000783735.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 2, "name": "Shigella flexneri strain FDAARGOS_74 chromosome, complete genome", "reads_in": 320390, "reads_removed_percent": 0.0006242392084646837, "reads_retained_percent": 99.99937576079154}, {"sample": "GCA_016903355.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Streptococcus anginosus strain FDAARGOS_1357 chromosome, complete genome", "reads_in": 132094, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904515.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Klebsiella oxytoca strain FDAARGOS_1331 chromosome, complete genome", "reads_in": 400534, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903515.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1330 chromosome, complete genome", "reads_in": 365880, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903535.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1373 chromosome, complete genome", "reads_in": 321672, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903555.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus epidermidis strain FDAARGOS_1363 chromosome, complete genome", "reads_in": 167308, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009730495.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 2, "name": "Clostridioides difficile strain FDAARGOS_723 chromosome, complete genome", "reads_in": 278540, "reads_removed_percent": 0.0007180297264306742, "reads_retained_percent": 99.99928197027357}, {"sample": "GCA_016904655.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1284 chromosome, complete genome", "reads_in": 337852, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127175.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 2, "name": "Veillonella parvula strain FDAARGOS_1046 chromosome, complete genome", "reads_in": 142154, "reads_removed_percent": 0.0014069248842804283, "reads_retained_percent": 99.99859307511572}, {"sample": "GCA_001559135.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 2, "name": "Serratia liquefaciens strain FDAARGOS_125 chromosome, complete genome", "reads_in": 352316, "reads_removed_percent": 0.0005676722033628902, "reads_retained_percent": 99.99943232779664}, {"sample": "GCA_013267355.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 2, "name": "Achromobacter pestifer strain FDAARGOS_790 chromosome, complete genome", "reads_in": 478488, "reads_removed_percent": 0.00041798331410610086, "reads_retained_percent": 99.99958201668589}, {"sample": "GCA_016903995.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 2, "name": "Escherichia coli strain FDAARGOS_1337 chromosome, complete genome", "reads_in": 331158, "reads_removed_percent": 0.0006039413210612457, "reads_retained_percent": 99.99939605867894}, {"sample": "GCA_002073715.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Neisseria mucosa strain FDAARGOS_260 chromosome, complete genome", "reads_in": 185596, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002208885.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 2, "name": "Stenotrophomonas maltophilia strain FDAARGOS_325 chromosome, complete genome", "reads_in": 323434, "reads_removed_percent": 0.0006183641793998157, "reads_retained_percent": 99.9993816358206}, {"sample": "GCA_016403145.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 2, "name": "Agrobacterium tumefaciens strain FDAARGOS_1048 chromosome 1, complete sequence", "reads_in": 395512, "reads_removed_percent": 0.0005056736584477841, "reads_retained_percent": 99.99949432634155}, {"sample": "GCA_020097455.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 2, "name": "Gemella haemolysans strain FDAARGOS_1500 chromosome, complete genome", "reads_in": 128550, "reads_removed_percent": 0.0015558148580318942, "reads_retained_percent": 99.99844418514196}, {"sample": "GCA_013267455.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 2, "name": "Bacillus cereus strain FDAARGOS_781 chromosome, complete genome", "reads_in": 376156, "reads_removed_percent": 0.0005316942970469699, "reads_retained_percent": 99.99946830570295}, {"sample": "GCA_016599815.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 2, "name": "Anaerococcus obesiensis strain FDAARGOS_989 chromosome, complete genome", "reads_in": 126308, "reads_removed_percent": 0.0015834309782436584, "reads_retained_percent": 99.99841656902176}, {"sample": "GCA_003855135.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 2, "name": "Shigella flexneri strain FDAARGOS_535 chromosome, complete genome", "reads_in": 324836, "reads_removed_percent": 0.0006156953047076062, "reads_retained_percent": 99.9993843046953}, {"sample": "GCA_016028495.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 2, "name": "Salmonella enterica strain FDAARGOS_878 chromosome, complete genome", "reads_in": 331248, "reads_removed_percent": 0.0006037772303530889, "reads_retained_percent": 99.99939622276965}, {"sample": "GCA_003854935.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 2, "name": "Cupriavidus pauculus strain FDAARGOS_614 chromosome 1, complete sequence", "reads_in": 411854, "reads_removed_percent": 0.00048560897793878415, "reads_retained_percent": 99.99951439102206}, {"sample": "GCA_002591135.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 2, "name": "Escherichia coli strain FDAARGOS_448 chromosome, complete genome", "reads_in": 363122, "reads_removed_percent": 0.0005507790770044227, "reads_retained_percent": 99.999449220923}, {"sample": "GCA_016889805.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 2, "name": "Rhodanobacter sp. FDAARGOS 1247 strain FDAARGOS_1247 chromosome, complete genome", "reads_in": 261508, "reads_removed_percent": 0.0007647949584716338, "reads_retained_percent": 99.99923520504153}, {"sample": "GCA_008693745.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 2, "name": "Vibrio parahaemolyticus strain FDAARGOS_662 chromosome 1, complete sequence", "reads_in": 343608, "reads_removed_percent": 0.0005820586249447045, "reads_retained_percent": 99.99941794137506}, {"sample": "GCA_002953335.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 2, "name": "Vibrio diabolicus strain FDAARGOS_96 chromosome 1, complete sequence", "reads_in": 344910, "reads_removed_percent": 0.0005798614131222638, "reads_retained_percent": 99.99942013858688}, {"sample": "GCA_009730295.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 2, "name": "Gemella sanguinis strain FDAARGOS_742 chromosome, complete genome", "reads_in": 119774, "reads_removed_percent": 0.0016698114782841016, "reads_retained_percent": 99.99833018852172}, {"sample": "GCA_002951115.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 2, "name": "Stenotrophomonas maltophilia strain FDAARGOS_92 chromosome, complete genome", "reads_in": 321348, "reads_removed_percent": 0.0006223782316989681, "reads_retained_percent": 99.9993776217683}, {"sample": "GCA_002209085.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 2, "name": "Burkholderia cepacia strain FDAARGOS_345 chromosome 1, complete sequence", "reads_in": 571324, "reads_removed_percent": 0.0003500640617232954, "reads_retained_percent": 99.99964993593828}, {"sample": "GCA_016903275.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 2, "name": "Escherichia coli strain FDAARGOS_1282 chromosome, complete genome", "reads_in": 352200, "reads_removed_percent": 0.0005678591709256105, "reads_retained_percent": 99.99943214082907}, {"sample": "GCA_019930825.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 2, "name": "Aeromonas enteropelogenes strain FDAARGOS_1459 chromosome, complete genome", "reads_in": 294464, "reads_removed_percent": 0.0006792001738752445, "reads_retained_percent": 99.99932079982612}, {"sample": "GCA_001019275.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 2, "name": "Staphylococcus aureus strain FDAARGOS_7 chromosome, complete genome", "reads_in": 181938, "reads_removed_percent": 0.001099275577394497, "reads_retained_percent": 99.99890072442261}, {"sample": "GCA_016903235.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 2, "name": "Klebsiella pneumoniae strain FDAARGOS_1324 chromosome, complete genome", "reads_in": 376682, "reads_removed_percent": 0.0005309518373588332, "reads_retained_percent": 99.99946904816264}, {"sample": "GCA_016127275.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 2, "name": "Streptococcus infantarius strain FDAARGOS_1019 chromosome, complete genome", "reads_in": 130582, "reads_removed_percent": 0.001531604662204592, "reads_retained_percent": 99.9984683953378}, {"sample": "GCA_019930665.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 2, "name": "Bacteroides nordii strain FDAARGOS_1461 chromosome, complete genome", "reads_in": 378572, "reads_removed_percent": 0.0005283010893568463, "reads_retained_percent": 99.99947169891064}, {"sample": "GCA_002209005.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 2, "name": "Campylobacter jejuni strain FDAARGOS_262 chromosome, complete genome", "reads_in": 117896, "reads_removed_percent": 0.0016964103956029044, "reads_retained_percent": 99.99830358960439}, {"sample": "GCA_016889885.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 2, "name": "Streptococcus constellatus strain FDAARGOS_1208 chromosome, complete genome", "reads_in": 131912, "reads_removed_percent": 0.001516162290011523, "reads_retained_percent": 99.99848383770998}, {"sample": "GCA_016903395.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 2, "name": "Klebsiella pneumoniae strain FDAARGOS_1313 chromosome, complete genome", "reads_in": 385540, "reads_removed_percent": 0.0005187529179851636, "reads_retained_percent": 99.99948124708202}, {"sample": "GCA_001559175.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 2, "name": "Streptococcus pyogenes strain FDAARGOS_149 chromosome, complete genome", "reads_in": 122642, "reads_removed_percent": 0.0016307627077184, "reads_retained_percent": 99.99836923729228}, {"sample": "GCA_016903575.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus epidermidis strain FDAARGOS_1364 chromosome, complete genome", "reads_in": 163244, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002944935.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 2, "name": "Escherichia coli strain FDAARGOS_144 chromosome, complete genome", "reads_in": 334812, "reads_removed_percent": 0.0005973501547136901, "reads_retained_percent": 99.99940264984528}, {"sample": "GCA_001558775.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus lugdunensis strain FDAARGOS_141 chromosome, complete genome", "reads_in": 171456, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904095.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1319 chromosome, complete genome", "reads_in": 368102, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904115.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1290 chromosome, complete genome", "reads_in": 347130, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904135.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus epidermidis strain FDAARGOS_1361 chromosome, complete genome", "reads_in": 168278, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904155.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Coprococcus comes strain FDAARGOS_1339 chromosome, complete genome", "reads_in": 224872, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903955.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1309 chromosome, complete genome", "reads_in": 373694, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903935.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1288 chromosome, complete genome", "reads_in": 378792, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903715.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Klebsiella oxytoca strain FDAARGOS_1332 chromosome, complete genome", "reads_in": 403376, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904335.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1370 chromosome, complete genome", "reads_in": 356214, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904175.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1372 chromosome, complete genome", "reads_in": 321568, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903735.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1325 chromosome, complete genome", "reads_in": 366224, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904415.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1314 chromosome, complete genome", "reads_in": 380180, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001558895.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Morganella morganii strain FDAARGOS_172 chromosome, complete genome", "reads_in": 260538, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903775.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1307 chromosome, complete genome", "reads_in": 378308, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903795.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1308 chromosome, complete genome", "reads_in": 370052, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001558935.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Citrobacter amalonaticus strain FDAARGOS_165 chromosome, complete genome", "reads_in": 338936, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903835.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1299 chromosome, complete genome", "reads_in": 344708, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903855.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1291 chromosome, complete genome", "reads_in": 348814, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889185.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1257 chromosome, complete genome", "reads_in": 354122, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903875.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1329 chromosome, complete genome", "reads_in": 370504, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904375.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Bordetella pertussis strain FDAARGOS_1366 chromosome, complete genome", "reads_in": 273696, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903895.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1321 chromosome, complete genome", "reads_in": 362858, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903915.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1300 chromosome, complete genome", "reads_in": 363838, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904355.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Bordetella pertussis strain FDAARGOS_1365 chromosome, complete genome", "reads_in": 273672, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904435.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1301 chromosome, complete genome", "reads_in": 347014, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001558795.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus aureus strain FDAARGOS_159 chromosome, complete genome", "reads_in": 189278, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003711125.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Enterococcus avium strain FDAARGOS_184 chromosome, complete genome", "reads_in": 248226, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903315.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1289 chromosome, complete genome", "reads_in": 368416, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904075.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1303 chromosome, complete genome", "reads_in": 352828, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904055.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1293 chromosome, complete genome", "reads_in": 342718, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903755.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1317 chromosome, complete genome", "reads_in": 351878, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904215.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Bordetella pertussis strain FDAARGOS_1367 chromosome, complete genome", "reads_in": 273706, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001558755.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Achromobacter xylosoxidans strain FDAARGOS_147 chromosome, complete genome", "reads_in": 459130, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001558875.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Enterococcus gallinarum strain FDAARGOS_163 chromosome, complete genome", "reads_in": 254766, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904535.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1297 chromosome, complete genome", "reads_in": 345586, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904035.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1322 chromosome, complete genome", "reads_in": 363140, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903475.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1315 chromosome, complete genome", "reads_in": 368824, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903495.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1312 chromosome, complete genome", "reads_in": 371120, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904595.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1310 chromosome, complete genome", "reads_in": 372866, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903455.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1318 chromosome, complete genome", "reads_in": 346980, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903435.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1302 chromosome, complete genome", "reads_in": 342060, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903595.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1279 chromosome, complete genome", "reads_in": 356446, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001558855.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Providencia stuartii strain FDAARGOS_145 chromosome, complete genome", "reads_in": 287008, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903415.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Klebsiella oxytoca strain FDAARGOS_1334 chromosome, complete genome", "reads_in": 415644, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904615.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1283 chromosome, complete genome", "reads_in": 347664, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001559055.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Bordetella bronchiseptica strain FDAARGOS_176 chromosome, complete genome", "reads_in": 355948, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904635.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1320 chromosome, complete genome", "reads_in": 351422, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001558915.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Achromobacter xylosoxidans strain FDAARGOS_162 chromosome, complete genome", "reads_in": 437716, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904015.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Ruminococcus] gnavus strain FDAARGOS_1342 chromosome, complete genome", "reads_in": 213130, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001558815.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Staphylococcus lugdunensis strain FDAARGOS_143 chromosome, complete genome", "reads_in": 174258, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904275.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1328 chromosome, complete genome", "reads_in": 383876, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903335.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1295 chromosome, complete genome", "reads_in": 342824, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904295.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1306 chromosome, complete genome", "reads_in": 370864, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903975.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Klebsiella oxytoca strain FDAARGOS_1333 chromosome, complete genome", "reads_in": 404192, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904255.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1371 chromosome, complete genome", "reads_in": 342696, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001559035.2", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Bartonella bacilliformis strain ATCC 35685D-5 chromosome, complete genome", "reads_in": 96228, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904195.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1296 chromosome, complete genome", "reads_in": 333376, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904455.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1298 chromosome, complete genome", "reads_in": 357678, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904315.1", "platform": "illumina", "tool": "ncbi-scrubber", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1316 chromosome, complete genome", "reads_in": 365988, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_008693605.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Citrobacter portucalensis strain FDAARGOS_617 chromosome, complete genome", "reads_in": 328624, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002209005.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 6, "name": "Campylobacter jejuni strain FDAARGOS_262 chromosome, complete genome", "reads_in": 117896, "reads_removed_percent": 0.005089231186808713, "reads_retained_percent": 99.99491076881318}, {"sample": "GCA_008693825.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Pseudomonas oryzihabitans strain FDAARGOS_657 chromosome, complete genome", "reads_in": 350992, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016728785.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 6, "name": "Thomasclavelia ramosa strain FDAARGOS_1105 chromosome, complete genome", "reads_in": 236248, "reads_removed_percent": 0.0025397040398225596, "reads_retained_percent": 99.99746029596018}, {"sample": "GCA_008693805.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Providencia stuartii strain FDAARGOS_645 chromosome, complete genome", "reads_in": 301984, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_008693645.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Citrobacter werkmanii strain FDAARGOS_616 chromosome, complete genome", "reads_in": 336620, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_008693705.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Aeromonas veronii strain FDAARGOS_632 chromosome, complete genome", "reads_in": 304126, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002208665.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Haemophilus] ducreyi strain FDAARGOS_297 chromosome, complete genome", "reads_in": 113420, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_008693725.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Streptococcus dysgalactiae strain FDAARGOS_654 chromosome, complete genome", "reads_in": 150236, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_008693685.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Vibrio vulnificus strain FDAARGOS_663 chromosome 1, complete sequence", "reads_in": 331654, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_008693845.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Pasteurella multocida strain FDAARGOS_644 chromosome, complete genome", "reads_in": 150004, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002209025.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 6, "name": "Campylobacter jejuni strain FDAARGOS_263 chromosome, complete genome", "reads_in": 109432, "reads_removed_percent": 0.0054828569339864025, "reads_retained_percent": 99.99451714306602}, {"sample": "GCA_008693625.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Vibrio parahaemolyticus strain FDAARGOS_667 chromosome 1, complete sequence", "reads_in": 333544, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_008693785.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Roseomonas mucosa strain FDAARGOS_658 chromosome 1, complete sequence", "reads_in": 333084, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_008693665.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Bordetella bronchiseptica strain FDAARGOS_634 chromosome, complete genome", "reads_in": 341072, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735905.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 6, "name": "Prevotella melaninogenica strain FDAARGOS_1567 chromosome 1, complete sequence", "reads_in": 220450, "reads_removed_percent": 0.0027217056021773644, "reads_retained_percent": 99.99727829439782}, {"sample": "GCA_019048685.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 6, "name": "Acinetobacter nosocomialis strain FDAARGOS 1395 chromosome, complete genome", "reads_in": 260174, "reads_removed_percent": 0.002306148961848609, "reads_retained_percent": 99.99769385103815}, {"sample": "GCA_016904375.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Bordetella pertussis strain FDAARGOS_1366 chromosome, complete genome", "reads_in": 273696, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002073235.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 6, "name": "Neisseria meningitidis strain FDAARGOS_214 chromosome, complete genome", "reads_in": 159830, "reads_removed_percent": 0.0037539886129012074, "reads_retained_percent": 99.9962460113871}, {"sample": "GCA_008693585.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Streptococcus agalactiae strain FDAARGOS_670 chromosome, complete genome", "reads_in": 147382, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019930865.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 4, "name": "Campylobacter coli strain FDAARGOS_1466 chromosome, complete genome", "reads_in": 122190, "reads_removed_percent": 0.003273590310172682, "reads_retained_percent": 99.99672640968983}, {"sample": "GCA_020097195.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Enterobacter hormaechei subsp. oharae strain FDAARGOS_1533 chromosome, complete genome", "reads_in": 315438, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013267675.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 6, "name": "Staphylococcus capitis strain FDAARGOS_753 chromosome, complete genome", "reads_in": 166064, "reads_removed_percent": 0.003613064842470373, "reads_retained_percent": 99.99638693515753}, {"sample": "GCA_002278015.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Enterococcus hirae strain FDAARGOS_234 chromosome, complete genome", "reads_in": 189710, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904395.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1277 chromosome, complete genome", "reads_in": 316756, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904415.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1314 chromosome, complete genome", "reads_in": 380180, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019931045.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Cupriavidus pauculus strain FDAARGOS_1472 chromosome 1, complete sequence", "reads_in": 432224, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013267695.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 6, "name": "Streptococcus gallolyticus strain FDAARGOS_755 chromosome, complete genome", "reads_in": 150534, "reads_removed_percent": 0.0039858105145681375, "reads_retained_percent": 99.99601418948544}, {"sample": "GCA_001018735.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Staphylococcus aureus strain FDAARGOS_12 chromosome, complete genome", "reads_in": 181716, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904455.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1298 chromosome, complete genome", "reads_in": 357678, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019931005.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Granulicatella adiacens strain FDAARGOS_1477 chromosome, complete genome", "reads_in": 131028, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020097175.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Aeromonas enteropelogenes strain FDAARGOS_1536 chromosome, complete genome", "reads_in": 288620, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002208685.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Kocuria rhizophila strain FDAARGOS_302 chromosome, complete genome", "reads_in": 179858, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_006364495.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Bacillus subtilis strain FDAARGOS_606 chromosome, complete genome", "reads_in": 269708, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_008693965.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Pseudomonas luteola strain FDAARGOS_637 chromosome 1, complete sequence", "reads_in": 393426, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019046965.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 6, "name": "Acinetobacter pittii strain FDAARGOS 1398 chromosome, complete genome", "reads_in": 255868, "reads_removed_percent": 0.002344959119546016, "reads_retained_percent": 99.99765504088046}, {"sample": "GCA_008693545.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Bordetella hinzii strain FDAARGOS_621 chromosome, complete genome", "reads_in": 324030, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013267255.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 6, "name": "Bacillus cereus strain FDAARGOS_799 chromosome, complete genome", "reads_in": 412650, "reads_removed_percent": 0.0014540167211922936, "reads_retained_percent": 99.9985459832788}, {"sample": "GCA_009730015.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Salmonella enterica strain FDAARGOS_712 chromosome, complete genome", "reads_in": 311360, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_006364635.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Legionella pneumophila strain FDAARGOS_779 chromosome, complete genome", "reads_in": 227280, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_006364355.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Vibrio furnissii strain FDAARGOS_777 chromosome 1, complete sequence", "reads_in": 332890, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_008693565.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Klebsiella michiganensis strain FDAARGOS_647 chromosome, complete genome", "reads_in": 443772, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003798285.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 6, "name": "Yersinia pseudotuberculosis strain FDAARGOS_583 chromosome, complete genome", "reads_in": 310146, "reads_removed_percent": 0.0019345727496082492, "reads_retained_percent": 99.9980654272504}, {"sample": "GCA_006364035.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Listeria monocytogenes strain FDAARGOS_607 chromosome, complete genome", "reads_in": 199470, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_006364655.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Listeria monocytogenes strain FDAARGOS_778 chromosome, complete genome", "reads_in": 196732, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003955985.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Corynebacterium jeikeium strain FDAARGOS_574 chromosome, complete genome", "reads_in": 168544, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002208745.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Pseudomonas protegens strain FDAARGOS_307 chromosome, complete genome", "reads_in": 471722, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904355.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Bordetella pertussis strain FDAARGOS_1365 chromosome, complete genome", "reads_in": 273672, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003955815.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Francisella tularensis strain FDAARGOS_595 chromosome, complete genome", "reads_in": 126280, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_006364295.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_775 chromosome, complete genome", "reads_in": 371594, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_008693885.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Klebsiella aerogenes strain FDAARGOS_641 chromosome, complete genome", "reads_in": 352422, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_008693905.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Enterobacter hormaechei strain FDAARGOS_642 chromosome, complete genome", "reads_in": 313208, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_008693925.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Dermabacter vaginalis strain FDAARGOS_640 chromosome, complete genome", "reads_in": 152620, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013267535.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 6, "name": "Finegoldia magna strain FDAARGOS_764 chromosome, complete genome", "reads_in": 139338, "reads_removed_percent": 0.004306075873056883, "reads_retained_percent": 99.99569392412694}, {"sample": "GCA_009729955.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Salmonella enterica strain FDAARGOS_688 chromosome, complete genome", "reads_in": 308930, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003798305.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 6, "name": "Yersinia pseudotuberculosis strain FDAARGOS_579 chromosome, complete genome", "reads_in": 312098, "reads_removed_percent": 0.0019224730693564202, "reads_retained_percent": 99.99807752693064}, {"sample": "GCA_002209065.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 6, "name": "Campylobacter jejuni strain FDAARGOS_266 chromosome, complete genome", "reads_in": 108030, "reads_removed_percent": 0.005554012774229381, "reads_retained_percent": 99.99444598722577}, {"sample": "GCA_009729935.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Salmonella enterica strain FDAARGOS_707 chromosome, complete genome", "reads_in": 303738, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009729915.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Salmonella enterica strain FDAARGOS_708 chromosome, complete genome", "reads_in": 342448, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002983785.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 6, "name": "Enterococcus faecium strain FDAARGOS_323 chromosome, complete genome", "reads_in": 180694, "reads_removed_percent": 0.0033205308421973057, "reads_retained_percent": 99.9966794691578}, {"sample": "GCA_009729895.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Salmonella enterica strain FDAARGOS_710 chromosome, complete genome", "reads_in": 307402, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_008693985.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Stenotrophomonas maltophilia strain FDAARGOS_649 chromosome, complete genome", "reads_in": 308392, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013267515.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 6, "name": "Streptococcus intermedius strain FDAARGOS_769 chromosome, complete genome", "reads_in": 128892, "reads_removed_percent": 0.004655060050274649, "reads_retained_percent": 99.99534493994973}, {"sample": "GCA_002983965.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 6, "name": "Salmonella enterica subsp. enterica serovar Typhimurium strain FDAARGOS_320 chromosome, complete genome", "reads_in": 335804, "reads_removed_percent": 0.0017867565603745042, "reads_retained_percent": 99.99821324343962}, {"sample": "GCA_008693865.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Paracoccus yeei strain FDAARGOS_643 chromosome, complete genome", "reads_in": 312652, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_008693945.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Citrobacter koseri strain FDAARGOS_646 chromosome, complete genome", "reads_in": 305402, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016027435.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 6, "name": "Burkholderia oklahomensis strain FDAARGOS_900 chromosome 1, complete sequence", "reads_in": 487968, "reads_removed_percent": 0.0012295888254967538, "reads_retained_percent": 99.9987704111745}, {"sample": "GCA_016889725.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 6, "name": "Escherichia coli strain FDAARGOS_1252 chromosome, complete genome", "reads_in": 362544, "reads_removed_percent": 0.0016549715344896067, "reads_retained_percent": 99.99834502846551}, {"sample": "GCA_003812225.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 6, "name": "Acinetobacter sp. FDAARGOS_494 chromosome, complete genome", "reads_in": 263550, "reads_removed_percent": 0.0022766078542970974, "reads_retained_percent": 99.9977233921457}, {"sample": "GCA_016889865.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 4, "name": "Escherichia coli strain FDAARGOS_1253 chromosome, complete genome", "reads_in": 313026, "reads_removed_percent": 0.0012778491243538876, "reads_retained_percent": 99.99872215087565}, {"sample": "GCA_020097615.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Corynebacterium falsenii strain FDAARGOS_1493 chromosome, complete genome", "reads_in": 170792, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020097275.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Bacteroides fragilis strain FDAARGOS_1512 chromosome, complete genome", "reads_in": 349070, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889685.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 4, "name": "Escherichia coli strain FDAARGOS_1267 chromosome, complete genome", "reads_in": 362318, "reads_removed_percent": 0.0011040025612859422, "reads_retained_percent": 99.99889599743871}, {"sample": "GCA_016904255.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1371 chromosome, complete genome", "reads_in": 342696, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020097375.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Porphyromonas endodontalis strain FDAARGOS_1506 chromosome, complete genome", "reads_in": 140196, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127415.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 4, "name": "Delftia acidovorans strain FDAARGOS_997 chromosome, complete genome", "reads_in": 453112, "reads_removed_percent": 0.0008827839474566994, "reads_retained_percent": 99.99911721605254}, {"sample": "GCA_020099255.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Enterobacter bugandensis strain FDAARGOS_1496 chromosome, complete genome", "reads_in": 334858, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904135.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Staphylococcus epidermidis strain FDAARGOS_1361 chromosome, complete genome", "reads_in": 168278, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020097415.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Klebsiella quasipneumoniae strain FDAARGOS_1504 chromosome, complete genome", "reads_in": 344172, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020099335.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Citrobacter amalonaticus strain FDAARGOS_1489 chromosome, complete genome", "reads_in": 338134, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903975.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Klebsiella oxytoca strain FDAARGOS_1333 chromosome, complete genome", "reads_in": 404192, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020450285.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "UNVERIFIED_ORG: Hafnia paralvei strain FDAARGOS_1502 chromosome, complete genome", "reads_in": 322462, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020149725.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Campylobacter coli strain FDAARGOS_1464 chromosome, complete genome", "reads_in": 107916, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020099375.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Acinetobacter variabilis strain FDAARGOS_1487 chromosome, complete genome", "reads_in": 232316, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016026375.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 4, "name": "Escherichia coli strain FDAARGOS_941 chromosome, complete genome", "reads_in": 337554, "reads_removed_percent": 0.0011849955858914427, "reads_retained_percent": 99.99881500441411}, {"sample": "GCA_016904015.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Ruminococcus] gnavus strain FDAARGOS_1342 chromosome, complete genome", "reads_in": 213130, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889825.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 4, "name": "Acinetobacter pittii strain FDAARGOS_1214 chromosome, complete genome", "reads_in": 273260, "reads_removed_percent": 0.0014638073629510355, "reads_retained_percent": 99.99853619263705}, {"sample": "GCA_020099355.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Burkholderia vietnamiensis strain FDAARGOS_1488 chromosome 1, complete sequence", "reads_in": 462034, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_000783955.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Morganella morganii strain FDAARGOS_63 chromosome, complete genome", "reads_in": 266668, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020097495.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Corynebacterium freneyi strain FDAARGOS_1495 chromosome, complete genome", "reads_in": 192122, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001188185.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Vibrio parahaemolyticus strain FDAARGOS_51 chromosome 1, complete sequence", "reads_in": 343280, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003955795.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Francisella tularensis strain FDAARGOS_598 chromosome, complete genome", "reads_in": 126240, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048185.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Lelliottia amnigena strain FDAARGOS 1446 chromosome, complete genome", "reads_in": 327628, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019047085.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 4, "name": "Aeromonas sp. FDAARGOS 1404 chromosome, complete genome", "reads_in": 315912, "reads_removed_percent": 0.001266175390615108, "reads_retained_percent": 99.99873382460939}, {"sample": "GCA_002587105.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 4, "name": "Campylobacter jejuni strain FDAARGOS_422 chromosome, complete genome", "reads_in": 112690, "reads_removed_percent": 0.003549560741858195, "reads_retained_percent": 99.99645043925814}, {"sample": "GCA_019930745.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Mycolicibacterium smegmatis strain FDAARGOS_1484 chromosome, complete genome", "reads_in": 462514, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020097475.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Escherichia fergusonii strain FDAARGOS_1499 chromosome, complete genome", "reads_in": 309728, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904295.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1306 chromosome, complete genome", "reads_in": 370864, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020099235.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Enterobacter bugandensis strain FDAARGOS_1497 chromosome, complete genome", "reads_in": 334858, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904155.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Coprococcus comes strain FDAARGOS_1339 chromosome, complete genome", "reads_in": 224872, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020099215.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Enterobacter ludwigii strain FDAARGOS_1498 chromosome, complete genome", "reads_in": 315380, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013267315.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 4, "name": "Bacillus thuringiensis strain FDAARGOS_795 chromosome, complete genome", "reads_in": 381992, "reads_removed_percent": 0.0010471423485308594, "reads_retained_percent": 99.99895285765147}, {"sample": "GCA_020097655.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Corynebacterium coyleae strain FDAARGOS_1492 chromosome, complete genome", "reads_in": 172188, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904215.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Bordetella pertussis strain FDAARGOS_1367 chromosome, complete genome", "reads_in": 273706, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009730295.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 4, "name": "Gemella sanguinis strain FDAARGOS_742 chromosome, complete genome", "reads_in": 119774, "reads_removed_percent": 0.003339622956568203, "reads_retained_percent": 99.99666037704343}, {"sample": "GCA_016904035.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1322 chromosome, complete genome", "reads_in": 363140, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020099275.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Corynebacterium falsenii strain FDAARGOS_1494 chromosome, complete genome", "reads_in": 178492, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020736245.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 4, "name": "Burkholderia dolosa strain FDAARGOS_1544 chromosome 1, complete sequence", "reads_in": 427432, "reads_removed_percent": 0.0009358213704168149, "reads_retained_percent": 99.99906417862958}, {"sample": "GCA_013267575.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 4, "name": "Staphylococcus hominis strain FDAARGOS_762 chromosome, complete genome", "reads_in": 150368, "reads_removed_percent": 0.002660140455416046, "reads_retained_percent": 99.99733985954458}, {"sample": "GCA_019930785.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Burkholderia dolosa strain FDAARGOS_1463 chromosome 1, complete sequence", "reads_in": 413044, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904635.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1320 chromosome, complete genome", "reads_in": 351422, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019930765.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Legionella pneumophila strain FDAARGOS_1483 chromosome, complete genome", "reads_in": 227276, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904655.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1284 chromosome, complete genome", "reads_in": 337852, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016026235.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 4, "name": "Escherichia coli strain FDAARGOS_944 chromosome, complete genome", "reads_in": 355392, "reads_removed_percent": 0.0011255177381595534, "reads_retained_percent": 99.99887448226184}, {"sample": "GCA_019930805.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Legionella pneumophila strain FDAARGOS_1482 chromosome, complete genome", "reads_in": 229162, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904475.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1292 chromosome, complete genome", "reads_in": 341372, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013267615.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 4, "name": "Parabacteroides distasonis strain FDAARGOS_759 chromosome, complete genome", "reads_in": 328402, "reads_removed_percent": 0.001218019378688315, "reads_retained_percent": 99.99878198062132}, {"sample": "GCA_016026335.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 4, "name": "Escherichia coli strain FDAARGOS_943 chromosome, complete genome", "reads_in": 362216, "reads_removed_percent": 0.0011043134483291737, "reads_retained_percent": 99.99889568655168}, {"sample": "GCA_000784015.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Proteus mirabilis strain FDAARGOS_60 chromosome, complete genome", "reads_in": 273704, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020097315.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Aeromonas enteropelogenes strain FDAARGOS_1509 chromosome, complete genome", "reads_in": 302706, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904315.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1316 chromosome, complete genome", "reads_in": 365988, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_000783595.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 4, "name": "Proteus mirabilis strain FDAARGOS_80 chromosome, complete genome", "reads_in": 269616, "reads_removed_percent": 0.001483591478250549, "reads_retained_percent": 99.99851640852175}, {"sample": "GCA_019930965.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Enterobacter ludwigii strain FDAARGOS_1475 chromosome, complete genome", "reads_in": 315382, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904595.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1310 chromosome, complete genome", "reads_in": 372866, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019930825.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Aeromonas enteropelogenes strain FDAARGOS_1459 chromosome, complete genome", "reads_in": 294464, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904495.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 4, "name": "Escherichia coli strain FDAARGOS_1281 chromosome, complete genome", "reads_in": 372684, "reads_removed_percent": 0.0010732953386783442, "reads_retained_percent": 99.99892670466132}, {"sample": "GCA_016904075.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1303 chromosome, complete genome", "reads_in": 352828, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903155.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 4, "name": "Klebsiella pneumoniae strain FDAARGOS_1326 chromosome, complete genome", "reads_in": 368106, "reads_removed_percent": 0.001086643521159666, "reads_retained_percent": 99.99891335647884}, {"sample": "GCA_020099315.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Citrobacter europaeus strain FDAARGOS_1490 chromosome, complete genome", "reads_in": 339568, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001558255.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Grimontia hollisae strain FDAARGOS_111 chromosome 1, complete sequence", "reads_in": 266562, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904095.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1319 chromosome, complete genome", "reads_in": 368102, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_000783935.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Raoultella planticola strain FDAARGOS_64 chromosome, complete genome", "reads_in": 388262, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019930925.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Bartonella henselae strain FDAARGOS_1462 chromosome, complete genome", "reads_in": 129782, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001188125.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 4, "name": "Salmonella enterica subsp. enterica serovar Heidelberg strain FDAARGOS_54 chromosome, complete genome", "reads_in": 316752, "reads_removed_percent": 0.0012628175986260544, "reads_retained_percent": 99.99873718240137}, {"sample": "GCA_019930885.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Legionella anisa strain FDAARGOS_1480 chromosome, complete genome", "reads_in": 296538, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904515.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Klebsiella oxytoca strain FDAARGOS_1331 chromosome, complete genome", "reads_in": 400534, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904535.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1297 chromosome, complete genome", "reads_in": 345586, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001559175.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 4, "name": "Streptococcus pyogenes strain FDAARGOS_149 chromosome, complete genome", "reads_in": 122642, "reads_removed_percent": 0.0032615254154368, "reads_retained_percent": 99.99673847458456}, {"sample": "GCA_016904555.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1285 chromosome, complete genome", "reads_in": 345962, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019930845.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Eikenella corrodens strain FDAARGOS_1474 chromosome, complete genome", "reads_in": 146172, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904575.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1276 chromosome, complete genome", "reads_in": 338828, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020097255.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Thomasclavelia ramosa strain FDAARGOS_1519 chromosome, complete genome", "reads_in": 254472, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003955775.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Francisella tularensis strain FDAARGOS_599 chromosome, complete genome", "reads_in": 126190, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002953335.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Vibrio diabolicus strain FDAARGOS_96 chromosome 1, complete sequence", "reads_in": 344910, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003855135.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Shigella flexneri strain FDAARGOS_535 chromosome, complete genome", "reads_in": 324836, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002943995.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Pseudomonas monteilii strain FDAARGOS_171 chromosome, complete genome", "reads_in": 393152, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002754815.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Roseomonas sp. FDAARGOS_362 chromosome, complete genome", "reads_in": 344946, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002591295.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Pasteurella multocida strain FDAARGOS_385 chromosome, complete genome", "reads_in": 156448, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002591275.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Chryseobacterium indologenes strain FDAARGOS_379 chromosome, complete genome", "reads_in": 325922, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002587225.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 8, "name": "Campylobacter jejuni strain FDAARGOS_421 chromosome, complete genome", "reads_in": 118520, "reads_removed_percent": 0.006749915626054674, "reads_retained_percent": 99.99325008437394}, {"sample": "GCA_002951455.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Elizabethkingia anophelis strain FDAARGOS_134 chromosome, complete genome", "reads_in": 266560, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002209185.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Corynebacterium jeikeium strain FDAARGOS_328 chromosome, complete genome", "reads_in": 176726, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002591255.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Brucella canis strain FDAARGOS_420 chromosome 1, complete sequence", "reads_in": 220852, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016888925.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 8, "name": "Escherichia coli strain FDAARGOS_1260 chromosome, complete genome", "reads_in": 345352, "reads_removed_percent": 0.0023164771016238503, "reads_retained_percent": 99.99768352289837}, {"sample": "GCA_016888905.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 8, "name": "Parabacteroides distasonis strain FDAARGOS_1234 chromosome, complete genome", "reads_in": 320802, "reads_removed_percent": 0.0024937500389648442, "reads_retained_percent": 99.99750624996103}, {"sample": "GCA_002591195.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Yersinia frederiksenii strain FDAARGOS_418 chromosome, complete genome", "reads_in": 328936, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002591175.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Staphylococcus capitis strain FDAARGOS_378 chromosome, complete genome", "reads_in": 165676, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_021183585.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 8, "name": "Nocardia asteroides strain FDAARGOS_1622 chromosome, complete genome", "reads_in": 465562, "reads_removed_percent": 0.001718353302030664, "reads_retained_percent": 99.99828164669798}, {"sample": "GCA_002209205.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Salmonella enterica subsp. enterica serovar Hadar strain FDAARGOS_313 chromosome, complete genome", "reads_in": 329616, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002591235.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Pseudomonas sp. FDAARGOS_380 chromosome, complete genome", "reads_in": 433290, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_000783575.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 8, "name": "Proteus mirabilis strain FDAARGOS_81 chromosome, complete genome", "reads_in": 271822, "reads_removed_percent": 0.0029431024714703007, "reads_retained_percent": 99.99705689752852}, {"sample": "GCA_002944995.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Staphylococcus epidermidis strain FDAARGOS_153 chromosome, complete genome", "reads_in": 170132, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002951115.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Stenotrophomonas maltophilia strain FDAARGOS_92 chromosome, complete genome", "reads_in": 321348, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_012273555.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Chromobacterium violaceum strain FDAARGOS_635 chromosome, complete genome", "reads_in": 320050, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009730315.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 6, "name": "Gemella morbillorum strain FDAARGOS_741 chromosome, complete genome", "reads_in": 118630, "reads_removed_percent": 0.0050577425609036495, "reads_retained_percent": 99.99494225743909}, {"sample": "GCA_002073715.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Neisseria mucosa strain FDAARGOS_260 chromosome, complete genome", "reads_in": 185596, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003813025.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 6, "name": "Pseudomonas aeruginosa strain FDAARGOS_570 chromosome, complete genome", "reads_in": 477010, "reads_removed_percent": 0.0012578352655080605, "reads_retained_percent": 99.9987421647345}, {"sample": "GCA_009734165.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Bacteroides ovatus strain FDAARGOS_733 chromosome, complete genome", "reads_in": 438092, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002073755.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Citrobacter braakii strain FDAARGOS_253 chromosome, complete genome", "reads_in": 349620, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020736145.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 8, "name": "Campylobacter jejuni strain FDAARGOS_1546 chromosome, complete genome", "reads_in": 108536, "reads_removed_percent": 0.0073708262696248255, "reads_retained_percent": 99.99262917373038}, {"sample": "GCA_016725925.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 6, "name": "Sphingobacterium multivorum strain FDAARGOS_1141 chromosome, complete genome", "reads_in": 419822, "reads_removed_percent": 0.0014291771274492523, "reads_retained_percent": 99.99857082287255}, {"sample": "GCA_009730555.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Mammaliicoccus fleurettii strain FDAARGOS_682 chromosome, complete genome", "reads_in": 168504, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009730535.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Streptococcus gordonii strain FDAARGOS_683 chromosome, complete genome", "reads_in": 146440, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009730515.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Streptococcus mitis strain FDAARGOS_684 chromosome, complete genome", "reads_in": 124590, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009730475.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Corynebacterium xerosis strain FDAARGOS_674 chromosome, complete genome", "reads_in": 188102, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003854935.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Cupriavidus pauculus strain FDAARGOS_614 chromosome 1, complete sequence", "reads_in": 411854, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889985.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Corynebacterium sp. FDAARGOS 1242 strain FDAARGOS_1242 chromosome, complete genome", "reads_in": 185000, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009730575.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Shewanella putrefaciens strain FDAARGOS_681 chromosome, complete genome", "reads_in": 310642, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002951395.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Elizabethkingia anophelis strain FDAARGOS_132 chromosome, complete genome", "reads_in": 266560, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016128155.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 8, "name": "Lactococcus garvieae strain FDAARGOS_1062 chromosome, complete genome", "reads_in": 135644, "reads_removed_percent": 0.0058977912771667005, "reads_retained_percent": 99.99410220872284}, {"sample": "GCA_002209165.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Mammaliicoccus sciuri strain FDAARGOS_285 chromosome, complete genome", "reads_in": 190256, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002386365.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Rothia mucilaginosa strain FDAARGOS_369 chromosome, complete genome", "reads_in": 160584, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016026175.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 8, "name": "Listeria welshimeri strain FDAARGOS_947 chromosome, complete genome", "reads_in": 195648, "reads_removed_percent": 0.0040889761203794575, "reads_retained_percent": 99.99591102387961}, {"sample": "GCA_002386345.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Streptococcus oralis strain FDAARGOS_367 chromosome, complete genome", "reads_in": 136880, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016026535.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 8, "name": "Delftia acidovorans strain FDAARGOS_939 chromosome, complete genome", "reads_in": 423412, "reads_removed_percent": 0.0018894126760696439, "reads_retained_percent": 99.99811058732394}, {"sample": "GCA_002386325.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Salmonella enterica subsp. enterica serovar Saintpaul strain FDAARGOS_373 chromosome, complete genome", "reads_in": 322540, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002393265.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Enterococcus sp. FDAARGOS_375 chromosome, complete genome", "reads_in": 236622, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009806675.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 8, "name": "Bacillus anthracis strain FDAARGOS_705 chromosome, complete genome", "reads_in": 348874, "reads_removed_percent": 0.002293091488617667, "reads_retained_percent": 99.99770690851138}, {"sample": "GCA_016028675.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 8, "name": "Proteus penneri strain FDAARGOS_874 chromosome, complete genome", "reads_in": 256720, "reads_removed_percent": 0.0031162355874104083, "reads_retained_percent": 99.99688376441259}, {"sample": "GCA_016026875.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 8, "name": "Aeromonas hydrophila strain FDAARGOS_916 chromosome, complete genome", "reads_in": 315580, "reads_removed_percent": 0.002535014893212498, "reads_retained_percent": 99.99746498510679}, {"sample": "GCA_002386285.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Streptococcus gordonii strain FDAARGOS_371 chromosome, complete genome", "reads_in": 141420, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002386265.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Weissella paramesenteroides strain FDAARGOS_414 chromosome, complete genome", "reads_in": 129860, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048085.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 8, "name": "Aeromonas sp. FDAARGOS 1416 chromosome, complete genome", "reads_in": 301256, "reads_removed_percent": 0.0026555487691531455, "reads_retained_percent": 99.99734445123084}, {"sample": "GCA_019047745.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1381 chromosome, complete genome", "reads_in": 349546, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002209265.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Staphylococcus saprophyticus strain FDAARGOS_355 chromosome, complete genome", "reads_in": 175946, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002386305.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Morganella morganii strain FDAARGOS_365 chromosome, complete genome", "reads_in": 271714, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009806635.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 8, "name": "Bacillus anthracis strain FDAARGOS_700 chromosome, complete genome", "reads_in": 348884, "reads_removed_percent": 0.0022930257621444376, "reads_retained_percent": 99.99770697423786}, {"sample": "GCA_002386385.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Citrobacter werkmanii strain FDAARGOS_364 chromosome, complete genome", "reads_in": 325674, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002393245.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Citrobacter koseri strain FDAARGOS_393 chromosome, complete genome", "reads_in": 325632, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002591095.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Yersinia frederiksenii strain FDAARGOS_417 chromosome, complete genome", "reads_in": 333378, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009730035.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 8, "name": "Burkholderia multivorans strain FDAARGOS_719 chromosome 1, complete sequence", "reads_in": 437710, "reads_removed_percent": 0.0018276941353864429, "reads_retained_percent": 99.99817230586461}, {"sample": "GCA_002983625.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Yersinia intermedia strain FDAARGOS_358 chromosome, complete genome", "reads_in": 330710, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002954055.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Staphylococcus epidermidis strain FDAARGOS_161 chromosome, complete genome", "reads_in": 167716, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002953535.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Vibrio alginolyticus strain FDAARGOS_110 chromosome 1, complete sequence", "reads_in": 351246, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002953515.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Vibrio alginolyticus strain FDAARGOS_108 chromosome 1, complete sequence", "reads_in": 363068, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002953395.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Vibrio diabolicus strain FDAARGOS_105 chromosome 1, complete sequence", "reads_in": 361742, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002953375.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Vibrio fluvialis strain FDAARGOS_100 chromosome 1, complete sequence", "reads_in": 324504, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002951475.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Pseudomonas fulva strain FDAARGOS_167 chromosome, complete genome", "reads_in": 324340, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002953355.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Vibrio diabolicus strain FDAARGOS_99 chromosome 1, complete sequence", "reads_in": 342120, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002953295.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Klebsiella quasipneumoniae strain FDAARGOS_93 chromosome, complete genome", "reads_in": 367536, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_006364675.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 8, "name": "Staphylococcus aureus strain FDAARGOS_773 chromosome, complete genome", "reads_in": 185804, "reads_removed_percent": 0.004305612365718714, "reads_retained_percent": 99.99569438763429}, {"sample": "GCA_002951635.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_156 chromosome, complete genome", "reads_in": 370044, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002591115.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Klebsiella aerogenes strain FDAARGOS_363 chromosome, complete genome", "reads_in": 349698, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020097355.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 8, "name": "Proteus vulgaris strain FDAARGOS_1507 chromosome, complete genome", "reads_in": 264242, "reads_removed_percent": 0.0030275277964895817, "reads_retained_percent": 99.99697247220351}, {"sample": "GCA_012273615.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Chryseobacterium gallinarum strain FDAARGOS_636 chromosome, complete genome", "reads_in": 289264, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904275.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 6, "name": "Klebsiella pneumoniae strain FDAARGOS_1328 chromosome, complete genome", "reads_in": 383876, "reads_removed_percent": 0.0015630047202742554, "reads_retained_percent": 99.99843699527973}, {"sample": "GCA_013267235.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Bacillus cereus strain FDAARGOS_802 chromosome, complete genome", "reads_in": 404570, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013267355.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Achromobacter pestifer strain FDAARGOS_790 chromosome, complete genome", "reads_in": 478488, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002208765.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Psychrobacter cryohalolentis strain FDAARGOS_308 chromosome, complete genome", "reads_in": 207242, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013267495.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Streptococcus salivarius strain FDAARGOS_771 chromosome, complete genome", "reads_in": 133300, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013267655.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Mycobacterium tuberculosis strain FDAARGOS_756 chromosome, complete genome", "reads_in": 294306, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013267635.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Mycobacterium tuberculosis strain FDAARGOS_757 chromosome, complete genome", "reads_in": 294528, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013267595.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Prevotella melaninogenica strain FDAARGOS_760 chromosome 1, complete sequence", "reads_in": 223532, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013267555.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Bacteroides fragilis strain FDAARGOS_763 chromosome, complete genome", "reads_in": 373716, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002073835.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Streptococcus salivarius strain FDAARGOS_259 chromosome, complete genome", "reads_in": 150622, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002073575.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Neisseria meningitidis strain FDAARGOS_209 chromosome, complete genome", "reads_in": 145416, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013267475.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Bacillus cereus strain FDAARGOS_780 chromosome, complete genome", "reads_in": 376156, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013267455.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Bacillus cereus strain FDAARGOS_781 chromosome, complete genome", "reads_in": 376156, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013267415.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Abiotrophia defectiva strain FDAARGOS_785 chromosome, complete genome", "reads_in": 136456, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013267395.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Achromobacter denitrificans strain FDAARGOS_786 chromosome, complete genome", "reads_in": 449124, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002073555.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Neisseria meningitidis strain FDAARGOS_211 chromosome, complete genome", "reads_in": 153720, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904335.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 6, "name": "Escherichia coli strain FDAARGOS_1370 chromosome, complete genome", "reads_in": 356214, "reads_removed_percent": 0.0016843807374218867, "reads_retained_percent": 99.99831561926258}, {"sample": "GCA_013267715.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Staphylococcus warneri strain FDAARGOS_754 chromosome, complete genome", "reads_in": 164000, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013267735.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Rothia dentocariosa strain FDAARGOS_752 chromosome, complete genome", "reads_in": 168894, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002208725.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Prevotella melaninogenica strain FDAARGOS_306 chromosome 1, complete sequence", "reads_in": 211214, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002208705.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Mycobacterium avium subsp. paratuberculosis strain FDAARGOS_305 chromosome, complete genome", "reads_in": 322166, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003855115.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Shigella sonnei strain FDAARGOS_524 chromosome, complete genome", "reads_in": 330206, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_006364695.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_772 chromosome, complete genome", "reads_in": 358738, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_008693525.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Burkholderia multivorans strain FDAARGOS_622 chromosome 1, complete sequence", "reads_in": 431910, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_008693405.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_630 chromosome, complete genome", "reads_in": 376186, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_008693505.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Streptococcus agalactiae strain FDAARGOS_669 chromosome, complete genome", "reads_in": 137712, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_008693485.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Burkholderia multivorans strain FDAARGOS_623 chromosome 1, complete sequence", "reads_in": 431910, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_006364755.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Salmonella enterica strain FDAARGOS_765 chromosome, complete genome", "reads_in": 298806, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009729975.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 6, "name": "Salmonella enterica strain FDAARGOS_687 chromosome, complete genome", "reads_in": 337762, "reads_removed_percent": 0.0017763987659949903, "reads_retained_percent": 99.998223601234}, {"sample": "GCA_008693445.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_629 chromosome, complete genome", "reads_in": 381246, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_008693425.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_631 chromosome 1, complete sequence", "reads_in": 364570, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_006364735.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Pseudomonas aeruginosa strain FDAARGOS_767 chromosome, complete genome", "reads_in": 417578, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_006365335.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Salmonella enterica strain FDAARGOS_768 chromosome, complete genome", "reads_in": 330318, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_006364815.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Enterococcus faecalis strain FDAARGOS_611 chromosome, complete genome", "reads_in": 189696, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_006364795.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Pseudomonas aeruginosa strain FDAARGOS_610 chromosome, complete genome", "reads_in": 452814, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016026615.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 6, "name": "Aeromonas allosaccharophila strain FDAARGOS_933 chromosome, complete genome", "reads_in": 308694, "reads_removed_percent": 0.00194367237458454, "reads_retained_percent": 99.99805632762542}, {"sample": "GCA_013364135.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Pantoea ananatis strain FDAARGOS_680 chromosome, complete genome", "reads_in": 351950, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016724665.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 6, "name": "Staphylococcus carnosus strain FDAARGOS_1147 chromosome, complete genome", "reads_in": 179336, "reads_removed_percent": 0.0033456751572467328, "reads_retained_percent": 99.99665432484275}, {"sample": "GCA_013364115.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Shigella flexneri strain FDAARGOS_691 chromosome, complete genome", "reads_in": 302920, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002073795.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Streptococcus sp. 'group B' strain FDAARGOS_229 chromosome, complete genome", "reads_in": 145218, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002073815.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Streptococcus sp. FDAARGOS_192 chromosome, complete genome", "reads_in": 162368, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009730275.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Lactobacillus crispatus strain FDAARGOS_743 chromosome, complete genome", "reads_in": 159130, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009730255.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Lactobacillus jensenii strain FDAARGOS_749 chromosome, complete genome", "reads_in": 107268, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009730435.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Gordonia bronchialis strain FDAARGOS_676 chromosome, complete genome", "reads_in": 353264, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009730215.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Mycobacterium tuberculosis strain FDAARGOS_751 chromosome, complete genome", "reads_in": 296132, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009730335.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Schaalia odontolytica strain FDAARGOS_732 chromosome, complete genome", "reads_in": 162176, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009730175.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Burkholderia multivorans strain FDAARGOS_726 chromosome 1, complete sequence", "reads_in": 434580, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009730115.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Staphylococcus hominis strain FDAARGOS_747 chromosome, complete genome", "reads_in": 149472, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009730095.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Acinetobacter lwoffii strain FDAARGOS_552 chromosome, complete genome", "reads_in": 232364, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009730075.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Yersinia intermedia strain FDAARGOS_729 chromosome, complete genome", "reads_in": 328594, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_006364235.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 6, "name": "Streptococcus pyogenes strain FDAARGOS_774 chromosome, complete genome", "reads_in": 127658, "reads_removed_percent": 0.004700057967381598, "reads_retained_percent": 99.99529994203262}, {"sample": "GCA_009730415.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Micrococcus luteus strain FDAARGOS_677 chromosome, complete genome", "reads_in": 165218, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019047185.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 6, "name": "Acinetobacter pittii strain FDAARGOS 1397 chromosome, complete genome", "reads_in": 278826, "reads_removed_percent": 0.0021518796668890277, "reads_retained_percent": 99.99784812033312}, {"sample": "GCA_009730155.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Enterococcus gallinarum strain FDAARGOS_728 chromosome, complete genome", "reads_in": 218328, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009730355.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Citrobacter portucalensis strain FDAARGOS_738 chromosome, complete genome", "reads_in": 344180, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009730375.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Capnocytophaga sp. FDAARGOS_737 chromosome, complete genome", "reads_in": 176848, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009730235.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Mycobacterium tuberculosis strain FDAARGOS_750 chromosome, complete genome", "reads_in": 295644, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013347325.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Cupriavidus gilardii strain FDAARGOS_639 chromosome 1, complete sequence", "reads_in": 384002, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013343155.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Pseudomonas sp. FDAARGOS_761 chromosome, complete genome", "reads_in": 425656, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016725285.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 6, "name": "Staphylococcus pasteuri strain FDAARGOS_1151 chromosome, complete genome", "reads_in": 175648, "reads_removed_percent": 0.0034159227546001095, "reads_retained_percent": 99.9965840772454}, {"sample": "GCA_013343135.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Achromobacter xylosoxidans strain FDAARGOS_789 chromosome, complete genome", "reads_in": 453762, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013343115.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Streptococcus sanguinis strain FDAARGOS_770 chromosome, complete genome", "reads_in": 157256, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009730455.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 6, "name": "Cupriavidus metallidurans strain FDAARGOS_675 chromosome, complete genome", "reads_in": 462760, "reads_removed_percent": 0.0012965684155933962, "reads_retained_percent": 99.9987034315844}, {"sample": "GCA_013343095.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Achromobacter denitrificans strain FDAARGOS_787 chromosome, complete genome", "reads_in": 454436, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013343075.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Bacillus thuringiensis strain FDAARGOS_791 chromosome, complete genome", "reads_in": 355852, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002073455.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Legionella longbeachae strain FDAARGOS_201 chromosome, complete genome", "reads_in": 277516, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013331815.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Bacillus tropicus strain FDAARGOS_782 chromosome, complete genome", "reads_in": 353154, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013267835.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Neisseria mucosa strain FDAARGOS_758 chromosome, complete genome", "reads_in": 148318, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002073495.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Leptospira interrogans serovar Copenhageni strain FDAARGOS_203 chromosome, complete genome", "reads_in": 308718, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013267795.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Bacillus thuringiensis strain FDAARGOS_794 chromosome, complete genome", "reads_in": 356574, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013267375.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Achromobacter denitrificans strain FDAARGOS_788 chromosome, complete genome", "reads_in": 449126, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009730055.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Yersinia intermedia strain FDAARGOS_730 chromosome, complete genome", "reads_in": 328594, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002073635.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Paracoccus yeei strain FDAARGOS_252 chromosome, complete genome", "reads_in": 321974, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903935.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 4, "name": "Escherichia coli strain FDAARGOS_1288 chromosome, complete genome", "reads_in": 378792, "reads_removed_percent": 0.001055988510845002, "reads_retained_percent": 99.99894401148916}, {"sample": "GCA_021183605.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Nocardia asteroides strain FDAARGOS_1621 chromosome, complete genome", "reads_in": 477942, "reads_removed_percent": 0.00041846081742136074, "reads_retained_percent": 99.99958153918259}, {"sample": "GCA_019048245.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Enterobacter hormaechei strain FDAARGOS 1433 chromosome, complete genome", "reads_in": 327728, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020736105.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Corynebacterium diphtheriae strain FDAARGOS_1553 chromosome, complete genome", "reads_in": 166080, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016890085.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Chromobacterium violaceum strain FDAARGOS_1273 chromosome, complete genome", "reads_in": 320536, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020736125.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Corynebacterium diphtheriae strain FDAARGOS_1552 chromosome, complete genome", "reads_in": 157962, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735565.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Hoylesella buccalis strain FDAARGOS_1588 chromosome, complete genome", "reads_in": 220560, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016890105.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1266 chromosome, complete genome", "reads_in": 347734, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903595.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Escherichia coli strain FDAARGOS_1279 chromosome, complete genome", "reads_in": 356446, "reads_removed_percent": 0.0005610948081897399, "reads_retained_percent": 99.99943890519181}, {"sample": "GCA_020735485.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Mycobacteroides abscessus subsp. abscessus strain FDAARGOS_1603 chromosome, complete genome", "reads_in": 322226, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735545.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Gordonia otitidis strain FDAARGOS_1600 chromosome, complete genome", "reads_in": 340588, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_021183685.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Mycobacterium kansasii strain FDAARGOS_1615 chromosome, complete genome", "reads_in": 430348, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735505.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Mycobacteroides abscessus strain FDAARGOS_1602 chromosome, complete genome", "reads_in": 322236, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016728085.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Escherichia coli strain FDAARGOS_1077 chromosome, complete genome", "reads_in": 339426, "reads_removed_percent": 0.0005892300530896278, "reads_retained_percent": 99.9994107699469}, {"sample": "GCA_020735965.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Bordetella parapertussis strain FDAARGOS_1540 chromosome, complete genome", "reads_in": 318370, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_021183665.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Mycolicibacterium smegmatis strain FDAARGOS_1618 chromosome, complete genome", "reads_in": 459422, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735525.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Helicobacter pylori strain FDAARGOS_1601 chromosome, complete genome", "reads_in": 109606, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735945.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Parabacteroides distasonis strain FDAARGOS_1565 chromosome, complete genome", "reads_in": 320804, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904115.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Escherichia coli strain FDAARGOS_1290 chromosome, complete genome", "reads_in": 347130, "reads_removed_percent": 0.0005761530262437703, "reads_retained_percent": 99.99942384697376}, {"sample": "GCA_016727265.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Enterococcus hirae strain FDAARGOS_1125 chromosome, complete genome", "reads_in": 204300, "reads_removed_percent": 0.0009789525208027412, "reads_retained_percent": 99.9990210474792}, {"sample": "GCA_002591135.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Escherichia coli strain FDAARGOS_448 chromosome, complete genome", "reads_in": 363122, "reads_removed_percent": 0.0005507790770044227, "reads_retained_percent": 99.999449220923}, {"sample": "GCA_002073335.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Vibrio cholerae strain FDAARGOS_223 chromosome 1, complete sequence", "reads_in": 269486, "reads_removed_percent": 0.0007421535812621064, "reads_retained_percent": 99.99925784641874}, {"sample": "GCA_016906205.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Streptomyces californicus strain FDAARGOS_1212 chromosome", "reads_in": 538842, "reads_removed_percent": 0.00037116631591449815, "reads_retained_percent": 99.99962883368408}, {"sample": "GCA_016890005.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Escherichia coli strain FDAARGOS_1250 chromosome, complete genome", "reads_in": 337234, "reads_removed_percent": 0.0005930600117425882, "reads_retained_percent": 99.99940693998826}, {"sample": "GCA_002591155.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Proteus vulgaris strain FDAARGOS_366 chromosome, complete genome", "reads_in": 264132, "reads_removed_percent": 0.0007571971589962594, "reads_retained_percent": 99.99924280284101}, {"sample": "GCA_016906265.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Streptomyces californicus strain FDAARGOS_1210 chromosome", "reads_in": 548102, "reads_removed_percent": 0.00036489558512831555, "reads_retained_percent": 99.99963510441488}, {"sample": "GCA_002073535.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Neisseria gonorrhoeae strain FDAARGOS_204 chromosome, complete genome", "reads_in": 147496, "reads_removed_percent": 0.001355968975429842, "reads_retained_percent": 99.99864403102457}, {"sample": "GCA_016889045.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Escherichia coli strain FDAARGOS_1255 chromosome, complete genome", "reads_in": 337566, "reads_removed_percent": 0.0005924767304764106, "reads_retained_percent": 99.99940752326953}, {"sample": "GCA_013267435.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Niallia circulans strain FDAARGOS_783 chromosome, complete genome", "reads_in": 345304, "reads_removed_percent": 0.0005791997775872853, "reads_retained_percent": 99.99942080022241}, {"sample": "GCA_016903815.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Escherichia coli strain FDAARGOS_1280 chromosome, complete genome", "reads_in": 366450, "reads_removed_percent": 0.0005457770500750443, "reads_retained_percent": 99.99945422294992}, {"sample": "GCA_002591215.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Staphylococcus lugdunensis strain FDAARGOS_381 chromosome, complete genome", "reads_in": 171380, "reads_removed_percent": 0.0011669973159061735, "reads_retained_percent": 99.9988330026841}, {"sample": "GCA_002944935.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Escherichia coli strain FDAARGOS_144 chromosome, complete genome", "reads_in": 334812, "reads_removed_percent": 0.0005973501547136901, "reads_retained_percent": 99.99940264984528}, {"sample": "GCA_016903855.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Escherichia coli strain FDAARGOS_1291 chromosome, complete genome", "reads_in": 348814, "reads_removed_percent": 0.0005733714816492458, "reads_retained_percent": 99.99942662851835}, {"sample": "GCA_016728005.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Serratia liquefaciens strain FDAARGOS_1081 chromosome, complete genome", "reads_in": 367766, "reads_removed_percent": 0.0005438240620394489, "reads_retained_percent": 99.99945617593796}, {"sample": "GCA_002951295.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Acinetobacter nosocomialis strain FDAARGOS_129 chromosome, complete genome", "reads_in": 266846, "reads_removed_percent": 0.000749495963964234, "reads_retained_percent": 99.99925050403604}, {"sample": "GCA_016727905.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Yersinia enterocolitica strain FDAARGOS_1090 chromosome, complete genome", "reads_in": 306216, "reads_removed_percent": 0.0006531337356637145, "reads_retained_percent": 99.99934686626433}, {"sample": "GCA_016903875.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1329 chromosome, complete genome", "reads_in": 370504, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903355.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Streptococcus anginosus strain FDAARGOS_1357 chromosome, complete genome", "reads_in": 132094, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889205.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Burkholderia dolosa strain FDAARGOS_1272 chromosome 1, complete sequence", "reads_in": 427274, "reads_removed_percent": 0.00046808371209107035, "reads_retained_percent": 99.9995319162879}, {"sample": "GCA_021183745.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Mycobacterium intracellulare strain FDAARGOS_1612 chromosome, complete genome", "reads_in": 371710, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903375.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Escherichia coli strain FDAARGOS_1286 chromosome, complete genome", "reads_in": 344596, "reads_removed_percent": 0.0005803897897828181, "reads_retained_percent": 99.99941961021021}, {"sample": "GCA_016127135.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Lactococcus lactis strain FDAARGOS_1064 chromosome, complete genome", "reads_in": 167518, "reads_removed_percent": 0.0011939015508781146, "reads_retained_percent": 99.99880609844912}, {"sample": "GCA_016028835.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Lactococcus lactis strain FDAARGOS_865 chromosome, complete genome", "reads_in": 171814, "reads_removed_percent": 0.0011640494953845436, "reads_retained_percent": 99.99883595050461}, {"sample": "GCA_019047725.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Escherichia coli strain FDAARGOS 1374 chromosome, complete genome", "reads_in": 349518, "reads_removed_percent": 0.0005722165954257005, "reads_retained_percent": 99.99942778340457}, {"sample": "GCA_016890045.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Escherichia coli strain FDAARGOS_1265 chromosome, complete genome", "reads_in": 361686, "reads_removed_percent": 0.0005529658322412259, "reads_retained_percent": 99.99944703416776}, {"sample": "GCA_020735985.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Bordetella holmesii strain FDAARGOS_1539 chromosome, complete genome", "reads_in": 245874, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903895.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1321 chromosome, complete genome", "reads_in": 362858, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016890065.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Acinetobacter pittii strain FDAARGOS_1216 chromosome, complete genome", "reads_in": 261924, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735445.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Prevotella copri DSM 18205 strain FDAARGOS_1573 chromosome, complete genome", "reads_in": 236774, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_021183765.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Mycobacterium intracellulare strain FDAARGOS_1611 chromosome, complete genome", "reads_in": 365956, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020736085.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Corynebacterium diphtheriae subsp. diphtheriae strain FDAARGOS_1554 chromosome, complete genome", "reads_in": 164242, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903295.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1323 chromosome, complete genome", "reads_in": 366060, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903495.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1312 chromosome, complete genome", "reads_in": 371120, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735465.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Collinsella stercoris DSM 13279 strain FDAARGOS_1572 chromosome, complete genome", "reads_in": 164050, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735425.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Mycobacteroides abscessus subsp. abscessus strain FDAARGOS_1604 chromosome, complete genome", "reads_in": 339370, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889105.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Escherichia coli strain FDAARGOS_1254 chromosome, complete genome", "reads_in": 326142, "reads_removed_percent": 0.0006132298201396938, "reads_retained_percent": 99.99938677017985}, {"sample": "GCA_020736005.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Mycobacterium intracellulare strain FDAARGOS_1564 chromosome, complete genome", "reads_in": 376386, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735405.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Mycobacterium avium subsp. avium strain FDAARGOS_1606 chromosome, complete genome", "reads_in": 330704, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_021183725.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Mycobacterium ostraviense strain FDAARGOS_1613 chromosome, complete genome", "reads_in": 408516, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016725185.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Streptococcus anginosus strain FDAARGOS_1155 chromosome, complete genome", "reads_in": 152270, "reads_removed_percent": 0.0013134563604124251, "reads_retained_percent": 99.9986865436396}, {"sample": "GCA_016904175.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Escherichia coli strain FDAARGOS_1372 chromosome, complete genome", "reads_in": 321568, "reads_removed_percent": 0.0006219524330779182, "reads_retained_percent": 99.99937804756692}, {"sample": "GCA_020735385.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Granulicatella elegans strain FDAARGOS_1559 chromosome, complete genome", "reads_in": 126042, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_000783435.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Achromobacter insolitus strain FDAARGOS_88 chromosome, complete genome", "reads_in": 459536, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904055.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Escherichia coli strain FDAARGOS_1293 chromosome, complete genome", "reads_in": 342718, "reads_removed_percent": 0.000583570165558856, "reads_retained_percent": 99.99941642983444}, {"sample": "GCA_016726985.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Myroides odoratus strain FDAARGOS_1131 chromosome, complete genome", "reads_in": 282832, "reads_removed_percent": 0.0007071335633874527, "reads_retained_percent": 99.99929286643662}, {"sample": "GCA_020735365.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Blautia massiliensis strain FDAARGOS_1576 chromosome, complete genome", "reads_in": 248852, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903335.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1295 chromosome, complete genome", "reads_in": 342824, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889085.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Escherichia coli strain FDAARGOS_1256 chromosome, complete genome", "reads_in": 346170, "reads_removed_percent": 0.0005777508160730278, "reads_retained_percent": 99.99942224918392}, {"sample": "GCA_016903955.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Klebsiella pneumoniae strain FDAARGOS_1309 chromosome, complete genome", "reads_in": 373694, "reads_removed_percent": 0.0005351972469453617, "reads_retained_percent": 99.99946480275305}, {"sample": "GCA_020735345.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Mycobacteroides abscessus subsp. abscessus strain FDAARGOS_1605 chromosome, complete genome", "reads_in": 335566, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_021183785.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Mycobacterium intracellulare subsp. intracellulare strain FDAARGOS_1610 chromosome, complete genome", "reads_in": 360844, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903995.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Escherichia coli strain FDAARGOS_1337 chromosome, complete genome", "reads_in": 331158, "reads_removed_percent": 0.0006039413210612457, "reads_retained_percent": 99.99939605867894}, {"sample": "GCA_002393365.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Escherichia coli strain FDAARGOS_403 chromosome, complete genome", "reads_in": 365028, "reads_removed_percent": 0.0005479031745509933, "reads_retained_percent": 99.99945209682545}, {"sample": "GCA_016903175.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Bordetella pertussis strain FDAARGOS_1369 chromosome, complete genome", "reads_in": 273602, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048145.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Escherichia coli strain FDAARGOS 1386 chromosome, complete genome", "reads_in": 346162, "reads_removed_percent": 0.0005777641682218152, "reads_retained_percent": 99.99942223583177}, {"sample": "GCA_021183645.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Nocardia gipuzkoensis strain FDAARGOS_1619 chromosome, complete genome", "reads_in": 541242, "reads_removed_percent": 0.0003695204732818222, "reads_retained_percent": 99.99963047952672}, {"sample": "GCA_020097295.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Aeromonas enteropelogenes strain FDAARGOS_1510 chromosome, complete genome", "reads_in": 292826, "reads_removed_percent": 0.0006829994604304263, "reads_retained_percent": 99.99931700053956}, {"sample": "GCA_009729995.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Shigella sonnei strain FDAARGOS_715 chromosome, complete genome", "reads_in": 340654, "reads_removed_percent": 0.000587105978500179, "reads_retained_percent": 99.9994128940215}, {"sample": "GCA_020099175.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Klebsiella quasipneumoniae strain FDAARGOS_1503 chromosome, complete genome", "reads_in": 366590, "reads_removed_percent": 0.0005455686188930413, "reads_retained_percent": 99.99945443138111}, {"sample": "GCA_001018655.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Staphylococcus aureus strain FDAARGOS_6 chromosome, complete genome", "reads_in": 183952, "reads_removed_percent": 0.0010872401496042445, "reads_retained_percent": 99.9989127598504}, {"sample": "GCA_009730195.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Burkholderia multivorans strain FDAARGOS_722 chromosome 1, complete sequence", "reads_in": 439364, "reads_removed_percent": 0.00045520343041305155, "reads_retained_percent": 99.99954479656958}, {"sample": "GCA_000783815.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Salmonella enterica strain FDAARGOS_70 chromosome, complete genome", "reads_in": 319106, "reads_removed_percent": 0.0006267509855659248, "reads_retained_percent": 99.99937324901444}, {"sample": "GCA_008693465.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Klebsiella variicola strain FDAARGOS_627 chromosome, complete genome", "reads_in": 397174, "reads_removed_percent": 0.0005035576346890783, "reads_retained_percent": 99.99949644236531}, {"sample": "GCA_020097235.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Enterobacter bugandensis strain FDAARGOS_1520 chromosome, complete genome", "reads_in": 334858, "reads_removed_percent": 0.0005972680957301304, "reads_retained_percent": 99.99940273190427}, {"sample": "GCA_009730135.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Staphylococcus hominis strain FDAARGOS_746 chromosome, complete genome", "reads_in": 158146, "reads_removed_percent": 0.001264654180314393, "reads_retained_percent": 99.99873534581968}, {"sample": "GCA_003614835.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Burkholderia thailandensis strain FDAARGOS_242 chromosome 1, complete sequence", "reads_in": 443554, "reads_removed_percent": 0.0004509033849317107, "reads_retained_percent": 99.99954909661507}, {"sample": "GCA_003546905.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Yersinia pseudotuberculosis strain FDAARGOS_342 chromosome, complete genome", "reads_in": 327036, "reads_removed_percent": 0.0006115534681197177, "reads_retained_percent": 99.99938844653188}, {"sample": "GCA_003798385.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Yersinia pseudotuberculosis strain FDAARGOS_584 chromosome, complete genome", "reads_in": 312628, "reads_removed_percent": 0.00063973796333022, "reads_retained_percent": 99.99936026203667}, {"sample": "GCA_002984195.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Providencia rettgeri strain FDAARGOS_330 chromosome, complete genome", "reads_in": 301240, "reads_removed_percent": 0.0006639224538573894, "reads_retained_percent": 99.99933607754615}, {"sample": "GCA_003812105.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Klebsiella pneumoniae strain FDAARGOS_531 chromosome, complete genome", "reads_in": 384250, "reads_removed_percent": 0.000520494469746259, "reads_retained_percent": 99.99947950553026}, {"sample": "GCA_002209105.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Escherichia coli O104:H4 strain FDAARGOS_348 chromosome, complete genome", "reads_in": 362832, "reads_removed_percent": 0.0005512192970851524, "reads_retained_percent": 99.99944878070292}, {"sample": "GCA_003812065.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Acinetobacter baumannii strain FDAARGOS_540 chromosome, complete genome", "reads_in": 260684, "reads_removed_percent": 0.0007672124104279511, "reads_retained_percent": 99.99923278758958}, {"sample": "GCA_003812505.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Staphylococcus hominis strain FDAARGOS_575 chromosome, complete genome", "reads_in": 150494, "reads_removed_percent": 0.0013289566361449625, "reads_retained_percent": 99.99867104336386}, {"sample": "GCA_008694005.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Streptococcus pyogenes strain FDAARGOS_668 chromosome, complete genome", "reads_in": 122756, "reads_removed_percent": 0.0016292482648505979, "reads_retained_percent": 99.99837075173515}, {"sample": "GCA_002953975.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Vibrio alginolyticus strain FDAARGOS_114 chromosome 1, complete sequence", "reads_in": 345218, "reads_removed_percent": 0.0005793440666477414, "reads_retained_percent": 99.99942065593335}, {"sample": "GCA_003813045.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Burkholderia dolosa strain FDAARGOS_562 chromosome 1, complete sequence", "reads_in": 427274, "reads_removed_percent": 0.00046808371209107035, "reads_retained_percent": 99.9995319162879}, {"sample": "GCA_003812385.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Acinetobacter baumannii strain FDAARGOS_533 chromosome, complete genome", "reads_in": 259026, "reads_removed_percent": 0.000772123261757507, "reads_retained_percent": 99.99922787673825}, {"sample": "GCA_002073655.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Neisseria gonorrhoeae strain FDAARGOS_207 chromosome, complete genome", "reads_in": 145954, "reads_removed_percent": 0.0013702947504008114, "reads_retained_percent": 99.9986297052496}, {"sample": "GCA_002073675.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Neisseria meningitidis strain FDAARGOS_212 chromosome, complete genome", "reads_in": 149660, "reads_removed_percent": 0.0013363624214887077, "reads_retained_percent": 99.99866363757852}, {"sample": "GCA_009806655.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Bacillus anthracis strain FDAARGOS_706 chromosome, complete genome", "reads_in": 348660, "reads_removed_percent": 0.0005736247346985602, "reads_retained_percent": 99.9994263752653}, {"sample": "GCA_003812525.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Proteus vulgaris strain FDAARGOS_556 chromosome, complete genome", "reads_in": 277230, "reads_removed_percent": 0.0007214226454568409, "reads_retained_percent": 99.99927857735454}, {"sample": "GCA_002073695.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Streptococcus agalactiae strain FDAARGOS_254 chromosome, complete genome", "reads_in": 147914, "reads_removed_percent": 0.0013521370526116529, "reads_retained_percent": 99.9986478629474}, {"sample": "GCA_003812625.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Escherichia coli strain FDAARGOS_536 chromosome, complete genome", "reads_in": 340302, "reads_removed_percent": 0.0005877132664515636, "reads_retained_percent": 99.99941228673354}, {"sample": "GCA_003812765.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Gardnerella vaginalis strain FDAARGOS_568 chromosome, complete genome", "reads_in": 114440, "reads_removed_percent": 0.0017476406850751487, "reads_retained_percent": 99.99825235931492}, {"sample": "GCA_009806595.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Bacillus anthracis strain FDAARGOS_698 chromosome, complete genome", "reads_in": 348384, "reads_removed_percent": 0.0005740791770000919, "reads_retained_percent": 99.999425920823}, {"sample": "GCA_020221755.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Lactococcus lactis strain FDAARGOS_867 chromosome, complete genome", "reads_in": 178994, "reads_removed_percent": 0.0011173558890242132, "reads_retained_percent": 99.99888264411098}, {"sample": "GCA_000783915.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Serratia marcescens strain FDAARGOS_65 chromosome, complete genome", "reads_in": 349894, "reads_removed_percent": 0.0005716016850817676, "reads_retained_percent": 99.99942839831492}, {"sample": "GCA_002983705.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Helicobacter pylori strain FDAARGOS_300 chromosome, complete genome", "reads_in": 109716, "reads_removed_percent": 0.001822888184038791, "reads_retained_percent": 99.99817711181596}, {"sample": "GCA_003956005.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Staphylococcus haemolyticus strain FDAARGOS_517 chromosome, complete genome", "reads_in": 177968, "reads_removed_percent": 0.0011237975366357998, "reads_retained_percent": 99.99887620246336}, {"sample": "GCA_013267815.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Bacillus thuringiensis strain FDAARGOS_792 chromosome, complete genome", "reads_in": 355252, "reads_removed_percent": 0.0005629806447254344, "reads_retained_percent": 99.99943701935527}, {"sample": "GCA_003798445.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Yersinia pseudotuberculosis strain FDAARGOS_580 chromosome, complete genome", "reads_in": 332068, "reads_removed_percent": 0.0006022862787139983, "reads_retained_percent": 99.99939771372128}, {"sample": "GCA_016027455.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Burkholderia humptydooensis strain FDAARGOS_899 chromosome 1, complete sequence", "reads_in": 485776, "reads_removed_percent": 0.0004117123941899147, "reads_retained_percent": 99.9995882876058}, {"sample": "GCA_016026195.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Escherichia coli strain FDAARGOS_946 chromosome, complete genome", "reads_in": 359310, "reads_removed_percent": 0.0005566224151846595, "reads_retained_percent": 99.99944337758481}, {"sample": "GCA_016904615.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Escherichia coli strain FDAARGOS_1283 chromosome, complete genome", "reads_in": 347664, "reads_removed_percent": 0.0005752680749229141, "reads_retained_percent": 99.99942473192507}, {"sample": "GCA_002587885.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Klebsiella pneumoniae strain FDAARGOS_446 chromosome, complete genome", "reads_in": 384644, "reads_removed_percent": 0.0005199613148781731, "reads_retained_percent": 99.99948003868512}, {"sample": "GCA_016127515.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Acinetobacter baumannii strain FDAARGOS_1036 chromosome, complete genome", "reads_in": 270026, "reads_removed_percent": 0.0007406694170191019, "reads_retained_percent": 99.99925933058299}, {"sample": "GCA_016027015.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Bacillus cereus strain FDAARGOS_918 chromosome, complete genome", "reads_in": 363314, "reads_removed_percent": 0.0005504880076187541, "reads_retained_percent": 99.99944951199238}, {"sample": "GCA_016127435.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Salmonella enterica strain FDAARGOS_1067 chromosome, complete genome", "reads_in": 337820, "reads_removed_percent": 0.0005920312592504884, "reads_retained_percent": 99.99940796874075}, {"sample": "GCA_016904435.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Escherichia coli strain FDAARGOS_1301 chromosome, complete genome", "reads_in": 347014, "reads_removed_percent": 0.0005763456229431665, "reads_retained_percent": 99.99942365437705}, {"sample": "GCA_016027055.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Acinetobacter johnsonii strain FDAARGOS_910 chromosome, complete genome", "reads_in": 241162, "reads_removed_percent": 0.0008293180517660328, "reads_retained_percent": 99.99917068194823}, {"sample": "GCA_021183805.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Mycobacterium avium subsp. avium strain FDAARGOS_1609 chromosome, complete genome", "reads_in": 330370, "reads_removed_percent": 0.0006053818445984805, "reads_retained_percent": 99.9993946181554}, {"sample": "GCA_016027135.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Thomasclavelia ramosa strain FDAARGOS_906 chromosome, complete genome", "reads_in": 216294, "reads_removed_percent": 0.0009246673509205063, "reads_retained_percent": 99.99907533264908}, {"sample": "GCA_016127215.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Haemophilus parainfluenzae strain FDAARGOS_1000 chromosome, complete genome", "reads_in": 142780, "reads_removed_percent": 0.0014007564084605687, "reads_retained_percent": 99.99859924359154}, {"sample": "GCA_002277995.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Elizabethkingia anophelis strain FDAARGOS_198 chromosome, complete genome", "reads_in": 271614, "reads_removed_percent": 0.0007363390694146841, "reads_retained_percent": 99.99926366093058}, {"sample": "GCA_002073255.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Pasteurella multocida strain FDAARGOS_218 chromosome, complete genome", "reads_in": 155702, "reads_removed_percent": 0.0012845050159920875, "reads_retained_percent": 99.99871549498401}, {"sample": "GCA_021183705.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Mycobacterium kansasii strain FDAARGOS_1614 chromosome, complete genome", "reads_in": 430260, "reads_removed_percent": 0.0004648352159159578, "reads_retained_percent": 99.99953516478408}, {"sample": "GCA_016127315.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Achromobacter deleyi strain FDAARGOS_1050 chromosome, complete genome", "reads_in": 465818, "reads_removed_percent": 0.00042935223628112263, "reads_retained_percent": 99.99957064776372}, {"sample": "GCA_016026055.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Burkholderia glumae strain FDAARGOS_949 chromosome 1, complete sequence", "reads_in": 454724, "reads_removed_percent": 0.0004398272358617535, "reads_retained_percent": 99.99956017276413}, {"sample": "GCA_016026215.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Escherichia coli strain FDAARGOS_945 chromosome, complete genome", "reads_in": 371304, "reads_removed_percent": 0.0005386421907655183, "reads_retained_percent": 99.99946135780924}, {"sample": "GCA_002591075.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Klebsiella pneumoniae strain FDAARGOS_439 chromosome, complete genome", "reads_in": 383526, "reads_removed_percent": 0.0005214770315441457, "reads_retained_percent": 99.99947852296846}, {"sample": "GCA_016889325.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Escherichia coli strain FDAARGOS_1258 chromosome, complete genome", "reads_in": 348734, "reads_removed_percent": 0.0005735030137583373, "reads_retained_percent": 99.99942649698625}, {"sample": "GCA_016128135.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Streptococcus dysgalactiae strain FDAARGOS_1016 chromosome, complete genome", "reads_in": 151136, "reads_removed_percent": 0.001323311454583951, "reads_retained_percent": 99.99867668854542}, {"sample": "GCA_008693365.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Yersinia pseudotuberculosis strain FDAARGOS_665 chromosome, complete genome", "reads_in": 291134, "reads_removed_percent": 0.0006869688871790996, "reads_retained_percent": 99.99931303111282}, {"sample": "GCA_016889345.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Acinetobacter pittii strain FDAARGOS_1215 chromosome, complete genome", "reads_in": 259280, "reads_removed_percent": 0.0007713668620796051, "reads_retained_percent": 99.99922863313792}, {"sample": "GCA_013267335.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Bacillus thuringiensis strain FDAARGOS_793 chromosome, complete genome", "reads_in": 354256, "reads_removed_percent": 0.000564563479517637, "reads_retained_percent": 99.99943543652049}, {"sample": "GCA_019048305.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Acinetobacter lwoffii strain FDAARGOS 1393 chromosome, complete genome", "reads_in": 233634, "reads_removed_percent": 0.00085603978872938, "reads_retained_percent": 99.99914396021127}, {"sample": "GCA_021183565.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Nocardia yamanashiensis strain FDAARGOS_1623 chromosome, complete genome", "reads_in": 564586, "reads_removed_percent": 0.00035424186926349575, "reads_retained_percent": 99.99964575813074}, {"sample": "GCA_016889385.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Haemophilus parahaemolyticus strain FDAARGOS_1199 chromosome, complete genome", "reads_in": 139564, "reads_removed_percent": 0.0014330343068413057, "reads_retained_percent": 99.99856696569316}, {"sample": "GCA_016127895.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Moraxella nonliquefaciens strain FDAARGOS_1006 chromosome, complete genome", "reads_in": 154890, "reads_removed_percent": 0.001291238943766544, "reads_retained_percent": 99.99870876105624}, {"sample": "GCA_002386205.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Burkholderia thailandensis strain FDAARGOS_426 chromosome 1, complete sequence", "reads_in": 448658, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001019415.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Staphylococcus aureus strain FDAARGOS_24 chromosome, complete genome", "reads_in": 179832, "reads_removed_percent": 0.001112149116953601, "reads_retained_percent": 99.99888785088305}, {"sample": "GCA_016903315.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Escherichia coli strain FDAARGOS_1289 chromosome, complete genome", "reads_in": 368416, "reads_removed_percent": 0.000542864587857205, "reads_retained_percent": 99.99945713541214}, {"sample": "GCA_019048225.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Acinetobacter pittii strain FDAARGOS 1396 chromosome, complete genome", "reads_in": 261056, "reads_removed_percent": 0.000766119146849718, "reads_retained_percent": 99.99923388085315}, {"sample": "GCA_016889245.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Mammaliicoccus vitulinus strain FDAARGOS_1207 chromosome, complete genome", "reads_in": 176858, "reads_removed_percent": 0.001130850739010958, "reads_retained_percent": 99.99886914926098}, {"sample": "GCA_016889265.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Escherichia coli strain FDAARGOS_1264 chromosome, complete genome", "reads_in": 336128, "reads_removed_percent": 0.000595011424219345, "reads_retained_percent": 99.99940498857578}, {"sample": "GCA_016906185.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Streptomyces californicus strain FDAARGOS_1209 chromosome", "reads_in": 535448, "reads_removed_percent": 0.00037351899717619636, "reads_retained_percent": 99.99962648100282}, {"sample": "GCA_020735765.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Streptococcus anginosus subsp. anginosus strain FDAARGOS_1569 chromosome, complete genome", "reads_in": 125088, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903455.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1318 chromosome, complete genome", "reads_in": 346980, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903275.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1282 chromosome, complete genome", "reads_in": 352200, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019046905.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Enterobacter bugandensis strain FDAARGOS 1427 chromosome, complete genome", "reads_in": 335846, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019056115.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS 1383 chromosome, complete genome", "reads_in": 337060, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019046925.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Aeromonas sp. FDAARGOS 1402 chromosome, complete genome", "reads_in": 328168, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048805.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Citrobacter braakii strain FDAARGOS 1421 chromosome, complete genome", "reads_in": 371974, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735745.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 4, "name": "Lachnospira eligens strain FDAARGOS_1570 chromosome, complete genome", "reads_in": 184852, "reads_removed_percent": 0.002163893276783589, "reads_retained_percent": 99.99783610672321}, {"sample": "GCA_019048745.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Aeromonas sp. FDAARGOS 1410 chromosome, complete genome", "reads_in": 300686, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016939475.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Acinetobacter calcoaceticus strain FDAARGOS_1218 chromosome, complete genome", "reads_in": 261016, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019046945.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Streptococcus gordonii strain FDAARGOS 1454 chromosome, complete genome", "reads_in": 145842, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003020185.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 4, "name": "Burkholderia thailandensis strain FDAARGOS_238 chromosome 1, complete sequence", "reads_in": 451244, "reads_removed_percent": 0.0008864383792360673, "reads_retained_percent": 99.99911356162076}, {"sample": "GCA_016903555.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 4, "name": "Staphylococcus epidermidis strain FDAARGOS_1363 chromosome, complete genome", "reads_in": 167308, "reads_removed_percent": 0.002390800200827217, "reads_retained_percent": 99.99760919979917}, {"sample": "GCA_019048705.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Aeromonas sp. FDAARGOS 1409 chromosome, complete genome", "reads_in": 291864, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002984285.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 4, "name": "Salmonella enterica subsp. enterica serovar Typhimurium strain FDAARGOS_317 chromosome, complete genome", "reads_in": 346190, "reads_removed_percent": 0.001155434876801756, "reads_retained_percent": 99.99884456512319}, {"sample": "GCA_019047145.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1379 chromosome, complete genome", "reads_in": 335034, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048445.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS 1382 chromosome, complete genome", "reads_in": 337060, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048725.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Aeromonas sp. FDAARGOS 1407 chromosome, complete genome", "reads_in": 313436, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016027595.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 4, "name": "Serratia plymuthica strain FDAARGOS_896 chromosome, complete genome", "reads_in": 368456, "reads_removed_percent": 0.0010856113077273813, "reads_retained_percent": 99.99891438869227}, {"sample": "GCA_019930605.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Prevotella nigrescens strain FDAARGOS_1486 chromosome, complete genome", "reads_in": 178206, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016939455.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Leuconostoc falkenbergense strain FDAARGOS_1201 chromosome, complete genome", "reads_in": 126594, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048505.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS 1387 chromosome, complete genome", "reads_in": 348454, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019047125.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Aeromonas sp. FDAARGOS 1411 chromosome, complete genome", "reads_in": 302630, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048485.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Enterobacter asburiae strain FDAARGOS 1432 chromosome, complete genome", "reads_in": 326758, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048465.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Jonesia denitrificans strain FDAARGOS 1440 chromosome, complete genome", "reads_in": 183310, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001188045.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Listeria monocytogenes strain FDAARGOS_57 chromosome unnamed, complete genome", "reads_in": 195692, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019046985.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Aeromonas sp. FDAARGOS 1417 chromosome, complete genome", "reads_in": 319550, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019930625.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Nocardia asteroides strain FDAARGOS_1485 chromosome, complete genome", "reads_in": 494394, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003798405.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 4, "name": "Yersinia pseudotuberculosis strain FDAARGOS_582 chromosome, complete genome", "reads_in": 322164, "reads_removed_percent": 0.0012416036552811611, "reads_retained_percent": 99.99875839634471}, {"sample": "GCA_016906225.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Streptomyces californicus strain FDAARGOS_1211 chromosome", "reads_in": 536880, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019930685.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Legionella longbeachae strain FDAARGOS_1481 chromosome, complete genome", "reads_in": 277696, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016906245.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Streptomyces californicus strain FDAARGOS_1213 chromosome", "reads_in": 538938, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016890025.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 4, "name": "Staphylococcus carnosus strain FDAARGOS_1204 chromosome, complete genome", "reads_in": 176972, "reads_removed_percent": 0.0022602445584612254, "reads_retained_percent": 99.99773975544154}, {"sample": "GCA_019930665.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Bacteroides nordii strain FDAARGOS_1461 chromosome, complete genome", "reads_in": 378572, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019930645.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Gordonia bronchialis strain FDAARGOS_1476 chromosome, complete genome", "reads_in": 353808, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001019275.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Staphylococcus aureus strain FDAARGOS_7 chromosome, complete genome", "reads_in": 181938, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019047165.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1384 chromosome, complete genome", "reads_in": 347978, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019047785.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Enterobacter cancerogenus strain FDAARGOS 1428 chromosome, complete genome", "reads_in": 321448, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003798245.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 4, "name": "Burkholderia pseudomallei strain FDAARGOS_593 chromosome 1, complete sequence", "reads_in": 483702, "reads_removed_percent": 0.0008269554395061422, "reads_retained_percent": 99.9991730445605}, {"sample": "GCA_019048165.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Corynebacterium coyleae strain FDAARGOS 1425 chromosome, complete genome", "reads_in": 167350, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048425.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS 1380 chromosome, complete genome", "reads_in": 349396, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048405.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Acinetobacter seifertii strain FDAARGOS 1400 chromosome, complete genome", "reads_in": 266604, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048385.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Pantoea agglomerans strain FDAARGOS 1447 chromosome, complete genome", "reads_in": 312802, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048365.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Aeromonas sp. FDAARGOS 1415 chromosome, complete genome", "reads_in": 289762, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019047325.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Aeromonas sp. FDAARGOS 1403 chromosome, complete genome", "reads_in": 321318, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002208785.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 4, "name": "Bacillus anthracis strain FDAARGOS_341 chromosome, complete genome", "reads_in": 360696, "reads_removed_percent": 0.0011089671080355757, "reads_retained_percent": 99.99889103289196}, {"sample": "GCA_019047465.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Lelliottia amnigena strain FDAARGOS 1444 chromosome, complete genome", "reads_in": 300368, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016727305.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 4, "name": "Enterococcus casseliflavus strain FDAARGOS_1122 chromosome, complete genome", "reads_in": 243400, "reads_removed_percent": 0.0016433853738701727, "reads_retained_percent": 99.99835661462613}, {"sample": "GCA_019048325.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Enterococcus durans strain FDAARGOS 1437 chromosome, complete genome", "reads_in": 206380, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019047505.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Streptococcus gordonii strain FDAARGOS 1455 chromosome, complete genome", "reads_in": 143654, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019047765.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Citrobacter pasteurii strain FDAARGOS 1424 chromosome, complete genome", "reads_in": 334754, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048285.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS 1389 chromosome, complete genome", "reads_in": 345514, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048265.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Aeromonas sp. FDAARGOS 1405 chromosome, complete genome", "reads_in": 307614, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048345.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS 1377 chromosome, complete genome", "reads_in": 356918, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048025.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Staphylococcus haemolyticus strain FDAARGOS 1453 chromosome, complete genome", "reads_in": 171250, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048045.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Citrobacter braakii strain FDAARGOS 1422 chromosome, complete genome", "reads_in": 321184, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048005.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1385 chromosome, complete genome", "reads_in": 348010, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903735.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 4, "name": "Klebsiella pneumoniae strain FDAARGOS_1325 chromosome, complete genome", "reads_in": 366224, "reads_removed_percent": 0.0010922277076324872, "reads_retained_percent": 99.99890777229237}, {"sample": "GCA_019047805.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Corynebacterium freneyi strain FDAARGOS 1426 chromosome, complete genome", "reads_in": 192120, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019047825.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Streptococcus mitis strain FDAARGOS 1456 chromosome, complete genome", "reads_in": 124590, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016026915.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 4, "name": "Staphylococcus epidermidis strain FDAARGOS_913 chromosome, complete genome", "reads_in": 175120, "reads_removed_percent": 0.0022841480127912287, "reads_retained_percent": 99.99771585198721}, {"sample": "GCA_019048125.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Klebsiella aerogenes strain FDAARGOS 1442 chromosome, complete genome", "reads_in": 352118, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019047885.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Providencia rettgeri strain FDAARGOS 1449 chromosome, complete genome", "reads_in": 318484, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019047905.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1388 chromosome, complete genome", "reads_in": 345864, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019047925.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Klebsiella aerogenes strain FDAARGOS 1441 chromosome, complete genome", "reads_in": 346180, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812805.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 4, "name": "Streptococcus agalactiae strain FDAARGOS_512 chromosome, complete genome", "reads_in": 142276, "reads_removed_percent": 0.0028114369254125782, "reads_retained_percent": 99.99718856307459}, {"sample": "GCA_019048105.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Providencia rettgeri strain FDAARGOS 1450 chromosome, complete genome", "reads_in": 309462, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019047945.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Enterobacter hormaechei strain FDAARGOS 1434 chromosome, complete genome", "reads_in": 318836, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019047965.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1378 chromosome, complete genome", "reads_in": 333822, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812885.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 4, "name": "Pseudomonas aeruginosa strain FDAARGOS_571 chromosome, complete genome", "reads_in": 466652, "reads_removed_percent": 0.0008571697967650412, "reads_retained_percent": 99.99914283020324}, {"sample": "GCA_019048065.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Citrobacter farmeri strain FDAARGOS 1423 chromosome, complete genome", "reads_in": 330986, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019047985.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Klebsiella quasipneumoniae strain FDAARGOS 1443 chromosome, complete genome", "reads_in": 359656, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019047105.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Enterobacter cloacae strain FDAARGOS 1431 chromosome, complete genome", "reads_in": 373508, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020097335.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 4, "name": "Streptococcus pneumoniae strain FDAARGOS_1508 chromosome, complete genome", "reads_in": 141108, "reads_removed_percent": 0.0028347081667942286, "reads_retained_percent": 99.99716529183321}, {"sample": "GCA_019048525.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Acinetobacter lwoffii strain FDAARGOS 1394 chromosome, complete genome", "reads_in": 231122, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048545.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Providencia rettgeri strain FDAARGOS 1451 chromosome, complete genome", "reads_in": 292254, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020736065.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 4, "name": "Finegoldia magna strain FDAARGOS_1556 chromosome, complete genome", "reads_in": 115922, "reads_removed_percent": 0.0034505960904746297, "reads_retained_percent": 99.99654940390953}, {"sample": "GCA_020735585.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Campylobacter fetus subsp. testudinum strain FDAARGOS_1591 chromosome, complete genome", "reads_in": 118366, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735705.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Coprococcus eutactus ATCC 27759 strain FDAARGOS_1581 chromosome, complete genome", "reads_in": 207556, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903755.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1317 chromosome, complete genome", "reads_in": 351878, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020736165.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Capnocytophaga ochracea strain FDAARGOS_1550 chromosome, complete genome", "reads_in": 180882, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735605.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Parabacteroides merdae strain FDAARGOS_1586 chromosome, complete genome", "reads_in": 291720, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020736265.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Bordetella pertussis strain FDAARGOS_1543 chromosome, complete genome", "reads_in": 273102, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903775.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1307 chromosome, complete genome", "reads_in": 378308, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903115.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1327 chromosome, complete genome", "reads_in": 366464, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_021183625.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Nocardia asteroides strain FDAARGOS_1620 chromosome, complete genome", "reads_in": 465792, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735925.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Bordetella parapertussis strain FDAARGOS_1541 chromosome, complete genome", "reads_in": 318372, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903795.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1308 chromosome, complete genome", "reads_in": 370052, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_000783735.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 4, "name": "Shigella flexneri strain FDAARGOS_74 chromosome, complete genome", "reads_in": 320390, "reads_removed_percent": 0.0012484784169293675, "reads_retained_percent": 99.99875152158307}, {"sample": "GCA_016903095.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Bordetella pertussis strain FDAARGOS_1368 chromosome, complete genome", "reads_in": 273672, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_000783755.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Citrobacter freundii strain FDAARGOS_73 chromosome, complete genome", "reads_in": 363416, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904195.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 4, "name": "Escherichia coli strain FDAARGOS_1296 chromosome, complete genome", "reads_in": 333376, "reads_removed_percent": 0.0011998464196582837, "reads_retained_percent": 99.99880015358035}, {"sample": "GCA_016903195.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1305 chromosome, complete genome", "reads_in": 384390, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903255.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1311 chromosome, complete genome", "reads_in": 370730, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735865.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Parabacteroides johnsonii strain FDAARGOS_1580 chromosome, complete genome", "reads_in": 317744, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903515.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1330 chromosome, complete genome", "reads_in": 365880, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019047545.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Escherichia fergusonii strain FDAARGOS 1438 chromosome, complete genome", "reads_in": 302878, "reads_removed_percent": 0.0006603318828042974, "reads_retained_percent": 99.9993396681172}, {"sample": "GCA_019047205.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Acinetobacter pittii strain FDAARGOS 1399 chromosome, complete genome", "reads_in": 265342, "reads_removed_percent": 0.0007537442244348803, "reads_retained_percent": 99.99924625577556}, {"sample": "GCA_016903435.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1302 chromosome, complete genome", "reads_in": 342060, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903535.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1373 chromosome, complete genome", "reads_in": 321672, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735725.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Alistipes senegalensis strain FDAARGOS_1578 chromosome, complete genome", "reads_in": 268220, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735785.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Prevotella melaninogenica strain FDAARGOS_1566 chromosome 1, complete sequence", "reads_in": 211214, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020736205.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Campylobacter rectus strain FDAARGOS_1549 chromosome, complete genome", "reads_in": 171454, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903235.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1324 chromosome, complete genome", "reads_in": 376682, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735845.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Tsukamurella paurometabola strain FDAARGOS_1558 chromosome, complete genome", "reads_in": 313538, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903475.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1315 chromosome, complete genome", "reads_in": 368824, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735805.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Helicobacter pylori strain FDAARGOS_1563 chromosome, complete genome", "reads_in": 110868, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020736225.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Campylobacter curvus strain FDAARGOS_1547 chromosome, complete genome", "reads_in": 131660, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001558275.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Staphylococcus saprophyticus strain FDAARGOS_168 chromosome, complete genome", "reads_in": 176676, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_000783875.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Proteus mirabilis strain FDAARGOS_67 chromosome, complete genome", "reads_in": 263698, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019047645.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1376 chromosome, complete genome", "reads_in": 356636, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003956025.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 4, "name": "Staphylococcus cohnii strain FDAARGOS_538 chromosome, complete genome", "reads_in": 180056, "reads_removed_percent": 0.0022215310792197984, "reads_retained_percent": 99.99777846892079}, {"sample": "GCA_016903135.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Acinetobacter baumannii strain FDAARGOS_1360 chromosome, complete genome", "reads_in": 271340, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904235.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 4, "name": "Escherichia coli strain FDAARGOS_1287 chromosome, complete genome", "reads_in": 369658, "reads_removed_percent": 0.0010820812751245745, "reads_retained_percent": 99.99891791872487}, {"sample": "GCA_019047025.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Enterobacter roggenkampii strain FDAARGOS 1430 chromosome, complete genome", "reads_in": 327818, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048625.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Enterobacter hormaechei strain FDAARGOS 1435 chromosome, complete genome", "reads_in": 317498, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020097455.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 4, "name": "Gemella haemolysans strain FDAARGOS_1500 chromosome, complete genome", "reads_in": 128550, "reads_removed_percent": 0.0031116297160637884, "reads_retained_percent": 99.99688837028394}, {"sample": "GCA_002951195.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 4, "name": "Pandoraea apista strain FDAARGOS_126 chromosome, complete genome", "reads_in": 355100, "reads_removed_percent": 0.0011264432554210083, "reads_retained_percent": 99.99887355674458}, {"sample": "GCA_019048605.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS 1375 chromosome, complete genome", "reads_in": 349518, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048645.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Streptococcus mutans strain FDAARGOS 1458 chromosome, complete genome", "reads_in": 134624, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019047045.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Aeromonas sp. FDAARGOS 1419 chromosome, complete genome", "reads_in": 305918, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048565.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Aeromonas sp. FDAARGOS 1406 chromosome, complete genome", "reads_in": 309722, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002393465.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 4, "name": "Escherichia coli strain FDAARGOS_401 chromosome, complete genome", "reads_in": 359158, "reads_removed_percent": 0.0011137159690164217, "reads_retained_percent": 99.99888628403099}, {"sample": "GCA_008693385.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 4, "name": "Cupriavidus pauculus strain FDAARGOS_664 chromosome 1, complete sequence", "reads_in": 470942, "reads_removed_percent": 0.0008493614924980146, "reads_retained_percent": 99.9991506385075}, {"sample": "GCA_019047065.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Aeromonas sp. FDAARGOS 1414 chromosome, complete genome", "reads_in": 299588, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_000783995.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 4, "name": "Citrobacter freundii strain FDAARGOS_61 chromosome, complete genome", "reads_in": 350566, "reads_removed_percent": 0.0011410119635104374, "reads_retained_percent": 99.99885898803649}, {"sample": "GCA_001188025.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Listeria monocytogenes strain FDAARGOS_58 chromosome, complete genome", "reads_in": 193778, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048585.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Aeromonas sp. FDAARGOS 1418 chromosome, complete genome", "reads_in": 296936, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016028775.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 4, "name": "Moraxella nonliquefaciens strain FDAARGOS_869 chromosome, complete genome", "reads_in": 156250, "reads_removed_percent": 0.0025599999999999998, "reads_retained_percent": 99.99744}, {"sample": "GCA_019047005.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Enterobacter ludwigii strain FDAARGOS 1436 chromosome, complete genome", "reads_in": 329562, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048665.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Aeromonas sp. FDAARGOS 1408 chromosome, complete genome", "reads_in": 322962, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020736185.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Campylobacter gracilis strain FDAARGOS_1548 chromosome, complete genome", "reads_in": 152116, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903415.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Klebsiella oxytoca strain FDAARGOS_1334 chromosome, complete genome", "reads_in": 415644, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903575.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Staphylococcus epidermidis strain FDAARGOS_1364 chromosome, complete genome", "reads_in": 163244, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_000783775.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Aeromonas caviae strain FDAARGOS_72 chromosome, complete genome", "reads_in": 301220, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016599795.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 4, "name": "Staphylococcus pasteuri strain FDAARGOS_1069 chromosome, complete genome", "reads_in": 165654, "reads_removed_percent": 0.002414671544303186, "reads_retained_percent": 99.9975853284557}, {"sample": "GCA_016903615.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1304 chromosome, complete genome", "reads_in": 351638, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903395.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1313 chromosome, complete genome", "reads_in": 385540, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735885.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Bordetella pertussis strain FDAARGOS_1542 chromosome, complete genome", "reads_in": 274846, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903715.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Klebsiella oxytoca strain FDAARGOS_1332 chromosome, complete genome", "reads_in": 403376, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_008693745.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 4, "name": "Vibrio parahaemolyticus strain FDAARGOS_662 chromosome 1, complete sequence", "reads_in": 343608, "reads_removed_percent": 0.001164117249889409, "reads_retained_percent": 99.99883588275011}, {"sample": "GCA_020735645.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Acidaminococcus fermentans DSM 20731 strain FDAARGOS_1571 chromosome, complete genome", "reads_in": 155546, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016403145.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 4, "name": "Agrobacterium tumefaciens strain FDAARGOS_1048 chromosome 1, complete sequence", "reads_in": 395512, "reads_removed_percent": 0.0010113473168955683, "reads_retained_percent": 99.99898865268311}, {"sample": "GCA_000783675.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Enterobacter cloacae complex sp. strain FDAARGOS_77 chromosome, complete genome", "reads_in": 325134, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735285.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Mycobacterium avium subsp. avium strain FDAARGOS_1607 chromosome, complete genome", "reads_in": 330462, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889885.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 4, "name": "Streptococcus constellatus strain FDAARGOS_1208 chromosome, complete genome", "reads_in": 131912, "reads_removed_percent": 0.003032324580023046, "reads_retained_percent": 99.99696767541998}, {"sample": "GCA_016725905.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 4, "name": "Proteus mirabilis strain FDAARGOS_1079 chromosome, complete genome", "reads_in": 269036, "reads_removed_percent": 0.001486789871987392, "reads_retained_percent": 99.99851321012801}, {"sample": "GCA_002386165.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Salmonella enterica subsp. enterica serovar Paratyphi A strain FDAARGOS_368 chromosome, complete genome", "reads_in": 305866, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003546885.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Klebsiella aerogenes strain FDAARGOS_327 chromosome, complete genome", "reads_in": 360012, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016728745.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Corynebacterium amycolatum strain FDAARGOS_1107 chromosome, complete genome", "reads_in": 180436, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889645.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Salmonella enterica strain FDAARGOS_1271 chromosome, complete genome", "reads_in": 320272, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001018685.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 30, "name": "Staphylococcus aureus strain FDAARGOS_4 chromosome, complete genome", "reads_in": 187452, "reads_removed_percent": 0.016004097048844507, "reads_retained_percent": 99.98399590295115}, {"sample": "GCA_016727585.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 14, "name": "Staphylococcus aureus strain FDAARGOS_1086 chromosome, complete genome", "reads_in": 185098, "reads_removed_percent": 0.007563560924483247, "reads_retained_percent": 99.99243643907552}, {"sample": "GCA_016127635.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Enterococcus casseliflavus strain FDAARGOS_998 chromosome, complete genome", "reads_in": 238722, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812725.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 30, "name": "Streptococcus pyogenes strain FDAARGOS_514 chromosome, complete genome", "reads_in": 126808, "reads_removed_percent": 0.023657813387168002, "reads_retained_percent": 99.97634218661283}, {"sample": "GCA_016127615.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Corynebacterium amycolatum strain FDAARGOS_991 chromosome, complete genome", "reads_in": 188596, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001018975.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 30, "name": "Staphylococcus aureus strain FDAARGOS_30 chromosome, complete genome", "reads_in": 190776, "reads_removed_percent": 0.01572524845892565, "reads_retained_percent": 99.98427475154108}, {"sample": "GCA_016724885.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Streptococcus dysgalactiae strain FDAARGOS_1157 chromosome, complete genome", "reads_in": 143412, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016725005.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Streptococcus constellatus strain FDAARGOS_1156 chromosome, complete genome", "reads_in": 126884, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016725165.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Streptococcus lutetiensis strain FDAARGOS_1158 chromosome, complete genome", "reads_in": 119568, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001018845.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 30, "name": "Staphylococcus aureus strain FDAARGOS_21 chromosome, complete genome", "reads_in": 192082, "reads_removed_percent": 0.01561832967170271, "reads_retained_percent": 99.9843816703283}, {"sample": "GCA_016725205.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Staphylococcus pasteuri strain FDAARGOS_1152 chromosome, complete genome", "reads_in": 170422, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001558855.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Providencia stuartii strain FDAARGOS_145 chromosome, complete genome", "reads_in": 287008, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001019255.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 30, "name": "Staphylococcus aureus strain FDAARGOS_1 chromosome, complete genome", "reads_in": 190366, "reads_removed_percent": 0.015759116648981438, "reads_retained_percent": 99.98424088335102}, {"sample": "GCA_016725225.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Mobiluncus curtisii strain FDAARGOS_1127 chromosome, complete genome", "reads_in": 143540, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127875.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Streptococcus constellatus strain FDAARGOS_1015 chromosome, complete genome", "reads_in": 135906, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009732965.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 14, "name": "Aerococcus viridans strain FDAARGOS_672 chromosome, complete genome", "reads_in": 138814, "reads_removed_percent": 0.010085438068206377, "reads_retained_percent": 99.9899145619318}, {"sample": "GCA_001018775.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 30, "name": "Staphylococcus aureus strain FDAARGOS_14 chromosome, complete genome", "reads_in": 184314, "reads_removed_percent": 0.016276571502978614, "reads_retained_percent": 99.98372342849702}, {"sample": "GCA_016725245.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Mammaliicoccus vitulinus strain FDAARGOS_1153 chromosome, complete genome", "reads_in": 169568, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016724865.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Peribacillus psychrosaccharolyticus strain FDAARGOS_1161 chromosome, complete genome", "reads_in": 312086, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001559035.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Bartonella bacilliformis strain ATCC 35685D-5 chromosome, complete genome", "reads_in": 96228, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016725265.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Staphylococcus equorum strain FDAARGOS_1150 chromosome, complete genome", "reads_in": 186494, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009806615.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 14, "name": "Bacillus anthracis strain FDAARGOS_701 chromosome, complete genome", "reads_in": 348494, "reads_removed_percent": 0.004017285806929244, "reads_retained_percent": 99.99598271419308}, {"sample": "GCA_001558815.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Staphylococcus lugdunensis strain FDAARGOS_143 chromosome, complete genome", "reads_in": 174258, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016725305.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Staphylococcus equorum strain FDAARGOS_1149 chromosome, complete genome", "reads_in": 182404, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003813085.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 28, "name": "Alcaligenes faecalis strain FDAARGOS_491 chromosome, complete genome", "reads_in": 271526, "reads_removed_percent": 0.010312087976841997, "reads_retained_percent": 99.98968791202316}, {"sample": "GCA_016127655.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Brevundimonas diminuta strain FDAARGOS_1026 chromosome, complete genome", "reads_in": 236606, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001558795.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 14, "name": "Staphylococcus aureus strain FDAARGOS_159 chromosome, complete genome", "reads_in": 189278, "reads_removed_percent": 0.007396527858493855, "reads_retained_percent": 99.99260347214151}, {"sample": "GCA_016889625.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1262 chromosome, complete genome", "reads_in": 348356, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889965.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Staphylococcus condimenti strain FDAARGOS_1205 chromosome, complete genome", "reads_in": 180552, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127775.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Pediococcus pentosaceus strain FDAARGOS_1009 chromosome, complete genome", "reads_in": 116796, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001559075.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Citrobacter amalonaticus strain FDAARGOS_122 chromosome, complete genome", "reads_in": 326982, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001019485.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 28, "name": "Staphylococcus aureus strain FDAARGOS_35 chromosome, complete genome", "reads_in": 197060, "reads_removed_percent": 0.014208870394803614, "reads_retained_percent": 99.98579112960519}, {"sample": "GCA_001558295.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Shigella sonnei strain FDAARGOS_90 chromosome, complete genome", "reads_in": 332556, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889845.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1248 chromosome, complete genome", "reads_in": 337768, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001019205.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 28, "name": "Staphylococcus aureus strain FDAARGOS_48 chromosome, complete genome", "reads_in": 189826, "reads_removed_percent": 0.014750350320820118, "reads_retained_percent": 99.98524964967918}, {"sample": "GCA_016889705.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Amycolatopsis sp. FDAARGOS 1241 strain FDAARGOS_1241 chromosome, complete genome", "reads_in": 653208, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127735.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Streptococcus oralis strain FDAARGOS_1075 chromosome, complete genome", "reads_in": 132994, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127715.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Staphylococcus saccharolyticus strain FDAARGOS_1074 chromosome, complete genome", "reads_in": 156434, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889285.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Kytococcus sedentarius strain FDAARGOS_1200 chromosome, complete genome", "reads_in": 163250, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889945.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 28, "name": "Staphylococcus aureus strain FDAARGOS_1245 chromosome, complete genome", "reads_in": 192584, "reads_removed_percent": 0.014539110206455364, "reads_retained_percent": 99.98546088979354}, {"sample": "GCA_016725325.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Sphingobacterium spiritivorum strain FDAARGOS_1145 chromosome, complete genome", "reads_in": 342354, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889305.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Acinetobacter pittii strain FDAARGOS_1217 chromosome, complete genome", "reads_in": 263804, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889365.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Corynebacterium kroppenstedtii strain FDAARGOS_1192 chromosome, complete genome", "reads_in": 163120, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001018915.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 14, "name": "Staphylococcus aureus strain FDAARGOS_27 chromosome, complete genome", "reads_in": 182690, "reads_removed_percent": 0.0076632546937435, "reads_retained_percent": 99.99233674530626}, {"sample": "GCA_016027155.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 28, "name": "Clostridium perfringens strain FDAARGOS_904 chromosome, complete genome", "reads_in": 224056, "reads_removed_percent": 0.012496875781054738, "reads_retained_percent": 99.98750312421895}, {"sample": "GCA_001559115.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Staphylococcus simulans strain FDAARGOS_124 chromosome, complete genome", "reads_in": 176504, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889405.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Corynebacterium kroppenstedtii strain FDAARGOS_1194 chromosome, complete genome", "reads_in": 167458, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016027715.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 28, "name": "Lactococcus garvieae strain FDAARGOS_893 chromosome, complete genome", "reads_in": 140684, "reads_removed_percent": 0.01990276079724773, "reads_retained_percent": 99.98009723920275}, {"sample": "GCA_016889665.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Enterocloster bolteae strain FDAARGOS_1231 chromosome, complete genome", "reads_in": 440932, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889425.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Corynebacterium amycolatum strain FDAARGOS_1189 chromosome, complete genome", "reads_in": 164996, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127755.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 14, "name": "Burkholderia ambifaria strain FDAARGOS_1027 chromosome 1, complete sequence", "reads_in": 501574, "reads_removed_percent": 0.0027912132606554567, "reads_retained_percent": 99.99720878673935}, {"sample": "GCA_016889445.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Corynebacterium striatum strain FDAARGOS_1197 chromosome, complete genome", "reads_in": 200318, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889465.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Corynebacterium macginleyi strain FDAARGOS_1195 chromosome, complete genome", "reads_in": 166236, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889605.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1268 chromosome, complete genome", "reads_in": 333912, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127695.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1059 chromosome, complete genome", "reads_in": 361050, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016128175.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Rothia kristinae strain FDAARGOS_1001 chromosome, complete genome", "reads_in": 167590, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016725645.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Sphingobacterium spiritivorum strain FDAARGOS_1144 chromosome, complete genome", "reads_in": 342552, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013267275.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 14, "name": "Bacillus cereus strain FDAARGOS_798 chromosome, complete genome", "reads_in": 412640, "reads_removed_percent": 0.0033927879022877085, "reads_retained_percent": 99.99660721209771}, {"sample": "GCA_001558735.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Pantoea vagans strain FDAARGOS_160 chromosome, complete genome", "reads_in": 320590, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016727325.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Enterococcus casseliflavus strain FDAARGOS_1121 chromosome, complete genome", "reads_in": 243364, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016727345.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Enterococcus casseliflavus strain FDAARGOS_1120 chromosome, complete genome", "reads_in": 236566, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127975.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Corynebacterium urealyticum strain FDAARGOS_994 chromosome, complete genome", "reads_in": 158948, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016727365.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Corynebacterium ulcerans strain FDAARGOS_1118 chromosome, complete genome", "reads_in": 163584, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020099295.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 34, "name": "Clostridium sporogenes strain FDAARGOS_1491 chromosome, complete genome", "reads_in": 271160, "reads_removed_percent": 0.01253872252544623, "reads_retained_percent": 99.98746127747455}, {"sample": "GCA_016127955.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Actinomyces oris strain FDAARGOS_1051 chromosome, complete genome", "reads_in": 212692, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016727385.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Staphylococcus epidermidis strain FDAARGOS_1083 chromosome, complete genome", "reads_in": 165308, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127475.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 34, "name": "Anaerococcus vaginalis strain FDAARGOS_988 chromosome, complete genome", "reads_in": 126264, "reads_removed_percent": 0.02692770702654755, "reads_retained_percent": 99.97307229297346}, {"sample": "GCA_016727405.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Bacillus cereus strain FDAARGOS_1084 chromosome, complete genome", "reads_in": 354920, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016727425.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1076 chromosome, complete genome", "reads_in": 360454, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016128035.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 34, "name": "Alcaligenes faecalis strain FDAARGOS_1024 chromosome, complete genome", "reads_in": 271528, "reads_removed_percent": 0.012521728882472527, "reads_retained_percent": 99.98747827111752}, {"sample": "GCA_001018835.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 34, "name": "Staphylococcus aureus strain FDAARGOS_19 chromosome, complete genome", "reads_in": 191936, "reads_removed_percent": 0.017714238079359786, "reads_retained_percent": 99.98228576192064}, {"sample": "GCA_016727445.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Morganella morganii strain FDAARGOS_1085 chromosome, complete genome", "reads_in": 265002, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016727765.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Yersinia enterocolitica strain FDAARGOS_1082 chromosome, complete genome", "reads_in": 302752, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127915.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Streptococcus oralis strain FDAARGOS_1021 chromosome, complete genome", "reads_in": 127050, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016728025.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Listeria monocytogenes strain FDAARGOS_1080 chromosome, complete genome", "reads_in": 197582, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001559135.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Serratia liquefaciens strain FDAARGOS_125 chromosome, complete genome", "reads_in": 352316, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016726045.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 12, "name": "Sphingobacterium multivorum strain FDAARGOS_1140 chromosome, complete genome", "reads_in": 377044, "reads_removed_percent": 0.0031826524225289352, "reads_retained_percent": 99.99681734757748}, {"sample": "GCA_016728045.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Serratia ureilytica strain FDAARGOS_1089 chromosome, complete genome", "reads_in": 346696, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002386185.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 12, "name": "Staphylococcus simulans strain FDAARGOS_383 chromosome, complete genome", "reads_in": 191464, "reads_removed_percent": 0.00626749676179334, "reads_retained_percent": 99.9937325032382}, {"sample": "GCA_000626615.3", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 12, "name": "Staphylococcus aureus strain FDAARGOS_5 chromosome, complete genome", "reads_in": 187346, "reads_removed_percent": 0.006405260854248289, "reads_retained_percent": 99.99359473914575}, {"sample": "GCA_002208945.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 12, "name": "Enterococcus faecalis strain FDAARGOS_338 chromosome, complete genome", "reads_in": 190734, "reads_removed_percent": 0.006291484475762056, "reads_retained_percent": 99.99370851552423}, {"sample": "GCA_016127075.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Macrococcus caseolyticus strain FDAARGOS_1005 chromosome, complete genome", "reads_in": 144448, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016728065.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Pseudomonas fluorescens strain FDAARGOS_1088 chromosome, complete genome", "reads_in": 408974, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001558475.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Vibrio mimicus strain FDAARGOS_112 chromosome 1, complete sequence", "reads_in": 295914, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127935.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Staphylococcus pettenkoferi strain FDAARGOS_1071 chromosome, complete genome", "reads_in": 166242, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001558355.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Salmonella enterica strain FDAARGOS_94 chromosome, complete genome", "reads_in": 330094, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016725945.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Acinetobacter johnsonii strain FDAARGOS_1093 chromosome, complete genome", "reads_in": 257722, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127995.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Schaalia meyeri strain FDAARGOS_985 chromosome, complete genome", "reads_in": 135536, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016725825.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Enterobacter hormaechei strain FDAARGOS_1078 chromosome, complete genome", "reads_in": 338684, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001558775.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Staphylococcus lugdunensis strain FDAARGOS_141 chromosome, complete genome", "reads_in": 171456, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003827915.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 30, "name": "Staphylococcus aureus strain FDAARGOS_504 chromosome, complete genome", "reads_in": 192972, "reads_removed_percent": 0.01554629687208507, "reads_retained_percent": 99.98445370312791}, {"sample": "GCA_001558755.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Achromobacter xylosoxidans strain FDAARGOS_147 chromosome, complete genome", "reads_in": 459130, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_022213445.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 12, "name": "Burkholderia vietnamiensis strain FDAARGOS_721 chromosome 1, complete sequence", "reads_in": 467904, "reads_removed_percent": 0.0025646286417726713, "reads_retained_percent": 99.99743537135822}, {"sample": "GCA_016128115.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Providencia stuartii strain FDAARGOS_1040 chromosome, complete genome", "reads_in": 289014, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016724825.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Weizmannia coagulans strain FDAARGOS_1160 chromosome, complete genome", "reads_in": 222856, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016128235.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Escherichia fergusonii strain FDAARGOS_1032 chromosome, complete genome", "reads_in": 309820, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016128095.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Streptococcus dysgalactiae strain FDAARGOS_1017 chromosome, complete genome", "reads_in": 144956, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001558935.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Citrobacter amalonaticus strain FDAARGOS_165 chromosome, complete genome", "reads_in": 338936, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016128055.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Pediococcus acidilactici strain FDAARGOS_1008 chromosome, complete genome", "reads_in": 133972, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016128255.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Lactococcus garvieae strain FDAARGOS_1063 chromosome, complete genome", "reads_in": 147538, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016728105.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Corynebacterium striatum strain FDAARGOS_1115 chromosome, complete genome", "reads_in": 193656, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016128275.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Citrobacter koseri strain FDAARGOS_1029 chromosome, complete genome", "reads_in": 315772, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812125.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 32, "name": "Bacillus sp. FDAARGOS_527 chromosome, complete genome", "reads_in": 380076, "reads_removed_percent": 0.00841936875782738, "reads_retained_percent": 99.99158063124217}, {"sample": "GCA_016598875.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Burkholderia glumae strain FDAARGOS_921 chromosome 1, complete sequence", "reads_in": 456378, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016599755.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Corynebacterium kefirresidentii strain FDAARGOS_1055 chromosome, complete genome", "reads_in": 168214, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001559055.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Bordetella bronchiseptica strain FDAARGOS_176 chromosome, complete genome", "reads_in": 355948, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016599775.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Stutzerimonas balearica strain FDAARGOS_1013 chromosome, complete genome", "reads_in": 294082, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016724845.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Sphingobacterium spiritivorum strain FDAARGOS_1146 chromosome, complete genome", "reads_in": 351684, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016128015.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Brucella anthropi strain FDAARGOS_1039 chromosome 1, complete sequence", "reads_in": 344930, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016599835.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Elizabethkingia bruuniana strain FDAARGOS_1031 chromosome, complete genome", "reads_in": 308800, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016724565.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Staphylococcus condimenti strain FDAARGOS_1148 chromosome, complete genome", "reads_in": 177956, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001558895.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Morganella morganii strain FDAARGOS_172 chromosome, complete genome", "reads_in": 260538, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001558875.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Enterococcus gallinarum strain FDAARGOS_163 chromosome, complete genome", "reads_in": 254766, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016724805.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Sutterella wadsworthensis strain FDAARGOS_1159 chromosome, complete genome", "reads_in": 201768, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735825.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 32, "name": "Haemophilus influenzae strain FDAARGOS_1562 chromosome, complete genome", "reads_in": 122840, "reads_removed_percent": 0.026050146532074245, "reads_retained_percent": 99.97394985346793}, {"sample": "GCA_016127175.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Veillonella parvula strain FDAARGOS_1046 chromosome, complete genome", "reads_in": 142154, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127795.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Stutzerimonas stutzeri strain FDAARGOS_1065 chromosome, complete genome", "reads_in": 295130, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002386245.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 14, "name": "Staphylococcus aureus strain FDAARGOS_412 chromosome, complete genome", "reads_in": 184640, "reads_removed_percent": 0.0075823223570190635, "reads_retained_percent": 99.99241767764298}, {"sample": "GCA_016027835.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Serratia plymuthica strain FDAARGOS_889 chromosome, complete genome", "reads_in": 362786, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016026595.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Brevibacterium casei strain FDAARGOS_936 chromosome, complete genome", "reads_in": 257772, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016028295.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Staphylococcus auricularis strain FDAARGOS_882 chromosome, complete genome", "reads_in": 147772, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016027855.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Atlantibacter hermannii strain FDAARGOS_888 chromosome, complete genome", "reads_in": 303658, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127335.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 18, "name": "Staphylococcus pseudintermedius strain FDAARGOS_1072 chromosome, complete genome", "reads_in": 189290, "reads_removed_percent": 0.009509218659200168, "reads_retained_percent": 99.9904907813408}, {"sample": "GCA_016026575.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Fannyhessea vaginae strain FDAARGOS_934 chromosome, complete genome", "reads_in": 96468, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001019535.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 18, "name": "Staphylococcus aureus strain FDAARGOS_42 chromosome, complete genome", "reads_in": 187956, "reads_removed_percent": 0.00957670944263551, "reads_retained_percent": 99.99042329055736}, {"sample": "GCA_001593585.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Klebsiella aerogenes strain FDAARGOS_139 chromosome, complete genome", "reads_in": 353188, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016028175.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Streptococcus oralis strain FDAARGOS_886 chromosome, complete genome", "reads_in": 127586, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016725865.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 18, "name": "Sphingobacterium multivorum strain FDAARGOS_1143 chromosome, complete genome", "reads_in": 401680, "reads_removed_percent": 0.004481179047998407, "reads_retained_percent": 99.995518820952}, {"sample": "GCA_002073475.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 22, "name": "Haemophilus influenzae strain FDAARGOS_199 chromosome, complete genome", "reads_in": 122046, "reads_removed_percent": 0.018025990200416234, "reads_retained_percent": 99.98197400979959}, {"sample": "GCA_016028235.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Staphylococcus equorum strain FDAARGOS_884 chromosome, complete genome", "reads_in": 190484, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812865.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 18, "name": "Chryseobacterium indologenes strain FDAARGOS_510 chromosome, complete genome", "reads_in": 328196, "reads_removed_percent": 0.005484527538422162, "reads_retained_percent": 99.99451547246157}, {"sample": "GCA_016028255.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Streptococcus oralis strain FDAARGOS_885 chromosome, complete genome", "reads_in": 134954, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016026555.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Brevibacterium casei strain FDAARGOS_937 chromosome, complete genome", "reads_in": 258770, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016026735.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Serratia rubidaea strain FDAARGOS_926 chromosome, complete genome", "reads_in": 333000, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016028275.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Staphylococcus carnosus strain FDAARGOS_883 chromosome, complete genome", "reads_in": 171424, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016026415.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Corynebacterium amycolatum strain FDAARGOS_938 chromosome, complete genome", "reads_in": 186842, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002073415.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Bacillus sp. FDAARGOS_235 chromosome, complete genome", "reads_in": 373892, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016128215.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Pediococcus pentosaceus strain FDAARGOS_1010 chromosome, complete genome", "reads_in": 129894, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009730495.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 18, "name": "Clostridioides difficile strain FDAARGOS_723 chromosome, complete genome", "reads_in": 278540, "reads_removed_percent": 0.006462267537876068, "reads_retained_percent": 99.99353773246213}, {"sample": "GCA_001019435.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 24, "name": "Staphylococcus aureus strain FDAARGOS_25 chromosome, complete genome", "reads_in": 191900, "reads_removed_percent": 0.012506513809275666, "reads_retained_percent": 99.98749348619073}, {"sample": "GCA_016728825.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Achromobacter xylosoxidans strain FDAARGOS_1091 chromosome, complete genome", "reads_in": 460132, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016026395.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Edwardsiella hoshinae strain FDAARGOS_940 chromosome, complete genome", "reads_in": 254474, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016888885.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Moraxella osloensis strain FDAARGOS_1202 chromosome, complete genome", "reads_in": 183946, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016027815.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Delftia lacustris strain FDAARGOS_890 chromosome, complete genome", "reads_in": 484968, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013267775.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 22, "name": "Bacillus cereus strain FDAARGOS_797 chromosome, complete genome", "reads_in": 374962, "reads_removed_percent": 0.005867261215803201, "reads_retained_percent": 99.9941327387842}, {"sample": "GCA_016027775.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Delftia acidovorans strain FDAARGOS_891 chromosome, complete genome", "reads_in": 476700, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002073215.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Moraxella catarrhalis strain FDAARGOS_213 chromosome, complete genome", "reads_in": 127200, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019930985.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 22, "name": "Campylobacter coli strain FDAARGOS_1465 chromosome, complete genome", "reads_in": 128994, "reads_removed_percent": 0.01705505682434842, "reads_retained_percent": 99.98294494317565}, {"sample": "GCA_016028635.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Stutzerimonas degradans strain FDAARGOS_876 chromosome, complete genome", "reads_in": 269474, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016028655.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Stutzerimonas stutzeri strain FDAARGOS_875 chromosome, complete genome", "reads_in": 302358, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016028495.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Salmonella enterica strain FDAARGOS_878 chromosome, complete genome", "reads_in": 331248, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016028475.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Serratia rubidaea strain FDAARGOS_880 chromosome, complete genome", "reads_in": 337078, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001559255.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Hafnia paralvei strain FDAARGOS_158 chromosome, complete genome", "reads_in": 300914, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016028715.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Neisseria cinerea strain FDAARGOS_871 chromosome, complete genome", "reads_in": 122194, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016027615.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Corynebacterium minutissimum strain FDAARGOS_894 chromosome, complete genome", "reads_in": 179730, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013267295.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 20, "name": "Bacillus thuringiensis strain FDAARGOS_796 chromosome, complete genome", "reads_in": 413816, "reads_removed_percent": 0.004833065903686662, "reads_retained_percent": 99.99516693409632}, {"sample": "GCA_016028735.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Oligella ureolytica strain FDAARGOS_872 chromosome, complete genome", "reads_in": 188856, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016128195.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Paraburkholderia ginsengisoli strain FDAARGOS_1049 chromosome 1, complete sequence", "reads_in": 439818, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019930705.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 20, "name": "Haemophilus influenzae strain FDAARGOS_1479 chromosome, complete genome", "reads_in": 126840, "reads_removed_percent": 0.015767896562598548, "reads_retained_percent": 99.9842321034374}, {"sample": "GCA_016888945.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Phocaeicola coprophilus strain FDAARGOS_1220 chromosome, complete genome", "reads_in": 274240, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016028755.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Moraxella osloensis strain FDAARGOS_870 chromosome, complete genome", "reads_in": 162388, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001018645.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 20, "name": "Staphylococcus aureus strain FDAARGOS_2 chromosome, complete genome", "reads_in": 187334, "reads_removed_percent": 0.0106761185903253, "reads_retained_percent": 99.98932388140967}, {"sample": "GCA_016026715.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Salmonella enterica strain FDAARGOS_928 chromosome, complete genome", "reads_in": 330094, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016028795.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Macrococcus caseolyticus strain FDAARGOS_868 chromosome, complete genome", "reads_in": 144730, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002073375.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Corynebacterium diphtheriae strain FDAARGOS_197 chromosome, complete genome", "reads_in": 165936, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016028855.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Rothia kristinae strain FDAARGOS_864 chromosome, complete genome", "reads_in": 158516, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903215.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 20, "name": "Acinetobacter baumannii strain FDAARGOS_1359 chromosome, complete genome", "reads_in": 259886, "reads_removed_percent": 0.0076956819528562525, "reads_retained_percent": 99.99230431804715}, {"sample": "GCA_016028695.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Pediococcus pentosaceus strain FDAARGOS_873 chromosome, complete genome", "reads_in": 116058, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016028435.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Sphingomonas paucimobilis strain FDAARGOS_881 chromosome, complete genome", "reads_in": 301070, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016027675.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Serratia plymuthica strain FDAARGOS_895 chromosome, complete genome", "reads_in": 356612, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016026635.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Staphylococcus pseudintermedius strain FDAARGOS_930 chromosome, complete genome", "reads_in": 170856, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016027695.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Enterobacter asburiae strain FDAARGOS_892 chromosome, complete genome", "reads_in": 317798, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002073395.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Staphylococcus lugdunensis strain FDAARGOS_222 chromosome, complete genome", "reads_in": 169354, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001019305.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 22, "name": "Staphylococcus aureus strain FDAARGOS_9 chromosome, complete genome", "reads_in": 187646, "reads_removed_percent": 0.011724204086418041, "reads_retained_percent": 99.98827579591358}, {"sample": "GCA_016888965.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1261 chromosome, complete genome", "reads_in": 336674, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016027975.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 18, "name": "Lactococcus lactis strain FDAARGOS_887 chromosome, complete genome", "reads_in": 172198, "reads_removed_percent": 0.010453083078781402, "reads_retained_percent": 99.98954691692121}, {"sample": "GCA_002073435.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Streptococcus gordonii strain FDAARGOS_257 chromosome, complete genome", "reads_in": 148180, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016728665.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Corynebacterium propinquum strain FDAARGOS_1112 chromosome, complete genome", "reads_in": 165342, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002073315.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Yersinia sp. FDAARGOS_228 chromosome, complete genome", "reads_in": 330870, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016728685.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Corynebacterium aurimucosum strain FDAARGOS_1110 chromosome, complete genome", "reads_in": 188632, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016728705.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Corynebacterium aurimucosum strain FDAARGOS_1109 chromosome, complete genome", "reads_in": 182518, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016026975.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Aerococcus urinae strain FDAARGOS_911 chromosome, complete genome", "reads_in": 131618, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016027375.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 16, "name": "Clostridium perfringens strain FDAARGOS_903 chromosome, complete genome", "reads_in": 223682, "reads_removed_percent": 0.007153011865058431, "reads_retained_percent": 99.99284698813494}, {"sample": "GCA_016026895.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Neisseria meningitidis strain FDAARGOS_914 chromosome, complete genome", "reads_in": 145422, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016728725.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Corynebacterium amycolatum strain FDAARGOS_1108 chromosome, complete genome", "reads_in": 165074, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016728645.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Corynebacterium glucuronolyticum strain FDAARGOS_1111 chromosome, complete genome", "reads_in": 196126, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016026855.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Bacillus tropicus strain FDAARGOS_920 chromosome, complete genome", "reads_in": 380104, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889225.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Corynebacterium glucuronolyticum strain FDAARGOS_1190 chromosome, complete genome", "reads_in": 188214, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889745.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Bacteroides caccae strain FDAARGOS_1219 chromosome, complete genome", "reads_in": 304730, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127675.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 16, "name": "Campylobacter coli strain FDAARGOS_1028 chromosome, complete genome", "reads_in": 125030, "reads_removed_percent": 0.012796928737103096, "reads_retained_percent": 99.9872030712629}, {"sample": "GCA_016026835.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Bacillus licheniformis strain FDAARGOS_923 chromosome, complete genome", "reads_in": 312892, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001019395.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 16, "name": "Staphylococcus aureus strain FDAARGOS_20 chromosome, complete genome", "reads_in": 187888, "reads_removed_percent": 0.008515711487694796, "reads_retained_percent": 99.99148428851231}, {"sample": "GCA_001558395.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Bordetella pertussis strain FDAARGOS_179 chromosome, complete genome", "reads_in": 273390, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889765.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Corynebacterium minutissimum strain FDAARGOS_1196 chromosome, complete genome", "reads_in": 181688, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889785.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Corynebacterium glucuronolyticum strain FDAARGOS_1191 chromosome, complete genome", "reads_in": 192446, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016126915.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Corynebacterium minutissimum strain FDAARGOS_992 chromosome, complete genome", "reads_in": 179734, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889805.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Rhodanobacter sp. FDAARGOS 1247 strain FDAARGOS_1247 chromosome, complete genome", "reads_in": 261508, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889145.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 26, "name": "Staphylococcus aureus strain FDAARGOS_1246 chromosome, complete genome", "reads_in": 192792, "reads_removed_percent": 0.013486036765010997, "reads_retained_percent": 99.98651396323498}, {"sample": "GCA_016127855.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Actinomyces naeslundii strain FDAARGOS_1037 chromosome, complete genome", "reads_in": 210142, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127835.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Salmonella enterica strain FDAARGOS_1066 chromosome, complete genome", "reads_in": 315766, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127255.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 26, "name": "Leuconostoc pseudomesenteroides strain FDAARGOS_1003 chromosome, complete genome", "reads_in": 140978, "reads_removed_percent": 0.018442593879896156, "reads_retained_percent": 99.9815574061201}, {"sample": "GCA_001558375.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Staphylococcus saprophyticus strain FDAARGOS_137 chromosome, complete genome", "reads_in": 171946, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127815.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Pediococcus acidilactici strain FDAARGOS_1007 chromosome, complete genome", "reads_in": 137886, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002073355.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Streptococcus intermedius strain FDAARGOS_233 chromosome, complete genome", "reads_in": 127600, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889925.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Bacteroides fragilis strain FDAARGOS_1225 chromosome, complete genome", "reads_in": 349930, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016027035.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Achromobacter xylosoxidans strain FDAARGOS_915 chromosome, complete genome", "reads_in": 454194, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016728625.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Corynebacterium propinquum strain FDAARGOS_1113 chromosome, complete genome", "reads_in": 164086, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001558435.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Vibrio harveyi strain FDAARGOS_107 chromosome 1, complete sequence", "reads_in": 402594, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812965.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 24, "name": "Burkholderia multivorans strain FDAARGOS_496 chromosome 1, complete sequence", "reads_in": 519760, "reads_removed_percent": 0.0046175157765122364, "reads_retained_percent": 99.99538248422348}, {"sample": "GCA_016888985.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Chromobacterium violaceum strain FDAARGOS_1274 chromosome, complete genome", "reads_in": 320536, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016728805.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Acinetobacter johnsonii strain FDAARGOS_1092 chromosome, complete genome", "reads_in": 275708, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889025.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1251 chromosome, complete genome", "reads_in": 349158, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889065.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Butyricimonas virosa strain FDAARGOS_1229 chromosome, complete genome", "reads_in": 320876, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016027575.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Bacillus tropicus strain FDAARGOS_897 chromosome, complete genome", "reads_in": 381134, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889125.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1249 chromosome, complete genome", "reads_in": 348824, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016027075.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Delftia acidovorans strain FDAARGOS_909 chromosome, complete genome", "reads_in": 447236, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020097395.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 16, "name": "Leclercia adecarboxylata strain FDAARGOS_1505 chromosome, complete genome", "reads_in": 337142, "reads_removed_percent": 0.004745774777393502, "reads_retained_percent": 99.99525422522261}, {"sample": "GCA_001558415.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Vibrio fluvialis strain ATCC 33809 chromosome 1, complete sequence", "reads_in": 321850, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889165.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Staphylococcus epidermidis strain FDAARGOS_1243 chromosome, complete genome", "reads_in": 168996, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016027095.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Sphingomonas paucimobilis strain FDAARGOS_908 chromosome, complete genome", "reads_in": 291138, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889185.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1257 chromosome, complete genome", "reads_in": 354122, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016027415.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Brevibacterium casei strain FDAARGOS_902 chromosome, complete genome", "reads_in": 264006, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016724785.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 24, "name": "Fusobacterium canifelinum strain FDAARGOS_1126 chromosome, complete genome", "reads_in": 156814, "reads_removed_percent": 0.015304755952912367, "reads_retained_percent": 99.98469524404709}, {"sample": "GCA_016728765.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Chryseobacterium sp. FDAARGOS 1104 strain FDAARGOS_1104 chromosome, complete genome", "reads_in": 246632, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016728205.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Corynebacterium striatum strain FDAARGOS_1116 chromosome, complete genome", "reads_in": 177712, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001019125.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 24, "name": "Staphylococcus aureus strain FDAARGOS_43 chromosome, complete genome", "reads_in": 188042, "reads_removed_percent": 0.012763106114591421, "reads_retained_percent": 99.98723689388541}, {"sample": "GCA_016728365.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Corynebacterium tuberculostearicum strain FDAARGOS_1117 chromosome, complete genome", "reads_in": 163456, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003050665.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 16, "name": "Helicobacter pylori strain FDAARGOS_298 chromosome, complete genome", "reads_in": 112060, "reads_removed_percent": 0.014278065322148849, "reads_retained_percent": 99.98572193467785}, {"sample": "GCA_016728505.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Corynebacterium riegelii strain FDAARGOS_1114 chromosome, complete genome", "reads_in": 167948, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001558495.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 16, "name": "Vibrio parahaemolyticus strain ATCC 17802 chromosome 1, complete sequence", "reads_in": 343498, "reads_removed_percent": 0.0046579601627957075, "reads_retained_percent": 99.9953420398372}, {"sample": "GCA_002073275.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Neisseria meningitidis strain FDAARGOS_215 chromosome, complete genome", "reads_in": 153722, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002073295.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Pasteurella multocida strain FDAARGOS_216 chromosome, complete genome", "reads_in": 153012, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016027115.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Serratia plymuthica strain FDAARGOS_907 chromosome, complete genome", "reads_in": 364598, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889005.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Clostridium] scindens strain FDAARGOS_1227 chromosome, complete genome", "reads_in": 241274, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016727125.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Enterococcus hirae strain FDAARGOS_1124 chromosome, complete genome", "reads_in": 195226, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001558915.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Achromobacter xylosoxidans strain FDAARGOS_162 chromosome, complete genome", "reads_in": 437716, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002984485.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Oligella urethralis strain FDAARGOS_329 chromosome, complete genome", "reads_in": 167422, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003020025.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Burkholderia thailandensis strain FDAARGOS_237 chromosome 1, complete sequence", "reads_in": 444854, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002209085.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Burkholderia cepacia strain FDAARGOS_345 chromosome 1, complete sequence", "reads_in": 571324, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_008693765.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 56, "name": "Staphylococcus aureus strain FDAARGOS_660 chromosome, complete genome", "reads_in": 191386, "reads_removed_percent": 0.029260238470943535, "reads_retained_percent": 99.97073976152906}, {"sample": "GCA_016026675.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 10, "name": "Clostridium perfringens strain FDAARGOS_931 chromosome, complete genome", "reads_in": 225034, "reads_removed_percent": 0.004443772940977807, "reads_retained_percent": 99.99555622705903}, {"sample": "GCA_003030425.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Enterococcus faecalis strain FDAARGOS_324 chromosome, complete genome", "reads_in": 210658, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001558515.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Vibrio vulnificus strain FDAARGOS_119 chromosome 1, complete sequence", "reads_in": 331920, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002209145.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Jonesia denitrificans strain FDAARGOS_301 chromosome, complete genome", "reads_in": 185920, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127275.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Streptococcus infantarius strain FDAARGOS_1019 chromosome, complete genome", "reads_in": 130582, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016599815.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 58, "name": "Anaerococcus obesiensis strain FDAARGOS_989 chromosome, complete genome", "reads_in": 126308, "reads_removed_percent": 0.04591949836906609, "reads_retained_percent": 99.95408050163094}, {"sample": "GCA_016026995.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 10, "name": "Clostridium perfringens strain FDAARGOS_905 chromosome, complete genome", "reads_in": 223168, "reads_removed_percent": 0.0044809291654717524, "reads_retained_percent": 99.99551907083453}, {"sample": "GCA_003812185.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Klebsiella aerogenes strain FDAARGOS_513 chromosome, complete genome", "reads_in": 351446, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003019945.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Burkholderia vietnamiensis strain FDAARGOS_239 chromosome 1, complete sequence", "reads_in": 462028, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812705.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Chryseobacterium arthrosphaerae strain FDAARGOS_519 chromosome, complete genome", "reads_in": 357458, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812745.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Serratia sp. FDAARGOS_506 chromosome, complete genome", "reads_in": 324182, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812785.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Cutibacterium acnes strain FDAARGOS_503 chromosome, complete genome", "reads_in": 166302, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001019575.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 64, "name": "Staphylococcus aureus strain FDAARGOS_47 chromosome, complete genome", "reads_in": 193428, "reads_removed_percent": 0.03308724693425978, "reads_retained_percent": 99.96691275306574}, {"sample": "GCA_002208845.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Citrobacter braakii strain FDAARGOS_290 chromosome, complete genome", "reads_in": 322318, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812685.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Streptococcus sp. FDAARGOS_520 chromosome, complete genome", "reads_in": 139974, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127535.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Streptococcus salivarius strain FDAARGOS_1045 chromosome, complete genome", "reads_in": 154588, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812845.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Klebsiella sp. FDAARGOS_511 chromosome, complete genome", "reads_in": 419348, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002208825.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Brevundimonas vesicularis strain FDAARGOS_289 chromosome, complete genome", "reads_in": 223036, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812905.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Pseudomonas aeruginosa strain FDAARGOS_501 chromosome, complete genome", "reads_in": 457722, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127495.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Kytococcus sedentarius strain FDAARGOS_1061 chromosome, complete genome", "reads_in": 185572, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812205.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 10, "name": "Acinetobacter sp. FDAARGOS_560 chromosome, complete genome", "reads_in": 277772, "reads_removed_percent": 0.003600074881557536, "reads_retained_percent": 99.99639992511844}, {"sample": "GCA_016127555.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Streptococcus oralis strain FDAARGOS_1020 chromosome, complete genome", "reads_in": 128770, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127455.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Staphylococcus equorum strain FDAARGOS_1068 chromosome, complete genome", "reads_in": 184822, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001018725.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 54, "name": "Staphylococcus aureus strain FDAARGOS_10 chromosome, complete genome", "reads_in": 191592, "reads_removed_percent": 0.02818489289740699, "reads_retained_percent": 99.9718151071026}, {"sample": "GCA_003019925.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Pluralibacter gergoviae strain FDAARGOS_186 chromosome, complete genome", "reads_in": 360538, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812085.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_566 chromosome 1, complete sequence", "reads_in": 377804, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016126995.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1058 chromosome, complete genome", "reads_in": 360064, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002208885.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Stenotrophomonas maltophilia strain FDAARGOS_325 chromosome, complete genome", "reads_in": 323434, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003798345.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Yersinia pestis strain FDAARGOS_602 chromosome, complete genome", "reads_in": 318966, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812145.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Enterobacter roggenkampii strain FDAARGOS_523 chromosome, complete genome", "reads_in": 346032, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003613595.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Burkholderia gladioli pv. gladioli strain FDAARGOS_188 chromosome 1, complete sequence", "reads_in": 592836, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016126975.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Pediococcus pentosaceus strain FDAARGOS_1011 chromosome, complete genome", "reads_in": 119050, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016026695.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 50, "name": "Lactococcus garvieae strain FDAARGOS_929 chromosome, complete genome", "reads_in": 138956, "reads_removed_percent": 0.03598261320130113, "reads_retained_percent": 99.9640173867987}, {"sample": "GCA_002209125.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Nocardia brasiliensis strain FDAARGOS_352 chromosome, complete genome", "reads_in": 595700, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016126955.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Pseudomonas aeruginosa strain FDAARGOS_1041 chromosome, complete genome", "reads_in": 456166, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002983865.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Roseobacter denitrificans strain FDAARGOS_309 chromosome, complete genome", "reads_in": 287254, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016028815.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 10, "name": "Lactococcus lactis strain FDAARGOS_866 chromosome, complete genome", "reads_in": 173712, "reads_removed_percent": 0.005756654692824906, "reads_retained_percent": 99.99424334530717}, {"sample": "GCA_016126935.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Stenotrophomonas maltophilia strain FDAARGOS_1044 chromosome, complete genome", "reads_in": 333618, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002984035.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Salmonella enterica subsp. enterica serovar Heidelberg strain FDAARGOS_319 chromosome, complete genome", "reads_in": 333918, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001018805.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 10, "name": "Staphylococcus aureus strain FDAARGOS_15 chromosome, complete genome", "reads_in": 186660, "reads_removed_percent": 0.005357334190506804, "reads_retained_percent": 99.9946426658095}, {"sample": "GCA_002984125.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Moraxella catarrhalis strain FDAARGOS_304 chromosome, complete genome", "reads_in": 129466, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812365.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 10, "name": "Burkholderia multivorans strain FDAARGOS_547 chromosome 1, complete sequence", "reads_in": 426274, "reads_removed_percent": 0.002345908969348353, "reads_retained_percent": 99.99765409103065}, {"sample": "GCA_020736025.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 10, "name": "Haemophilus influenzae strain FDAARGOS_1561 chromosome, complete genome", "reads_in": 122286, "reads_removed_percent": 0.008177550987030404, "reads_retained_percent": 99.99182244901297}, {"sample": "GCA_016127195.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Aeromonas jandaei strain FDAARGOS_986 chromosome, complete genome", "reads_in": 303406, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127235.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Enterococcus hirae strain FDAARGOS_1057 chromosome, complete genome", "reads_in": 188792, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002984395.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Klebsiella oxytoca strain FDAARGOS_335 chromosome, complete genome", "reads_in": 403320, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001559235.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Citrobacter sp. FDAARGOS_156 strain FDAARGOS_155 chromosome, complete genome", "reads_in": 315166, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002984565.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Staphylococcus cohnii strain FDAARGOS_334 chromosome, complete genome", "reads_in": 177418, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002073615.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 10, "name": "Neisseria gonorrhoeae strain FDAARGOS_205 chromosome, complete genome", "reads_in": 151930, "reads_removed_percent": 0.0065819785427499505, "reads_retained_percent": 99.99341802145725}, {"sample": "GCA_002208865.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 10, "name": "Escherichia coli O157 strain FDAARGOS_293 chromosome, complete genome", "reads_in": 379110, "reads_removed_percent": 0.002637756851573422, "reads_retained_percent": 99.99736224314843}, {"sample": "GCA_003812925.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Klebsiella oxytoca strain FDAARGOS_500 chromosome, complete genome", "reads_in": 391940, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812945.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_497 chromosome, complete genome", "reads_in": 367552, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002208805.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Staphylococcus pettenkoferi strain FDAARGOS_288 chromosome, complete genome", "reads_in": 166824, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812445.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Streptococcus sp. FDAARGOS_522 chromosome, complete genome", "reads_in": 140076, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003813065.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Bifidobacterium breve strain FDAARGOS_561 chromosome, complete genome", "reads_in": 151710, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002983665.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Providencia stuartii strain FDAARGOS_291 chromosome, complete genome", "reads_in": 292966, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001559215.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Klebsiella aerogenes strain FDAARGOS_152 chromosome, complete genome", "reads_in": 354476, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002983645.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Escherichia coli O104:H4 strain FDAARGOS_349 chromosome, complete genome", "reads_in": 366744, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735665.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 108, "name": "Lachnospiraceae bacterium GAM79 strain FDAARGOS_1585 chromosome, complete genome", "reads_in": 197388, "reads_removed_percent": 0.054714572314426405, "reads_retained_percent": 99.94528542768558}, {"sample": "GCA_016127295.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Leuconostoc mesenteroides strain FDAARGOS_1033 chromosome, complete genome", "reads_in": 138384, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812825.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 8, "name": "Fusobacterium necrophorum strain FDAARGOS_565 chromosome, complete genome", "reads_in": 178560, "reads_removed_percent": 0.004480286738351254, "reads_retained_percent": 99.99551971326164}, {"sample": "GCA_002393405.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Lelliottia amnigena strain FDAARGOS_395 chromosome, complete genome", "reads_in": 297974, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020097215.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 134, "name": "Clostridioides difficile strain FDAARGOS_1528 chromosome, complete genome", "reads_in": 291690, "reads_removed_percent": 0.04593918200829648, "reads_retained_percent": 99.9540608179917}, {"sample": "GCA_002393425.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Burkholderia gladioli pv. gladioli strain FDAARGOS_389 chromosome 1, complete sequence", "reads_in": 552478, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735685.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 100, "name": "Lachnospiraceae bacterium GAM79 strain FDAARGOS_1582 chromosome, complete genome", "reads_in": 198604, "reads_removed_percent": 0.0503514531429377, "reads_retained_percent": 99.94964854685706}, {"sample": "GCA_002393445.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Cedecea neteri strain FDAARGOS_392 chromosome, complete genome", "reads_in": 364620, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002393485.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Ralstonia pickettii strain FDAARGOS_410 chromosome 1, complete sequence", "reads_in": 321204, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002393385.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Pasteurella multocida strain FDAARGOS_384 chromosome, complete genome", "reads_in": 156448, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002393505.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Providencia alcalifaciens strain FDAARGOS_408 chromosome, complete genome", "reads_in": 266008, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002407165.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Staphylococcus lugdunensis strain FDAARGOS_377 chromosome, complete genome", "reads_in": 171442, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002407525.3", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Burkholderia cepacia strain FDAARGOS_388 chromosome 1, complete sequence", "reads_in": 571322, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002073735.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 274, "name": "Clostridioides difficile strain FDAARGOS_267 chromosome, complete genome", "reads_in": 285744, "reads_removed_percent": 0.09589002743714654, "reads_retained_percent": 99.90410997256285}, {"sample": "GCA_002209245.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Shewanella sp. FDAARGOS_354 chromosome, complete genome", "reads_in": 338000, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002587165.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_444 chromosome, complete genome", "reads_in": 392066, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019930725.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 2, "name": "Capnocytophaga ochracea strain FDAARGOS_1468 chromosome, complete genome", "reads_in": 174364, "reads_removed_percent": 0.001147025762198619, "reads_retained_percent": 99.9988529742378}, {"sample": "GCA_002588845.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Serratia fonticola strain FDAARGOS_411 chromosome, complete genome", "reads_in": 381692, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002589795.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_433 chromosome, complete genome", "reads_in": 341458, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016026755.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 184, "name": "Lysinibacillus macroides strain FDAARGOS_925 chromosome, complete genome", "reads_in": 324176, "reads_removed_percent": 0.056759291249197964, "reads_retained_percent": 99.9432407087508}, {"sample": "GCA_003812605.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Chryseobacterium indologenes strain FDAARGOS_537 chromosome, complete genome", "reads_in": 324424, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812585.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Burkholderia multivorans strain FDAARGOS_548 chromosome 1, complete sequence", "reads_in": 443592, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003798265.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 8, "name": "Burkholderia pseudomallei strain FDAARGOS_592 chromosome 1, complete sequence", "reads_in": 476446, "reads_removed_percent": 0.0016790989954790258, "reads_retained_percent": 99.99832090100452}, {"sample": "GCA_003812985.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Stenotrophomonas maltophilia strain FDAARGOS_507 chromosome, complete genome", "reads_in": 305172, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003813005.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Pseudomonas aeruginosa strain FDAARGOS_505 chromosome, complete genome", "reads_in": 468654, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812165.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Pseudomonas aeruginosa strain FDAARGOS_532 chromosome, complete genome", "reads_in": 462084, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812645.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Streptococcus pyogenes strain FDAARGOS_534 chromosome, complete genome", "reads_in": 122516, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019930945.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 78, "name": "Campylobacter hominis strain FDAARGOS_1467 chromosome, complete genome", "reads_in": 114540, "reads_removed_percent": 0.0680984808800419, "reads_retained_percent": 99.93190151911996}, {"sample": "GCA_003812245.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Acinetobacter sp. FDAARGOS_493 chromosome, complete genome", "reads_in": 268098, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903835.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 10, "name": "Escherichia coli strain FDAARGOS_1299 chromosome, complete genome", "reads_in": 344708, "reads_removed_percent": 0.002901006068904696, "reads_retained_percent": 99.99709899393109}, {"sample": "GCA_003812305.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Enterococcus sp. FDAARGOS_553 chromosome, complete genome", "reads_in": 218900, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812325.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Citrobacter freundii strain FDAARGOS_550 chromosome, complete genome", "reads_in": 331276, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812345.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Citrobacter freundii strain FDAARGOS_549 chromosome, complete genome", "reads_in": 340144, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003798365.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 8, "name": "Burkholderia pseudomallei strain FDAARGOS_594 chromosome 1, complete sequence", "reads_in": 478692, "reads_removed_percent": 0.0016712207431918646, "reads_retained_percent": 99.99832877925681}, {"sample": "GCA_003812405.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Citrobacter koseri strain FDAARGOS_530 chromosome, complete genome", "reads_in": 305400, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003019965.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 8, "name": "Burkholderia multivorans strain FDAARGOS_246 chromosome 1, complete sequence", "reads_in": 421526, "reads_removed_percent": 0.001897866323785484, "reads_retained_percent": 99.99810213367621}, {"sample": "GCA_016127395.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Corynebacterium glucuronolyticum strain FDAARGOS_1053 chromosome, complete genome", "reads_in": 194320, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812425.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Staphylococcus epidermidis strain FDAARGOS_529 chromosome, complete genome", "reads_in": 171828, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812665.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Enterococcus faecalis strain FDAARGOS_528 chromosome, complete genome", "reads_in": 197576, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016726485.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 8, "name": "Peptoniphilus harei strain FDAARGOS_1136 chromosome, complete genome", "reads_in": 128638, "reads_removed_percent": 0.006219002161103252, "reads_retained_percent": 99.9937809978389}, {"sample": "GCA_016127375.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Lactococcus garvieae strain FDAARGOS_1002 chromosome, complete genome", "reads_in": 152810, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812465.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Streptococcus sp. FDAARGOS_521 chromosome, complete genome", "reads_in": 143800, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812545.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Listeria monocytogenes strain FDAARGOS_555 chromosome, complete genome", "reads_in": 190138, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002983725.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 84, "name": "Campylobacter jejuni subsp. doylei strain FDAARGOS_295 chromosome, complete genome", "reads_in": 123004, "reads_removed_percent": 0.06829046209879354, "reads_retained_percent": 99.93170953790121}, {"sample": "GCA_020735625.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 8, "name": "Finegoldia magna strain FDAARGOS_1557 chromosome, complete genome", "reads_in": 132318, "reads_removed_percent": 0.006046040599162624, "reads_retained_percent": 99.99395395940084}, {"sample": "GCA_016127355.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Kingella denitrificans strain FDAARGOS_1060 chromosome, complete genome", "reads_in": 148004, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001559195.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Achromobacter xylosoxidans strain FDAARGOS_150 chromosome, complete genome", "reads_in": 417916, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812565.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Listeria monocytogenes strain FDAARGOS_554 chromosome, complete genome", "reads_in": 189124, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127595.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 46, "name": "Fusobacterium canifelinum strain FDAARGOS_999 chromosome, complete genome", "reads_in": 161678, "reads_removed_percent": 0.028451613701307537, "reads_retained_percent": 99.97154838629869}, {"sample": "GCA_016127015.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Corynebacterium tuberculostearicum strain FDAARGOS_993 chromosome, complete genome", "reads_in": 160572, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016028515.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Stutzerimonas frequens strain FDAARGOS_877 chromosome, complete genome", "reads_in": 292980, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003798205.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Yersinia pestis strain FDAARGOS_603 chromosome, complete genome", "reads_in": 328642, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009730395.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 10, "name": "Campylobacter coli strain FDAARGOS_735 chromosome, complete genome", "reads_in": 114436, "reads_removed_percent": 0.008738508860847986, "reads_retained_percent": 99.99126149113916}, {"sample": "GCA_016128075.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 10, "name": "Corynebacterium bovis strain FDAARGOS_1052 chromosome, complete genome", "reads_in": 176814, "reads_removed_percent": 0.005655660750845522, "reads_retained_percent": 99.99434433924915}, {"sample": "GCA_016726345.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Acinetobacter ursingii strain FDAARGOS_1096 chromosome, complete genome", "reads_in": 232482, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003711125.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Enterococcus avium strain FDAARGOS_184 chromosome, complete genome", "reads_in": 248226, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002208985.3", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Citrobacter koseri strain FDAARGOS_287 chromosome, complete genome", "reads_in": 328978, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016727005.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Myroides odoratus strain FDAARGOS_1132 chromosome, complete genome", "reads_in": 277848, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016726325.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Serratia plymuthica strain FDAARGOS_1138 chromosome, complete genome", "reads_in": 365626, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127095.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Corynebacterium urealyticum strain FDAARGOS_996 chromosome, complete genome", "reads_in": 156730, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016726305.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Bacteroides caccae strain FDAARGOS_1097 chromosome, complete genome", "reads_in": 304730, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016725465.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 10, "name": "Sphingobacterium multivorum strain FDAARGOS_1142 chromosome, complete genome", "reads_in": 436790, "reads_removed_percent": 0.0022894297030609675, "reads_retained_percent": 99.99771057029695}, {"sample": "GCA_016127055.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Staphylococcus felis strain FDAARGOS_1014 chromosome, complete genome", "reads_in": 165302, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003798225.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Yersinia pestis strain FDAARGOS_601 chromosome, complete genome", "reads_in": 328732, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002208965.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Salmonella enterica subsp. enterica serovar Typhimurium var. 5- strain FDAARGOS_312 chromosome, complete genome", "reads_in": 347590, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002073595.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 12, "name": "Neisseria meningitidis strain FDAARGOS_210 chromosome, complete genome", "reads_in": 151552, "reads_removed_percent": 0.007918074324324325, "reads_retained_percent": 99.99208192567568}, {"sample": "GCA_016726285.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Shigella boydii strain FDAARGOS_1139 chromosome, complete genome", "reads_in": 303264, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889905.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 38, "name": "Escherichia coli strain FDAARGOS_1259 chromosome, complete genome", "reads_in": 348288, "reads_removed_percent": 0.010910510841602352, "reads_retained_percent": 99.9890894891584}, {"sample": "GCA_003798325.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Propionibacterium acidifaciens strain FDAARGOS_576 chromosome, complete genome", "reads_in": 204852, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019930905.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 42, "name": "Haemophilus aegyptius strain FDAARGOS_1478 chromosome, complete genome", "reads_in": 132884, "reads_removed_percent": 0.0316065139520183, "reads_retained_percent": 99.96839348604799}, {"sample": "GCA_006364775.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 42, "name": "Staphylococcus aureus strain FDAARGOS_766 chromosome, complete genome", "reads_in": 199754, "reads_removed_percent": 0.021025861810026334, "reads_retained_percent": 99.97897413818997}, {"sample": "GCA_016726945.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Moraxella osloensis strain FDAARGOS_1130 chromosome, complete genome", "reads_in": 169780, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002073775.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 10, "name": "Vibrio parahaemolyticus strain FDAARGOS_191 chromosome 1, complete sequence", "reads_in": 344886, "reads_removed_percent": 0.002899508823205349, "reads_retained_percent": 99.9971004911768}, {"sample": "GCA_016127115.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Aggregatibacter segnis strain FDAARGOS_987 chromosome, complete genome", "reads_in": 137500, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003798465.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Cutibacterium acnes strain FDAARGOS_577 chromosome, complete genome", "reads_in": 166356, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020736045.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 40, "name": "Haemophilus influenzae strain FDAARGOS_1560 chromosome, complete genome", "reads_in": 126044, "reads_removed_percent": 0.03173494970010473, "reads_retained_percent": 99.96826505029989}, {"sample": "GCA_016726965.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Mobiluncus curtisii strain FDAARGOS_1128 chromosome, complete genome", "reads_in": 144282, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002983685.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 40, "name": "Streptobacillus moniliformis strain FDAARGOS_310 chromosome, complete genome", "reads_in": 112396, "reads_removed_percent": 0.03558845510516388, "reads_retained_percent": 99.96441154489483}, {"sample": "GCA_016726765.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Pediococcus acidilactici strain FDAARGOS_1133 chromosome, complete genome", "reads_in": 130226, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016726625.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Acinetobacter johnsonii strain FDAARGOS_1094 chromosome, complete genome", "reads_in": 234960, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002209225.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 10, "name": "Salmonella enterica subsp. enterica serovar Typhimurium strain FDAARGOS_321 chromosome, complete genome", "reads_in": 334714, "reads_removed_percent": 0.0029876252561888655, "reads_retained_percent": 99.99701237474382}, {"sample": "GCA_016726365.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Rothia aeria strain FDAARGOS_1137 chromosome, complete genome", "reads_in": 172262, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019931025.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 38, "name": "Clostridioides difficile strain FDAARGOS_1469 chromosome, complete genome", "reads_in": 269708, "reads_removed_percent": 0.014089311403443725, "reads_retained_percent": 99.98591068859656}, {"sample": "GCA_016026035.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Aggregatibacter actinomycetemcomitans strain FDAARGOS_948 chromosome, complete genome", "reads_in": 140358, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009806695.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 10, "name": "Bacillus anthracis strain FDAARGOS_704 chromosome, complete genome", "reads_in": 348496, "reads_removed_percent": 0.002869473394242689, "reads_retained_percent": 99.99713052660576}, {"sample": "GCA_016903915.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 10, "name": "Escherichia coli strain FDAARGOS_1300 chromosome, complete genome", "reads_in": 363838, "reads_removed_percent": 0.0027484759700745934, "reads_retained_percent": 99.99725152402992}, {"sample": "GCA_016025875.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Prescottella equi strain FDAARGOS_952 chromosome, complete genome", "reads_in": 347914, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002209045.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 12, "name": "Campylobacter jejuni strain FDAARGOS_265 chromosome, complete genome", "reads_in": 113074, "reads_removed_percent": 0.010612519235191114, "reads_retained_percent": 99.9893874807648}, {"sample": "GCA_016726885.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Pediococcus pentosaceus strain FDAARGOS_1134 chromosome, complete genome", "reads_in": 116092, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002208905.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Staphylococcus saprophyticus strain FDAARGOS_336 chromosome, complete genome", "reads_in": 175718, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002073515.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 12, "name": "Streptococcus equinus strain FDAARGOS_251 chromosome, complete genome", "reads_in": 154102, "reads_removed_percent": 0.007787050135624456, "reads_retained_percent": 99.99221294986438}, {"sample": "GCA_002208925.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Chryseobacterium indologenes strain FDAARGOS_337 chromosome, complete genome", "reads_in": 341292, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127155.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Brevibacterium casei strain FDAARGOS_990 chromosome, complete genome", "reads_in": 256754, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127035.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Leuconostoc pseudomesenteroides strain FDAARGOS_1004 chromosome, complete genome", "reads_in": 139694, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016726245.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Brevibacterium casei strain FDAARGOS_1100 chromosome, complete genome", "reads_in": 256458, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003614815.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Burkholderia thailandensis strain FDAARGOS_241 chromosome 1, complete sequence", "reads_in": 445608, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127575.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Cupriavidus necator strain FDAARGOS_1030 chromosome 1, complete sequence", "reads_in": 494398, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016726265.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 10, "name": "Campylobacter lari strain FDAARGOS_1101 chromosome, complete genome", "reads_in": 100892, "reads_removed_percent": 0.009911588629425525, "reads_retained_percent": 99.99008841137058}, {"sample": "GCA_020097435.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 10, "name": "Gemella morbillorum strain FDAARGOS_1501 chromosome, complete genome", "reads_in": 117396, "reads_removed_percent": 0.008518177791406862, "reads_retained_percent": 99.9914818222086}, {"sample": "GCA_003798425.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Yersinia pseudotuberculosis strain FDAARGOS_581 chromosome, complete genome", "reads_in": 339798, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001019495.2", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 38, "name": "Staphylococcus aureus strain FDAARGOS_40 chromosome, complete genome", "reads_in": 192622, "reads_removed_percent": 0.019727756954034326, "reads_retained_percent": 99.98027224304596}, {"sample": "GCA_016726205.1", "platform": "illumina", "tool": "kraken2-standard", "reads_removed": 0, "name": "Acinetobacter ursingii strain FDAARGOS_1095 chromosome, complete genome", "reads_in": 240834, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016726625.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Acinetobacter johnsonii strain FDAARGOS_1094 chromosome, complete genome", "reads_in": 234960, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001019535.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 12, "name": "Staphylococcus aureus strain FDAARGOS_42 chromosome, complete genome", "reads_in": 187956, "reads_removed_percent": 0.006384472961757006, "reads_retained_percent": 99.99361552703824}, {"sample": "GCA_016727265.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 10, "name": "Enterococcus hirae strain FDAARGOS_1125 chromosome, complete genome", "reads_in": 204300, "reads_removed_percent": 0.004894762604013705, "reads_retained_percent": 99.99510523739599}, {"sample": "GCA_003019945.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 10, "name": "Burkholderia vietnamiensis strain FDAARGOS_239 chromosome 1, complete sequence", "reads_in": 462028, "reads_removed_percent": 0.0021643709905027403, "reads_retained_percent": 99.9978356290095}, {"sample": "GCA_003812365.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 10, "name": "Burkholderia multivorans strain FDAARGOS_547 chromosome 1, complete sequence", "reads_in": 426274, "reads_removed_percent": 0.002345908969348353, "reads_retained_percent": 99.99765409103065}, {"sample": "GCA_016127635.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 10, "name": "Enterococcus casseliflavus strain FDAARGOS_998 chromosome, complete genome", "reads_in": 238722, "reads_removed_percent": 0.0041889729476127045, "reads_retained_percent": 99.99581102705238}, {"sample": "GCA_002073315.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 12, "name": "Yersinia sp. FDAARGOS_228 chromosome, complete genome", "reads_in": 330870, "reads_removed_percent": 0.0036268020672771783, "reads_retained_percent": 99.99637319793273}, {"sample": "GCA_001019255.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 12, "name": "Staphylococcus aureus strain FDAARGOS_1 chromosome, complete genome", "reads_in": 190366, "reads_removed_percent": 0.006303646659592574, "reads_retained_percent": 99.99369635334041}, {"sample": "GCA_002208965.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Salmonella enterica subsp. enterica serovar Typhimurium var. 5- strain FDAARGOS_312 chromosome, complete genome", "reads_in": 347590, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889005.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 10, "name": "Clostridium] scindens strain FDAARGOS_1227 chromosome, complete genome", "reads_in": 241274, "reads_removed_percent": 0.004144665401162164, "reads_retained_percent": 99.99585533459884}, {"sample": "GCA_001018915.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 12, "name": "Staphylococcus aureus strain FDAARGOS_27 chromosome, complete genome", "reads_in": 182690, "reads_removed_percent": 0.006568504023208715, "reads_retained_percent": 99.9934314959768}, {"sample": "GCA_016889385.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 10, "name": "Haemophilus parahaemolyticus strain FDAARGOS_1199 chromosome, complete genome", "reads_in": 139564, "reads_removed_percent": 0.007165171534206529, "reads_retained_percent": 99.99283482846579}, {"sample": "GCA_016904495.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 10, "name": "Escherichia coli strain FDAARGOS_1281 chromosome, complete genome", "reads_in": 372684, "reads_removed_percent": 0.00268323834669586, "reads_retained_percent": 99.99731676165331}, {"sample": "GCA_002591175.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 10, "name": "Staphylococcus capitis strain FDAARGOS_378 chromosome, complete genome", "reads_in": 165676, "reads_removed_percent": 0.006035877254400154, "reads_retained_percent": 99.9939641227456}, {"sample": "GCA_002073795.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 10, "name": "Streptococcus sp. 'group B' strain FDAARGOS_229 chromosome, complete genome", "reads_in": 145218, "reads_removed_percent": 0.006886198680604332, "reads_retained_percent": 99.9931138013194}, {"sample": "GCA_002208985.3", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Citrobacter koseri strain FDAARGOS_287 chromosome, complete genome", "reads_in": 328978, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003956005.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 10, "name": "Staphylococcus haemolyticus strain FDAARGOS_517 chromosome, complete genome", "reads_in": 177968, "reads_removed_percent": 0.005618987683178998, "reads_retained_percent": 99.99438101231682}, {"sample": "GCA_016726245.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Brevibacterium casei strain FDAARGOS_1100 chromosome, complete genome", "reads_in": 256458, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019930765.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 10, "name": "Legionella pneumophila strain FDAARGOS_1483 chromosome, complete genome", "reads_in": 227276, "reads_removed_percent": 0.004399936640912371, "reads_retained_percent": 99.99560006335909}, {"sample": "GCA_001019575.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 12, "name": "Staphylococcus aureus strain FDAARGOS_47 chromosome, complete genome", "reads_in": 193428, "reads_removed_percent": 0.006203858800173708, "reads_retained_percent": 99.99379614119982}, {"sample": "GCA_020735905.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 10, "name": "Prevotella melaninogenica strain FDAARGOS_1567 chromosome 1, complete sequence", "reads_in": 220450, "reads_removed_percent": 0.00453617600362894, "reads_retained_percent": 99.99546382399637}, {"sample": "GCA_000783915.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 12, "name": "Serratia marcescens strain FDAARGOS_65 chromosome, complete genome", "reads_in": 349894, "reads_removed_percent": 0.0034296101104906054, "reads_retained_percent": 99.99657038988951}, {"sample": "GCA_009729915.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 10, "name": "Salmonella enterica strain FDAARGOS_708 chromosome, complete genome", "reads_in": 342448, "reads_removed_percent": 0.0029201513806475727, "reads_retained_percent": 99.99707984861935}, {"sample": "GCA_000783995.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 12, "name": "Citrobacter freundii strain FDAARGOS_61 chromosome, complete genome", "reads_in": 350566, "reads_removed_percent": 0.003423035890531312, "reads_retained_percent": 99.99657696410947}, {"sample": "GCA_002073515.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 12, "name": "Streptococcus equinus strain FDAARGOS_251 chromosome, complete genome", "reads_in": 154102, "reads_removed_percent": 0.007787050135624456, "reads_retained_percent": 99.99221294986438}, {"sample": "GCA_016906265.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 10, "name": "Streptomyces californicus strain FDAARGOS_1210 chromosome", "reads_in": 548102, "reads_removed_percent": 0.0018244779256415776, "reads_retained_percent": 99.99817552207436}, {"sample": "GCA_003812505.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 10, "name": "Staphylococcus hominis strain FDAARGOS_575 chromosome, complete genome", "reads_in": 150494, "reads_removed_percent": 0.006644783180724813, "reads_retained_percent": 99.99335521681928}, {"sample": "GCA_019930725.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Capnocytophaga ochracea strain FDAARGOS_1468 chromosome, complete genome", "reads_in": 174364, "reads_removed_percent": 0.001147025762198619, "reads_retained_percent": 99.9988529742378}, {"sample": "GCA_001558815.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 10, "name": "Staphylococcus lugdunensis strain FDAARGOS_143 chromosome, complete genome", "reads_in": 174258, "reads_removed_percent": 0.005738617452283396, "reads_retained_percent": 99.99426138254772}, {"sample": "GCA_016127415.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Delftia acidovorans strain FDAARGOS_997 chromosome, complete genome", "reads_in": 453112, "reads_removed_percent": 0.0017655678949133988, "reads_retained_percent": 99.9982344321051}, {"sample": "GCA_016127255.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Leuconostoc pseudomesenteroides strain FDAARGOS_1003 chromosome, complete genome", "reads_in": 140978, "reads_removed_percent": 0.0056746442707372785, "reads_retained_percent": 99.99432535572926}, {"sample": "GCA_013267555.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Bacteroides fragilis strain FDAARGOS_763 chromosome, complete genome", "reads_in": 373716, "reads_removed_percent": 0.00214066296331974, "reads_retained_percent": 99.99785933703669}, {"sample": "GCA_002587165.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_444 chromosome, complete genome", "reads_in": 392066, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013267595.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Prevotella melaninogenica strain FDAARGOS_760 chromosome 1, complete sequence", "reads_in": 223532, "reads_removed_percent": 0.0035789059284576704, "reads_retained_percent": 99.99642109407154}, {"sample": "GCA_001188025.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Listeria monocytogenes strain FDAARGOS_58 chromosome, complete genome", "reads_in": 193778, "reads_removed_percent": 0.004128435632527945, "reads_retained_percent": 99.99587156436748}, {"sample": "GCA_001019305.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Staphylococcus aureus strain FDAARGOS_9 chromosome, complete genome", "reads_in": 187646, "reads_removed_percent": 0.004263346940515652, "reads_retained_percent": 99.99573665305948}, {"sample": "GCA_016127135.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Lactococcus lactis strain FDAARGOS_1064 chromosome, complete genome", "reads_in": 167518, "reads_removed_percent": 0.004775606203512459, "reads_retained_percent": 99.99522439379649}, {"sample": "GCA_002073475.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Haemophilus influenzae strain FDAARGOS_199 chromosome, complete genome", "reads_in": 122046, "reads_removed_percent": 0.006554905527424087, "reads_retained_percent": 99.99344509447258}, {"sample": "GCA_016127075.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Macrococcus caseolyticus strain FDAARGOS_1005 chromosome, complete genome", "reads_in": 144448, "reads_removed_percent": 0.005538325210456358, "reads_retained_percent": 99.99446167478955}, {"sample": "GCA_019048445.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Escherichia coli strain FDAARGOS 1382 chromosome, complete genome", "reads_in": 337060, "reads_removed_percent": 0.0023734646650447992, "reads_retained_percent": 99.99762653533496}, {"sample": "GCA_016126995.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Escherichia coli strain FDAARGOS_1058 chromosome, complete genome", "reads_in": 360064, "reads_removed_percent": 0.0022218272307145395, "reads_retained_percent": 99.99777817276929}, {"sample": "GCA_002393445.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Cedecea neteri strain FDAARGOS_392 chromosome, complete genome", "reads_in": 364620, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002209245.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Shewanella sp. FDAARGOS_354 chromosome, complete genome", "reads_in": 338000, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003955775.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Francisella tularensis strain FDAARGOS_599 chromosome, complete genome", "reads_in": 126190, "reads_removed_percent": 0.006339646564704017, "reads_retained_percent": 99.9936603534353}, {"sample": "GCA_016903575.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Staphylococcus epidermidis strain FDAARGOS_1364 chromosome, complete genome", "reads_in": 163244, "reads_removed_percent": 0.004900639533459116, "reads_retained_percent": 99.99509936046654}, {"sample": "GCA_002393385.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Pasteurella multocida strain FDAARGOS_384 chromosome, complete genome", "reads_in": 156448, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016727325.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Enterococcus casseliflavus strain FDAARGOS_1121 chromosome, complete genome", "reads_in": 243364, "reads_removed_percent": 0.003287256948439375, "reads_retained_percent": 99.99671274305156}, {"sample": "GCA_016727445.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Morganella morganii strain FDAARGOS_1085 chromosome, complete genome", "reads_in": 265002, "reads_removed_percent": 0.0030188451407913903, "reads_retained_percent": 99.99698115485921}, {"sample": "GCA_003812745.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Serratia sp. FDAARGOS_506 chromosome, complete genome", "reads_in": 324182, "reads_removed_percent": 0.002467749597448347, "reads_retained_percent": 99.99753225040256}, {"sample": "GCA_020735625.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Finegoldia magna strain FDAARGOS_1557 chromosome, complete genome", "reads_in": 132318, "reads_removed_percent": 0.006046040599162624, "reads_retained_percent": 99.99395395940084}, {"sample": "GCA_016903735.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Klebsiella pneumoniae strain FDAARGOS_1325 chromosome, complete genome", "reads_in": 366224, "reads_removed_percent": 0.0021844554152649744, "reads_retained_percent": 99.99781554458474}, {"sample": "GCA_003812065.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Acinetobacter baumannii strain FDAARGOS_540 chromosome, complete genome", "reads_in": 260684, "reads_removed_percent": 0.0030688496417118046, "reads_retained_percent": 99.9969311503583}, {"sample": "GCA_019048725.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Aeromonas sp. FDAARGOS 1407 chromosome, complete genome", "reads_in": 313436, "reads_removed_percent": 0.0025523551857476487, "reads_retained_percent": 99.99744764481426}, {"sample": "GCA_019047145.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Escherichia coli strain FDAARGOS_1379 chromosome, complete genome", "reads_in": 335034, "reads_removed_percent": 0.002387817355850451, "reads_retained_percent": 99.99761218264415}, {"sample": "GCA_019048405.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Acinetobacter seifertii strain FDAARGOS 1400 chromosome, complete genome", "reads_in": 266604, "reads_removed_percent": 0.0030007051657139428, "reads_retained_percent": 99.99699929483428}, {"sample": "GCA_016904135.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Staphylococcus epidermidis strain FDAARGOS_1361 chromosome, complete genome", "reads_in": 168278, "reads_removed_percent": 0.0047540379609931184, "reads_retained_percent": 99.99524596203901}, {"sample": "GCA_020099175.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Klebsiella quasipneumoniae strain FDAARGOS_1503 chromosome, complete genome", "reads_in": 366590, "reads_removed_percent": 0.002182274475572165, "reads_retained_percent": 99.99781772552443}, {"sample": "GCA_000783955.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Morganella morganii strain FDAARGOS_63 chromosome, complete genome", "reads_in": 266668, "reads_removed_percent": 0.0029999850000749996, "reads_retained_percent": 99.99700001499993}, {"sample": "GCA_002951475.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Pseudomonas fulva strain FDAARGOS_167 chromosome, complete genome", "reads_in": 324340, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020097415.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Klebsiella quasipneumoniae strain FDAARGOS_1504 chromosome, complete genome", "reads_in": 344172, "reads_removed_percent": 0.0023244191857559595, "reads_retained_percent": 99.99767558081425}, {"sample": "GCA_001558855.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Providencia stuartii strain FDAARGOS_145 chromosome, complete genome", "reads_in": 287008, "reads_removed_percent": 0.0027873787490244177, "reads_retained_percent": 99.99721262125098}, {"sample": "GCA_008693405.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Klebsiella pneumoniae strain FDAARGOS_630 chromosome, complete genome", "reads_in": 376186, "reads_removed_percent": 0.0021266075824193354, "reads_retained_percent": 99.99787339241757}, {"sample": "GCA_016724825.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Weizmannia coagulans strain FDAARGOS_1160 chromosome, complete genome", "reads_in": 222856, "reads_removed_percent": 0.0035897619987794813, "reads_retained_percent": 99.99641023800122}, {"sample": "GCA_016724665.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Staphylococcus carnosus strain FDAARGOS_1147 chromosome, complete genome", "reads_in": 179336, "reads_removed_percent": 0.00446090020966231, "reads_retained_percent": 99.99553909979034}, {"sample": "GCA_002386385.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Citrobacter werkmanii strain FDAARGOS_364 chromosome, complete genome", "reads_in": 325674, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_008693585.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Streptococcus agalactiae strain FDAARGOS_670 chromosome, complete genome", "reads_in": 147382, "reads_removed_percent": 0.005428071270575783, "reads_retained_percent": 99.99457192872943}, {"sample": "GCA_016904335.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Escherichia coli strain FDAARGOS_1370 chromosome, complete genome", "reads_in": 356214, "reads_removed_percent": 0.0022458409832291828, "reads_retained_percent": 99.99775415901676}, {"sample": "GCA_000783875.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Proteus mirabilis strain FDAARGOS_67 chromosome, complete genome", "reads_in": 263698, "reads_removed_percent": 0.003033773483302869, "reads_retained_percent": 99.99696622651669}, {"sample": "GCA_002386365.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Rothia mucilaginosa strain FDAARGOS_369 chromosome, complete genome", "reads_in": 160584, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016026215.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Escherichia coli strain FDAARGOS_945 chromosome, complete genome", "reads_in": 371304, "reads_removed_percent": 0.002154568763062073, "reads_retained_percent": 99.99784543123694}, {"sample": "GCA_002386325.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Salmonella enterica subsp. enterica serovar Saintpaul strain FDAARGOS_373 chromosome, complete genome", "reads_in": 322540, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016026555.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Brevibacterium casei strain FDAARGOS_937 chromosome, complete genome", "reads_in": 258770, "reads_removed_percent": 0.0030915484793445917, "reads_retained_percent": 99.99690845152065}, {"sample": "GCA_019048265.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Aeromonas sp. FDAARGOS 1405 chromosome, complete genome", "reads_in": 307614, "reads_removed_percent": 0.0026006618684455193, "reads_retained_percent": 99.99739933813156}, {"sample": "GCA_019048145.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Escherichia coli strain FDAARGOS 1386 chromosome, complete genome", "reads_in": 346162, "reads_removed_percent": 0.002311056672887261, "reads_retained_percent": 99.99768894332712}, {"sample": "GCA_016028295.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Staphylococcus auricularis strain FDAARGOS_882 chromosome, complete genome", "reads_in": 147772, "reads_removed_percent": 0.005413745499824054, "reads_retained_percent": 99.99458625450018}, {"sample": "GCA_016026875.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Aeromonas hydrophila strain FDAARGOS_916 chromosome, complete genome", "reads_in": 315580, "reads_removed_percent": 0.002535014893212498, "reads_retained_percent": 99.99746498510679}, {"sample": "GCA_016027815.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Delftia lacustris strain FDAARGOS_890 chromosome, complete genome", "reads_in": 484968, "reads_removed_percent": 0.0016495933752330052, "reads_retained_percent": 99.99835040662477}, {"sample": "GCA_016027775.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Delftia acidovorans strain FDAARGOS_891 chromosome, complete genome", "reads_in": 476700, "reads_removed_percent": 0.0016782043213761273, "reads_retained_percent": 99.99832179567862}, {"sample": "GCA_016027115.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Serratia plymuthica strain FDAARGOS_907 chromosome, complete genome", "reads_in": 364598, "reads_removed_percent": 0.002194197444857076, "reads_retained_percent": 99.99780580255515}, {"sample": "GCA_019048345.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Escherichia coli strain FDAARGOS 1377 chromosome, complete genome", "reads_in": 356918, "reads_removed_percent": 0.0022414111924867895, "reads_retained_percent": 99.99775858880751}, {"sample": "GCA_001019275.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Staphylococcus aureus strain FDAARGOS_7 chromosome, complete genome", "reads_in": 181938, "reads_removed_percent": 0.004397102309577988, "reads_retained_percent": 99.99560289769042}, {"sample": "GCA_003613595.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Burkholderia gladioli pv. gladioli strain FDAARGOS_188 chromosome 1, complete sequence", "reads_in": 592836, "reads_removed_percent": 0.001349445715172493, "reads_retained_percent": 99.99865055428482}, {"sample": "GCA_003030425.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Enterococcus faecalis strain FDAARGOS_324 chromosome, complete genome", "reads_in": 210658, "reads_removed_percent": 0.0037976245858215687, "reads_retained_percent": 99.99620237541419}, {"sample": "GCA_003812385.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Acinetobacter baumannii strain FDAARGOS_533 chromosome, complete genome", "reads_in": 259026, "reads_removed_percent": 0.003088493047030028, "reads_retained_percent": 99.99691150695297}, {"sample": "GCA_001558375.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 10, "name": "Staphylococcus saprophyticus strain FDAARGOS_137 chromosome, complete genome", "reads_in": 171946, "reads_removed_percent": 0.005815779372593722, "reads_retained_percent": 99.9941842206274}, {"sample": "GCA_016028795.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 10, "name": "Macrococcus caseolyticus strain FDAARGOS_868 chromosome, complete genome", "reads_in": 144730, "reads_removed_percent": 0.006909417536101706, "reads_retained_percent": 99.99309058246389}, {"sample": "GCA_019047205.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 10, "name": "Acinetobacter pittii strain FDAARGOS 1399 chromosome, complete genome", "reads_in": 265342, "reads_removed_percent": 0.0037687211221744013, "reads_retained_percent": 99.99623127887783}, {"sample": "GCA_019047165.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 10, "name": "Escherichia coli strain FDAARGOS_1384 chromosome, complete genome", "reads_in": 347978, "reads_removed_percent": 0.0028737448919184546, "reads_retained_percent": 99.99712625510809}, {"sample": "GCA_002984485.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Oligella urethralis strain FDAARGOS_329 chromosome, complete genome", "reads_in": 167422, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016028235.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 10, "name": "Staphylococcus equorum strain FDAARGOS_884 chromosome, complete genome", "reads_in": 190484, "reads_removed_percent": 0.005249784758824889, "reads_retained_percent": 99.99475021524117}, {"sample": "GCA_003546885.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Klebsiella aerogenes strain FDAARGOS_327 chromosome, complete genome", "reads_in": 360012, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001018845.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 10, "name": "Staphylococcus aureus strain FDAARGOS_21 chromosome, complete genome", "reads_in": 192082, "reads_removed_percent": 0.00520610989056757, "reads_retained_percent": 99.99479389010943}, {"sample": "GCA_002984035.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Salmonella enterica subsp. enterica serovar Heidelberg strain FDAARGOS_319 chromosome, complete genome", "reads_in": 333918, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001018735.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 10, "name": "Staphylococcus aureus strain FDAARGOS_12 chromosome, complete genome", "reads_in": 181716, "reads_removed_percent": 0.005503092738118822, "reads_retained_percent": 99.99449690726188}, {"sample": "GCA_003812645.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 10, "name": "Streptococcus pyogenes strain FDAARGOS_534 chromosome, complete genome", "reads_in": 122516, "reads_removed_percent": 0.008162199222958634, "reads_retained_percent": 99.99183780077703}, {"sample": "GCA_020735645.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 10, "name": "Acidaminococcus fermentans DSM 20731 strain FDAARGOS_1571 chromosome, complete genome", "reads_in": 155546, "reads_removed_percent": 0.00642896635079012, "reads_retained_percent": 99.9935710336492}, {"sample": "GCA_002983705.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Helicobacter pylori strain FDAARGOS_300 chromosome, complete genome", "reads_in": 109716, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016599795.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 10, "name": "Staphylococcus pasteuri strain FDAARGOS_1069 chromosome, complete genome", "reads_in": 165654, "reads_removed_percent": 0.006036678860757965, "reads_retained_percent": 99.99396332113925}, {"sample": "GCA_001558495.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 10, "name": "Vibrio parahaemolyticus strain ATCC 17802 chromosome 1, complete sequence", "reads_in": 343498, "reads_removed_percent": 0.0029112251017473175, "reads_retained_percent": 99.99708877489826}, {"sample": "GCA_003812805.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 10, "name": "Streptococcus agalactiae strain FDAARGOS_512 chromosome, complete genome", "reads_in": 142276, "reads_removed_percent": 0.007028592313531446, "reads_retained_percent": 99.99297140768647}, {"sample": "GCA_016889245.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 10, "name": "Mammaliicoccus vitulinus strain FDAARGOS_1207 chromosome, complete genome", "reads_in": 176858, "reads_removed_percent": 0.0056542536950547895, "reads_retained_percent": 99.99434574630494}, {"sample": "GCA_008694005.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 10, "name": "Streptococcus pyogenes strain FDAARGOS_668 chromosome, complete genome", "reads_in": 122756, "reads_removed_percent": 0.00814624132425299, "reads_retained_percent": 99.99185375867575}, {"sample": "GCA_000784015.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 10, "name": "Proteus mirabilis strain FDAARGOS_60 chromosome, complete genome", "reads_in": 273704, "reads_removed_percent": 0.003653581971765119, "reads_retained_percent": 99.99634641802824}, {"sample": "GCA_003812985.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 10, "name": "Stenotrophomonas maltophilia strain FDAARGOS_507 chromosome, complete genome", "reads_in": 305172, "reads_removed_percent": 0.003276840601365787, "reads_retained_percent": 99.99672315939864}, {"sample": "GCA_003798465.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Cutibacterium acnes strain FDAARGOS_577 chromosome, complete genome", "reads_in": 166356, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_006364815.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 10, "name": "Enterococcus faecalis strain FDAARGOS_611 chromosome, complete genome", "reads_in": 189696, "reads_removed_percent": 0.0052715924426450745, "reads_retained_percent": 99.99472840755736}, {"sample": "GCA_003813005.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 10, "name": "Pseudomonas aeruginosa strain FDAARGOS_505 chromosome, complete genome", "reads_in": 468654, "reads_removed_percent": 0.0021337703294968145, "reads_retained_percent": 99.9978662296705}, {"sample": "GCA_006364635.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 10, "name": "Legionella pneumophila strain FDAARGOS_779 chromosome, complete genome", "reads_in": 227280, "reads_removed_percent": 0.004399859204505456, "reads_retained_percent": 99.99560014079549}, {"sample": "GCA_003812685.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 10, "name": "Streptococcus sp. FDAARGOS_520 chromosome, complete genome", "reads_in": 139974, "reads_removed_percent": 0.007144183919870833, "reads_retained_percent": 99.99285581608012}, {"sample": "GCA_016904155.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 10, "name": "Coprococcus comes strain FDAARGOS_1339 chromosome, complete genome", "reads_in": 224872, "reads_removed_percent": 0.004446974278700772, "reads_retained_percent": 99.99555302572129}, {"sample": "GCA_000783775.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 10, "name": "Aeromonas caviae strain FDAARGOS_72 chromosome, complete genome", "reads_in": 301220, "reads_removed_percent": 0.0033198326804329066, "reads_retained_percent": 99.99668016731957}, {"sample": "GCA_003798405.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 10, "name": "Yersinia pseudotuberculosis strain FDAARGOS_582 chromosome, complete genome", "reads_in": 322164, "reads_removed_percent": 0.0031040091382029027, "reads_retained_percent": 99.9968959908618}, {"sample": "GCA_001558755.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Achromobacter xylosoxidans strain FDAARGOS_147 chromosome, complete genome", "reads_in": 459130, "reads_removed_percent": 0.001742425892448762, "reads_retained_percent": 99.99825757410756}, {"sample": "GCA_016725905.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Proteus mirabilis strain FDAARGOS_1079 chromosome, complete genome", "reads_in": 269036, "reads_removed_percent": 0.002973579743974784, "reads_retained_percent": 99.99702642025602}, {"sample": "GCA_003812405.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Citrobacter koseri strain FDAARGOS_530 chromosome, complete genome", "reads_in": 305400, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812845.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Klebsiella sp. FDAARGOS_511 chromosome, complete genome", "reads_in": 419348, "reads_removed_percent": 0.0019077234182588209, "reads_retained_percent": 99.99809227658174}, {"sample": "GCA_016727405.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Bacillus cereus strain FDAARGOS_1084 chromosome, complete genome", "reads_in": 354920, "reads_removed_percent": 0.002254029076975093, "reads_retained_percent": 99.99774597092302}, {"sample": "GCA_016726305.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Bacteroides caccae strain FDAARGOS_1097 chromosome, complete genome", "reads_in": 304730, "reads_removed_percent": 0.0026252748334591276, "reads_retained_percent": 99.99737472516654}, {"sample": "GCA_016727345.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Enterococcus casseliflavus strain FDAARGOS_1120 chromosome, complete genome", "reads_in": 236566, "reads_removed_percent": 0.0033817201119349357, "reads_retained_percent": 99.99661827988807}, {"sample": "GCA_003812225.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Acinetobacter sp. FDAARGOS_494 chromosome, complete genome", "reads_in": 263550, "reads_removed_percent": 0.0030354771390627964, "reads_retained_percent": 99.99696452286094}, {"sample": "GCA_016727905.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 10, "name": "Yersinia enterocolitica strain FDAARGOS_1090 chromosome, complete genome", "reads_in": 306216, "reads_removed_percent": 0.003265668678318573, "reads_retained_percent": 99.99673433132168}, {"sample": "GCA_016888905.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 10, "name": "Parabacteroides distasonis strain FDAARGOS_1234 chromosome, complete genome", "reads_in": 320802, "reads_removed_percent": 0.003117187548706056, "reads_retained_percent": 99.99688281245129}, {"sample": "GCA_016127315.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 10, "name": "Achromobacter deleyi strain FDAARGOS_1050 chromosome, complete genome", "reads_in": 465818, "reads_removed_percent": 0.0021467611814056133, "reads_retained_percent": 99.99785323881859}, {"sample": "GCA_002386265.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 10, "name": "Weissella paramesenteroides strain FDAARGOS_414 chromosome, complete genome", "reads_in": 129860, "reads_removed_percent": 0.0077006006468504536, "reads_retained_percent": 99.99229939935314}, {"sample": "GCA_016903855.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 10, "name": "Escherichia coli strain FDAARGOS_1291 chromosome, complete genome", "reads_in": 348814, "reads_removed_percent": 0.0028668574082462285, "reads_retained_percent": 99.99713314259175}, {"sample": "GCA_003812245.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 10, "name": "Acinetobacter sp. FDAARGOS_493 chromosome, complete genome", "reads_in": 268098, "reads_removed_percent": 0.003729979335914479, "reads_retained_percent": 99.99627002066408}, {"sample": "GCA_002208805.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Staphylococcus pettenkoferi strain FDAARGOS_288 chromosome, complete genome", "reads_in": 166824, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016026175.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 10, "name": "Listeria welshimeri strain FDAARGOS_947 chromosome, complete genome", "reads_in": 195648, "reads_removed_percent": 0.005111220150474321, "reads_retained_percent": 99.99488877984953}, {"sample": "GCA_002278015.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 10, "name": "Enterococcus hirae strain FDAARGOS_234 chromosome, complete genome", "reads_in": 189710, "reads_removed_percent": 0.005271203415739814, "reads_retained_percent": 99.99472879658425}, {"sample": "GCA_000626615.3", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 10, "name": "Staphylococcus aureus strain FDAARGOS_5 chromosome, complete genome", "reads_in": 187346, "reads_removed_percent": 0.005337717378540241, "reads_retained_percent": 99.99466228262146}, {"sample": "GCA_019048025.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 10, "name": "Staphylococcus haemolyticus strain FDAARGOS 1453 chromosome, complete genome", "reads_in": 171250, "reads_removed_percent": 0.005839416058394161, "reads_retained_percent": 99.9941605839416}, {"sample": "GCA_003812785.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Cutibacterium acnes strain FDAARGOS_503 chromosome, complete genome", "reads_in": 166302, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002984395.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 10, "name": "Klebsiella oxytoca strain FDAARGOS_335 chromosome, complete genome", "reads_in": 403320, "reads_removed_percent": 0.002479420807299415, "reads_retained_percent": 99.9975205791927}, {"sample": "GCA_003812765.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Gardnerella vaginalis strain FDAARGOS_568 chromosome, complete genome", "reads_in": 114440, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001593585.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 10, "name": "Klebsiella aerogenes strain FDAARGOS_139 chromosome, complete genome", "reads_in": 353188, "reads_removed_percent": 0.002831353273610655, "reads_retained_percent": 99.9971686467264}, {"sample": "GCA_002208845.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Citrobacter braakii strain FDAARGOS_290 chromosome, complete genome", "reads_in": 322318, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013267815.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 10, "name": "Bacillus thuringiensis strain FDAARGOS_792 chromosome, complete genome", "reads_in": 355252, "reads_removed_percent": 0.0028149032236271716, "reads_retained_percent": 99.99718509677638}, {"sample": "GCA_016726965.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Mobiluncus curtisii strain FDAARGOS_1128 chromosome, complete genome", "reads_in": 144282, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003546905.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 14, "name": "Yersinia pseudotuberculosis strain FDAARGOS_342 chromosome, complete genome", "reads_in": 327036, "reads_removed_percent": 0.0042808742768380235, "reads_retained_percent": 99.99571912572316}, {"sample": "GCA_001019495.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 12, "name": "Staphylococcus aureus strain FDAARGOS_40 chromosome, complete genome", "reads_in": 192622, "reads_removed_percent": 0.006229817985484524, "reads_retained_percent": 99.99377018201452}, {"sample": "GCA_016127875.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 22, "name": "Streptococcus constellatus strain FDAARGOS_1015 chromosome, complete genome", "reads_in": 135906, "reads_removed_percent": 0.016187659117331096, "reads_retained_percent": 99.98381234088266}, {"sample": "GCA_013331815.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 22, "name": "Bacillus tropicus strain FDAARGOS_782 chromosome, complete genome", "reads_in": 353154, "reads_removed_percent": 0.006229576898463559, "reads_retained_percent": 99.99377042310154}, {"sample": "GCA_013267835.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 22, "name": "Neisseria mucosa strain FDAARGOS_758 chromosome, complete genome", "reads_in": 148318, "reads_removed_percent": 0.01483299397241063, "reads_retained_percent": 99.9851670060276}, {"sample": "GCA_009730315.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 22, "name": "Gemella morbillorum strain FDAARGOS_741 chromosome, complete genome", "reads_in": 118630, "reads_removed_percent": 0.018545056056646716, "reads_retained_percent": 99.98145494394335}, {"sample": "GCA_016026535.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Delftia acidovorans strain FDAARGOS_939 chromosome, complete genome", "reads_in": 423412, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812525.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 22, "name": "Proteus vulgaris strain FDAARGOS_556 chromosome, complete genome", "reads_in": 277230, "reads_removed_percent": 0.00793564910002525, "reads_retained_percent": 99.99206435089998}, {"sample": "GCA_013267255.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 24, "name": "Bacillus cereus strain FDAARGOS_799 chromosome, complete genome", "reads_in": 412650, "reads_removed_percent": 0.0058160668847691745, "reads_retained_percent": 99.99418393311522}, {"sample": "GCA_020097355.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 24, "name": "Proteus vulgaris strain FDAARGOS_1507 chromosome, complete genome", "reads_in": 264242, "reads_removed_percent": 0.009082583389468744, "reads_retained_percent": 99.99091741661053}, {"sample": "GCA_016026855.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 24, "name": "Bacillus tropicus strain FDAARGOS_920 chromosome, complete genome", "reads_in": 380104, "reads_removed_percent": 0.0063140614147706945, "reads_retained_percent": 99.99368593858523}, {"sample": "GCA_016026335.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_943 chromosome, complete genome", "reads_in": 362216, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127235.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 24, "name": "Enterococcus hirae strain FDAARGOS_1057 chromosome, complete genome", "reads_in": 188792, "reads_removed_percent": 0.012712403067926606, "reads_retained_percent": 99.98728759693208}, {"sample": "GCA_013267275.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 24, "name": "Bacillus cereus strain FDAARGOS_798 chromosome, complete genome", "reads_in": 412640, "reads_removed_percent": 0.0058162078324932144, "reads_retained_percent": 99.9941837921675}, {"sample": "GCA_016728765.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 24, "name": "Chryseobacterium sp. FDAARGOS 1104 strain FDAARGOS_1104 chromosome, complete genome", "reads_in": 246632, "reads_removed_percent": 0.009731097343410426, "reads_retained_percent": 99.9902689026566}, {"sample": "GCA_000783575.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 24, "name": "Proteus mirabilis strain FDAARGOS_81 chromosome, complete genome", "reads_in": 271822, "reads_removed_percent": 0.008829307414410901, "reads_retained_percent": 99.99117069258558}, {"sample": "GCA_020149725.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 24, "name": "Campylobacter coli strain FDAARGOS_1464 chromosome, complete genome", "reads_in": 107916, "reads_removed_percent": 0.022239519626376072, "reads_retained_percent": 99.97776048037362}, {"sample": "GCA_016127675.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 24, "name": "Campylobacter coli strain FDAARGOS_1028 chromosome, complete genome", "reads_in": 125030, "reads_removed_percent": 0.019195393105654643, "reads_retained_percent": 99.98080460689434}, {"sample": "GCA_016026235.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 24, "name": "Escherichia coli strain FDAARGOS_944 chromosome, complete genome", "reads_in": 355392, "reads_removed_percent": 0.00675310642895732, "reads_retained_percent": 99.99324689357104}, {"sample": "GCA_016027135.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 26, "name": "Thomasclavelia ramosa strain FDAARGOS_906 chromosome, complete genome", "reads_in": 216294, "reads_removed_percent": 0.012020675561966582, "reads_retained_percent": 99.98797932443803}, {"sample": "GCA_016027095.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Sphingomonas paucimobilis strain FDAARGOS_908 chromosome, complete genome", "reads_in": 291138, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019047085.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 26, "name": "Aeromonas sp. FDAARGOS 1404 chromosome, complete genome", "reads_in": 315912, "reads_removed_percent": 0.008230140038998201, "reads_retained_percent": 99.991769859961}, {"sample": "GCA_002587225.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 26, "name": "Campylobacter jejuni strain FDAARGOS_421 chromosome, complete genome", "reads_in": 118520, "reads_removed_percent": 0.02193722578467769, "reads_retained_percent": 99.97806277421532}, {"sample": "GCA_002073495.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 26, "name": "Leptospira interrogans serovar Copenhageni strain FDAARGOS_203 chromosome, complete genome", "reads_in": 308718, "reads_removed_percent": 0.008421925511308055, "reads_retained_percent": 99.99157807448869}, {"sample": "GCA_019930985.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 26, "name": "Campylobacter coli strain FDAARGOS_1465 chromosome, complete genome", "reads_in": 128994, "reads_removed_percent": 0.020155976246957225, "reads_retained_percent": 99.97984402375305}, {"sample": "GCA_002073295.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Pasteurella multocida strain FDAARGOS_216 chromosome, complete genome", "reads_in": 153012, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001019435.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 26, "name": "Staphylococcus aureus strain FDAARGOS_25 chromosome, complete genome", "reads_in": 191900, "reads_removed_percent": 0.01354872329338197, "reads_retained_percent": 99.98645127670662}, {"sample": "GCA_020097255.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 26, "name": "Thomasclavelia ramosa strain FDAARGOS_1519 chromosome, complete genome", "reads_in": 254472, "reads_removed_percent": 0.010217234116130654, "reads_retained_percent": 99.98978276588387}, {"sample": "GCA_013267295.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 26, "name": "Bacillus thuringiensis strain FDAARGOS_796 chromosome, complete genome", "reads_in": 413816, "reads_removed_percent": 0.006282985674792661, "reads_retained_percent": 99.99371701432521}, {"sample": "GCA_002587105.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 26, "name": "Campylobacter jejuni strain FDAARGOS_422 chromosome, complete genome", "reads_in": 112690, "reads_removed_percent": 0.02307214482207827, "reads_retained_percent": 99.97692785517792}, {"sample": "GCA_016026835.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Bacillus licheniformis strain FDAARGOS_923 chromosome, complete genome", "reads_in": 312892, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812705.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 22, "name": "Chryseobacterium arthrosphaerae strain FDAARGOS_519 chromosome, complete genome", "reads_in": 357458, "reads_removed_percent": 0.006154569208130746, "reads_retained_percent": 99.99384543079186}, {"sample": "GCA_016027015.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 22, "name": "Bacillus cereus strain FDAARGOS_918 chromosome, complete genome", "reads_in": 363314, "reads_removed_percent": 0.006055368083806295, "reads_retained_percent": 99.9939446319162}, {"sample": "GCA_022213445.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 22, "name": "Burkholderia vietnamiensis strain FDAARGOS_721 chromosome 1, complete sequence", "reads_in": 467904, "reads_removed_percent": 0.004701819176583231, "reads_retained_percent": 99.99529818082341}, {"sample": "GCA_019048085.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 22, "name": "Aeromonas sp. FDAARGOS 1416 chromosome, complete genome", "reads_in": 301256, "reads_removed_percent": 0.00730275911517115, "reads_retained_percent": 99.99269724088482}, {"sample": "GCA_002073415.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 18, "name": "Bacillus sp. FDAARGOS_235 chromosome, complete genome", "reads_in": 373892, "reads_removed_percent": 0.004814224428444577, "reads_retained_percent": 99.99518577557156}, {"sample": "GCA_016725865.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 18, "name": "Sphingobacterium multivorum strain FDAARGOS_1143 chromosome, complete genome", "reads_in": 401680, "reads_removed_percent": 0.004481179047998407, "reads_retained_percent": 99.995518820952}, {"sample": "GCA_002951455.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 18, "name": "Elizabethkingia anophelis strain FDAARGOS_134 chromosome, complete genome", "reads_in": 266560, "reads_removed_percent": 0.006752701080432173, "reads_retained_percent": 99.99324729891957}, {"sample": "GCA_016724865.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 18, "name": "Peribacillus psychrosaccharolyticus strain FDAARGOS_1161 chromosome, complete genome", "reads_in": 312086, "reads_removed_percent": 0.00576764097075806, "reads_retained_percent": 99.99423235902924}, {"sample": "GCA_002209125.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 18, "name": "Nocardia brasiliensis strain FDAARGOS_352 chromosome, complete genome", "reads_in": 595700, "reads_removed_percent": 0.003021655195568239, "reads_retained_percent": 99.99697834480443}, {"sample": "GCA_008693705.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 20, "name": "Aeromonas veronii strain FDAARGOS_632 chromosome, complete genome", "reads_in": 304126, "reads_removed_percent": 0.006576221697585869, "reads_retained_percent": 99.99342377830241}, {"sample": "GCA_016726985.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 20, "name": "Myroides odoratus strain FDAARGOS_1131 chromosome, complete genome", "reads_in": 282832, "reads_removed_percent": 0.007071335633874526, "reads_retained_percent": 99.99292866436612}, {"sample": "GCA_003798365.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 20, "name": "Burkholderia pseudomallei strain FDAARGOS_594 chromosome 1, complete sequence", "reads_in": 478692, "reads_removed_percent": 0.004178051857979662, "reads_retained_percent": 99.99582194814202}, {"sample": "GCA_016028275.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Staphylococcus carnosus strain FDAARGOS_883 chromosome, complete genome", "reads_in": 171424, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002386185.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 20, "name": "Staphylococcus simulans strain FDAARGOS_383 chromosome, complete genome", "reads_in": 191464, "reads_removed_percent": 0.010445827936322233, "reads_retained_percent": 99.98955417206368}, {"sample": "GCA_016726045.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 20, "name": "Sphingobacterium multivorum strain FDAARGOS_1140 chromosome, complete genome", "reads_in": 377044, "reads_removed_percent": 0.005304420704214893, "reads_retained_percent": 99.99469557929578}, {"sample": "GCA_016127295.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 20, "name": "Leuconostoc mesenteroides strain FDAARGOS_1033 chromosome, complete genome", "reads_in": 138384, "reads_removed_percent": 0.014452537865649207, "reads_retained_percent": 99.98554746213435}, {"sample": "GCA_013267795.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 20, "name": "Bacillus thuringiensis strain FDAARGOS_794 chromosome, complete genome", "reads_in": 356574, "reads_removed_percent": 0.005608933909931739, "reads_retained_percent": 99.99439106609007}, {"sample": "GCA_016028755.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Moraxella osloensis strain FDAARGOS_870 chromosome, complete genome", "reads_in": 162388, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002951395.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 26, "name": "Elizabethkingia anophelis strain FDAARGOS_132 chromosome, complete genome", "reads_in": 266560, "reads_removed_percent": 0.009753901560624249, "reads_retained_percent": 99.99024609843937}, {"sample": "GCA_016889705.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 20, "name": "Amycolatopsis sp. FDAARGOS 1241 strain FDAARGOS_1241 chromosome, complete genome", "reads_in": 653208, "reads_removed_percent": 0.003061811857784963, "reads_retained_percent": 99.99693818814221}, {"sample": "GCA_019046985.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 20, "name": "Aeromonas sp. FDAARGOS 1417 chromosome, complete genome", "reads_in": 319550, "reads_removed_percent": 0.0062588014395243315, "reads_retained_percent": 99.99374119856047}, {"sample": "GCA_016904015.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 20, "name": "Ruminococcus] gnavus strain FDAARGOS_1342 chromosome, complete genome", "reads_in": 213130, "reads_removed_percent": 0.009383944071693334, "reads_retained_percent": 99.99061605592831}, {"sample": "GCA_016027455.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 20, "name": "Burkholderia humptydooensis strain FDAARGOS_899 chromosome 1, complete sequence", "reads_in": 485776, "reads_removed_percent": 0.0041171239418991475, "reads_retained_percent": 99.9958828760581}, {"sample": "GCA_001018775.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 20, "name": "Staphylococcus aureus strain FDAARGOS_14 chromosome, complete genome", "reads_in": 184314, "reads_removed_percent": 0.010851047668652409, "reads_retained_percent": 99.98914895233135}, {"sample": "GCA_016127275.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 20, "name": "Streptococcus infantarius strain FDAARGOS_1019 chromosome, complete genome", "reads_in": 130582, "reads_removed_percent": 0.015316046622045918, "reads_retained_percent": 99.98468395337795}, {"sample": "GCA_016028515.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Stutzerimonas frequens strain FDAARGOS_877 chromosome, complete genome", "reads_in": 292980, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020097455.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 20, "name": "Gemella haemolysans strain FDAARGOS_1500 chromosome, complete genome", "reads_in": 128550, "reads_removed_percent": 0.015558148580318941, "reads_retained_percent": 99.98444185141967}, {"sample": "GCA_003711125.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 22, "name": "Enterococcus avium strain FDAARGOS_184 chromosome, complete genome", "reads_in": 248226, "reads_removed_percent": 0.008862891075068687, "reads_retained_percent": 99.99113710892493}, {"sample": "GCA_002591275.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 22, "name": "Chryseobacterium indologenes strain FDAARGOS_379 chromosome, complete genome", "reads_in": 325922, "reads_removed_percent": 0.006750081307797572, "reads_retained_percent": 99.9932499186922}, {"sample": "GCA_016027615.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Corynebacterium minutissimum strain FDAARGOS_894 chromosome, complete genome", "reads_in": 179730, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002591155.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 22, "name": "Proteus vulgaris strain FDAARGOS_366 chromosome, complete genome", "reads_in": 264132, "reads_removed_percent": 0.008329168748958855, "reads_retained_percent": 99.99167083125104}, {"sample": "GCA_002209025.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 22, "name": "Campylobacter jejuni strain FDAARGOS_263 chromosome, complete genome", "reads_in": 109432, "reads_removed_percent": 0.02010380875795014, "reads_retained_percent": 99.97989619124205}, {"sample": "GCA_002209265.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 22, "name": "Staphylococcus saprophyticus strain FDAARGOS_355 chromosome, complete genome", "reads_in": 175946, "reads_removed_percent": 0.012503836404351336, "reads_retained_percent": 99.98749616359565}, {"sample": "GCA_002393425.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 22, "name": "Burkholderia gladioli pv. gladioli strain FDAARGOS_389 chromosome 1, complete sequence", "reads_in": 552478, "reads_removed_percent": 0.003982059014114589, "reads_retained_percent": 99.99601794098588}, {"sample": "GCA_019047065.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 20, "name": "Aeromonas sp. FDAARGOS 1414 chromosome, complete genome", "reads_in": 299588, "reads_removed_percent": 0.006675834813143384, "reads_retained_percent": 99.99332416518686}, {"sample": "GCA_016026395.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 26, "name": "Edwardsiella hoshinae strain FDAARGOS_940 chromosome, complete genome", "reads_in": 254474, "reads_removed_percent": 0.010217153815321015, "reads_retained_percent": 99.98978284618468}, {"sample": "GCA_019048745.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 26, "name": "Aeromonas sp. FDAARGOS 1410 chromosome, complete genome", "reads_in": 300686, "reads_removed_percent": 0.008646894102153076, "reads_retained_percent": 99.99135310589784}, {"sample": "GCA_013267675.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 26, "name": "Staphylococcus capitis strain FDAARGOS_753 chromosome, complete genome", "reads_in": 166064, "reads_removed_percent": 0.015656614317371614, "reads_retained_percent": 99.98434338568264}, {"sample": "GCA_020736145.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 36, "name": "Campylobacter jejuni strain FDAARGOS_1546 chromosome, complete genome", "reads_in": 108536, "reads_removed_percent": 0.033168718213311715, "reads_retained_percent": 99.96683128178668}, {"sample": "GCA_016725925.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 36, "name": "Sphingobacterium multivorum strain FDAARGOS_1141 chromosome, complete genome", "reads_in": 419822, "reads_removed_percent": 0.008575062764695515, "reads_retained_percent": 99.99142493723531}, {"sample": "GCA_009730295.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 38, "name": "Gemella sanguinis strain FDAARGOS_742 chromosome, complete genome", "reads_in": 119774, "reads_removed_percent": 0.03172641808739793, "reads_retained_percent": 99.9682735819126}, {"sample": "GCA_016127155.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Brevibacterium casei strain FDAARGOS_990 chromosome, complete genome", "reads_in": 256754, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812125.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 38, "name": "Bacillus sp. FDAARGOS_527 chromosome, complete genome", "reads_in": 380076, "reads_removed_percent": 0.009998000399920015, "reads_retained_percent": 99.99000199960008}, {"sample": "GCA_016026675.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 38, "name": "Clostridium perfringens strain FDAARGOS_931 chromosome, complete genome", "reads_in": 225034, "reads_removed_percent": 0.01688633717571567, "reads_retained_percent": 99.98311366282428}, {"sample": "GCA_003812865.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 40, "name": "Chryseobacterium indologenes strain FDAARGOS_510 chromosome, complete genome", "reads_in": 328196, "reads_removed_percent": 0.012187838974271472, "reads_retained_percent": 99.98781216102573}, {"sample": "GCA_016127475.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 42, "name": "Anaerococcus vaginalis strain FDAARGOS_988 chromosome, complete genome", "reads_in": 126264, "reads_removed_percent": 0.033263638091617566, "reads_retained_percent": 99.96673636190839}, {"sample": "GCA_002209045.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 42, "name": "Campylobacter jejuni strain FDAARGOS_265 chromosome, complete genome", "reads_in": 113074, "reads_removed_percent": 0.037143817323168904, "reads_retained_percent": 99.96285618267683}, {"sample": "GCA_003812825.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 44, "name": "Fusobacterium necrophorum strain FDAARGOS_565 chromosome, complete genome", "reads_in": 178560, "reads_removed_percent": 0.024641577060931903, "reads_retained_percent": 99.97535842293907}, {"sample": "GCA_002209005.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 44, "name": "Campylobacter jejuni strain FDAARGOS_262 chromosome, complete genome", "reads_in": 117896, "reads_removed_percent": 0.03732102870326389, "reads_retained_percent": 99.96267897129674}, {"sample": "GCA_009730395.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 44, "name": "Campylobacter coli strain FDAARGOS_735 chromosome, complete genome", "reads_in": 114436, "reads_removed_percent": 0.038449438987731135, "reads_retained_percent": 99.96155056101227}, {"sample": "GCA_016026995.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 46, "name": "Clostridium perfringens strain FDAARGOS_905 chromosome, complete genome", "reads_in": 223168, "reads_removed_percent": 0.02061227416117006, "reads_retained_percent": 99.97938772583883}, {"sample": "GCA_016724785.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 48, "name": "Fusobacterium canifelinum strain FDAARGOS_1126 chromosome, complete genome", "reads_in": 156814, "reads_removed_percent": 0.030609511905824734, "reads_retained_percent": 99.96939048809418}, {"sample": "GCA_019930905.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 36, "name": "Haemophilus aegyptius strain FDAARGOS_1478 chromosome, complete genome", "reads_in": 132884, "reads_removed_percent": 0.027091297673158543, "reads_retained_percent": 99.97290870232685}, {"sample": "GCA_002983685.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 50, "name": "Streptobacillus moniliformis strain FDAARGOS_310 chromosome, complete genome", "reads_in": 112396, "reads_removed_percent": 0.04448556888145486, "reads_retained_percent": 99.95551443111854}, {"sample": "GCA_016126915.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Corynebacterium minutissimum strain FDAARGOS_992 chromosome, complete genome", "reads_in": 179734, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016027375.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 56, "name": "Clostridium perfringens strain FDAARGOS_903 chromosome, complete genome", "reads_in": 223682, "reads_removed_percent": 0.025035541527704507, "reads_retained_percent": 99.9749644584723}, {"sample": "GCA_003798265.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 62, "name": "Burkholderia pseudomallei strain FDAARGOS_592 chromosome 1, complete sequence", "reads_in": 476446, "reads_removed_percent": 0.01301301721496245, "reads_retained_percent": 99.98698698278504}, {"sample": "GCA_016127495.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Kytococcus sedentarius strain FDAARGOS_1061 chromosome, complete genome", "reads_in": 185572, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019930945.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 66, "name": "Campylobacter hominis strain FDAARGOS_1467 chromosome, complete genome", "reads_in": 114540, "reads_removed_percent": 0.057621791513881616, "reads_retained_percent": 99.94237820848612}, {"sample": "GCA_016127595.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 70, "name": "Fusobacterium canifelinum strain FDAARGOS_999 chromosome, complete genome", "reads_in": 161678, "reads_removed_percent": 0.04329593389329408, "reads_retained_percent": 99.95670406610671}, {"sample": "GCA_016027435.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 72, "name": "Burkholderia oklahomensis strain FDAARGOS_900 chromosome 1, complete sequence", "reads_in": 487968, "reads_removed_percent": 0.014755065905961047, "reads_retained_percent": 99.98524493409404}, {"sample": "GCA_009730495.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 78, "name": "Clostridioides difficile strain FDAARGOS_723 chromosome, complete genome", "reads_in": 278540, "reads_removed_percent": 0.028003159330796296, "reads_retained_percent": 99.9719968406692}, {"sample": "GCA_019931025.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 84, "name": "Clostridioides difficile strain FDAARGOS_1469 chromosome, complete genome", "reads_in": 269708, "reads_removed_percent": 0.031144793628665075, "reads_retained_percent": 99.96885520637133}, {"sample": "GCA_016127355.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Kingella denitrificans strain FDAARGOS_1060 chromosome, complete genome", "reads_in": 148004, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002073735.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 94, "name": "Clostridioides difficile strain FDAARGOS_267 chromosome, complete genome", "reads_in": 285744, "reads_removed_percent": 0.0328965787558094, "reads_retained_percent": 99.96710342124419}, {"sample": "GCA_020099295.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 98, "name": "Clostridium sporogenes strain FDAARGOS_1491 chromosome, complete genome", "reads_in": 271160, "reads_removed_percent": 0.03614102374981561, "reads_retained_percent": 99.96385897625018}, {"sample": "GCA_001559195.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Achromobacter xylosoxidans strain FDAARGOS_150 chromosome, complete genome", "reads_in": 417916, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020097215.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 126, "name": "Clostridioides difficile strain FDAARGOS_1528 chromosome, complete genome", "reads_in": 291690, "reads_removed_percent": 0.04319654427645788, "reads_retained_percent": 99.95680345572354}, {"sample": "GCA_016027155.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 50, "name": "Clostridium perfringens strain FDAARGOS_904 chromosome, complete genome", "reads_in": 224056, "reads_removed_percent": 0.022315849609026314, "reads_retained_percent": 99.97768415039097}, {"sample": "GCA_003812605.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 18, "name": "Chryseobacterium indologenes strain FDAARGOS_537 chromosome, complete genome", "reads_in": 324424, "reads_removed_percent": 0.005548294824057406, "reads_retained_percent": 99.99445170517595}, {"sample": "GCA_020097435.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 36, "name": "Gemella morbillorum strain FDAARGOS_1501 chromosome, complete genome", "reads_in": 117396, "reads_removed_percent": 0.030665440049064706, "reads_retained_percent": 99.96933455995094}, {"sample": "GCA_016889885.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 34, "name": "Streptococcus constellatus strain FDAARGOS_1208 chromosome, complete genome", "reads_in": 131912, "reads_removed_percent": 0.025774758930195887, "reads_retained_percent": 99.9742252410698}, {"sample": "GCA_016028855.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Rothia kristinae strain FDAARGOS_864 chromosome, complete genome", "reads_in": 158516, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003798245.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 26, "name": "Burkholderia pseudomallei strain FDAARGOS_593 chromosome 1, complete sequence", "reads_in": 483702, "reads_removed_percent": 0.005375210356789924, "reads_retained_percent": 99.99462478964321}, {"sample": "GCA_016128255.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 26, "name": "Lactococcus garvieae strain FDAARGOS_1063 chromosome, complete genome", "reads_in": 147538, "reads_removed_percent": 0.017622578589922597, "reads_retained_percent": 99.98237742141008}, {"sample": "GCA_016889945.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 26, "name": "Staphylococcus aureus strain FDAARGOS_1245 chromosome, complete genome", "reads_in": 192584, "reads_removed_percent": 0.013500602334565697, "reads_retained_percent": 99.98649939766544}, {"sample": "GCA_016026615.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 28, "name": "Aeromonas allosaccharophila strain FDAARGOS_933 chromosome, complete genome", "reads_in": 308694, "reads_removed_percent": 0.00907047108139452, "reads_retained_percent": 99.9909295289186}, {"sample": "GCA_020735705.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 28, "name": "Coprococcus eutactus ATCC 27759 strain FDAARGOS_1581 chromosome, complete genome", "reads_in": 207556, "reads_removed_percent": 0.013490335138468652, "reads_retained_percent": 99.98650966486153}, {"sample": "GCA_016728785.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 28, "name": "Thomasclavelia ramosa strain FDAARGOS_1105 chromosome, complete genome", "reads_in": 236248, "reads_removed_percent": 0.01185195218583861, "reads_retained_percent": 99.98814804781416}, {"sample": "GCA_013267535.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 28, "name": "Finegoldia magna strain FDAARGOS_764 chromosome, complete genome", "reads_in": 139338, "reads_removed_percent": 0.020095020740932123, "reads_retained_percent": 99.97990497925906}, {"sample": "GCA_002208925.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 28, "name": "Chryseobacterium indologenes strain FDAARGOS_337 chromosome, complete genome", "reads_in": 341292, "reads_removed_percent": 0.00820411846747067, "reads_retained_percent": 99.99179588153252}, {"sample": "GCA_016027575.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 28, "name": "Bacillus tropicus strain FDAARGOS_897 chromosome, complete genome", "reads_in": 381134, "reads_removed_percent": 0.007346497557289562, "reads_retained_percent": 99.9926535024427}, {"sample": "GCA_016127855.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Actinomyces naeslundii strain FDAARGOS_1037 chromosome, complete genome", "reads_in": 210142, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002983725.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 28, "name": "Campylobacter jejuni subsp. doylei strain FDAARGOS_295 chromosome, complete genome", "reads_in": 123004, "reads_removed_percent": 0.022763487366264514, "reads_retained_percent": 99.97723651263374}, {"sample": "GCA_001559075.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Citrobacter amalonaticus strain FDAARGOS_122 chromosome, complete genome", "reads_in": 326982, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127655.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Brevundimonas diminuta strain FDAARGOS_1026 chromosome, complete genome", "reads_in": 236606, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016026755.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 34, "name": "Lysinibacillus macroides strain FDAARGOS_925 chromosome, complete genome", "reads_in": 324176, "reads_removed_percent": 0.010488129904743102, "reads_retained_percent": 99.98951187009526}, {"sample": "GCA_016028675.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 28, "name": "Proteus penneri strain FDAARGOS_874 chromosome, complete genome", "reads_in": 256720, "reads_removed_percent": 0.010906824555936428, "reads_retained_percent": 99.98909317544407}, {"sample": "GCA_016599835.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 30, "name": "Elizabethkingia bruuniana strain FDAARGOS_1031 chromosome, complete genome", "reads_in": 308800, "reads_removed_percent": 0.009715025906735751, "reads_retained_percent": 99.99028497409327}, {"sample": "GCA_016127615.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Corynebacterium amycolatum strain FDAARGOS_991 chromosome, complete genome", "reads_in": 188596, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003813085.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 30, "name": "Alcaligenes faecalis strain FDAARGOS_491 chromosome, complete genome", "reads_in": 271526, "reads_removed_percent": 0.011048665689473568, "reads_retained_percent": 99.98895133431053}, {"sample": "GCA_019048105.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 30, "name": "Providencia rettgeri strain FDAARGOS 1450 chromosome, complete genome", "reads_in": 309462, "reads_removed_percent": 0.009694243558175155, "reads_retained_percent": 99.99030575644183}, {"sample": "GCA_020097395.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 30, "name": "Leclercia adecarboxylata strain FDAARGOS_1505 chromosome, complete genome", "reads_in": 337142, "reads_removed_percent": 0.008898327707612817, "reads_retained_percent": 99.99110167229239}, {"sample": "GCA_020735585.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 32, "name": "Campylobacter fetus subsp. testudinum strain FDAARGOS_1591 chromosome, complete genome", "reads_in": 118366, "reads_removed_percent": 0.027034790395890712, "reads_retained_percent": 99.97296520960411}, {"sample": "GCA_016599815.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 32, "name": "Anaerococcus obesiensis strain FDAARGOS_989 chromosome, complete genome", "reads_in": 126308, "reads_removed_percent": 0.025334895651898535, "reads_retained_percent": 99.9746651043481}, {"sample": "GCA_002209065.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 32, "name": "Campylobacter jejuni strain FDAARGOS_266 chromosome, complete genome", "reads_in": 108030, "reads_removed_percent": 0.029621401462556696, "reads_retained_percent": 99.97037859853744}, {"sample": "GCA_013267695.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 32, "name": "Streptococcus gallolyticus strain FDAARGOS_755 chromosome, complete genome", "reads_in": 150534, "reads_removed_percent": 0.021257656077696734, "reads_retained_percent": 99.9787423439223}, {"sample": "GCA_019930865.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 32, "name": "Campylobacter coli strain FDAARGOS_1466 chromosome, complete genome", "reads_in": 122190, "reads_removed_percent": 0.026188722481381457, "reads_retained_percent": 99.97381127751862}, {"sample": "GCA_001558935.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Citrobacter amalonaticus strain FDAARGOS_165 chromosome, complete genome", "reads_in": 338936, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127755.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 32, "name": "Burkholderia ambifaria strain FDAARGOS_1027 chromosome 1, complete sequence", "reads_in": 501574, "reads_removed_percent": 0.00637991602435533, "reads_retained_percent": 99.99362008397564}, {"sample": "GCA_016127995.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Schaalia meyeri strain FDAARGOS_985 chromosome, complete genome", "reads_in": 135536, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003020185.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 34, "name": "Burkholderia thailandensis strain FDAARGOS_238 chromosome 1, complete sequence", "reads_in": 451244, "reads_removed_percent": 0.007534726223506573, "reads_retained_percent": 99.99246527377649}, {"sample": "GCA_013267235.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 28, "name": "Bacillus cereus strain FDAARGOS_802 chromosome, complete genome", "reads_in": 404570, "reads_removed_percent": 0.006920928393108732, "reads_retained_percent": 99.99307907160689}, {"sample": "GCA_002208865.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 12, "name": "Escherichia coli O157 strain FDAARGOS_293 chromosome, complete genome", "reads_in": 379110, "reads_removed_percent": 0.0031653082218881068, "reads_retained_percent": 99.9968346917781}, {"sample": "GCA_016028175.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Streptococcus oralis strain FDAARGOS_886 chromosome, complete genome", "reads_in": 127586, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_008693465.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 18, "name": "Klebsiella variicola strain FDAARGOS_627 chromosome, complete genome", "reads_in": 397174, "reads_removed_percent": 0.004532018712201705, "reads_retained_percent": 99.9954679812878}, {"sample": "GCA_019048325.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 12, "name": "Enterococcus durans strain FDAARGOS 1437 chromosome, complete genome", "reads_in": 206380, "reads_removed_percent": 0.0058145169105533485, "reads_retained_percent": 99.99418548308945}, {"sample": "GCA_020735825.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 12, "name": "Haemophilus influenzae strain FDAARGOS_1562 chromosome, complete genome", "reads_in": 122840, "reads_removed_percent": 0.009768804949527842, "reads_retained_percent": 99.99023119505047}, {"sample": "GCA_016903335.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 12, "name": "Escherichia coli strain FDAARGOS_1295 chromosome, complete genome", "reads_in": 342824, "reads_removed_percent": 0.003500338366042051, "reads_retained_percent": 99.99649966163396}, {"sample": "GCA_002591215.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 12, "name": "Staphylococcus lugdunensis strain FDAARGOS_381 chromosome, complete genome", "reads_in": 171380, "reads_removed_percent": 0.00700198389543704, "reads_retained_percent": 99.99299801610456}, {"sample": "GCA_016939475.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 12, "name": "Acinetobacter calcoaceticus strain FDAARGOS_1218 chromosome, complete genome", "reads_in": 261016, "reads_removed_percent": 0.0045974193152910165, "reads_retained_percent": 99.99540258068471}, {"sample": "GCA_002073615.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 14, "name": "Neisseria gonorrhoeae strain FDAARGOS_205 chromosome, complete genome", "reads_in": 151930, "reads_removed_percent": 0.009214769959849932, "reads_retained_percent": 99.99078523004015}, {"sample": "GCA_001018645.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 14, "name": "Staphylococcus aureus strain FDAARGOS_2 chromosome, complete genome", "reads_in": 187334, "reads_removed_percent": 0.00747328301322771, "reads_retained_percent": 99.99252671698677}, {"sample": "GCA_013267435.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 14, "name": "Niallia circulans strain FDAARGOS_783 chromosome, complete genome", "reads_in": 345304, "reads_removed_percent": 0.004054398443110998, "reads_retained_percent": 99.99594560155688}, {"sample": "GCA_001018685.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 14, "name": "Staphylococcus aureus strain FDAARGOS_4 chromosome, complete genome", "reads_in": 187452, "reads_removed_percent": 0.007468578622794102, "reads_retained_percent": 99.99253142137721}, {"sample": "GCA_016904075.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 14, "name": "Escherichia coli strain FDAARGOS_1303 chromosome, complete genome", "reads_in": 352828, "reads_removed_percent": 0.003967939052456154, "reads_retained_percent": 99.99603206094754}, {"sample": "GCA_009806695.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 14, "name": "Bacillus anthracis strain FDAARGOS_704 chromosome, complete genome", "reads_in": 348496, "reads_removed_percent": 0.004017262751939764, "reads_retained_percent": 99.99598273724806}, {"sample": "GCA_009806675.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 14, "name": "Bacillus anthracis strain FDAARGOS_705 chromosome, complete genome", "reads_in": 348874, "reads_removed_percent": 0.004012910105080918, "reads_retained_percent": 99.99598708989492}, {"sample": "GCA_013267575.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 14, "name": "Staphylococcus hominis strain FDAARGOS_762 chromosome, complete genome", "reads_in": 150368, "reads_removed_percent": 0.009310491593956161, "reads_retained_percent": 99.99068950840605}, {"sample": "GCA_020736025.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 14, "name": "Haemophilus influenzae strain FDAARGOS_1561 chromosome, complete genome", "reads_in": 122286, "reads_removed_percent": 0.011448571381842566, "reads_retained_percent": 99.98855142861815}, {"sample": "GCA_009734165.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 14, "name": "Bacteroides ovatus strain FDAARGOS_733 chromosome, complete genome", "reads_in": 438092, "reads_removed_percent": 0.0031956757941254345, "reads_retained_percent": 99.99680432420587}, {"sample": "GCA_016725225.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Mobiluncus curtisii strain FDAARGOS_1127 chromosome, complete genome", "reads_in": 143540, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009730095.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 14, "name": "Acinetobacter lwoffii strain FDAARGOS_552 chromosome, complete genome", "reads_in": 232364, "reads_removed_percent": 0.00602502969478921, "reads_retained_percent": 99.99397497030522}, {"sample": "GCA_009806635.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 14, "name": "Bacillus anthracis strain FDAARGOS_700 chromosome, complete genome", "reads_in": 348884, "reads_removed_percent": 0.004012795083752766, "reads_retained_percent": 99.99598720491625}, {"sample": "GCA_016725165.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Streptococcus lutetiensis strain FDAARGOS_1158 chromosome, complete genome", "reads_in": 119568, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016026695.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 14, "name": "Lactococcus garvieae strain FDAARGOS_929 chromosome, complete genome", "reads_in": 138956, "reads_removed_percent": 0.010075131696364316, "reads_retained_percent": 99.98992486830363}, {"sample": "GCA_016889745.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 14, "name": "Bacteroides caccae strain FDAARGOS_1219 chromosome, complete genome", "reads_in": 304730, "reads_removed_percent": 0.004594230958553473, "reads_retained_percent": 99.99540576904144}, {"sample": "GCA_016906185.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 14, "name": "Streptomyces californicus strain FDAARGOS_1209 chromosome", "reads_in": 535448, "reads_removed_percent": 0.002614632980233375, "reads_retained_percent": 99.99738536701976}, {"sample": "GCA_016727005.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 14, "name": "Myroides odoratus strain FDAARGOS_1132 chromosome, complete genome", "reads_in": 277848, "reads_removed_percent": 0.005038726210014109, "reads_retained_percent": 99.99496127378998}, {"sample": "GCA_016726485.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 14, "name": "Peptoniphilus harei strain FDAARGOS_1136 chromosome, complete genome", "reads_in": 128638, "reads_removed_percent": 0.01088325378193069, "reads_retained_percent": 99.98911674621807}, {"sample": "GCA_016725305.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 14, "name": "Staphylococcus equorum strain FDAARGOS_1149 chromosome, complete genome", "reads_in": 182404, "reads_removed_percent": 0.007675270279160544, "reads_retained_percent": 99.99232472972083}, {"sample": "GCA_019047105.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 14, "name": "Enterobacter cloacae strain FDAARGOS 1431 chromosome, complete genome", "reads_in": 373508, "reads_removed_percent": 0.003748246356169078, "reads_retained_percent": 99.99625175364383}, {"sample": "GCA_016889465.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Corynebacterium macginleyi strain FDAARGOS_1195 chromosome, complete genome", "reads_in": 166236, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016724845.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 14, "name": "Sphingobacterium spiritivorum strain FDAARGOS_1146 chromosome, complete genome", "reads_in": 351684, "reads_removed_percent": 0.0039808464416919735, "reads_retained_percent": 99.99601915355831}, {"sample": "GCA_019048225.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 14, "name": "Acinetobacter pittii strain FDAARGOS 1396 chromosome, complete genome", "reads_in": 261056, "reads_removed_percent": 0.005362834027948027, "reads_retained_percent": 99.99463716597205}, {"sample": "GCA_019048525.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 12, "name": "Acinetobacter lwoffii strain FDAARGOS 1394 chromosome, complete genome", "reads_in": 231122, "reads_removed_percent": 0.005192063066259378, "reads_retained_percent": 99.99480793693374}, {"sample": "GCA_019047045.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 12, "name": "Aeromonas sp. FDAARGOS 1419 chromosome, complete genome", "reads_in": 305918, "reads_removed_percent": 0.0039226197870017455, "reads_retained_percent": 99.996077380213}, {"sample": "GCA_002943995.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 12, "name": "Pseudomonas monteilii strain FDAARGOS_171 chromosome, complete genome", "reads_in": 393152, "reads_removed_percent": 0.003052254598730262, "reads_retained_percent": 99.99694774540127}, {"sample": "GCA_019047185.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 12, "name": "Acinetobacter pittii strain FDAARGOS 1397 chromosome, complete genome", "reads_in": 278826, "reads_removed_percent": 0.004303759333778055, "reads_retained_percent": 99.99569624066622}, {"sample": "GCA_016728045.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Serratia ureilytica strain FDAARGOS_1089 chromosome, complete genome", "reads_in": 346696, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001019485.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 12, "name": "Staphylococcus aureus strain FDAARGOS_35 chromosome, complete genome", "reads_in": 197060, "reads_removed_percent": 0.006089515883487263, "reads_retained_percent": 99.99391048411651}, {"sample": "GCA_016725945.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 12, "name": "Acinetobacter johnsonii strain FDAARGOS_1093 chromosome, complete genome", "reads_in": 257722, "reads_removed_percent": 0.004656179914791907, "reads_retained_percent": 99.99534382008521}, {"sample": "GCA_003614815.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 12, "name": "Burkholderia thailandensis strain FDAARGOS_241 chromosome 1, complete sequence", "reads_in": 445608, "reads_removed_percent": 0.0026929498572736575, "reads_retained_percent": 99.99730705014272}, {"sample": "GCA_008693685.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 12, "name": "Vibrio vulnificus strain FDAARGOS_663 chromosome 1, complete sequence", "reads_in": 331654, "reads_removed_percent": 0.003618228635867501, "reads_retained_percent": 99.99638177136413}, {"sample": "GCA_016727765.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Yersinia enterocolitica strain FDAARGOS_1082 chromosome, complete genome", "reads_in": 302752, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003798225.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 12, "name": "Yersinia pestis strain FDAARGOS_601 chromosome, complete genome", "reads_in": 328732, "reads_removed_percent": 0.0036503899833298854, "reads_retained_percent": 99.99634961001667}, {"sample": "GCA_016725245.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 12, "name": "Mammaliicoccus vitulinus strain FDAARGOS_1153 chromosome, complete genome", "reads_in": 169568, "reads_removed_percent": 0.007076806944706549, "reads_retained_percent": 99.99292319305529}, {"sample": "GCA_016725205.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 12, "name": "Staphylococcus pasteuri strain FDAARGOS_1152 chromosome, complete genome", "reads_in": 170422, "reads_removed_percent": 0.007041344427362664, "reads_retained_percent": 99.99295865557264}, {"sample": "GCA_016128235.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 12, "name": "Escherichia fergusonii strain FDAARGOS_1032 chromosome, complete genome", "reads_in": 309820, "reads_removed_percent": 0.0038732167064747273, "reads_retained_percent": 99.99612678329352}, {"sample": "GCA_016127335.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 12, "name": "Staphylococcus pseudintermedius strain FDAARGOS_1072 chromosome, complete genome", "reads_in": 189290, "reads_removed_percent": 0.006339479106133446, "reads_retained_percent": 99.99366052089387}, {"sample": "GCA_003798385.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 12, "name": "Yersinia pseudotuberculosis strain FDAARGOS_584 chromosome, complete genome", "reads_in": 312628, "reads_removed_percent": 0.00383842777998132, "reads_retained_percent": 99.99616157222002}, {"sample": "GCA_016028735.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 12, "name": "Oligella ureolytica strain FDAARGOS_872 chromosome, complete genome", "reads_in": 188856, "reads_removed_percent": 0.006354047528275512, "reads_retained_percent": 99.99364595247172}, {"sample": "GCA_013347325.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 12, "name": "Cupriavidus gilardii strain FDAARGOS_639 chromosome 1, complete sequence", "reads_in": 384002, "reads_removed_percent": 0.003124983724043104, "reads_retained_percent": 99.99687501627595}, {"sample": "GCA_016889445.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Corynebacterium striatum strain FDAARGOS_1197 chromosome, complete genome", "reads_in": 200318, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013267715.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 12, "name": "Staphylococcus warneri strain FDAARGOS_754 chromosome, complete genome", "reads_in": 164000, "reads_removed_percent": 0.007317073170731707, "reads_retained_percent": 99.99268292682927}, {"sample": "GCA_016599775.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Stutzerimonas balearica strain FDAARGOS_1013 chromosome, complete genome", "reads_in": 294082, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016724805.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Sutterella wadsworthensis strain FDAARGOS_1159 chromosome, complete genome", "reads_in": 201768, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_012273615.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 12, "name": "Chryseobacterium gallinarum strain FDAARGOS_636 chromosome, complete genome", "reads_in": 289264, "reads_removed_percent": 0.004148459538691299, "reads_retained_percent": 99.9958515404613}, {"sample": "GCA_009806615.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 12, "name": "Bacillus anthracis strain FDAARGOS_701 chromosome, complete genome", "reads_in": 348494, "reads_removed_percent": 0.003443387834510781, "reads_retained_percent": 99.99655661216549}, {"sample": "GCA_003956025.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 12, "name": "Staphylococcus cohnii strain FDAARGOS_538 chromosome, complete genome", "reads_in": 180056, "reads_removed_percent": 0.0066645932376593955, "reads_retained_percent": 99.99333540676234}, {"sample": "GCA_009806595.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 12, "name": "Bacillus anthracis strain FDAARGOS_698 chromosome, complete genome", "reads_in": 348384, "reads_removed_percent": 0.0034444750620005516, "reads_retained_percent": 99.996555524938}, {"sample": "GCA_008693365.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 12, "name": "Yersinia pseudotuberculosis strain FDAARGOS_665 chromosome, complete genome", "reads_in": 291134, "reads_removed_percent": 0.004121813323074598, "reads_retained_percent": 99.99587818667692}, {"sample": "GCA_016889965.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 12, "name": "Staphylococcus condimenti strain FDAARGOS_1205 chromosome, complete genome", "reads_in": 180552, "reads_removed_percent": 0.006646284726837697, "reads_retained_percent": 99.99335371527316}, {"sample": "GCA_016890065.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 12, "name": "Acinetobacter pittii strain FDAARGOS_1216 chromosome, complete genome", "reads_in": 261924, "reads_removed_percent": 0.0045814816511659875, "reads_retained_percent": 99.99541851834883}, {"sample": "GCA_008693745.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 12, "name": "Vibrio parahaemolyticus strain FDAARGOS_662 chromosome 1, complete sequence", "reads_in": 343608, "reads_removed_percent": 0.0034923517496682262, "reads_retained_percent": 99.99650764825033}, {"sample": "GCA_002591095.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 12, "name": "Yersinia frederiksenii strain FDAARGOS_417 chromosome, complete genome", "reads_in": 333378, "reads_removed_percent": 0.003599517664632939, "reads_retained_percent": 99.99640048233536}, {"sample": "GCA_020097275.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 12, "name": "Bacteroides fragilis strain FDAARGOS_1512 chromosome, complete genome", "reads_in": 349070, "reads_removed_percent": 0.0034377059042598906, "reads_retained_percent": 99.99656229409574}, {"sample": "GCA_019930605.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 12, "name": "Prevotella nigrescens strain FDAARGOS_1486 chromosome, complete genome", "reads_in": 178206, "reads_removed_percent": 0.006733780007407158, "reads_retained_percent": 99.99326621999259}, {"sample": "GCA_019047885.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 12, "name": "Providencia rettgeri strain FDAARGOS 1449 chromosome, complete genome", "reads_in": 318484, "reads_removed_percent": 0.0037678501902764345, "reads_retained_percent": 99.99623214980973}, {"sample": "GCA_013267335.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 12, "name": "Bacillus thuringiensis strain FDAARGOS_793 chromosome, complete genome", "reads_in": 354256, "reads_removed_percent": 0.0033873808771058213, "reads_retained_percent": 99.9966126191229}, {"sample": "GCA_016128115.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 14, "name": "Providencia stuartii strain FDAARGOS_1040 chromosome, complete genome", "reads_in": 289014, "reads_removed_percent": 0.004844055997287328, "reads_retained_percent": 99.99515594400272}, {"sample": "GCA_016128035.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 14, "name": "Alcaligenes faecalis strain FDAARGOS_1024 chromosome, complete genome", "reads_in": 271528, "reads_removed_percent": 0.005156006010429864, "reads_retained_percent": 99.99484399398958}, {"sample": "GCA_019048545.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 14, "name": "Providencia rettgeri strain FDAARGOS 1451 chromosome, complete genome", "reads_in": 292254, "reads_removed_percent": 0.0047903535965290464, "reads_retained_percent": 99.99520964640347}, {"sample": "GCA_016127715.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 16, "name": "Staphylococcus saccharolyticus strain FDAARGOS_1074 chromosome, complete genome", "reads_in": 156434, "reads_removed_percent": 0.010227955559533094, "reads_retained_percent": 99.98977204444047}, {"sample": "GCA_016890025.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 16, "name": "Staphylococcus carnosus strain FDAARGOS_1204 chromosome, complete genome", "reads_in": 176972, "reads_removed_percent": 0.009040978233844902, "reads_retained_percent": 99.99095902176616}, {"sample": "GCA_016728645.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Corynebacterium glucuronolyticum strain FDAARGOS_1111 chromosome, complete genome", "reads_in": 196126, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016027975.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 16, "name": "Lactococcus lactis strain FDAARGOS_887 chromosome, complete genome", "reads_in": 172198, "reads_removed_percent": 0.009291629403361246, "reads_retained_percent": 99.99070837059664}, {"sample": "GCA_016728625.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Corynebacterium propinquum strain FDAARGOS_1113 chromosome, complete genome", "reads_in": 164086, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001018835.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 16, "name": "Staphylococcus aureus strain FDAARGOS_19 chromosome, complete genome", "reads_in": 191936, "reads_removed_percent": 0.00833611203734578, "reads_retained_percent": 99.99166388796266}, {"sample": "GCA_016028835.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 16, "name": "Lactococcus lactis strain FDAARGOS_865 chromosome, complete genome", "reads_in": 171814, "reads_removed_percent": 0.009312395963076349, "reads_retained_percent": 99.99068760403692}, {"sample": "GCA_016727385.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 16, "name": "Staphylococcus epidermidis strain FDAARGOS_1083 chromosome, complete genome", "reads_in": 165308, "reads_removed_percent": 0.009678902412466425, "reads_retained_percent": 99.99032109758754}, {"sample": "GCA_016728365.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Corynebacterium tuberculostearicum strain FDAARGOS_1117 chromosome, complete genome", "reads_in": 163456, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013267775.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 16, "name": "Bacillus cereus strain FDAARGOS_797 chromosome, complete genome", "reads_in": 374962, "reads_removed_percent": 0.004267099066038692, "reads_retained_percent": 99.99573290093396}, {"sample": "GCA_002208945.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 16, "name": "Enterococcus faecalis strain FDAARGOS_338 chromosome, complete genome", "reads_in": 190734, "reads_removed_percent": 0.008388645967682741, "reads_retained_percent": 99.99161135403232}, {"sample": "GCA_016728205.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Corynebacterium striatum strain FDAARGOS_1116 chromosome, complete genome", "reads_in": 177712, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013267455.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 16, "name": "Bacillus cereus strain FDAARGOS_781 chromosome, complete genome", "reads_in": 376156, "reads_removed_percent": 0.004253554376375759, "reads_retained_percent": 99.99574644562362}, {"sample": "GCA_003812965.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 16, "name": "Burkholderia multivorans strain FDAARGOS_496 chromosome 1, complete sequence", "reads_in": 519760, "reads_removed_percent": 0.003078343851008158, "reads_retained_percent": 99.996921656149}, {"sample": "GCA_016728665.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Corynebacterium propinquum strain FDAARGOS_1112 chromosome, complete genome", "reads_in": 165342, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735365.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 16, "name": "Blautia massiliensis strain FDAARGOS_1576 chromosome, complete genome", "reads_in": 248852, "reads_removed_percent": 0.0064295243759342905, "reads_retained_percent": 99.99357047562407}, {"sample": "GCA_021183585.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 16, "name": "Nocardia asteroides strain FDAARGOS_1622 chromosome, complete genome", "reads_in": 465562, "reads_removed_percent": 0.003436706604061328, "reads_retained_percent": 99.99656329339594}, {"sample": "GCA_008693565.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 16, "name": "Klebsiella michiganensis strain FDAARGOS_647 chromosome, complete genome", "reads_in": 443772, "reads_removed_percent": 0.0036054550534959395, "reads_retained_percent": 99.9963945449465}, {"sample": "GCA_001018655.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 16, "name": "Staphylococcus aureus strain FDAARGOS_6 chromosome, complete genome", "reads_in": 183952, "reads_removed_percent": 0.008697921196833956, "reads_retained_percent": 99.99130207880317}, {"sample": "GCA_008693805.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 16, "name": "Providencia stuartii strain FDAARGOS_645 chromosome, complete genome", "reads_in": 301984, "reads_removed_percent": 0.00529829394934831, "reads_retained_percent": 99.99470170605065}, {"sample": "GCA_020735445.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 16, "name": "Prevotella copri DSM 18205 strain FDAARGOS_1573 chromosome, complete genome", "reads_in": 236774, "reads_removed_percent": 0.006757498711851808, "reads_retained_percent": 99.99324250128815}, {"sample": "GCA_016888985.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Chromobacterium violaceum strain FDAARGOS_1274 chromosome, complete genome", "reads_in": 320536, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002393505.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 18, "name": "Providencia alcalifaciens strain FDAARGOS_408 chromosome, complete genome", "reads_in": 266008, "reads_removed_percent": 0.00676671378304412, "reads_retained_percent": 99.99323328621695}, {"sample": "GCA_016888965.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1261 chromosome, complete genome", "reads_in": 336674, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889145.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 18, "name": "Staphylococcus aureus strain FDAARGOS_1246 chromosome, complete genome", "reads_in": 192792, "reads_removed_percent": 0.00933648699116146, "reads_retained_percent": 99.99066351300884}, {"sample": "GCA_006364735.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 18, "name": "Pseudomonas aeruginosa strain FDAARGOS_767 chromosome, complete genome", "reads_in": 417578, "reads_removed_percent": 0.004310571917102913, "reads_retained_percent": 99.9956894280829}, {"sample": "GCA_016889905.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 18, "name": "Escherichia coli strain FDAARGOS_1259 chromosome, complete genome", "reads_in": 348288, "reads_removed_percent": 0.005168136714443219, "reads_retained_percent": 99.99483186328555}, {"sample": "GCA_000783595.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 18, "name": "Proteus mirabilis strain FDAARGOS_80 chromosome, complete genome", "reads_in": 269616, "reads_removed_percent": 0.00667616165212747, "reads_retained_percent": 99.99332383834788}, {"sample": "GCA_020736045.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 18, "name": "Haemophilus influenzae strain FDAARGOS_1560 chromosome, complete genome", "reads_in": 126044, "reads_removed_percent": 0.014280727365047126, "reads_retained_percent": 99.98571927263495}, {"sample": "GCA_013267475.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 18, "name": "Bacillus cereus strain FDAARGOS_780 chromosome, complete genome", "reads_in": 376156, "reads_removed_percent": 0.004785248673422729, "reads_retained_percent": 99.99521475132657}, {"sample": "GCA_003955795.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 16, "name": "Francisella tularensis strain FDAARGOS_598 chromosome, complete genome", "reads_in": 126240, "reads_removed_percent": 0.012674271229404309, "reads_retained_percent": 99.9873257287706}, {"sample": "GCA_016725645.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 18, "name": "Sphingobacterium spiritivorum strain FDAARGOS_1144 chromosome, complete genome", "reads_in": 342552, "reads_removed_percent": 0.005254676662229384, "reads_retained_percent": 99.99474532333777}, {"sample": "GCA_016728685.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Corynebacterium aurimucosum strain FDAARGOS_1110 chromosome, complete genome", "reads_in": 188632, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016906205.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 16, "name": "Streptomyces californicus strain FDAARGOS_1212 chromosome", "reads_in": 538842, "reads_removed_percent": 0.002969330527315985, "reads_retained_percent": 99.99703066947268}, {"sample": "GCA_019930805.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 14, "name": "Legionella pneumophila strain FDAARGOS_1482 chromosome, complete genome", "reads_in": 229162, "reads_removed_percent": 0.006109215314930049, "reads_retained_percent": 99.99389078468506}, {"sample": "GCA_016027715.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 14, "name": "Lactococcus garvieae strain FDAARGOS_893 chromosome, complete genome", "reads_in": 140684, "reads_removed_percent": 0.009951380398623866, "reads_retained_percent": 99.99004861960138}, {"sample": "GCA_001558395.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Bordetella pertussis strain FDAARGOS_179 chromosome, complete genome", "reads_in": 273390, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013343075.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 14, "name": "Bacillus thuringiensis strain FDAARGOS_791 chromosome, complete genome", "reads_in": 355852, "reads_removed_percent": 0.0039342198442048944, "reads_retained_percent": 99.99606578015579}, {"sample": "GCA_016725285.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 14, "name": "Staphylococcus pasteuri strain FDAARGOS_1151 chromosome, complete genome", "reads_in": 175648, "reads_removed_percent": 0.007970486427400255, "reads_retained_percent": 99.9920295135726}, {"sample": "GCA_016127455.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 14, "name": "Staphylococcus equorum strain FDAARGOS_1068 chromosome, complete genome", "reads_in": 184822, "reads_removed_percent": 0.007574855807209099, "reads_retained_percent": 99.9924251441928}, {"sample": "GCA_016889285.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Kytococcus sedentarius strain FDAARGOS_1200 chromosome, complete genome", "reads_in": 163250, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002953335.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 14, "name": "Vibrio diabolicus strain FDAARGOS_96 chromosome 1, complete sequence", "reads_in": 344910, "reads_removed_percent": 0.004059029891855847, "reads_retained_percent": 99.99594097010815}, {"sample": "GCA_002951115.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 14, "name": "Stenotrophomonas maltophilia strain FDAARGOS_92 chromosome, complete genome", "reads_in": 321348, "reads_removed_percent": 0.004356647621892777, "reads_retained_percent": 99.99564335237811}, {"sample": "GCA_016889645.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Salmonella enterica strain FDAARGOS_1271 chromosome, complete genome", "reads_in": 320272, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002983645.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 14, "name": "Escherichia coli O104:H4 strain FDAARGOS_349 chromosome, complete genome", "reads_in": 366744, "reads_removed_percent": 0.0038173766987326307, "reads_retained_percent": 99.99618262330127}, {"sample": "GCA_016889205.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Burkholderia dolosa strain FDAARGOS_1272 chromosome 1, complete sequence", "reads_in": 427274, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812905.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 14, "name": "Pseudomonas aeruginosa strain FDAARGOS_501 chromosome, complete genome", "reads_in": 457722, "reads_removed_percent": 0.003058625104320963, "reads_retained_percent": 99.99694137489568}, {"sample": "GCA_002983665.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 14, "name": "Providencia stuartii strain FDAARGOS_291 chromosome, complete genome", "reads_in": 292966, "reads_removed_percent": 0.0047787115228388275, "reads_retained_percent": 99.99522128847715}, {"sample": "GCA_002208905.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 16, "name": "Staphylococcus saprophyticus strain FDAARGOS_336 chromosome, complete genome", "reads_in": 175718, "reads_removed_percent": 0.009105498582956782, "reads_retained_percent": 99.99089450141705}, {"sample": "GCA_002208785.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 14, "name": "Bacillus anthracis strain FDAARGOS_341 chromosome, complete genome", "reads_in": 360696, "reads_removed_percent": 0.003881384878124515, "reads_retained_percent": 99.99611861512187}, {"sample": "GCA_002386205.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 14, "name": "Burkholderia thailandensis strain FDAARGOS_426 chromosome 1, complete sequence", "reads_in": 448658, "reads_removed_percent": 0.0031204168876961965, "reads_retained_percent": 99.9968795831123}, {"sample": "GCA_006364775.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 14, "name": "Staphylococcus aureus strain FDAARGOS_766 chromosome, complete genome", "reads_in": 199754, "reads_removed_percent": 0.007008620603342111, "reads_retained_percent": 99.99299137939666}, {"sample": "GCA_003827915.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 14, "name": "Staphylococcus aureus strain FDAARGOS_504 chromosome, complete genome", "reads_in": 192972, "reads_removed_percent": 0.0072549385403063655, "reads_retained_percent": 99.9927450614597}, {"sample": "GCA_002277995.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 16, "name": "Elizabethkingia anophelis strain FDAARGOS_198 chromosome, complete genome", "reads_in": 271614, "reads_removed_percent": 0.005890712555317473, "reads_retained_percent": 99.99410928744469}, {"sample": "GCA_016889785.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Corynebacterium glucuronolyticum strain FDAARGOS_1191 chromosome, complete genome", "reads_in": 192446, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889765.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Corynebacterium minutissimum strain FDAARGOS_1196 chromosome, complete genome", "reads_in": 181688, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019930705.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 16, "name": "Haemophilus influenzae strain FDAARGOS_1479 chromosome, complete genome", "reads_in": 126840, "reads_removed_percent": 0.01261431725007884, "reads_retained_percent": 99.98738568274992}, {"sample": "GCA_016724565.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 16, "name": "Staphylococcus condimenti strain FDAARGOS_1148 chromosome, complete genome", "reads_in": 177956, "reads_removed_percent": 0.008990986535997662, "reads_retained_percent": 99.99100901346401}, {"sample": "GCA_016725265.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 16, "name": "Staphylococcus equorum strain FDAARGOS_1150 chromosome, complete genome", "reads_in": 186494, "reads_removed_percent": 0.008579364483575878, "reads_retained_percent": 99.99142063551642}, {"sample": "GCA_016725465.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 16, "name": "Sphingobacterium multivorum strain FDAARGOS_1142 chromosome, complete genome", "reads_in": 436790, "reads_removed_percent": 0.0036630875248975484, "reads_retained_percent": 99.9963369124751}, {"sample": "GCA_019047125.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 16, "name": "Aeromonas sp. FDAARGOS 1411 chromosome, complete genome", "reads_in": 302630, "reads_removed_percent": 0.005286984106004032, "reads_retained_percent": 99.99471301589399}, {"sample": "GCA_016026915.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 16, "name": "Staphylococcus epidermidis strain FDAARGOS_913 chromosome, complete genome", "reads_in": 175120, "reads_removed_percent": 0.009136592051164915, "reads_retained_percent": 99.99086340794884}, {"sample": "GCA_016728105.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Corynebacterium striatum strain FDAARGOS_1115 chromosome, complete genome", "reads_in": 193656, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016906245.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 16, "name": "Streptomyces californicus strain FDAARGOS_1213 chromosome", "reads_in": 538938, "reads_removed_percent": 0.002968801606121669, "reads_retained_percent": 99.99703119839388}, {"sample": "GCA_002984195.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 14, "name": "Providencia rettgeri strain FDAARGOS_330 chromosome, complete genome", "reads_in": 301240, "reads_removed_percent": 0.0046474571770017264, "reads_retained_percent": 99.995352542823}, {"sample": "GCA_001558795.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Staphylococcus aureus strain FDAARGOS_159 chromosome, complete genome", "reads_in": 189278, "reads_removed_percent": 0.004226587347710775, "reads_retained_percent": 99.99577341265228}, {"sample": "GCA_009729955.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Salmonella enterica strain FDAARGOS_688 chromosome, complete genome", "reads_in": 308930, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002209145.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Jonesia denitrificans strain FDAARGOS_301 chromosome, complete genome", "reads_in": 185920, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016026635.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Staphylococcus pseudintermedius strain FDAARGOS_930 chromosome, complete genome", "reads_in": 170856, "reads_removed_percent": 0.00234115278363066, "reads_retained_percent": 99.99765884721637}, {"sample": "GCA_016127015.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Corynebacterium tuberculostearicum strain FDAARGOS_993 chromosome, complete genome", "reads_in": 160572, "reads_removed_percent": 0.0012455471688712853, "reads_retained_percent": 99.99875445283112}, {"sample": "GCA_016127035.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Leuconostoc pseudomesenteroides strain FDAARGOS_1004 chromosome, complete genome", "reads_in": 139694, "reads_removed_percent": 0.0014317007172820594, "reads_retained_percent": 99.99856829928272}, {"sample": "GCA_019047985.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Klebsiella quasipneumoniae strain FDAARGOS 1443 chromosome, complete genome", "reads_in": 359656, "reads_removed_percent": 0.0005560869275085081, "reads_retained_percent": 99.9994439130725}, {"sample": "GCA_016903475.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1315 chromosome, complete genome", "reads_in": 368824, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016028635.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Stutzerimonas degradans strain FDAARGOS_876 chromosome, complete genome", "reads_in": 269474, "reads_removed_percent": 0.0007421866302500427, "reads_retained_percent": 99.99925781336975}, {"sample": "GCA_016903175.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Bordetella pertussis strain FDAARGOS_1369 chromosome, complete genome", "reads_in": 273602, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903455.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1318 chromosome, complete genome", "reads_in": 346980, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889805.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Rhodanobacter sp. FDAARGOS 1247 strain FDAARGOS_1247 chromosome, complete genome", "reads_in": 261508, "reads_removed_percent": 0.0007647949584716338, "reads_retained_percent": 99.99923520504153}, {"sample": "GCA_016028495.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Salmonella enterica strain FDAARGOS_878 chromosome, complete genome", "reads_in": 331248, "reads_removed_percent": 0.0006037772303530889, "reads_retained_percent": 99.99939622276965}, {"sample": "GCA_016027855.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Atlantibacter hermannii strain FDAARGOS_888 chromosome, complete genome", "reads_in": 303658, "reads_removed_percent": 0.000658635702006863, "reads_retained_percent": 99.99934136429799}, {"sample": "GCA_016127095.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Corynebacterium urealyticum strain FDAARGOS_996 chromosome, complete genome", "reads_in": 156730, "reads_removed_percent": 0.0012760798826006508, "reads_retained_percent": 99.9987239201174}, {"sample": "GCA_020735725.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Alistipes senegalensis strain FDAARGOS_1578 chromosome, complete genome", "reads_in": 268220, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016028715.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Neisseria cinerea strain FDAARGOS_871 chromosome, complete genome", "reads_in": 122194, "reads_removed_percent": 0.0016367415748727435, "reads_retained_percent": 99.99836325842513}, {"sample": "GCA_016028775.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Moraxella nonliquefaciens strain FDAARGOS_869 chromosome, complete genome", "reads_in": 156250, "reads_removed_percent": 0.0012799999999999999, "reads_retained_percent": 99.99872}, {"sample": "GCA_001558355.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Salmonella enterica strain FDAARGOS_94 chromosome, complete genome", "reads_in": 330094, "reads_removed_percent": 0.000605888019776185, "reads_retained_percent": 99.99939411198022}, {"sample": "GCA_006364495.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Bacillus subtilis strain FDAARGOS_606 chromosome, complete genome", "reads_in": 269708, "reads_removed_percent": 0.001483085410888813, "reads_retained_percent": 99.99851691458912}, {"sample": "GCA_016726205.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Acinetobacter ursingii strain FDAARGOS_1095 chromosome, complete genome", "reads_in": 240834, "reads_removed_percent": 0.001660895056345865, "reads_retained_percent": 99.99833910494365}, {"sample": "GCA_016903275.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Escherichia coli strain FDAARGOS_1282 chromosome, complete genome", "reads_in": 352200, "reads_removed_percent": 0.001135718341851221, "reads_retained_percent": 99.99886428165814}, {"sample": "GCA_001558895.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Morganella morganii strain FDAARGOS_172 chromosome, complete genome", "reads_in": 260538, "reads_removed_percent": 0.001535284680161819, "reads_retained_percent": 99.99846471531984}, {"sample": "GCA_002754815.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Roseomonas sp. FDAARGOS_362 chromosome, complete genome", "reads_in": 344946, "reads_removed_percent": 0.0011596017927443716, "reads_retained_percent": 99.99884039820725}, {"sample": "GCA_006364675.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Staphylococcus aureus strain FDAARGOS_773 chromosome, complete genome", "reads_in": 185804, "reads_removed_percent": 0.002152806182859357, "reads_retained_percent": 99.99784719381714}, {"sample": "GCA_002951195.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Pandoraea apista strain FDAARGOS_126 chromosome, complete genome", "reads_in": 355100, "reads_removed_percent": 0.0011264432554210083, "reads_retained_percent": 99.99887355674458}, {"sample": "GCA_016725325.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Sphingobacterium spiritivorum strain FDAARGOS_1145 chromosome, complete genome", "reads_in": 342354, "reads_removed_percent": 0.001168381266174778, "reads_retained_percent": 99.99883161873383}, {"sample": "GCA_020097655.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Corynebacterium coyleae strain FDAARGOS_1492 chromosome, complete genome", "reads_in": 172188, "reads_removed_percent": 0.002323042256138639, "reads_retained_percent": 99.99767695774386}, {"sample": "GCA_016904115.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Escherichia coli strain FDAARGOS_1290 chromosome, complete genome", "reads_in": 347130, "reads_removed_percent": 0.0011523060524875406, "reads_retained_percent": 99.99884769394751}, {"sample": "GCA_019047725.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Escherichia coli strain FDAARGOS 1374 chromosome, complete genome", "reads_in": 349518, "reads_removed_percent": 0.001144433190851401, "reads_retained_percent": 99.99885556680915}, {"sample": "GCA_002951635.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Klebsiella pneumoniae strain FDAARGOS_156 chromosome, complete genome", "reads_in": 370044, "reads_removed_percent": 0.0010809525353741717, "reads_retained_percent": 99.99891904746463}, {"sample": "GCA_002208745.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Pseudomonas protegens strain FDAARGOS_307 chromosome, complete genome", "reads_in": 471722, "reads_removed_percent": 0.0008479570594545092, "reads_retained_percent": 99.99915204294055}, {"sample": "GCA_001558735.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Pantoea vagans strain FDAARGOS_160 chromosome, complete genome", "reads_in": 320590, "reads_removed_percent": 0.0012476995539474093, "reads_retained_percent": 99.99875230044606}, {"sample": "GCA_002208765.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Psychrobacter cryohalolentis strain FDAARGOS_308 chromosome, complete genome", "reads_in": 207242, "reads_removed_percent": 0.0019301106918481776, "reads_retained_percent": 99.99806988930816}, {"sample": "GCA_016904095.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Klebsiella pneumoniae strain FDAARGOS_1319 chromosome, complete genome", "reads_in": 368102, "reads_removed_percent": 0.0010866553292293985, "reads_retained_percent": 99.99891334467077}, {"sample": "GCA_003855135.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Shigella flexneri strain FDAARGOS_535 chromosome, complete genome", "reads_in": 324836, "reads_removed_percent": 0.0012313906094152125, "reads_retained_percent": 99.99876860939058}, {"sample": "GCA_002208665.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Haemophilus] ducreyi strain FDAARGOS_297 chromosome, complete genome", "reads_in": 113420, "reads_removed_percent": 0.003526714865103157, "reads_retained_percent": 99.9964732851349}, {"sample": "GCA_019047945.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Enterobacter hormaechei strain FDAARGOS 1434 chromosome, complete genome", "reads_in": 318836, "reads_removed_percent": 0.0006272817373194997, "reads_retained_percent": 99.99937271826268}, {"sample": "GCA_016903095.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Bordetella pertussis strain FDAARGOS_1368 chromosome, complete genome", "reads_in": 273672, "reads_removed_percent": 0.0007308018357742114, "reads_retained_percent": 99.99926919816423}, {"sample": "GCA_001558515.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Vibrio vulnificus strain FDAARGOS_119 chromosome 1, complete sequence", "reads_in": 331920, "reads_removed_percent": 0.0006025548324897565, "reads_retained_percent": 99.99939744516752}, {"sample": "GCA_016904215.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Bordetella pertussis strain FDAARGOS_1367 chromosome, complete genome", "reads_in": 273706, "reads_removed_percent": 0.00073071105492755, "reads_retained_percent": 99.99926928894507}, {"sample": "GCA_016890005.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1250 chromosome, complete genome", "reads_in": 337234, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904195.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Escherichia coli strain FDAARGOS_1296 chromosome, complete genome", "reads_in": 333376, "reads_removed_percent": 0.0005999232098291418, "reads_retained_percent": 99.99940007679017}, {"sample": "GCA_019048005.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Escherichia coli strain FDAARGOS_1385 chromosome, complete genome", "reads_in": 348010, "reads_removed_percent": 0.0005746961294215683, "reads_retained_percent": 99.99942530387058}, {"sample": "GCA_001558415.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Vibrio fluvialis strain ATCC 33809 chromosome 1, complete sequence", "reads_in": 321850, "reads_removed_percent": 0.0006214074879602299, "reads_retained_percent": 99.99937859251204}, {"sample": "GCA_016904055.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Escherichia coli strain FDAARGOS_1293 chromosome, complete genome", "reads_in": 342718, "reads_removed_percent": 0.000583570165558856, "reads_retained_percent": 99.99941642983444}, {"sample": "GCA_016889185.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Escherichia coli strain FDAARGOS_1257 chromosome, complete genome", "reads_in": 354122, "reads_removed_percent": 0.000564777110713257, "reads_retained_percent": 99.99943522288929}, {"sample": "GCA_020735505.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Mycobacteroides abscessus strain FDAARGOS_1602 chromosome, complete genome", "reads_in": 322236, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903915.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Escherichia coli strain FDAARGOS_1300 chromosome, complete genome", "reads_in": 363838, "reads_removed_percent": 0.0005496951940149187, "reads_retained_percent": 99.99945030480599}, {"sample": "GCA_016728085.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Escherichia coli strain FDAARGOS_1077 chromosome, complete genome", "reads_in": 339426, "reads_removed_percent": 0.0005892300530896278, "reads_retained_percent": 99.9994107699469}, {"sample": "GCA_016903615.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Escherichia coli strain FDAARGOS_1304 chromosome, complete genome", "reads_in": 351638, "reads_removed_percent": 0.0005687667430709992, "reads_retained_percent": 99.99943123325693}, {"sample": "GCA_016903595.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Escherichia coli strain FDAARGOS_1279 chromosome, complete genome", "reads_in": 356446, "reads_removed_percent": 0.0005610948081897399, "reads_retained_percent": 99.99943890519181}, {"sample": "GCA_016903755.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Klebsiella pneumoniae strain FDAARGOS_1317 chromosome, complete genome", "reads_in": 351878, "reads_removed_percent": 0.0005683788131113624, "reads_retained_percent": 99.99943162118689}, {"sample": "GCA_016903875.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1329 chromosome, complete genome", "reads_in": 370504, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889865.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Escherichia coli strain FDAARGOS_1253 chromosome, complete genome", "reads_in": 313026, "reads_removed_percent": 0.0006389245621769438, "reads_retained_percent": 99.99936107543782}, {"sample": "GCA_020735345.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Mycobacteroides abscessus subsp. abscessus strain FDAARGOS_1605 chromosome, complete genome", "reads_in": 335566, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889085.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Escherichia coli strain FDAARGOS_1256 chromosome, complete genome", "reads_in": 346170, "reads_removed_percent": 0.0005777508160730278, "reads_retained_percent": 99.99942224918392}, {"sample": "GCA_016027695.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Enterobacter asburiae strain FDAARGOS_892 chromosome, complete genome", "reads_in": 317798, "reads_removed_percent": 0.0006293305810609255, "reads_retained_percent": 99.99937066941894}, {"sample": "GCA_001559235.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Citrobacter sp. FDAARGOS_156 strain FDAARGOS_155 chromosome, complete genome", "reads_in": 315166, "reads_removed_percent": 0.0006345862180565162, "reads_retained_percent": 99.99936541378194}, {"sample": "GCA_020735565.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Hoylesella buccalis strain FDAARGOS_1588 chromosome, complete genome", "reads_in": 220560, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903115.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Klebsiella pneumoniae strain FDAARGOS_1327 chromosome, complete genome", "reads_in": 366464, "reads_removed_percent": 0.0005457561997904296, "reads_retained_percent": 99.99945424380022}, {"sample": "GCA_001559255.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Hafnia paralvei strain FDAARGOS_158 chromosome, complete genome", "reads_in": 300914, "reads_removed_percent": 0.0006646417248782043, "reads_retained_percent": 99.99933535827512}, {"sample": "GCA_019047325.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Aeromonas sp. FDAARGOS 1403 chromosome, complete genome", "reads_in": 321318, "reads_removed_percent": 0.0006224363403232935, "reads_retained_percent": 99.99937756365968}, {"sample": "GCA_016726945.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Moraxella osloensis strain FDAARGOS_1130 chromosome, complete genome", "reads_in": 169780, "reads_removed_percent": 0.0011779950524207798, "reads_retained_percent": 99.99882200494758}, {"sample": "GCA_020735425.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Mycobacteroides abscessus subsp. abscessus strain FDAARGOS_1604 chromosome, complete genome", "reads_in": 339370, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016028655.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Stutzerimonas stutzeri strain FDAARGOS_875 chromosome, complete genome", "reads_in": 302358, "reads_removed_percent": 0.0006614675318661983, "reads_retained_percent": 99.99933853246813}, {"sample": "GCA_021183745.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Mycobacterium intracellulare strain FDAARGOS_1612 chromosome, complete genome", "reads_in": 371710, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016890085.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Chromobacterium violaceum strain FDAARGOS_1273 chromosome, complete genome", "reads_in": 320536, "reads_removed_percent": 0.0006239548755833978, "reads_retained_percent": 99.99937604512442}, {"sample": "GCA_019047465.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Lelliottia amnigena strain FDAARGOS 1444 chromosome, complete genome", "reads_in": 300368, "reads_removed_percent": 0.0006658498908006179, "reads_retained_percent": 99.9993341501092}, {"sample": "GCA_016726285.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Shigella boydii strain FDAARGOS_1139 chromosome, complete genome", "reads_in": 303264, "reads_removed_percent": 0.0006594914002321409, "reads_retained_percent": 99.99934050859977}, {"sample": "GCA_016726365.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Rothia aeria strain FDAARGOS_1137 chromosome, complete genome", "reads_in": 172262, "reads_removed_percent": 0.001161022163913109, "reads_retained_percent": 99.99883897783609}, {"sample": "GCA_020735985.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Bordetella holmesii strain FDAARGOS_1539 chromosome, complete genome", "reads_in": 245874, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735545.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Gordonia otitidis strain FDAARGOS_1600 chromosome, complete genome", "reads_in": 340588, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904375.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Bordetella pertussis strain FDAARGOS_1366 chromosome, complete genome", "reads_in": 273696, "reads_removed_percent": 0.0014614755056705248, "reads_retained_percent": 99.99853852449434}, {"sample": "GCA_019048305.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Acinetobacter lwoffii strain FDAARGOS 1393 chromosome, complete genome", "reads_in": 233634, "reads_removed_percent": 0.00171207957745876, "reads_retained_percent": 99.99828792042254}, {"sample": "GCA_020735805.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Helicobacter pylori strain FDAARGOS_1563 chromosome, complete genome", "reads_in": 110868, "reads_removed_percent": 0.003607894072230039, "reads_retained_percent": 99.99639210592777}, {"sample": "GCA_002073215.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Moraxella catarrhalis strain FDAARGOS_213 chromosome, complete genome", "reads_in": 127200, "reads_removed_percent": 0.003144654088050314, "reads_retained_percent": 99.99685534591195}, {"sample": "GCA_020735785.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Prevotella melaninogenica strain FDAARGOS_1566 chromosome 1, complete sequence", "reads_in": 211214, "reads_removed_percent": 0.001893813857035992, "reads_retained_percent": 99.99810618614296}, {"sample": "GCA_016903555.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Staphylococcus epidermidis strain FDAARGOS_1363 chromosome, complete genome", "reads_in": 167308, "reads_removed_percent": 0.002390800200827217, "reads_retained_percent": 99.99760919979917}, {"sample": "GCA_003050665.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Helicobacter pylori strain FDAARGOS_298 chromosome, complete genome", "reads_in": 112060, "reads_removed_percent": 0.003569516330537212, "reads_retained_percent": 99.99643048366946}, {"sample": "GCA_016027415.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Brevibacterium casei strain FDAARGOS_902 chromosome, complete genome", "reads_in": 264006, "reads_removed_percent": 0.0015151170806724091, "reads_retained_percent": 99.99848488291933}, {"sample": "GCA_016903355.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Streptococcus anginosus strain FDAARGOS_1357 chromosome, complete genome", "reads_in": 132094, "reads_removed_percent": 0.0030281466228594786, "reads_retained_percent": 99.99697185337715}, {"sample": "GCA_016903515.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Klebsiella pneumoniae strain FDAARGOS_1330 chromosome, complete genome", "reads_in": 365880, "reads_removed_percent": 0.0010932546190007652, "reads_retained_percent": 99.99890674538099}, {"sample": "GCA_002984285.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Salmonella enterica subsp. enterica serovar Typhimurium strain FDAARGOS_317 chromosome, complete genome", "reads_in": 346190, "reads_removed_percent": 0.001155434876801756, "reads_retained_percent": 99.99884456512319}, {"sample": "GCA_020735745.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Lachnospira eligens strain FDAARGOS_1570 chromosome, complete genome", "reads_in": 184852, "reads_removed_percent": 0.002163893276783589, "reads_retained_percent": 99.99783610672321}, {"sample": "GCA_019046925.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Aeromonas sp. FDAARGOS 1402 chromosome, complete genome", "reads_in": 328168, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735765.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Streptococcus anginosus subsp. anginosus strain FDAARGOS_1569 chromosome, complete genome", "reads_in": 125088, "reads_removed_percent": 0.003197748784855462, "reads_retained_percent": 99.99680225121514}, {"sample": "GCA_016728825.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Achromobacter xylosoxidans strain FDAARGOS_1091 chromosome, complete genome", "reads_in": 460132, "reads_removed_percent": 0.0008693157615640729, "reads_retained_percent": 99.99913068423844}, {"sample": "GCA_019046905.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Enterobacter bugandensis strain FDAARGOS 1427 chromosome, complete genome", "reads_in": 335846, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903495.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Klebsiella pneumoniae strain FDAARGOS_1312 chromosome, complete genome", "reads_in": 371120, "reads_removed_percent": 0.0010778184953653806, "reads_retained_percent": 99.99892218150464}, {"sample": "GCA_003812185.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Klebsiella aerogenes strain FDAARGOS_513 chromosome, complete genome", "reads_in": 351446, "reads_removed_percent": 0.0011381549370315781, "reads_retained_percent": 99.99886184506296}, {"sample": "GCA_019047805.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Corynebacterium freneyi strain FDAARGOS 1426 chromosome, complete genome", "reads_in": 192120, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735285.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Mycobacterium avium subsp. avium strain FDAARGOS_1607 chromosome, complete genome", "reads_in": 330462, "reads_removed_percent": 0.0012104266148604075, "reads_retained_percent": 99.99878957338514}, {"sample": "GCA_016028435.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Sphingomonas paucimobilis strain FDAARGOS_881 chromosome, complete genome", "reads_in": 301070, "reads_removed_percent": 0.0013285946789783106, "reads_retained_percent": 99.99867140532102}, {"sample": "GCA_019047965.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1378 chromosome, complete genome", "reads_in": 333822, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812885.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Pseudomonas aeruginosa strain FDAARGOS_571 chromosome, complete genome", "reads_in": 466652, "reads_removed_percent": 0.0008571697967650412, "reads_retained_percent": 99.99914283020324}, {"sample": "GCA_016026375.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Escherichia coli strain FDAARGOS_941 chromosome, complete genome", "reads_in": 337554, "reads_removed_percent": 0.0011849955858914427, "reads_retained_percent": 99.99881500441411}, {"sample": "GCA_020099375.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Acinetobacter variabilis strain FDAARGOS_1487 chromosome, complete genome", "reads_in": 232316, "reads_removed_percent": 0.0017217927305910913, "reads_retained_percent": 99.99827820726941}, {"sample": "GCA_002953355.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Vibrio diabolicus strain FDAARGOS_99 chromosome 1, complete sequence", "reads_in": 342120, "reads_removed_percent": 0.00116918040453642, "reads_retained_percent": 99.99883081959547}, {"sample": "GCA_002209105.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Escherichia coli O104:H4 strain FDAARGOS_348 chromosome, complete genome", "reads_in": 362832, "reads_removed_percent": 0.0011024385941703048, "reads_retained_percent": 99.99889756140583}, {"sample": "GCA_019047785.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Enterobacter cancerogenus strain FDAARGOS 1428 chromosome, complete genome", "reads_in": 321448, "reads_removed_percent": 0.0012443692292376993, "reads_retained_percent": 99.99875563077076}, {"sample": "GCA_016028255.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Streptococcus oralis strain FDAARGOS_885 chromosome, complete genome", "reads_in": 134954, "reads_removed_percent": 0.0029639729092876092, "reads_retained_percent": 99.99703602709072}, {"sample": "GCA_002953395.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Vibrio diabolicus strain FDAARGOS_105 chromosome 1, complete sequence", "reads_in": 361742, "reads_removed_percent": 0.0011057604591117427, "reads_retained_percent": 99.99889423954089}, {"sample": "GCA_016726765.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Pediococcus acidilactici strain FDAARGOS_1133 chromosome, complete genome", "reads_in": 130226, "reads_removed_percent": 0.0030715832475849674, "reads_retained_percent": 99.99692841675241}, {"sample": "GCA_016889925.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Bacteroides fragilis strain FDAARGOS_1225 chromosome, complete genome", "reads_in": 349930, "reads_removed_percent": 0.0011430857600091447, "reads_retained_percent": 99.99885691424}, {"sample": "GCA_019047825.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Streptococcus mitis strain FDAARGOS 1456 chromosome, complete genome", "reads_in": 124590, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812725.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Streptococcus pyogenes strain FDAARGOS_514 chromosome, complete genome", "reads_in": 126808, "reads_removed_percent": 0.003154375118289067, "reads_retained_percent": 99.99684562488171}, {"sample": "GCA_019047905.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1388 chromosome, complete genome", "reads_in": 345864, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_000783435.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Achromobacter insolitus strain FDAARGOS_88 chromosome, complete genome", "reads_in": 459536, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002983965.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Salmonella enterica subsp. enterica serovar Typhimurium strain FDAARGOS_320 chromosome, complete genome", "reads_in": 335804, "reads_removed_percent": 0.0011911710402496695, "reads_retained_percent": 99.99880882895975}, {"sample": "GCA_003812145.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Enterobacter roggenkampii strain FDAARGOS_523 chromosome, complete genome", "reads_in": 346032, "reads_removed_percent": 0.001155962454339483, "reads_retained_percent": 99.99884403754567}, {"sample": "GCA_019047505.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Streptococcus gordonii strain FDAARGOS 1455 chromosome, complete genome", "reads_in": 143654, "reads_removed_percent": 0.002784468236178596, "reads_retained_percent": 99.99721553176383}, {"sample": "GCA_016724885.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Streptococcus dysgalactiae strain FDAARGOS_1157 chromosome, complete genome", "reads_in": 143412, "reads_removed_percent": 0.0027891668758541826, "reads_retained_percent": 99.99721083312414}, {"sample": "GCA_016904235.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Escherichia coli strain FDAARGOS_1287 chromosome, complete genome", "reads_in": 369658, "reads_removed_percent": 0.0010820812751245745, "reads_retained_percent": 99.99891791872487}, {"sample": "GCA_019048585.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Aeromonas sp. FDAARGOS 1418 chromosome, complete genome", "reads_in": 296936, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002209165.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Mammaliicoccus sciuri strain FDAARGOS_285 chromosome, complete genome", "reads_in": 190256, "reads_removed_percent": 0.002102430409553444, "reads_retained_percent": 99.99789756959045}, {"sample": "GCA_006364795.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Pseudomonas aeruginosa strain FDAARGOS_610 chromosome, complete genome", "reads_in": 452814, "reads_removed_percent": 0.0008833649136289956, "reads_retained_percent": 99.99911663508637}, {"sample": "GCA_020097335.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Streptococcus pneumoniae strain FDAARGOS_1508 chromosome, complete genome", "reads_in": 141108, "reads_removed_percent": 0.0028347081667942286, "reads_retained_percent": 99.99716529183321}, {"sample": "GCA_016028475.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Serratia rubidaea strain FDAARGOS_880 chromosome, complete genome", "reads_in": 337078, "reads_removed_percent": 0.0011866689608933244, "reads_retained_percent": 99.9988133310391}, {"sample": "GCA_016026595.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Brevibacterium casei strain FDAARGOS_936 chromosome, complete genome", "reads_in": 257772, "reads_removed_percent": 0.0015517589187343855, "reads_retained_percent": 99.99844824108126}, {"sample": "GCA_019048625.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Enterobacter hormaechei strain FDAARGOS 1435 chromosome, complete genome", "reads_in": 317498, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016725185.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Streptococcus anginosus strain FDAARGOS_1155 chromosome, complete genome", "reads_in": 152270, "reads_removed_percent": 0.0026269127208248502, "reads_retained_percent": 99.99737308727917}, {"sample": "GCA_008693525.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Burkholderia multivorans strain FDAARGOS_622 chromosome 1, complete sequence", "reads_in": 431910, "reads_removed_percent": 0.0009261188673566252, "reads_retained_percent": 99.99907388113265}, {"sample": "GCA_008693545.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Bordetella hinzii strain FDAARGOS_621 chromosome, complete genome", "reads_in": 324030, "reads_removed_percent": 0.001234453599975311, "reads_retained_percent": 99.99876554640002}, {"sample": "GCA_019931045.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Cupriavidus pauculus strain FDAARGOS_1472 chromosome 1, complete sequence", "reads_in": 432224, "reads_removed_percent": 0.0009254460650033316, "reads_retained_percent": 99.999074553935}, {"sample": "GCA_002208705.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Mycobacterium avium subsp. paratuberculosis strain FDAARGOS_305 chromosome, complete genome", "reads_in": 322166, "reads_removed_percent": 0.0012415959474308277, "reads_retained_percent": 99.99875840405257}, {"sample": "GCA_016027055.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Acinetobacter johnsonii strain FDAARGOS_910 chromosome, complete genome", "reads_in": 241162, "reads_removed_percent": 0.0016586361035320655, "reads_retained_percent": 99.99834136389647}, {"sample": "GCA_020097315.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Aeromonas enteropelogenes strain FDAARGOS_1509 chromosome, complete genome", "reads_in": 302706, "reads_removed_percent": 0.00132141417745271, "reads_retained_percent": 99.99867858582255}, {"sample": "GCA_016889105.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Escherichia coli strain FDAARGOS_1254 chromosome, complete genome", "reads_in": 326142, "reads_removed_percent": 0.0012264596402793875, "reads_retained_percent": 99.99877354035972}, {"sample": "GCA_016903835.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Escherichia coli strain FDAARGOS_1299 chromosome, complete genome", "reads_in": 344708, "reads_removed_percent": 0.0011604024275618785, "reads_retained_percent": 99.99883959757244}, {"sample": "GCA_002209085.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Burkholderia cepacia strain FDAARGOS_345 chromosome 1, complete sequence", "reads_in": 571324, "reads_removed_percent": 0.0007001281234465908, "reads_retained_percent": 99.99929987187656}, {"sample": "GCA_002208885.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Stenotrophomonas maltophilia strain FDAARGOS_325 chromosome, complete genome", "reads_in": 323434, "reads_removed_percent": 0.0012367283587996314, "reads_retained_percent": 99.9987632716412}, {"sample": "GCA_016727585.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Staphylococcus aureus strain FDAARGOS_1086 chromosome, complete genome", "reads_in": 185098, "reads_removed_percent": 0.002161017406995213, "reads_retained_percent": 99.99783898259301}, {"sample": "GCA_019048065.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Citrobacter farmeri strain FDAARGOS 1423 chromosome, complete genome", "reads_in": 330986, "reads_removed_percent": 0.0012085103297420434, "reads_retained_percent": 99.99879148967025}, {"sample": "GCA_003798445.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Yersinia pseudotuberculosis strain FDAARGOS_580 chromosome, complete genome", "reads_in": 332068, "reads_removed_percent": 0.0012045725574279967, "reads_retained_percent": 99.99879542744257}, {"sample": "GCA_016903995.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Escherichia coli strain FDAARGOS_1337 chromosome, complete genome", "reads_in": 331158, "reads_removed_percent": 0.0012078826421224914, "reads_retained_percent": 99.99879211735788}, {"sample": "GCA_003798425.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Yersinia pseudotuberculosis strain FDAARGOS_581 chromosome, complete genome", "reads_in": 339798, "reads_removed_percent": 0.0011771699656854956, "reads_retained_percent": 99.99882283003431}, {"sample": "GCA_003798345.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Yersinia pestis strain FDAARGOS_602 chromosome, complete genome", "reads_in": 318966, "reads_removed_percent": 0.0012540521560291693, "reads_retained_percent": 99.99874594784397}, {"sample": "GCA_016728065.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Pseudomonas fluorescens strain FDAARGOS_1088 chromosome, complete genome", "reads_in": 408974, "reads_removed_percent": 0.0009780572848151715, "reads_retained_percent": 99.99902194271519}, {"sample": "GCA_016027675.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Serratia plymuthica strain FDAARGOS_895 chromosome, complete genome", "reads_in": 356612, "reads_removed_percent": 0.0011216672461947439, "reads_retained_percent": 99.9988783327538}, {"sample": "GCA_019048465.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Jonesia denitrificans strain FDAARGOS 1440 chromosome, complete genome", "reads_in": 183310, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003798305.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Yersinia pseudotuberculosis strain FDAARGOS_579 chromosome, complete genome", "reads_in": 312098, "reads_removed_percent": 0.0012816487129042801, "reads_retained_percent": 99.9987183512871}, {"sample": "GCA_002386165.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Salmonella enterica subsp. enterica serovar Paratyphi A strain FDAARGOS_368 chromosome, complete genome", "reads_in": 305866, "reads_removed_percent": 0.0013077622226726738, "reads_retained_percent": 99.99869223777733}, {"sample": "GCA_002983865.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Roseobacter denitrificans strain FDAARGOS_309 chromosome, complete genome", "reads_in": 287254, "reads_removed_percent": 0.0013924958399186781, "reads_retained_percent": 99.99860750416008}, {"sample": "GCA_020099335.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Citrobacter amalonaticus strain FDAARGOS_1489 chromosome, complete genome", "reads_in": 338134, "reads_removed_percent": 0.0011829629673443072, "reads_retained_percent": 99.99881703703265}, {"sample": "GCA_016903535.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Escherichia coli strain FDAARGOS_1373 chromosome, complete genome", "reads_in": 321672, "reads_removed_percent": 0.0006217513492004278, "reads_retained_percent": 99.9993782486508}, {"sample": "GCA_016728705.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Corynebacterium aurimucosum strain FDAARGOS_1109 chromosome, complete genome", "reads_in": 182518, "reads_removed_percent": 0.0010957823337972145, "reads_retained_percent": 99.99890421766621}, {"sample": "GCA_000783935.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Raoultella planticola strain FDAARGOS_64 chromosome, complete genome", "reads_in": 388262, "reads_removed_percent": 0.0005151160814089456, "reads_retained_percent": 99.9994848839186}, {"sample": "GCA_016128155.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Lactococcus garvieae strain FDAARGOS_1062 chromosome, complete genome", "reads_in": 135644, "reads_removed_percent": 0.0014744478192916751, "reads_retained_percent": 99.99852555218071}, {"sample": "GCA_006364655.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Listeria monocytogenes strain FDAARGOS_778 chromosome, complete genome", "reads_in": 196732, "reads_removed_percent": 0.0010166114307789277, "reads_retained_percent": 99.99898338856921}, {"sample": "GCA_003812085.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Klebsiella pneumoniae strain FDAARGOS_566 chromosome 1, complete sequence", "reads_in": 377804, "reads_removed_percent": 0.0005293750198515632, "reads_retained_percent": 99.99947062498015}, {"sample": "GCA_000783735.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Shigella flexneri strain FDAARGOS_74 chromosome, complete genome", "reads_in": 320390, "reads_removed_percent": 0.0006242392084646837, "reads_retained_percent": 99.99937576079154}, {"sample": "GCA_003812105.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Klebsiella pneumoniae strain FDAARGOS_531 chromosome, complete genome", "reads_in": 384250, "reads_removed_percent": 0.000520494469746259, "reads_retained_percent": 99.99947950553026}, {"sample": "GCA_016904475.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Escherichia coli strain FDAARGOS_1292 chromosome, complete genome", "reads_in": 341372, "reads_removed_percent": 0.0005858711317858524, "reads_retained_percent": 99.99941412886821}, {"sample": "GCA_009730035.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Burkholderia multivorans strain FDAARGOS_719 chromosome 1, complete sequence", "reads_in": 437710, "reads_removed_percent": 0.0004569235338466107, "reads_retained_percent": 99.99954307646615}, {"sample": "GCA_003798325.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Propionibacterium acidifaciens strain FDAARGOS_576 chromosome, complete genome", "reads_in": 204852, "reads_removed_percent": 0.0009763146076191591, "reads_retained_percent": 99.99902368539239}, {"sample": "GCA_019930825.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Aeromonas enteropelogenes strain FDAARGOS_1459 chromosome, complete genome", "reads_in": 294464, "reads_removed_percent": 0.0006792001738752445, "reads_retained_percent": 99.99932079982612}, {"sample": "GCA_009730175.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Burkholderia multivorans strain FDAARGOS_726 chromosome 1, complete sequence", "reads_in": 434580, "reads_removed_percent": 0.0004602144599383313, "reads_retained_percent": 99.99953978554007}, {"sample": "GCA_009730215.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Mycobacterium tuberculosis strain FDAARGOS_751 chromosome, complete genome", "reads_in": 296132, "reads_removed_percent": 0.0006753744951575649, "reads_retained_percent": 99.99932462550484}, {"sample": "GCA_003019965.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Burkholderia multivorans strain FDAARGOS_246 chromosome 1, complete sequence", "reads_in": 421526, "reads_removed_percent": 0.000474466580946371, "reads_retained_percent": 99.99952553341906}, {"sample": "GCA_008693825.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Pseudomonas oryzihabitans strain FDAARGOS_657 chromosome, complete genome", "reads_in": 350992, "reads_removed_percent": 0.0005698135570041483, "reads_retained_percent": 99.99943018644299}, {"sample": "GCA_002984125.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Moraxella catarrhalis strain FDAARGOS_304 chromosome, complete genome", "reads_in": 129466, "reads_removed_percent": 0.001544807130829716, "reads_retained_percent": 99.99845519286917}, {"sample": "GCA_003798285.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Yersinia pseudotuberculosis strain FDAARGOS_583 chromosome, complete genome", "reads_in": 310146, "reads_removed_percent": 0.0006448575832027496, "reads_retained_percent": 99.9993551424168}, {"sample": "GCA_006365335.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Salmonella enterica strain FDAARGOS_768 chromosome, complete genome", "reads_in": 330318, "reads_removed_percent": 0.0006054771462651143, "reads_retained_percent": 99.99939452285373}, {"sample": "GCA_000783815.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Salmonella enterica strain FDAARGOS_70 chromosome, complete genome", "reads_in": 319106, "reads_removed_percent": 0.0006267509855659248, "reads_retained_percent": 99.99937324901444}, {"sample": "GCA_002073375.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Corynebacterium diphtheriae strain FDAARGOS_197 chromosome, complete genome", "reads_in": 165936, "reads_removed_percent": 0.001205283964902131, "reads_retained_percent": 99.99879471603509}, {"sample": "GCA_016889405.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Corynebacterium kroppenstedtii strain FDAARGOS_1194 chromosome, complete genome", "reads_in": 167458, "reads_removed_percent": 0.0011943293243679013, "reads_retained_percent": 99.99880567067564}, {"sample": "GCA_016904515.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Klebsiella oxytoca strain FDAARGOS_1331 chromosome, complete genome", "reads_in": 400534, "reads_removed_percent": 0.0004993333899244508, "reads_retained_percent": 99.99950066661007}, {"sample": "GCA_016026575.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Fannyhessea vaginae strain FDAARGOS_934 chromosome, complete genome", "reads_in": 96468, "reads_removed_percent": 0.002073226354853423, "reads_retained_percent": 99.99792677364515}, {"sample": "GCA_002407525.3", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Burkholderia cepacia strain FDAARGOS_388 chromosome 1, complete sequence", "reads_in": 571322, "reads_removed_percent": 0.00035006528717605837, "reads_retained_percent": 99.99964993471282}, {"sample": "GCA_019048285.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Escherichia coli strain FDAARGOS 1389 chromosome, complete genome", "reads_in": 345514, "reads_removed_percent": 0.0005788477456774545, "reads_retained_percent": 99.99942115225433}, {"sample": "GCA_016889325.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Escherichia coli strain FDAARGOS_1258 chromosome, complete genome", "reads_in": 348734, "reads_removed_percent": 0.0005735030137583373, "reads_retained_percent": 99.99942649698625}, {"sample": "GCA_016904555.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Escherichia coli strain FDAARGOS_1285 chromosome, complete genome", "reads_in": 345962, "reads_removed_percent": 0.0005780981726316763, "reads_retained_percent": 99.99942190182738}, {"sample": "GCA_016127975.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Corynebacterium urealyticum strain FDAARGOS_994 chromosome, complete genome", "reads_in": 158948, "reads_removed_percent": 0.0012582731459345195, "reads_retained_percent": 99.99874172685406}, {"sample": "GCA_019048705.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Aeromonas sp. FDAARGOS 1409 chromosome, complete genome", "reads_in": 291864, "reads_removed_percent": 0.0006852506646931447, "reads_retained_percent": 99.99931474933531}, {"sample": "GCA_002407165.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Staphylococcus lugdunensis strain FDAARGOS_377 chromosome, complete genome", "reads_in": 171442, "reads_removed_percent": 0.0011665752849360134, "reads_retained_percent": 99.99883342471506}, {"sample": "GCA_002073395.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Staphylococcus lugdunensis strain FDAARGOS_222 chromosome, complete genome", "reads_in": 169354, "reads_removed_percent": 0.0011809582295074224, "reads_retained_percent": 99.99881904177049}, {"sample": "GCA_016128015.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Brucella anthropi strain FDAARGOS_1039 chromosome 1, complete sequence", "reads_in": 344930, "reads_removed_percent": 0.0005798277911460296, "reads_retained_percent": 99.99942017220886}, {"sample": "GCA_016728745.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Corynebacterium amycolatum strain FDAARGOS_1107 chromosome, complete genome", "reads_in": 180436, "reads_removed_percent": 0.0011084262564011616, "reads_retained_percent": 99.9988915737436}, {"sample": "GCA_016026415.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Corynebacterium amycolatum strain FDAARGOS_938 chromosome, complete genome", "reads_in": 186842, "reads_removed_percent": 0.0010704231382665569, "reads_retained_percent": 99.99892957686173}, {"sample": "GCA_016889225.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Corynebacterium glucuronolyticum strain FDAARGOS_1190 chromosome, complete genome", "reads_in": 188214, "reads_removed_percent": 0.001062620208911133, "reads_retained_percent": 99.99893737979109}, {"sample": "GCA_008693865.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Paracoccus yeei strain FDAARGOS_643 chromosome, complete genome", "reads_in": 312652, "reads_removed_percent": 0.0006396888553407623, "reads_retained_percent": 99.99936031114466}, {"sample": "GCA_008693885.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Klebsiella aerogenes strain FDAARGOS_641 chromosome, complete genome", "reads_in": 352422, "reads_removed_percent": 0.0005675014613162628, "reads_retained_percent": 99.99943249853868}, {"sample": "GCA_020736005.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Mycobacterium intracellulare strain FDAARGOS_1564 chromosome, complete genome", "reads_in": 376386, "reads_removed_percent": 0.0005313693920602785, "reads_retained_percent": 99.99946863060794}, {"sample": "GCA_020099275.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Corynebacterium falsenii strain FDAARGOS_1494 chromosome, complete genome", "reads_in": 178492, "reads_removed_percent": 0.0011204983976872915, "reads_retained_percent": 99.99887950160232}, {"sample": "GCA_020735465.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Collinsella stercoris DSM 13279 strain FDAARGOS_1572 chromosome, complete genome", "reads_in": 164050, "reads_removed_percent": 0.00121914050594331, "reads_retained_percent": 99.99878085949406}, {"sample": "GCA_000783675.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Enterobacter cloacae complex sp. strain FDAARGOS_77 chromosome, complete genome", "reads_in": 325134, "reads_removed_percent": 0.0006151309921447772, "reads_retained_percent": 99.99938486900786}, {"sample": "GCA_003813065.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Bifidobacterium breve strain FDAARGOS_561 chromosome, complete genome", "reads_in": 151710, "reads_removed_percent": 0.001318304660206974, "reads_retained_percent": 99.9986816953398}, {"sample": "GCA_003813045.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Burkholderia dolosa strain FDAARGOS_562 chromosome 1, complete sequence", "reads_in": 427274, "reads_removed_percent": 0.00046808371209107035, "reads_retained_percent": 99.9995319162879}, {"sample": "GCA_020099255.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Enterobacter bugandensis strain FDAARGOS_1496 chromosome, complete genome", "reads_in": 334858, "reads_removed_percent": 0.0005972680957301304, "reads_retained_percent": 99.99940273190427}, {"sample": "GCA_002073755.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Citrobacter braakii strain FDAARGOS_253 chromosome, complete genome", "reads_in": 349620, "reads_removed_percent": 0.0005720496539099594, "reads_retained_percent": 99.99942795034609}, {"sample": "GCA_008693785.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Roseomonas mucosa strain FDAARGOS_658 chromosome 1, complete sequence", "reads_in": 333084, "reads_removed_percent": 0.0006004491359536934, "reads_retained_percent": 99.99939955086404}, {"sample": "GCA_002953975.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Vibrio alginolyticus strain FDAARGOS_114 chromosome 1, complete sequence", "reads_in": 345218, "reads_removed_percent": 0.0005793440666477414, "reads_retained_percent": 99.99942065593335}, {"sample": "GCA_002954055.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Staphylococcus epidermidis strain FDAARGOS_161 chromosome, complete genome", "reads_in": 167716, "reads_removed_percent": 0.001192492069927735, "reads_retained_percent": 99.99880750793007}, {"sample": "GCA_002983625.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Yersinia intermedia strain FDAARGOS_358 chromosome, complete genome", "reads_in": 330710, "reads_removed_percent": 0.0006047594569260077, "reads_retained_percent": 99.99939524054308}, {"sample": "GCA_009730275.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Lactobacillus crispatus strain FDAARGOS_743 chromosome, complete genome", "reads_in": 159130, "reads_removed_percent": 0.0012568340350656696, "reads_retained_percent": 99.99874316596494}, {"sample": "GCA_009730355.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Citrobacter portucalensis strain FDAARGOS_738 chromosome, complete genome", "reads_in": 344180, "reads_removed_percent": 0.0005810912894415712, "reads_retained_percent": 99.99941890871055}, {"sample": "GCA_020097175.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Aeromonas enteropelogenes strain FDAARGOS_1536 chromosome, complete genome", "reads_in": 288620, "reads_removed_percent": 0.000692952671332548, "reads_retained_percent": 99.99930704732867}, {"sample": "GCA_008693725.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Streptococcus dysgalactiae strain FDAARGOS_654 chromosome, complete genome", "reads_in": 150236, "reads_removed_percent": 0.0013312388508746238, "reads_retained_percent": 99.99866876114912}, {"sample": "GCA_019930745.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Mycolicibacterium smegmatis strain FDAARGOS_1484 chromosome, complete genome", "reads_in": 462514, "reads_removed_percent": 0.0004324193429820503, "reads_retained_percent": 99.99956758065701}, {"sample": "GCA_020099235.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Enterobacter bugandensis strain FDAARGOS_1497 chromosome, complete genome", "reads_in": 334858, "reads_removed_percent": 0.0005972680957301304, "reads_retained_percent": 99.99940273190427}, {"sample": "GCA_003019925.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Pluralibacter gergoviae strain FDAARGOS_186 chromosome, complete genome", "reads_in": 360538, "reads_removed_percent": 0.000554726547548386, "reads_retained_percent": 99.99944527345245}, {"sample": "GCA_020735405.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Mycobacterium avium subsp. avium strain FDAARGOS_1606 chromosome, complete genome", "reads_in": 330704, "reads_removed_percent": 0.0006047704291450965, "reads_retained_percent": 99.99939522957085}, {"sample": "GCA_008693985.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Stenotrophomonas maltophilia strain FDAARGOS_649 chromosome, complete genome", "reads_in": 308392, "reads_removed_percent": 0.0006485252535733742, "reads_retained_percent": 99.99935147474643}, {"sample": "GCA_009729895.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Salmonella enterica strain FDAARGOS_710 chromosome, complete genome", "reads_in": 307402, "reads_removed_percent": 0.0006506138541714108, "reads_retained_percent": 99.99934938614582}, {"sample": "GCA_019930885.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Legionella anisa strain FDAARGOS_1480 chromosome, complete genome", "reads_in": 296538, "reads_removed_percent": 0.0006744498175613243, "reads_retained_percent": 99.99932555018243}, {"sample": "GCA_009729935.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Salmonella enterica strain FDAARGOS_707 chromosome, complete genome", "reads_in": 303738, "reads_removed_percent": 0.0006584622273143301, "reads_retained_percent": 99.99934153777268}, {"sample": "GCA_003812305.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Enterococcus sp. FDAARGOS_553 chromosome, complete genome", "reads_in": 218900, "reads_removed_percent": 0.0009136592051164914, "reads_retained_percent": 99.99908634079489}, {"sample": "GCA_020735385.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Granulicatella elegans strain FDAARGOS_1559 chromosome, complete genome", "reads_in": 126042, "reads_removed_percent": 0.0015867726630805603, "reads_retained_percent": 99.99841322733693}, {"sample": "GCA_009730235.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Mycobacterium tuberculosis strain FDAARGOS_750 chromosome, complete genome", "reads_in": 295644, "reads_removed_percent": 0.0006764892911745208, "reads_retained_percent": 99.99932351070882}, {"sample": "GCA_009730515.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Streptococcus mitis strain FDAARGOS_684 chromosome, complete genome", "reads_in": 124590, "reads_removed_percent": 0.0016052652700858816, "reads_retained_percent": 99.99839473472991}, {"sample": "GCA_019930625.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Nocardia asteroides strain FDAARGOS_1485 chromosome, complete genome", "reads_in": 494394, "reads_removed_percent": 0.0004045356537498433, "reads_retained_percent": 99.99959546434626}, {"sample": "GCA_020735925.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Bordetella parapertussis strain FDAARGOS_1541 chromosome, complete genome", "reads_in": 318372, "reads_removed_percent": 0.0006281959468797507, "reads_retained_percent": 99.99937180405313}, {"sample": "GCA_003812445.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Streptococcus sp. FDAARGOS_522 chromosome, complete genome", "reads_in": 140076, "reads_removed_percent": 0.0014277963391301864, "reads_retained_percent": 99.99857220366087}, {"sample": "GCA_003813025.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Pseudomonas aeruginosa strain FDAARGOS_570 chromosome, complete genome", "reads_in": 477010, "reads_removed_percent": 0.00041927842183602025, "reads_retained_percent": 99.99958072157817}, {"sample": "GCA_002208825.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Brevundimonas vesicularis strain FDAARGOS_289 chromosome, complete genome", "reads_in": 223036, "reads_removed_percent": 0.0008967162251833785, "reads_retained_percent": 99.99910328377482}, {"sample": "GCA_021183685.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Mycobacterium kansasii strain FDAARGOS_1615 chromosome, complete genome", "reads_in": 430348, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904315.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Klebsiella pneumoniae strain FDAARGOS_1316 chromosome, complete genome", "reads_in": 365988, "reads_removed_percent": 0.0005464660043498693, "reads_retained_percent": 99.99945353399565}, {"sample": "GCA_016025875.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Prescottella equi strain FDAARGOS_952 chromosome, complete genome", "reads_in": 347914, "reads_removed_percent": 0.0005748547054731917, "reads_retained_percent": 99.99942514529452}, {"sample": "GCA_013267615.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Parabacteroides distasonis strain FDAARGOS_759 chromosome, complete genome", "reads_in": 328402, "reads_removed_percent": 0.0006090096893441575, "reads_retained_percent": 99.99939099031066}, {"sample": "GCA_016903795.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Klebsiella pneumoniae strain FDAARGOS_1308 chromosome, complete genome", "reads_in": 370052, "reads_removed_percent": 0.0005404645833558527, "reads_retained_percent": 99.99945953541665}, {"sample": "GCA_019048665.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Aeromonas sp. FDAARGOS 1408 chromosome, complete genome", "reads_in": 322962, "reads_removed_percent": 0.0006192679014868622, "reads_retained_percent": 99.99938073209852}, {"sample": "GCA_020736185.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Campylobacter gracilis strain FDAARGOS_1548 chromosome, complete genome", "reads_in": 152116, "reads_removed_percent": 0.0013147860843040838, "reads_retained_percent": 99.99868521391569}, {"sample": "GCA_019048645.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Streptococcus mutans strain FDAARGOS 1458 chromosome, complete genome", "reads_in": 134624, "reads_removed_percent": 0.0014856192060850961, "reads_retained_percent": 99.99851438079392}, {"sample": "GCA_013267655.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Mycobacterium tuberculosis strain FDAARGOS_756 chromosome, complete genome", "reads_in": 294306, "reads_removed_percent": 0.0006795648066977907, "reads_retained_percent": 99.9993204351933}, {"sample": "GCA_019048565.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Aeromonas sp. FDAARGOS 1406 chromosome, complete genome", "reads_in": 309722, "reads_removed_percent": 0.0006457403736253802, "reads_retained_percent": 99.99935425962637}, {"sample": "GCA_020736105.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Corynebacterium diphtheriae strain FDAARGOS_1553 chromosome, complete genome", "reads_in": 166080, "reads_removed_percent": 0.001204238921001927, "reads_retained_percent": 99.998795761079}, {"sample": "GCA_016026895.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Neisseria meningitidis strain FDAARGOS_914 chromosome, complete genome", "reads_in": 145422, "reads_removed_percent": 0.0013753077251034919, "reads_retained_percent": 99.9986246922749}, {"sample": "GCA_001188125.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Salmonella enterica subsp. enterica serovar Heidelberg strain FDAARGOS_54 chromosome, complete genome", "reads_in": 316752, "reads_removed_percent": 0.0006314087993130272, "reads_retained_percent": 99.99936859120069}, {"sample": "GCA_002393245.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Citrobacter koseri strain FDAARGOS_393 chromosome, complete genome", "reads_in": 325632, "reads_removed_percent": 0.0006141902515723271, "reads_retained_percent": 99.99938580974843}, {"sample": "GCA_016889625.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Escherichia coli strain FDAARGOS_1262 chromosome, complete genome", "reads_in": 348356, "reads_removed_percent": 0.0005741253200748659, "reads_retained_percent": 99.99942587467993}, {"sample": "GCA_016127735.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Streptococcus oralis strain FDAARGOS_1075 chromosome, complete genome", "reads_in": 132994, "reads_removed_percent": 0.0015038272403266313, "reads_retained_percent": 99.99849617275967}, {"sample": "GCA_002393365.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Escherichia coli strain FDAARGOS_403 chromosome, complete genome", "reads_in": 365028, "reads_removed_percent": 0.0005479031745509933, "reads_retained_percent": 99.99945209682545}, {"sample": "GCA_016889605.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Escherichia coli strain FDAARGOS_1268 chromosome, complete genome", "reads_in": 333912, "reads_removed_percent": 0.0005989602050839743, "reads_retained_percent": 99.99940103979492}, {"sample": "GCA_019048125.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Klebsiella aerogenes strain FDAARGOS 1442 chromosome, complete genome", "reads_in": 352118, "reads_removed_percent": 0.000567991411969851, "reads_retained_percent": 99.99943200858803}, {"sample": "GCA_002073535.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Neisseria gonorrhoeae strain FDAARGOS_204 chromosome, complete genome", "reads_in": 147496, "reads_removed_percent": 0.001355968975429842, "reads_retained_percent": 99.99864403102457}, {"sample": "GCA_002591255.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Brucella canis strain FDAARGOS_420 chromosome 1, complete sequence", "reads_in": 220852, "reads_removed_percent": 0.0009055838298951334, "reads_retained_percent": 99.99909441617011}, {"sample": "GCA_016728725.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Corynebacterium amycolatum strain FDAARGOS_1108 chromosome, complete genome", "reads_in": 165074, "reads_removed_percent": 0.0012115778378181906, "reads_retained_percent": 99.99878842216218}, {"sample": "GCA_016728005.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Serratia liquefaciens strain FDAARGOS_1081 chromosome, complete genome", "reads_in": 367766, "reads_removed_percent": 0.0005438240620394489, "reads_retained_percent": 99.99945617593796}, {"sample": "GCA_020736085.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Corynebacterium diphtheriae subsp. diphtheriae strain FDAARGOS_1554 chromosome, complete genome", "reads_in": 164242, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016727125.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Enterococcus hirae strain FDAARGOS_1124 chromosome, complete genome", "reads_in": 195226, "reads_removed_percent": 0.001024453710059111, "reads_retained_percent": 99.99897554628994}, {"sample": "GCA_013267395.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Achromobacter denitrificans strain FDAARGOS_786 chromosome, complete genome", "reads_in": 449124, "reads_removed_percent": 0.0004453113171418138, "reads_retained_percent": 99.99955468868286}, {"sample": "GCA_016727365.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Corynebacterium ulcerans strain FDAARGOS_1118 chromosome, complete genome", "reads_in": 163584, "reads_removed_percent": 0.0012226134585289516, "reads_retained_percent": 99.99877738654148}, {"sample": "GCA_013267635.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Mycobacterium tuberculosis strain FDAARGOS_757 chromosome, complete genome", "reads_in": 294528, "reads_removed_percent": 0.0006790525858322468, "reads_retained_percent": 99.99932094741416}, {"sample": "GCA_013267375.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Achromobacter denitrificans strain FDAARGOS_788 chromosome, complete genome", "reads_in": 449126, "reads_removed_percent": 0.0004453093341289527, "reads_retained_percent": 99.99955469066587}, {"sample": "GCA_019048485.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Enterobacter asburiae strain FDAARGOS 1432 chromosome, complete genome", "reads_in": 326758, "reads_removed_percent": 0.0006120737671304145, "reads_retained_percent": 99.99938792623287}, {"sample": "GCA_002591295.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Pasteurella multocida strain FDAARGOS_385 chromosome, complete genome", "reads_in": 156448, "reads_removed_percent": 0.0012783800368173451, "reads_retained_percent": 99.99872161996318}, {"sample": "GCA_020736125.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Corynebacterium diphtheriae strain FDAARGOS_1552 chromosome, complete genome", "reads_in": 157962, "reads_removed_percent": 0.001266127296438384, "reads_retained_percent": 99.99873387270357}, {"sample": "GCA_013267495.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Streptococcus salivarius strain FDAARGOS_771 chromosome, complete genome", "reads_in": 133300, "reads_removed_percent": 0.0015003750937734434, "reads_retained_percent": 99.99849962490623}, {"sample": "GCA_002591135.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Escherichia coli strain FDAARGOS_448 chromosome, complete genome", "reads_in": 363122, "reads_removed_percent": 0.0005507790770044227, "reads_retained_percent": 99.999449220923}, {"sample": "GCA_016903415.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Klebsiella oxytoca strain FDAARGOS_1334 chromosome, complete genome", "reads_in": 415644, "reads_removed_percent": 0.0004811810106725948, "reads_retained_percent": 99.99951881898933}, {"sample": "GCA_016903895.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Klebsiella pneumoniae strain FDAARGOS_1321 chromosome, complete genome", "reads_in": 362858, "reads_removed_percent": 0.0005511798003626763, "reads_retained_percent": 99.99944882019963}, {"sample": "GCA_002209205.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Salmonella enterica subsp. enterica serovar Hadar strain FDAARGOS_313 chromosome, complete genome", "reads_in": 329616, "reads_removed_percent": 0.0006067666618125334, "reads_retained_percent": 99.99939323333818}, {"sample": "GCA_001559055.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Bordetella bronchiseptica strain FDAARGOS_176 chromosome, complete genome", "reads_in": 355948, "reads_removed_percent": 0.0005618798251429985, "reads_retained_percent": 99.99943812017486}, {"sample": "GCA_021183625.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Nocardia asteroides strain FDAARGOS_1620 chromosome, complete genome", "reads_in": 465792, "reads_removed_percent": 0.00042937620225336627, "reads_retained_percent": 99.99957062379775}, {"sample": "GCA_016026195.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Escherichia coli strain FDAARGOS_946 chromosome, complete genome", "reads_in": 359310, "reads_removed_percent": 0.0005566224151846595, "reads_retained_percent": 99.99944337758481}, {"sample": "GCA_016127515.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Acinetobacter baumannii strain FDAARGOS_1036 chromosome, complete genome", "reads_in": 270026, "reads_removed_percent": 0.0007406694170191019, "reads_retained_percent": 99.99925933058299}, {"sample": "GCA_016904615.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Escherichia coli strain FDAARGOS_1283 chromosome, complete genome", "reads_in": 347664, "reads_removed_percent": 0.0005752680749229141, "reads_retained_percent": 99.99942473192507}, {"sample": "GCA_016904255.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Escherichia coli strain FDAARGOS_1371 chromosome, complete genome", "reads_in": 342696, "reads_removed_percent": 0.0005836076289189253, "reads_retained_percent": 99.99941639237107}, {"sample": "GCA_016128215.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Pediococcus pentosaceus strain FDAARGOS_1010 chromosome, complete genome", "reads_in": 129894, "reads_removed_percent": 0.001539717000015397, "reads_retained_percent": 99.99846028299999}, {"sample": "GCA_016026035.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Aggregatibacter actinomycetemcomitans strain FDAARGOS_948 chromosome, complete genome", "reads_in": 140358, "reads_removed_percent": 0.0014249276849199904, "reads_retained_percent": 99.99857507231508}, {"sample": "GCA_002073355.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Streptococcus intermedius strain FDAARGOS_233 chromosome, complete genome", "reads_in": 127600, "reads_removed_percent": 0.001567398119122257, "reads_retained_percent": 99.99843260188088}, {"sample": "GCA_016904415.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Klebsiella pneumoniae strain FDAARGOS_1314 chromosome, complete genome", "reads_in": 380180, "reads_removed_percent": 0.000526066600031564, "reads_retained_percent": 99.99947393339997}, {"sample": "GCA_016128275.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Citrobacter koseri strain FDAARGOS_1029 chromosome, complete genome", "reads_in": 315772, "reads_removed_percent": 0.000633368379717011, "reads_retained_percent": 99.99936663162028}, {"sample": "GCA_016904355.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Bordetella pertussis strain FDAARGOS_1365 chromosome, complete genome", "reads_in": 273672, "reads_removed_percent": 0.0007308018357742114, "reads_retained_percent": 99.99926919816423}, {"sample": "GCA_016128175.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Rothia kristinae strain FDAARGOS_1001 chromosome, complete genome", "reads_in": 167590, "reads_removed_percent": 0.0011933886270063846, "reads_retained_percent": 99.998806611373}, {"sample": "GCA_002591075.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Klebsiella pneumoniae strain FDAARGOS_439 chromosome, complete genome", "reads_in": 383526, "reads_removed_percent": 0.0005214770315441457, "reads_retained_percent": 99.99947852296846}, {"sample": "GCA_019048425.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Escherichia coli strain FDAARGOS 1380 chromosome, complete genome", "reads_in": 349396, "reads_removed_percent": 0.0005724163985849867, "reads_retained_percent": 99.99942758360142}, {"sample": "GCA_020736205.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Campylobacter rectus strain FDAARGOS_1549 chromosome, complete genome", "reads_in": 171454, "reads_removed_percent": 0.0011664936367772113, "reads_retained_percent": 99.99883350636323}, {"sample": "GCA_016904395.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Escherichia coli strain FDAARGOS_1277 chromosome, complete genome", "reads_in": 316756, "reads_removed_percent": 0.0006314008258722802, "reads_retained_percent": 99.99936859917413}, {"sample": "GCA_002073335.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Vibrio cholerae strain FDAARGOS_223 chromosome 1, complete sequence", "reads_in": 269486, "reads_removed_percent": 0.0007421535812621064, "reads_retained_percent": 99.99925784641874}, {"sample": "GCA_019048185.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Lelliottia amnigena strain FDAARGOS 1446 chromosome, complete genome", "reads_in": 327628, "reads_removed_percent": 0.00061044843542066, "reads_retained_percent": 99.99938955156458}, {"sample": "GCA_001559175.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Streptococcus pyogenes strain FDAARGOS_149 chromosome, complete genome", "reads_in": 122642, "reads_removed_percent": 0.0016307627077184, "reads_retained_percent": 99.99836923729228}, {"sample": "GCA_016026735.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Serratia rubidaea strain FDAARGOS_926 chromosome, complete genome", "reads_in": 333000, "reads_removed_percent": 0.0006006006006006006, "reads_retained_percent": 99.9993993993994}, {"sample": "GCA_016026715.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Salmonella enterica strain FDAARGOS_928 chromosome, complete genome", "reads_in": 330094, "reads_removed_percent": 0.000605888019776185, "reads_retained_percent": 99.99939411198022}, {"sample": "GCA_016127835.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Salmonella enterica strain FDAARGOS_1066 chromosome, complete genome", "reads_in": 315766, "reads_removed_percent": 0.0006333804146108194, "reads_retained_percent": 99.99936661958539}, {"sample": "GCA_016026975.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Aerococcus urinae strain FDAARGOS_911 chromosome, complete genome", "reads_in": 131618, "reads_removed_percent": 0.001519548997857436, "reads_retained_percent": 99.99848045100214}, {"sample": "GCA_002386345.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Streptococcus oralis strain FDAARGOS_367 chromosome, complete genome", "reads_in": 136880, "reads_removed_percent": 0.0014611338398597313, "reads_retained_percent": 99.99853886616015}, {"sample": "GCA_016127575.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Cupriavidus necator strain FDAARGOS_1030 chromosome 1, complete sequence", "reads_in": 494398, "reads_removed_percent": 0.00040453238079442066, "reads_retained_percent": 99.9995954676192}, {"sample": "GCA_001559135.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Serratia liquefaciens strain FDAARGOS_125 chromosome, complete genome", "reads_in": 352316, "reads_removed_percent": 0.0005676722033628902, "reads_retained_percent": 99.99943232779664}, {"sample": "GCA_016889425.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Corynebacterium amycolatum strain FDAARGOS_1189 chromosome, complete genome", "reads_in": 164996, "reads_removed_percent": 0.0012121505975902446, "reads_retained_percent": 99.9987878494024}, {"sample": "GCA_021183805.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Mycobacterium avium subsp. avium strain FDAARGOS_1609 chromosome, complete genome", "reads_in": 330370, "reads_removed_percent": 0.0006053818445984805, "reads_retained_percent": 99.9993946181554}, {"sample": "GCA_021183605.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Nocardia asteroides strain FDAARGOS_1621 chromosome, complete genome", "reads_in": 477942, "reads_removed_percent": 0.00041846081742136074, "reads_retained_percent": 99.99958153918259}, {"sample": "GCA_019047025.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Enterobacter roggenkampii strain FDAARGOS 1430 chromosome, complete genome", "reads_in": 327818, "reads_removed_percent": 0.0006100946256764423, "reads_retained_percent": 99.99938990537433}, {"sample": "GCA_019047005.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Enterobacter ludwigii strain FDAARGOS 1436 chromosome, complete genome", "reads_in": 329562, "reads_removed_percent": 0.000606866082861495, "reads_retained_percent": 99.99939313391714}, {"sample": "GCA_016027035.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Achromobacter xylosoxidans strain FDAARGOS_915 chromosome, complete genome", "reads_in": 454194, "reads_removed_percent": 0.0004403404712523723, "reads_retained_percent": 99.99955965952874}, {"sample": "GCA_019046945.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Streptococcus gordonii strain FDAARGOS 1454 chromosome, complete genome", "reads_in": 145842, "reads_removed_percent": 0.0013713470742310171, "reads_retained_percent": 99.99862865292577}, {"sample": "GCA_002073255.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 2, "name": "Pasteurella multocida strain FDAARGOS_218 chromosome, complete genome", "reads_in": 155702, "reads_removed_percent": 0.0012845050159920875, "reads_retained_percent": 99.99871549498401}, {"sample": "GCA_003812945.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Escherichia coli strain FDAARGOS_497 chromosome, complete genome", "reads_in": 367552, "reads_removed_percent": 0.0010882813860351734, "reads_retained_percent": 99.99891171861397}, {"sample": "GCA_003812665.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Enterococcus faecalis strain FDAARGOS_528 chromosome, complete genome", "reads_in": 197576, "reads_removed_percent": 0.002024537393205653, "reads_retained_percent": 99.99797546260679}, {"sample": "GCA_016903935.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Escherichia coli strain FDAARGOS_1288 chromosome, complete genome", "reads_in": 378792, "reads_removed_percent": 0.001055988510845002, "reads_retained_percent": 99.99894401148916}, {"sample": "GCA_019930925.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Bartonella henselae strain FDAARGOS_1462 chromosome, complete genome", "reads_in": 129782, "reads_removed_percent": 0.004623137260945277, "reads_retained_percent": 99.99537686273905}, {"sample": "GCA_016889265.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Escherichia coli strain FDAARGOS_1264 chromosome, complete genome", "reads_in": 336128, "reads_removed_percent": 0.001785034272658035, "reads_retained_percent": 99.99821496572734}, {"sample": "GCA_013267415.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Abiotrophia defectiva strain FDAARGOS_785 chromosome, complete genome", "reads_in": 136456, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001558875.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Enterococcus gallinarum strain FDAARGOS_163 chromosome, complete genome", "reads_in": 254766, "reads_removed_percent": 0.0023551023291962037, "reads_retained_percent": 99.99764489767081}, {"sample": "GCA_001018725.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Staphylococcus aureus strain FDAARGOS_10 chromosome, complete genome", "reads_in": 191592, "reads_removed_percent": 0.0031316547663785546, "reads_retained_percent": 99.99686834523362}, {"sample": "GCA_019930965.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Enterobacter ludwigii strain FDAARGOS_1475 chromosome, complete genome", "reads_in": 315382, "reads_removed_percent": 0.0019024548008446899, "reads_retained_percent": 99.99809754519916}, {"sample": "GCA_013267355.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Achromobacter pestifer strain FDAARGOS_790 chromosome, complete genome", "reads_in": 478488, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002073575.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Neisseria meningitidis strain FDAARGOS_209 chromosome, complete genome", "reads_in": 145416, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016598875.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Burkholderia glumae strain FDAARGOS_921 chromosome 1, complete sequence", "reads_in": 456378, "reads_removed_percent": 0.0013146996568633895, "reads_retained_percent": 99.99868530034314}, {"sample": "GCA_016028815.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Lactococcus lactis strain FDAARGOS_866 chromosome, complete genome", "reads_in": 173712, "reads_removed_percent": 0.0034539928156949434, "reads_retained_percent": 99.9965460071843}, {"sample": "GCA_016403145.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Agrobacterium tumefaciens strain FDAARGOS_1048 chromosome 1, complete sequence", "reads_in": 395512, "reads_removed_percent": 0.0015170209753433523, "reads_retained_percent": 99.99848297902466}, {"sample": "GCA_008693765.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Staphylococcus aureus strain FDAARGOS_660 chromosome, complete genome", "reads_in": 191386, "reads_removed_percent": 0.0031350255504582368, "reads_retained_percent": 99.99686497444954}, {"sample": "GCA_001558915.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Achromobacter xylosoxidans strain FDAARGOS_162 chromosome, complete genome", "reads_in": 437716, "reads_removed_percent": 0.0013707518116769777, "reads_retained_percent": 99.99862924818832}, {"sample": "GCA_016128195.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Paraburkholderia ginsengisoli strain FDAARGOS_1049 chromosome 1, complete sequence", "reads_in": 439818, "reads_removed_percent": 0.0013642006466311066, "reads_retained_percent": 99.99863579935337}, {"sample": "GCA_016599755.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Corynebacterium kefirresidentii strain FDAARGOS_1055 chromosome, complete genome", "reads_in": 168214, "reads_removed_percent": 0.0035668850392951835, "reads_retained_percent": 99.99643311496071}, {"sample": "GCA_001019125.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Staphylococcus aureus strain FDAARGOS_43 chromosome, complete genome", "reads_in": 188042, "reads_removed_percent": 0.0031907765286478553, "reads_retained_percent": 99.99680922347135}, {"sample": "GCA_013364135.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Pantoea ananatis strain FDAARGOS_680 chromosome, complete genome", "reads_in": 351950, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013364115.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Shigella flexneri strain FDAARGOS_691 chromosome, complete genome", "reads_in": 302920, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020099355.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Burkholderia vietnamiensis strain FDAARGOS_1488 chromosome 1, complete sequence", "reads_in": 462034, "reads_removed_percent": 0.0012986057303142195, "reads_retained_percent": 99.99870139426969}, {"sample": "GCA_020735965.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Bordetella parapertussis strain FDAARGOS_1540 chromosome, complete genome", "reads_in": 318370, "reads_removed_percent": 0.0018845996796180546, "reads_retained_percent": 99.99811540032039}, {"sample": "GCA_003812925.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Klebsiella oxytoca strain FDAARGOS_500 chromosome, complete genome", "reads_in": 391940, "reads_removed_percent": 0.0015308465581466552, "reads_retained_percent": 99.99846915344186}, {"sample": "GCA_002073435.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Streptococcus gordonii strain FDAARGOS_257 chromosome, complete genome", "reads_in": 148180, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016906225.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Streptomyces californicus strain FDAARGOS_1211 chromosome", "reads_in": 536880, "reads_removed_percent": 0.001117568171658471, "reads_retained_percent": 99.99888243182833}, {"sample": "GCA_002386305.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Morganella morganii strain FDAARGOS_365 chromosome, complete genome", "reads_in": 271714, "reads_removed_percent": 0.0022082042147257776, "reads_retained_percent": 99.99779179578528}, {"sample": "GCA_020097295.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Aeromonas enteropelogenes strain FDAARGOS_1510 chromosome, complete genome", "reads_in": 292826, "reads_removed_percent": 0.0020489983812912787, "reads_retained_percent": 99.9979510016187}, {"sample": "GCA_019047765.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Citrobacter pasteurii strain FDAARGOS 1424 chromosome, complete genome", "reads_in": 334754, "reads_removed_percent": 0.0017923609575987145, "reads_retained_percent": 99.9982076390424}, {"sample": "GCA_003812625.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Escherichia coli strain FDAARGOS_536 chromosome, complete genome", "reads_in": 340302, "reads_removed_percent": 0.0017631397993546909, "reads_retained_percent": 99.99823686020065}, {"sample": "GCA_008693945.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Citrobacter koseri strain FDAARGOS_646 chromosome, complete genome", "reads_in": 305402, "reads_removed_percent": 0.001964623676334798, "reads_retained_percent": 99.99803537632367}, {"sample": "GCA_013343155.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Pseudomonas sp. FDAARGOS_761 chromosome, complete genome", "reads_in": 425656, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_008693485.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Burkholderia multivorans strain FDAARGOS_623 chromosome 1, complete sequence", "reads_in": 431910, "reads_removed_percent": 0.0013891783010349379, "reads_retained_percent": 99.99861082169896}, {"sample": "GCA_008693505.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Streptococcus agalactiae strain FDAARGOS_669 chromosome, complete genome", "reads_in": 137712, "reads_removed_percent": 0.00435691878703381, "reads_retained_percent": 99.99564308121296}, {"sample": "GCA_016128095.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Streptococcus dysgalactiae strain FDAARGOS_1017 chromosome, complete genome", "reads_in": 144956, "reads_removed_percent": 0.004139187063660697, "reads_retained_percent": 99.99586081293634}, {"sample": "GCA_016726265.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Campylobacter lari strain FDAARGOS_1101 chromosome, complete genome", "reads_in": 100892, "reads_removed_percent": 0.0059469531776553146, "reads_retained_percent": 99.99405304682234}, {"sample": "GCA_013267735.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Rothia dentocariosa strain FDAARGOS_752 chromosome, complete genome", "reads_in": 168894, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019047645.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Escherichia coli strain FDAARGOS_1376 chromosome, complete genome", "reads_in": 356636, "reads_removed_percent": 0.0016823876445451386, "reads_retained_percent": 99.99831761235545}, {"sample": "GCA_019930785.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Burkholderia dolosa strain FDAARGOS_1463 chromosome 1, complete sequence", "reads_in": 413044, "reads_removed_percent": 0.0014526297440466392, "reads_retained_percent": 99.99854737025595}, {"sample": "GCA_016903295.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Klebsiella pneumoniae strain FDAARGOS_1323 chromosome, complete genome", "reads_in": 366060, "reads_removed_percent": 0.00163907556138338, "reads_retained_percent": 99.99836092443861}, {"sample": "GCA_008693445.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Klebsiella pneumoniae strain FDAARGOS_629 chromosome, complete genome", "reads_in": 381246, "reads_removed_percent": 0.0015737870036669236, "reads_retained_percent": 99.99842621299634}, {"sample": "GCA_001019395.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Staphylococcus aureus strain FDAARGOS_20 chromosome, complete genome", "reads_in": 187888, "reads_removed_percent": 0.0031933918078855487, "reads_retained_percent": 99.99680660819212}, {"sample": "GCA_002393405.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Lelliottia amnigena strain FDAARGOS_395 chromosome, complete genome", "reads_in": 297974, "reads_removed_percent": 0.0020135985018827148, "reads_retained_percent": 99.99798640149811}, {"sample": "GCA_003955985.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Corynebacterium jeikeium strain FDAARGOS_574 chromosome, complete genome", "reads_in": 168544, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048605.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Escherichia coli strain FDAARGOS 1375 chromosome, complete genome", "reads_in": 349518, "reads_removed_percent": 0.0017166497862771016, "reads_retained_percent": 99.99828335021373}, {"sample": "GCA_002944935.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Escherichia coli strain FDAARGOS_144 chromosome, complete genome", "reads_in": 334812, "reads_removed_percent": 0.00179205046414107, "reads_retained_percent": 99.99820794953585}, {"sample": "GCA_006364035.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Listeria monocytogenes strain FDAARGOS_607 chromosome, complete genome", "reads_in": 199470, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889665.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Enterocloster bolteae strain FDAARGOS_1231 chromosome, complete genome", "reads_in": 440932, "reads_removed_percent": 0.001360754039171573, "reads_retained_percent": 99.99863924596083}, {"sample": "GCA_019048685.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Acinetobacter nosocomialis strain FDAARGOS 1395 chromosome, complete genome", "reads_in": 260174, "reads_removed_percent": 0.002306148961848609, "reads_retained_percent": 99.99769385103815}, {"sample": "GCA_006364295.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_775 chromosome, complete genome", "reads_in": 371594, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812325.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Citrobacter freundii strain FDAARGOS_550 chromosome, complete genome", "reads_in": 331276, "reads_removed_percent": 0.0018111785942839206, "reads_retained_percent": 99.99818882140572}, {"sample": "GCA_016728025.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Listeria monocytogenes strain FDAARGOS_1080 chromosome, complete genome", "reads_in": 197582, "reads_removed_percent": 0.0030367138706967234, "reads_retained_percent": 99.9969632861293}, {"sample": "GCA_006364355.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Vibrio furnissii strain FDAARGOS_777 chromosome 1, complete sequence", "reads_in": 332890, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_006364235.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Streptococcus pyogenes strain FDAARGOS_774 chromosome, complete genome", "reads_in": 127658, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_006364755.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Salmonella enterica strain FDAARGOS_765 chromosome, complete genome", "reads_in": 298806, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020736065.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Finegoldia magna strain FDAARGOS_1556 chromosome, complete genome", "reads_in": 115922, "reads_removed_percent": 0.005175894135711944, "reads_retained_percent": 99.99482410586428}, {"sample": "GCA_009730015.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Salmonella enterica strain FDAARGOS_712 chromosome, complete genome", "reads_in": 311360, "reads_removed_percent": 0.0019270298047276465, "reads_retained_percent": 99.99807297019527}, {"sample": "GCA_009730155.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Enterococcus gallinarum strain FDAARGOS_728 chromosome, complete genome", "reads_in": 218328, "reads_removed_percent": 0.0027481587336484555, "reads_retained_percent": 99.99725184126635}, {"sample": "GCA_016028695.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Pediococcus pentosaceus strain FDAARGOS_873 chromosome, complete genome", "reads_in": 116058, "reads_removed_percent": 0.005169828878664117, "reads_retained_percent": 99.99483017112134}, {"sample": "GCA_019047545.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Escherichia fergusonii strain FDAARGOS 1438 chromosome, complete genome", "reads_in": 302878, "reads_removed_percent": 0.001980995648412892, "reads_retained_percent": 99.99801900435159}, {"sample": "GCA_009730135.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Staphylococcus hominis strain FDAARGOS_746 chromosome, complete genome", "reads_in": 158146, "reads_removed_percent": 0.0037939625409431793, "reads_retained_percent": 99.99620603745906}, {"sample": "GCA_001019205.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Staphylococcus aureus strain FDAARGOS_48 chromosome, complete genome", "reads_in": 189826, "reads_removed_percent": 0.0031607893544614543, "reads_retained_percent": 99.99683921064553}, {"sample": "GCA_006364695.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Escherichia coli strain FDAARGOS_772 chromosome, complete genome", "reads_in": 358738, "reads_removed_percent": 0.0016725298128439138, "reads_retained_percent": 99.99832747018715}, {"sample": "GCA_008693385.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Cupriavidus pauculus strain FDAARGOS_664 chromosome 1, complete sequence", "reads_in": 470942, "reads_removed_percent": 0.001274042238747022, "reads_retained_percent": 99.99872595776125}, {"sample": "GCA_008693425.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_631 chromosome 1, complete sequence", "reads_in": 364570, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002073815.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Streptococcus sp. FDAARGOS_192 chromosome, complete genome", "reads_in": 162368, "reads_removed_percent": 0.0036953094205754828, "reads_retained_percent": 99.99630469057942}, {"sample": "GCA_016127935.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Staphylococcus pettenkoferi strain FDAARGOS_1071 chromosome, complete genome", "reads_in": 166242, "reads_removed_percent": 0.0036091962319991337, "reads_retained_percent": 99.996390803768}, {"sample": "GCA_016725005.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Streptococcus constellatus strain FDAARGOS_1156 chromosome, complete genome", "reads_in": 126884, "reads_removed_percent": 0.004728728602503074, "reads_retained_percent": 99.9952712713975}, {"sample": "GCA_009730055.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Yersinia intermedia strain FDAARGOS_730 chromosome, complete genome", "reads_in": 328594, "reads_removed_percent": 0.0018259615209042163, "reads_retained_percent": 99.99817403847909}, {"sample": "GCA_016903315.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Escherichia coli strain FDAARGOS_1289 chromosome, complete genome", "reads_in": 368416, "reads_removed_percent": 0.0016285937635716148, "reads_retained_percent": 99.99837140623643}, {"sample": "GCA_002073235.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Neisseria meningitidis strain FDAARGOS_214 chromosome, complete genome", "reads_in": 159830, "reads_removed_percent": 0.0037539886129012074, "reads_retained_percent": 99.9962460113871}, {"sample": "GCA_016904575.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Escherichia coli strain FDAARGOS_1276 chromosome, complete genome", "reads_in": 338828, "reads_removed_percent": 0.0017708099684795825, "reads_retained_percent": 99.99822919003152}, {"sample": "GCA_016903435.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Escherichia coli strain FDAARGOS_1302 chromosome, complete genome", "reads_in": 342060, "reads_removed_percent": 0.0017540782318891423, "reads_retained_percent": 99.99824592176812}, {"sample": "GCA_016726345.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Acinetobacter ursingii strain FDAARGOS_1096 chromosome, complete genome", "reads_in": 232482, "reads_removed_percent": 0.002580844968642734, "reads_retained_percent": 99.99741915503135}, {"sample": "GCA_016890045.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Escherichia coli strain FDAARGOS_1265 chromosome, complete genome", "reads_in": 361686, "reads_removed_percent": 0.0016588974967236775, "reads_retained_percent": 99.99834110250328}, {"sample": "GCA_016903375.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Escherichia coli strain FDAARGOS_1286 chromosome, complete genome", "reads_in": 344596, "reads_removed_percent": 0.0023215591591312723, "reads_retained_percent": 99.99767844084087}, {"sample": "GCA_016903155.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Klebsiella pneumoniae strain FDAARGOS_1326 chromosome, complete genome", "reads_in": 368106, "reads_removed_percent": 0.002173287042319332, "reads_retained_percent": 99.99782671295767}, {"sample": "GCA_002386245.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Staphylococcus aureus strain FDAARGOS_412 chromosome, complete genome", "reads_in": 184640, "reads_removed_percent": 0.004332755632582322, "reads_retained_percent": 99.99566724436741}, {"sample": "GCA_016889065.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Butyricimonas virosa strain FDAARGOS_1229 chromosome, complete genome", "reads_in": 320876, "reads_removed_percent": 0.0024931749336192174, "reads_retained_percent": 99.99750682506638}, {"sample": "GCA_002944995.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Staphylococcus epidermidis strain FDAARGOS_153 chromosome, complete genome", "reads_in": 170132, "reads_removed_percent": 0.004702231208708532, "reads_retained_percent": 99.99529776879129}, {"sample": "GCA_016889345.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Acinetobacter pittii strain FDAARGOS_1215 chromosome, complete genome", "reads_in": 259280, "reads_removed_percent": 0.0030854674483184203, "reads_retained_percent": 99.99691453255168}, {"sample": "GCA_020736245.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Burkholderia dolosa strain FDAARGOS_1544 chromosome 1, complete sequence", "reads_in": 427432, "reads_removed_percent": 0.0018716427408336299, "reads_retained_percent": 99.99812835725916}, {"sample": "GCA_021183725.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Mycobacterium ostraviense strain FDAARGOS_1613 chromosome, complete genome", "reads_in": 408516, "reads_removed_percent": 0.0019583076305456824, "reads_retained_percent": 99.99804169236945}, {"sample": "GCA_016728805.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Acinetobacter johnsonii strain FDAARGOS_1092 chromosome, complete genome", "reads_in": 275708, "reads_removed_percent": 0.0029016205550800125, "reads_retained_percent": 99.99709837944492}, {"sample": "GCA_002951295.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Acinetobacter nosocomialis strain FDAARGOS_129 chromosome, complete genome", "reads_in": 266846, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_021183765.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Mycobacterium intracellulare strain FDAARGOS_1611 chromosome, complete genome", "reads_in": 365956, "reads_removed_percent": 0.00218605515417154, "reads_retained_percent": 99.99781394484583}, {"sample": "GCA_002393465.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Escherichia coli strain FDAARGOS_401 chromosome, complete genome", "reads_in": 359158, "reads_removed_percent": 0.0022274319380328434, "reads_retained_percent": 99.99777256806196}, {"sample": "GCA_016726325.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Serratia plymuthica strain FDAARGOS_1138 chromosome, complete genome", "reads_in": 365626, "reads_removed_percent": 0.0016410211527626592, "reads_retained_percent": 99.99835897884724}, {"sample": "GCA_016127695.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Escherichia coli strain FDAARGOS_1059 chromosome, complete genome", "reads_in": 361050, "reads_removed_percent": 0.0016618196925633569, "reads_retained_percent": 99.99833818030744}, {"sample": "GCA_016889305.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Acinetobacter pittii strain FDAARGOS_1217 chromosome, complete genome", "reads_in": 263804, "reads_removed_percent": 0.0030325544722597077, "reads_retained_percent": 99.99696744552774}, {"sample": "GCA_002073775.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Vibrio parahaemolyticus strain FDAARGOS_191 chromosome 1, complete sequence", "reads_in": 344886, "reads_removed_percent": 0.0017397052939232091, "reads_retained_percent": 99.99826029470607}, {"sample": "GCA_020735865.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Parabacteroides johnsonii strain FDAARGOS_1580 chromosome, complete genome", "reads_in": 317744, "reads_removed_percent": 0.0025177501384762574, "reads_retained_percent": 99.99748224986152}, {"sample": "GCA_016888925.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Escherichia coli strain FDAARGOS_1260 chromosome, complete genome", "reads_in": 345352, "reads_removed_percent": 0.0023164771016238503, "reads_retained_percent": 99.99768352289837}, {"sample": "GCA_008693625.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Vibrio parahaemolyticus strain FDAARGOS_667 chromosome 1, complete sequence", "reads_in": 333544, "reads_removed_percent": 0.002398484158012136, "reads_retained_percent": 99.99760151584199}, {"sample": "GCA_020097235.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Enterobacter bugandensis strain FDAARGOS_1520 chromosome, complete genome", "reads_in": 334858, "reads_removed_percent": 0.0023890723829205216, "reads_retained_percent": 99.99761092761707}, {"sample": "GCA_016127915.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Streptococcus oralis strain FDAARGOS_1021 chromosome, complete genome", "reads_in": 127050, "reads_removed_percent": 0.006296733569460842, "reads_retained_percent": 99.99370326643054}, {"sample": "GCA_009730535.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Streptococcus gordonii strain FDAARGOS_683 chromosome, complete genome", "reads_in": 146440, "reads_removed_percent": 0.005462988254575253, "reads_retained_percent": 99.99453701174542}, {"sample": "GCA_019930665.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Bacteroides nordii strain FDAARGOS_1461 chromosome, complete genome", "reads_in": 378572, "reads_removed_percent": 0.002113204357427385, "reads_retained_percent": 99.99788679564257}, {"sample": "GCA_016127895.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Moraxella nonliquefaciens strain FDAARGOS_1006 chromosome, complete genome", "reads_in": 154890, "reads_removed_percent": 0.005164955775066176, "reads_retained_percent": 99.99483504422493}, {"sample": "GCA_002209225.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Salmonella enterica subsp. enterica serovar Typhimurium strain FDAARGOS_321 chromosome, complete genome", "reads_in": 334714, "reads_removed_percent": 0.0023901002049510926, "reads_retained_percent": 99.99760989979505}, {"sample": "GCA_009730115.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Staphylococcus hominis strain FDAARGOS_747 chromosome, complete genome", "reads_in": 149472, "reads_removed_percent": 0.005352172982230786, "reads_retained_percent": 99.99464782701777}, {"sample": "GCA_002984565.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Staphylococcus cohnii strain FDAARGOS_334 chromosome, complete genome", "reads_in": 177418, "reads_removed_percent": 0.004509125342411706, "reads_retained_percent": 99.99549087465759}, {"sample": "GCA_002073835.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Streptococcus salivarius strain FDAARGOS_259 chromosome, complete genome", "reads_in": 150622, "reads_removed_percent": 0.005311309104911633, "reads_retained_percent": 99.99468869089509}, {"sample": "GCA_016889685.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Escherichia coli strain FDAARGOS_1267 chromosome, complete genome", "reads_in": 362318, "reads_removed_percent": 0.0022080051225718844, "reads_retained_percent": 99.99779199487743}, {"sample": "GCA_016888885.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Moraxella osloensis strain FDAARGOS_1202 chromosome, complete genome", "reads_in": 183946, "reads_removed_percent": 0.004349102453981059, "reads_retained_percent": 99.99565089754601}, {"sample": "GCA_001558275.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Staphylococcus saprophyticus strain FDAARGOS_168 chromosome, complete genome", "reads_in": 176676, "reads_removed_percent": 0.004528062668387331, "reads_retained_percent": 99.99547193733162}, {"sample": "GCA_002209185.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Corynebacterium jeikeium strain FDAARGOS_328 chromosome, complete genome", "reads_in": 176726, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001018975.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Staphylococcus aureus strain FDAARGOS_30 chromosome, complete genome", "reads_in": 190776, "reads_removed_percent": 0.00419339958904684, "reads_retained_percent": 99.99580660041096}, {"sample": "GCA_013343115.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Streptococcus sanguinis strain FDAARGOS_770 chromosome, complete genome", "reads_in": 157256, "reads_removed_percent": 0.0038154347051940784, "reads_retained_percent": 99.99618456529481}, {"sample": "GCA_009730475.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Corynebacterium xerosis strain FDAARGOS_674 chromosome, complete genome", "reads_in": 188102, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903195.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Klebsiella pneumoniae strain FDAARGOS_1305 chromosome, complete genome", "reads_in": 384390, "reads_removed_percent": 0.001560914696011863, "reads_retained_percent": 99.99843908530399}, {"sample": "GCA_002953295.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Klebsiella quasipneumoniae strain FDAARGOS_93 chromosome, complete genome", "reads_in": 367536, "reads_removed_percent": 0.0016324931435287974, "reads_retained_percent": 99.99836750685647}, {"sample": "GCA_021183645.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Nocardia gipuzkoensis strain FDAARGOS_1619 chromosome, complete genome", "reads_in": 541242, "reads_removed_percent": 0.0011085614198454666, "reads_retained_percent": 99.99889143858016}, {"sample": "GCA_001558435.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Vibrio harveyi strain FDAARGOS_107 chromosome 1, complete sequence", "reads_in": 402594, "reads_removed_percent": 0.001490335176381168, "reads_retained_percent": 99.99850966482362}, {"sample": "GCA_019048805.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Citrobacter braakii strain FDAARGOS 1421 chromosome, complete genome", "reads_in": 371974, "reads_removed_percent": 0.0016130159634813188, "reads_retained_percent": 99.99838698403651}, {"sample": "GCA_020450285.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "UNVERIFIED_ORG: Hafnia paralvei strain FDAARGOS_1502 chromosome, complete genome", "reads_in": 322462, "reads_removed_percent": 0.0018606843597075006, "reads_retained_percent": 99.99813931564029}, {"sample": "GCA_016726885.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Pediococcus pentosaceus strain FDAARGOS_1134 chromosome, complete genome", "reads_in": 116092, "reads_removed_percent": 0.005168314784825828, "reads_retained_percent": 99.99483168521517}, {"sample": "GCA_019930685.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Legionella longbeachae strain FDAARGOS_1481 chromosome, complete genome", "reads_in": 277696, "reads_removed_percent": 0.0021606360912652684, "reads_retained_percent": 99.99783936390874}, {"sample": "GCA_009732965.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Aerococcus viridans strain FDAARGOS_672 chromosome, complete genome", "reads_in": 138814, "reads_removed_percent": 0.004322330600659876, "reads_retained_percent": 99.99567766939934}, {"sample": "GCA_002386285.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Streptococcus gordonii strain FDAARGOS_371 chromosome, complete genome", "reads_in": 141420, "reads_removed_percent": 0.004242681374628765, "reads_retained_percent": 99.99575731862537}, {"sample": "GCA_020221755.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Lactococcus lactis strain FDAARGOS_867 chromosome, complete genome", "reads_in": 178994, "reads_removed_percent": 0.0033520676670726396, "reads_retained_percent": 99.99664793233293}, {"sample": "GCA_002953375.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Vibrio fluvialis strain FDAARGOS_100 chromosome 1, complete sequence", "reads_in": 324504, "reads_removed_percent": 0.001848975667480216, "reads_retained_percent": 99.99815102433251}, {"sample": "GCA_009730335.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Schaalia odontolytica strain FDAARGOS_732 chromosome, complete genome", "reads_in": 162176, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735945.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Parabacteroides distasonis strain FDAARGOS_1565 chromosome, complete genome", "reads_in": 320804, "reads_removed_percent": 0.0018703008690664704, "reads_retained_percent": 99.99812969913093}, {"sample": "GCA_002588845.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Serratia fonticola strain FDAARGOS_411 chromosome, complete genome", "reads_in": 381692, "reads_removed_percent": 0.0015719480628360039, "reads_retained_percent": 99.99842805193717}, {"sample": "GCA_019056115.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Escherichia coli strain FDAARGOS 1383 chromosome, complete genome", "reads_in": 337060, "reads_removed_percent": 0.0017800984987835994, "reads_retained_percent": 99.99821990150122}, {"sample": "GCA_019048165.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Corynebacterium coyleae strain FDAARGOS 1425 chromosome, complete genome", "reads_in": 167350, "reads_removed_percent": 0.0035853002688975197, "reads_retained_percent": 99.9964146997311}, {"sample": "GCA_016126935.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Stenotrophomonas maltophilia strain FDAARGOS_1044 chromosome, complete genome", "reads_in": 333618, "reads_removed_percent": 0.0017984641116486522, "reads_retained_percent": 99.99820153588836}, {"sample": "GCA_016127815.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Pediococcus acidilactici strain FDAARGOS_1007 chromosome, complete genome", "reads_in": 137886, "reads_removed_percent": 0.004351420738871242, "reads_retained_percent": 99.99564857926113}, {"sample": "GCA_016889125.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Escherichia coli strain FDAARGOS_1249 chromosome, complete genome", "reads_in": 348824, "reads_removed_percent": 0.0017200651331330415, "reads_retained_percent": 99.99827993486687}, {"sample": "GCA_019047745.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Escherichia coli strain FDAARGOS_1381 chromosome, complete genome", "reads_in": 349546, "reads_removed_percent": 0.0017165122759236267, "reads_retained_percent": 99.99828348772408}, {"sample": "GCA_009730455.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Cupriavidus metallidurans strain FDAARGOS_675 chromosome, complete genome", "reads_in": 462760, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003854935.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Cupriavidus pauculus strain FDAARGOS_614 chromosome 1, complete sequence", "reads_in": 411854, "reads_removed_percent": 0.0014568269338163524, "reads_retained_percent": 99.99854317306618}, {"sample": "GCA_002393265.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Enterococcus sp. FDAARGOS_375 chromosome, complete genome", "reads_in": 236622, "reads_removed_percent": 0.002535689834419454, "reads_retained_percent": 99.99746431016558}, {"sample": "GCA_003855115.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Shigella sonnei strain FDAARGOS_524 chromosome, complete genome", "reads_in": 330206, "reads_removed_percent": 0.0018170475400204722, "reads_retained_percent": 99.99818295245998}, {"sample": "GCA_020099215.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Enterobacter ludwigii strain FDAARGOS_1498 chromosome, complete genome", "reads_in": 315380, "reads_removed_percent": 0.0019024668653687615, "reads_retained_percent": 99.99809753313463}, {"sample": "GCA_001188185.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Vibrio parahaemolyticus strain FDAARGOS_51 chromosome 1, complete sequence", "reads_in": 343280, "reads_removed_percent": 0.0017478443253320904, "reads_retained_percent": 99.99825215567466}, {"sample": "GCA_016725825.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Enterobacter hormaechei strain FDAARGOS_1078 chromosome, complete genome", "reads_in": 338684, "reads_removed_percent": 0.0017715628727663546, "reads_retained_percent": 99.99822843712724}, {"sample": "GCA_016889165.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Staphylococcus epidermidis strain FDAARGOS_1243 chromosome, complete genome", "reads_in": 168996, "reads_removed_percent": 0.0035503798906482997, "reads_retained_percent": 99.99644962010935}, {"sample": "GCA_003955815.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Francisella tularensis strain FDAARGOS_595 chromosome, complete genome", "reads_in": 126280, "reads_removed_percent": 0.004751346214760849, "reads_retained_percent": 99.99524865378524}, {"sample": "GCA_001558775.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Staphylococcus lugdunensis strain FDAARGOS_141 chromosome, complete genome", "reads_in": 171456, "reads_removed_percent": 0.0034994400895856665, "reads_retained_percent": 99.99650055991042}, {"sample": "GCA_002073595.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Neisseria meningitidis strain FDAARGOS_210 chromosome, complete genome", "reads_in": 151552, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009730575.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Shewanella putrefaciens strain FDAARGOS_681 chromosome, complete genome", "reads_in": 310642, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001558255.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Grimontia hollisae strain FDAARGOS_111 chromosome 1, complete sequence", "reads_in": 266562, "reads_removed_percent": 0.002250883471762667, "reads_retained_percent": 99.99774911652824}, {"sample": "GCA_001559215.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Klebsiella aerogenes strain FDAARGOS_152 chromosome, complete genome", "reads_in": 354476, "reads_removed_percent": 0.0016926392760017604, "reads_retained_percent": 99.998307360724}, {"sample": "GCA_016903815.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Escherichia coli strain FDAARGOS_1280 chromosome, complete genome", "reads_in": 366450, "reads_removed_percent": 0.0016373311502251332, "reads_retained_percent": 99.99836266884978}, {"sample": "GCA_003020025.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Burkholderia thailandensis strain FDAARGOS_237 chromosome 1, complete sequence", "reads_in": 444854, "reads_removed_percent": 0.0013487571203136311, "reads_retained_percent": 99.99865124287969}, {"sample": "GCA_016903975.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Klebsiella oxytoca strain FDAARGOS_1333 chromosome, complete genome", "reads_in": 404192, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016939455.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Leuconostoc falkenbergense strain FDAARGOS_1201 chromosome, complete genome", "reads_in": 126594, "reads_removed_percent": 0.0031597074110937327, "reads_retained_percent": 99.9968402925889}, {"sample": "GCA_001559115.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Staphylococcus simulans strain FDAARGOS_124 chromosome, complete genome", "reads_in": 176504, "reads_removed_percent": 0.002266237592349182, "reads_retained_percent": 99.99773376240765}, {"sample": "GCA_016903955.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1309 chromosome, complete genome", "reads_in": 373694, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009730375.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Capnocytophaga sp. FDAARGOS_737 chromosome, complete genome", "reads_in": 176848, "reads_removed_percent": 0.002261829367592509, "reads_retained_percent": 99.9977381706324}, {"sample": "GCA_009730075.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Yersinia intermedia strain FDAARGOS_729 chromosome, complete genome", "reads_in": 328594, "reads_removed_percent": 0.0012173076806028108, "reads_retained_percent": 99.9987826923194}, {"sample": "GCA_009730555.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Mammaliicoccus fleurettii strain FDAARGOS_682 chromosome, complete genome", "reads_in": 168504, "reads_removed_percent": 0.0023738308882875186, "reads_retained_percent": 99.99762616911171}, {"sample": "GCA_016903235.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Klebsiella pneumoniae strain FDAARGOS_1324 chromosome, complete genome", "reads_in": 376682, "reads_removed_percent": 0.0010619036747176664, "reads_retained_percent": 99.99893809632528}, {"sample": "GCA_001559035.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Bartonella bacilliformis strain ATCC 35685D-5 chromosome, complete genome", "reads_in": 96228, "reads_removed_percent": 0.00415679428025107, "reads_retained_percent": 99.99584320571975}, {"sample": "GCA_020097375.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Porphyromonas endodontalis strain FDAARGOS_1506 chromosome, complete genome", "reads_in": 140196, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009730195.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Burkholderia multivorans strain FDAARGOS_722 chromosome 1, complete sequence", "reads_in": 439364, "reads_removed_percent": 0.0009104068608261031, "reads_retained_percent": 99.99908959313917}, {"sample": "GCA_021183665.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Mycolicibacterium smegmatis strain FDAARGOS_1618 chromosome, complete genome", "reads_in": 459422, "reads_removed_percent": 0.0008706592196281414, "reads_retained_percent": 99.99912934078037}, {"sample": "GCA_002591115.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Klebsiella aerogenes strain FDAARGOS_363 chromosome, complete genome", "reads_in": 349698, "reads_removed_percent": 0.0011438441169237456, "reads_retained_percent": 99.99885615588308}, {"sample": "GCA_016127375.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Lactococcus garvieae strain FDAARGOS_1002 chromosome, complete genome", "reads_in": 152810, "reads_removed_percent": 0.0026176297362738043, "reads_retained_percent": 99.99738237026372}, {"sample": "GCA_016904595.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Klebsiella pneumoniae strain FDAARGOS_1310 chromosome, complete genome", "reads_in": 372866, "reads_removed_percent": 0.0010727714514061353, "reads_retained_percent": 99.9989272285486}, {"sample": "GCA_016889365.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Corynebacterium kroppenstedtii strain FDAARGOS_1192 chromosome, complete genome", "reads_in": 163120, "reads_removed_percent": 0.002452182442373713, "reads_retained_percent": 99.99754781755763}, {"sample": "GCA_013343095.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Achromobacter denitrificans strain FDAARGOS_787 chromosome, complete genome", "reads_in": 454436, "reads_removed_percent": 0.0008802119550387733, "reads_retained_percent": 99.99911978804496}, {"sample": "GCA_002073675.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Neisseria meningitidis strain FDAARGOS_212 chromosome, complete genome", "reads_in": 149660, "reads_removed_percent": 0.0026727248429774153, "reads_retained_percent": 99.99732727515702}, {"sample": "GCA_002589795.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Escherichia coli strain FDAARGOS_433 chromosome, complete genome", "reads_in": 341458, "reads_removed_percent": 0.0011714471472333348, "reads_retained_percent": 99.99882855285277}, {"sample": "GCA_016904295.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1306 chromosome, complete genome", "reads_in": 370864, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904655.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1284 chromosome, complete genome", "reads_in": 337852, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016904275.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1328 chromosome, complete genome", "reads_in": 383876, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_009730435.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Gordonia bronchialis strain FDAARGOS_676 chromosome, complete genome", "reads_in": 353264, "reads_removed_percent": 0.0011322976584084423, "reads_retained_percent": 99.99886770234158}, {"sample": "GCA_013343135.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Achromobacter xylosoxidans strain FDAARGOS_789 chromosome, complete genome", "reads_in": 453762, "reads_removed_percent": 0.0008815193868151144, "reads_retained_percent": 99.99911848061319}, {"sample": "GCA_009806655.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Bacillus anthracis strain FDAARGOS_706 chromosome, complete genome", "reads_in": 348660, "reads_removed_percent": 0.0011472494693971205, "reads_retained_percent": 99.9988527505306}, {"sample": "GCA_009730415.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Micrococcus luteus strain FDAARGOS_677 chromosome, complete genome", "reads_in": 165218, "reads_removed_percent": 0.002421043711944219, "reads_retained_percent": 99.99757895628805}, {"sample": "GCA_016904035.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1322 chromosome, complete genome", "reads_in": 363140, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020099315.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Citrobacter europaeus strain FDAARGOS_1490 chromosome, complete genome", "reads_in": 339568, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002073695.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Streptococcus agalactiae strain FDAARGOS_254 chromosome, complete genome", "reads_in": 147914, "reads_removed_percent": 0.0027042741052233058, "reads_retained_percent": 99.99729572589477}, {"sample": "GCA_016127535.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Streptococcus salivarius strain FDAARGOS_1045 chromosome, complete genome", "reads_in": 154588, "reads_removed_percent": 0.002587522964266308, "reads_retained_percent": 99.99741247703574}, {"sample": "GCA_002073715.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Neisseria mucosa strain FDAARGOS_260 chromosome, complete genome", "reads_in": 185596, "reads_removed_percent": 0.0021552188624754842, "reads_retained_percent": 99.99784478113753}, {"sample": "GCA_016127555.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Streptococcus oralis strain FDAARGOS_1020 chromosome, complete genome", "reads_in": 128770, "reads_removed_percent": 0.0031063135823561384, "reads_retained_percent": 99.99689368641765}, {"sample": "GCA_016890105.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Escherichia coli strain FDAARGOS_1266 chromosome, complete genome", "reads_in": 347734, "reads_removed_percent": 0.0011503045431277932, "reads_retained_percent": 99.99884969545687}, {"sample": "GCA_019930645.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Gordonia bronchialis strain FDAARGOS_1476 chromosome, complete genome", "reads_in": 353808, "reads_removed_percent": 0.0011305566861122417, "reads_retained_percent": 99.9988694433139}, {"sample": "GCA_020097475.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Escherichia fergusonii strain FDAARGOS_1499 chromosome, complete genome", "reads_in": 309728, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020097495.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Corynebacterium freneyi strain FDAARGOS_1495 chromosome, complete genome", "reads_in": 192122, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019048245.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Enterobacter hormaechei strain FDAARGOS 1433 chromosome, complete genome", "reads_in": 327728, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_003812545.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Listeria monocytogenes strain FDAARGOS_555 chromosome, complete genome", "reads_in": 190138, "reads_removed_percent": 0.002103735181815313, "reads_retained_percent": 99.99789626481818}, {"sample": "GCA_001558475.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Vibrio mimicus strain FDAARGOS_112 chromosome 1, complete sequence", "reads_in": 295914, "reads_removed_percent": 0.0013517440878092959, "reads_retained_percent": 99.99864825591219}, {"sample": "GCA_016889045.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Escherichia coli strain FDAARGOS_1255 chromosome, complete genome", "reads_in": 337566, "reads_removed_percent": 0.0011849534609528211, "reads_retained_percent": 99.99881504653905}, {"sample": "GCA_001558295.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Shigella sonnei strain FDAARGOS_90 chromosome, complete genome", "reads_in": 332556, "reads_removed_percent": 0.001202804941122698, "reads_retained_percent": 99.99879719505888}, {"sample": "GCA_020735885.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Bordetella pertussis strain FDAARGOS_1542 chromosome, complete genome", "reads_in": 274846, "reads_removed_percent": 0.001455360456401039, "reads_retained_percent": 99.9985446395436}, {"sample": "GCA_003812565.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Listeria monocytogenes strain FDAARGOS_554 chromosome, complete genome", "reads_in": 189124, "reads_removed_percent": 0.0021150144878492416, "reads_retained_percent": 99.99788498551214}, {"sample": "GCA_016027835.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Serratia plymuthica strain FDAARGOS_889 chromosome, complete genome", "reads_in": 362786, "reads_removed_percent": 0.0011025783795405556, "reads_retained_percent": 99.99889742162046}, {"sample": "GCA_019047925.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Klebsiella aerogenes strain FDAARGOS 1441 chromosome, complete genome", "reads_in": 346180, "reads_removed_percent": 0.001155468253509735, "reads_retained_percent": 99.99884453174649}, {"sample": "GCA_003812345.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Citrobacter freundii strain FDAARGOS_549 chromosome, complete genome", "reads_in": 340144, "reads_removed_percent": 0.0011759725292817158, "reads_retained_percent": 99.99882402747072}, {"sample": "GCA_019048365.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Aeromonas sp. FDAARGOS 1415 chromosome, complete genome", "reads_in": 289762, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016888945.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Phocaeicola coprophilus strain FDAARGOS_1220 chromosome, complete genome", "reads_in": 274240, "reads_removed_percent": 0.0014585764294049008, "reads_retained_percent": 99.9985414235706}, {"sample": "GCA_020735845.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Tsukamurella paurometabola strain FDAARGOS_1558 chromosome, complete genome", "reads_in": 313538, "reads_removed_percent": 0.001275762427520747, "reads_retained_percent": 99.99872423757247}, {"sample": "GCA_020735485.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Mycobacteroides abscessus subsp. abscessus strain FDAARGOS_1603 chromosome, complete genome", "reads_in": 322226, "reads_removed_percent": 0.0012413647564132005, "reads_retained_percent": 99.99875863524359}, {"sample": "GCA_003812425.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Staphylococcus epidermidis strain FDAARGOS_529 chromosome, complete genome", "reads_in": 171828, "reads_removed_percent": 0.0023279093046534903, "reads_retained_percent": 99.99767209069535}, {"sample": "GCA_016127955.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Actinomyces oris strain FDAARGOS_1051 chromosome, complete genome", "reads_in": 212692, "reads_removed_percent": 0.0018806537152314144, "reads_retained_percent": 99.99811934628477}, {"sample": "GCA_002953535.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Vibrio alginolyticus strain FDAARGOS_110 chromosome 1, complete sequence", "reads_in": 351246, "reads_removed_percent": 0.001138803004162325, "reads_retained_percent": 99.99886119699583}, {"sample": "GCA_021183705.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Mycobacterium kansasii strain FDAARGOS_1614 chromosome, complete genome", "reads_in": 430260, "reads_removed_percent": 0.0009296704318319156, "reads_retained_percent": 99.99907032956817}, {"sample": "GCA_002073635.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Paracoccus yeei strain FDAARGOS_252 chromosome, complete genome", "reads_in": 321974, "reads_removed_percent": 0.0012423363377167102, "reads_retained_percent": 99.99875766366229}, {"sample": "GCA_019048385.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Pantoea agglomerans strain FDAARGOS 1447 chromosome, complete genome", "reads_in": 312802, "reads_removed_percent": 0.0012787642022749216, "reads_retained_percent": 99.99872123579773}, {"sample": "GCA_016904175.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1372 chromosome, complete genome", "reads_in": 321568, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020097615.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Corynebacterium falsenii strain FDAARGOS_1493 chromosome, complete genome", "reads_in": 170792, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_013267315.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Bacillus thuringiensis strain FDAARGOS_795 chromosome, complete genome", "reads_in": 381992, "reads_removed_percent": 0.0010471423485308594, "reads_retained_percent": 99.99895285765147}, {"sample": "GCA_016889825.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Acinetobacter pittii strain FDAARGOS_1214 chromosome, complete genome", "reads_in": 273260, "reads_removed_percent": 0.0014638073629510355, "reads_retained_percent": 99.99853619263705}, {"sample": "GCA_003812585.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Burkholderia multivorans strain FDAARGOS_548 chromosome 1, complete sequence", "reads_in": 443592, "reads_removed_percent": 0.0009017295172140165, "reads_retained_percent": 99.99909827048279}, {"sample": "GCA_016903215.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Acinetobacter baumannii strain FDAARGOS_1359 chromosome, complete genome", "reads_in": 259886, "reads_removed_percent": 0.0015391363905712504, "reads_retained_percent": 99.99846086360942}, {"sample": "GCA_016127775.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Pediococcus pentosaceus strain FDAARGOS_1009 chromosome, complete genome", "reads_in": 116796, "reads_removed_percent": 0.003424774821055516, "reads_retained_percent": 99.99657522517894}, {"sample": "GCA_016127795.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Stutzerimonas stutzeri strain FDAARGOS_1065 chromosome, complete genome", "reads_in": 295130, "reads_removed_percent": 0.0013553349371463422, "reads_retained_percent": 99.99864466506286}, {"sample": "GCA_020736265.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Bordetella pertussis strain FDAARGOS_1543 chromosome, complete genome", "reads_in": 273102, "reads_removed_percent": 0.001464654231752239, "reads_retained_percent": 99.99853534576825}, {"sample": "GCA_009730255.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Lactobacillus jensenii strain FDAARGOS_749 chromosome, complete genome", "reads_in": 107268, "reads_removed_percent": 0.0037289778871611294, "reads_retained_percent": 99.99627102211284}, {"sample": "GCA_019048505.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Escherichia coli strain FDAARGOS 1387 chromosome, complete genome", "reads_in": 348454, "reads_removed_percent": 0.0011479277035132327, "reads_retained_percent": 99.99885207229649}, {"sample": "GCA_002208685.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Kocuria rhizophila strain FDAARGOS_302 chromosome, complete genome", "reads_in": 179858, "reads_removed_percent": 0.0022239766927242602, "reads_retained_percent": 99.99777602330728}, {"sample": "GCA_016126955.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Pseudomonas aeruginosa strain FDAARGOS_1041 chromosome, complete genome", "reads_in": 456166, "reads_removed_percent": 0.0008768737696364919, "reads_retained_percent": 99.99912312623036}, {"sample": "GCA_020736225.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Campylobacter curvus strain FDAARGOS_1547 chromosome, complete genome", "reads_in": 131660, "reads_removed_percent": 0.0030381285128360932, "reads_retained_percent": 99.99696187148716}, {"sample": "GCA_001188045.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Listeria monocytogenes strain FDAARGOS_57 chromosome unnamed, complete genome", "reads_in": 195692, "reads_removed_percent": 0.002044028371113791, "reads_retained_percent": 99.99795597162888}, {"sample": "GCA_002073655.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Neisseria gonorrhoeae strain FDAARGOS_207 chromosome, complete genome", "reads_in": 145954, "reads_removed_percent": 0.0027405895008016227, "reads_retained_percent": 99.9972594104992}, {"sample": "GCA_019046965.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Acinetobacter pittii strain FDAARGOS 1398 chromosome, complete genome", "reads_in": 255868, "reads_removed_percent": 0.002344959119546016, "reads_retained_percent": 99.99765504088046}, {"sample": "GCA_016127115.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Aggregatibacter segnis strain FDAARGOS_987 chromosome, complete genome", "reads_in": 137500, "reads_removed_percent": 0.004363636363636364, "reads_retained_percent": 99.99563636363636}, {"sample": "GCA_008693605.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Citrobacter portucalensis strain FDAARGOS_617 chromosome, complete genome", "reads_in": 328624, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_000783755.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Citrobacter freundii strain FDAARGOS_73 chromosome, complete genome", "reads_in": 363416, "reads_removed_percent": 0.001651000506306822, "reads_retained_percent": 99.9983489994937}, {"sample": "GCA_016127215.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Haemophilus parainfluenzae strain FDAARGOS_1000 chromosome, complete genome", "reads_in": 142780, "reads_removed_percent": 0.004202269225381706, "reads_retained_percent": 99.99579773077461}, {"sample": "GCA_008693645.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Citrobacter werkmanii strain FDAARGOS_616 chromosome, complete genome", "reads_in": 336620, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016728505.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Corynebacterium riegelii strain FDAARGOS_1114 chromosome, complete genome", "reads_in": 167948, "reads_removed_percent": 0.003572534355872056, "reads_retained_percent": 99.99642746564413}, {"sample": "GCA_008693665.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Bordetella bronchiseptica strain FDAARGOS_634 chromosome, complete genome", "reads_in": 341072, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127175.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Veillonella parvula strain FDAARGOS_1046 chromosome, complete genome", "reads_in": 142154, "reads_removed_percent": 0.0042207746528412846, "reads_retained_percent": 99.99577922534716}, {"sample": "GCA_020735685.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Lachnospiraceae bacterium GAM79 strain FDAARGOS_1582 chromosome, complete genome", "reads_in": 198604, "reads_removed_percent": 0.0030210871885762622, "reads_retained_percent": 99.99697891281143}, {"sample": "GCA_016903775.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Klebsiella pneumoniae strain FDAARGOS_1307 chromosome, complete genome", "reads_in": 378308, "reads_removed_percent": 0.001586009283441006, "reads_retained_percent": 99.99841399071656}, {"sample": "GCA_016903135.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Acinetobacter baumannii strain FDAARGOS_1360 chromosome, complete genome", "reads_in": 271340, "reads_removed_percent": 0.0022112478808874476, "reads_retained_percent": 99.9977887521191}, {"sample": "GCA_021183785.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Mycobacterium intracellulare subsp. intracellulare strain FDAARGOS_1610 chromosome, complete genome", "reads_in": 360844, "reads_removed_percent": 0.0016627683985323298, "reads_retained_percent": 99.99833723160147}, {"sample": "GCA_016889985.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Corynebacterium sp. FDAARGOS 1242 strain FDAARGOS_1242 chromosome, complete genome", "reads_in": 185000, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016127055.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Staphylococcus felis strain FDAARGOS_1014 chromosome, complete genome", "reads_in": 165302, "reads_removed_percent": 0.0036297201485765446, "reads_retained_percent": 99.99637027985142}, {"sample": "GCA_016727305.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Enterococcus casseliflavus strain FDAARGOS_1122 chromosome, complete genome", "reads_in": 243400, "reads_removed_percent": 0.0024650780608052587, "reads_retained_percent": 99.99753492193919}, {"sample": "GCA_020735665.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Lachnospiraceae bacterium GAM79 strain FDAARGOS_1585 chromosome, complete genome", "reads_in": 197388, "reads_removed_percent": 0.0030396984619125785, "reads_retained_percent": 99.99696030153808}, {"sample": "GCA_008693845.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Pasteurella multocida strain FDAARGOS_644 chromosome, complete genome", "reads_in": 150004, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016903395.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Klebsiella pneumoniae strain FDAARGOS_1313 chromosome, complete genome", "reads_in": 385540, "reads_removed_percent": 0.001556258753955491, "reads_retained_percent": 99.99844374124605}, {"sample": "GCA_016903715.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Klebsiella oxytoca strain FDAARGOS_1332 chromosome, complete genome", "reads_in": 403376, "reads_removed_percent": 0.0014874459561302605, "reads_retained_percent": 99.99851255404387}, {"sample": "GCA_008693905.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Enterobacter hormaechei strain FDAARGOS_642 chromosome, complete genome", "reads_in": 313208, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_016889025.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Escherichia coli strain FDAARGOS_1251 chromosome, complete genome", "reads_in": 349158, "reads_removed_percent": 0.001718419741205987, "reads_retained_percent": 99.99828158025879}, {"sample": "GCA_008693925.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Dermabacter vaginalis strain FDAARGOS_640 chromosome, complete genome", "reads_in": 152620, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_020735525.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Helicobacter pylori strain FDAARGOS_1601 chromosome, complete genome", "reads_in": 109606, "reads_removed_percent": 0.005474152874842618, "reads_retained_percent": 99.99452584712516}, {"sample": "GCA_016027075.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Delftia acidovorans strain FDAARGOS_909 chromosome, complete genome", "reads_in": 447236, "reads_removed_percent": 0.0013415735763668399, "reads_retained_percent": 99.99865842642363}, {"sample": "GCA_016727425.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Klebsiella pneumoniae strain FDAARGOS_1076 chromosome, complete genome", "reads_in": 360454, "reads_removed_percent": 0.0016645674621449617, "reads_retained_percent": 99.99833543253786}, {"sample": "GCA_003812165.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Pseudomonas aeruginosa strain FDAARGOS_532 chromosome, complete genome", "reads_in": 462084, "reads_removed_percent": 0.0012984652141169138, "reads_retained_percent": 99.99870153478588}, {"sample": "GCA_021183565.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Nocardia yamanashiensis strain FDAARGOS_1623 chromosome, complete genome", "reads_in": 564586, "reads_removed_percent": 0.0010627256077904873, "reads_retained_percent": 99.9989372743922}, {"sample": "GCA_013267515.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Streptococcus intermedius strain FDAARGOS_769 chromosome, complete genome", "reads_in": 128892, "reads_removed_percent": 0.004655060050274649, "reads_retained_percent": 99.99534493994973}, {"sample": "GCA_016127435.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Salmonella enterica strain FDAARGOS_1067 chromosome, complete genome", "reads_in": 337820, "reads_removed_percent": 0.001776093777751465, "reads_retained_percent": 99.99822390622225}, {"sample": "GCA_003812205.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Acinetobacter sp. FDAARGOS_560 chromosome, complete genome", "reads_in": 277772, "reads_removed_percent": 0.002160044928934522, "reads_retained_percent": 99.99783995507107}, {"sample": "GCA_002953515.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Vibrio alginolyticus strain FDAARGOS_108 chromosome 1, complete sequence", "reads_in": 363068, "reads_removed_percent": 0.0016525829872090076, "reads_retained_percent": 99.9983474170128}, {"sample": "GCA_020735605.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Parabacteroides merdae strain FDAARGOS_1586 chromosome, complete genome", "reads_in": 291720, "reads_removed_percent": 0.0020567667626491155, "reads_retained_percent": 99.99794323323735}, {"sample": "GCA_003798205.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Yersinia pestis strain FDAARGOS_603 chromosome, complete genome", "reads_in": 328642, "reads_removed_percent": 0.0018256948290236795, "reads_retained_percent": 99.99817430517098}, {"sample": "GCA_019048045.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Citrobacter braakii strain FDAARGOS 1422 chromosome, complete genome", "reads_in": 321184, "reads_removed_percent": 0.0018680880741257348, "reads_retained_percent": 99.99813191192588}, {"sample": "GCA_002983785.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Enterococcus faecium strain FDAARGOS_323 chromosome, complete genome", "reads_in": 180694, "reads_removed_percent": 0.0033205308421973057, "reads_retained_percent": 99.9966794691578}, {"sample": "GCA_002393485.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Ralstonia pickettii strain FDAARGOS_410 chromosome 1, complete sequence", "reads_in": 321204, "reads_removed_percent": 0.00124531450417803, "reads_retained_percent": 99.99875468549583}, {"sample": "GCA_016127195.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Aeromonas jandaei strain FDAARGOS_986 chromosome, complete genome", "reads_in": 303406, "reads_removed_percent": 0.0013183654904649217, "reads_retained_percent": 99.99868163450954}, {"sample": "GCA_016904635.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Klebsiella pneumoniae strain FDAARGOS_1320 chromosome, complete genome", "reads_in": 351422, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_001018805.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Staphylococcus aureus strain FDAARGOS_15 chromosome, complete genome", "reads_in": 186660, "reads_removed_percent": 0.0021429336762027215, "reads_retained_percent": 99.9978570663238}, {"sample": "GCA_016128135.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Streptococcus dysgalactiae strain FDAARGOS_1016 chromosome, complete genome", "reads_in": 151136, "reads_removed_percent": 0.002646622909167902, "reads_retained_percent": 99.99735337709083}, {"sample": "GCA_008693965.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Pseudomonas luteola strain FDAARGOS_637 chromosome 1, complete sequence", "reads_in": 393426, "reads_removed_percent": 0.0010167096226482235, "reads_retained_percent": 99.99898329037735}, {"sample": "GCA_002591235.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Pseudomonas sp. FDAARGOS_380 chromosome, complete genome", "reads_in": 433290, "reads_removed_percent": 0.0009231692400009233, "reads_retained_percent": 99.99907683076}, {"sample": "GCA_002073555.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Neisseria meningitidis strain FDAARGOS_211 chromosome, complete genome", "reads_in": 153720, "reads_removed_percent": 0.0026021337496747333, "reads_retained_percent": 99.99739786625032}, {"sample": "GCA_020736165.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Capnocytophaga ochracea strain FDAARGOS_1550 chromosome, complete genome", "reads_in": 180882, "reads_removed_percent": 0.0022113864287214868, "reads_retained_percent": 99.99778861357127}, {"sample": "GCA_016128075.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Corynebacterium bovis strain FDAARGOS_1052 chromosome, complete genome", "reads_in": 176814, "reads_removed_percent": 0.0022622643003382082, "reads_retained_percent": 99.99773773569966}, {"sample": "GCA_016903255.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Klebsiella pneumoniae strain FDAARGOS_1311 chromosome, complete genome", "reads_in": 370730, "reads_removed_percent": 0.0010789523372805006, "reads_retained_percent": 99.99892104766272}, {"sample": "GCA_016127395.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Corynebacterium glucuronolyticum strain FDAARGOS_1053 chromosome, complete genome", "reads_in": 194320, "reads_removed_percent": 0.0020584602717167557, "reads_retained_percent": 99.99794153972829}, {"sample": "GCA_016026055.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Burkholderia glumae strain FDAARGOS_949 chromosome 1, complete sequence", "reads_in": 454724, "reads_removed_percent": 0.000879654471723507, "reads_retained_percent": 99.99912034552828}, {"sample": "GCA_016904535.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Escherichia coli strain FDAARGOS_1297 chromosome, complete genome", "reads_in": 345586, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_002591195.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Yersinia frederiksenii strain FDAARGOS_418 chromosome, complete genome", "reads_in": 328936, "reads_removed_percent": 0.0012160420264124329, "reads_retained_percent": 99.99878395797359}, {"sample": "GCA_016904455.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Escherichia coli strain FDAARGOS_1298 chromosome, complete genome", "reads_in": 357678, "reads_removed_percent": 0.0011183243028645878, "reads_retained_percent": 99.99888167569713}, {"sample": "GCA_019931005.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Granulicatella adiacens strain FDAARGOS_1477 chromosome, complete genome", "reads_in": 131028, "reads_removed_percent": 0.0030527826113502456, "reads_retained_percent": 99.99694721738865}, {"sample": "GCA_016904435.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Escherichia coli strain FDAARGOS_1301 chromosome, complete genome", "reads_in": 347014, "reads_removed_percent": 0.001152691245886333, "reads_retained_percent": 99.99884730875411}, {"sample": "GCA_012273555.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Chromobacterium violaceum strain FDAARGOS_635 chromosome, complete genome", "reads_in": 320050, "reads_removed_percent": 0.0018747070770192159, "reads_retained_percent": 99.99812529292298}, {"sample": "GCA_003812465.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Streptococcus sp. FDAARGOS_521 chromosome, complete genome", "reads_in": 143800, "reads_removed_percent": 0.004172461752433936, "reads_retained_percent": 99.99582753824757}, {"sample": "GCA_002073275.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Neisseria meningitidis strain FDAARGOS_215 chromosome, complete genome", "reads_in": 153722, "reads_removed_percent": 0.0039031498419224314, "reads_retained_percent": 99.99609685015808}, {"sample": "GCA_003614835.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Burkholderia thailandensis strain FDAARGOS_242 chromosome 1, complete sequence", "reads_in": 443554, "reads_removed_percent": 0.001352710154795132, "reads_retained_percent": 99.99864728984521}, {"sample": "GCA_001019415.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Staphylococcus aureus strain FDAARGOS_24 chromosome, complete genome", "reads_in": 179832, "reads_removed_percent": 0.0033364473508608037, "reads_retained_percent": 99.99666355264914}, {"sample": "GCA_002073455.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Legionella longbeachae strain FDAARGOS_201 chromosome, complete genome", "reads_in": 277516, "reads_removed_percent": 0.0021620375041439053, "reads_retained_percent": 99.99783796249585}, {"sample": "GCA_016027595.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 6, "name": "Serratia plymuthica strain FDAARGOS_896 chromosome, complete genome", "reads_in": 368456, "reads_removed_percent": 0.001628416961591072, "reads_retained_percent": 99.9983715830384}, {"sample": "GCA_016128055.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Pediococcus acidilactici strain FDAARGOS_1008 chromosome, complete genome", "reads_in": 133972, "reads_removed_percent": 0.0029856985041650493, "reads_retained_percent": 99.99701430149584}, {"sample": "GCA_009729995.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Shigella sonnei strain FDAARGOS_715 chromosome, complete genome", "reads_in": 340654, "reads_removed_percent": 0.001174211957000358, "reads_retained_percent": 99.99882578804299}, {"sample": "GCA_020097195.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 0, "name": "Enterobacter hormaechei subsp. oharae strain FDAARGOS_1533 chromosome, complete genome", "reads_in": 315438, "reads_removed_percent": 0.0, "reads_retained_percent": 100.0}, {"sample": "GCA_019930845.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Eikenella corrodens strain FDAARGOS_1474 chromosome, complete genome", "reads_in": 146172, "reads_removed_percent": 0.002736502202884273, "reads_retained_percent": 99.99726349779712}, {"sample": "GCA_009729975.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Salmonella enterica strain FDAARGOS_687 chromosome, complete genome", "reads_in": 337762, "reads_removed_percent": 0.0011842658439966604, "reads_retained_percent": 99.998815734156}, {"sample": "GCA_016889845.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Escherichia coli strain FDAARGOS_1248 chromosome, complete genome", "reads_in": 337768, "reads_removed_percent": 0.001184244807086521, "reads_retained_percent": 99.99881575519291}, {"sample": "GCA_002587885.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Klebsiella pneumoniae strain FDAARGOS_446 chromosome, complete genome", "reads_in": 384644, "reads_removed_percent": 0.0010399226297563462, "reads_retained_percent": 99.99896007737024}, {"sample": "GCA_016889725.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 4, "name": "Escherichia coli strain FDAARGOS_1252 chromosome, complete genome", "reads_in": 362544, "reads_removed_percent": 0.0011033143563264046, "reads_retained_percent": 99.99889668564367}, {"sample": "GCA_002208725.2", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 8, "name": "Prevotella melaninogenica strain FDAARGOS_306 chromosome 1, complete sequence", "reads_in": 211214, "reads_removed_percent": 0.003787627714071984, "reads_retained_percent": 99.99621237228592}, {"sample": "GCA_016126975.1", "platform": "illumina", "tool": "kraken2-standard8", "reads_removed": 10, "name": "Pediococcus pentosaceus strain FDAARGOS_1011 chromosome, complete genome", "reads_in": 119050, "reads_removed_percent": 0.008399832003359933, "reads_retained_percent": 99.99160016799664}], "data-a50967b9c09b98afdf78ec5230ac7f78": [{"sample": "ERR3239334", "tool": "hostile", "reads_removed": 753275248, "reads_removed_percent": 99.35895919694029}, {"sample": "ERR3239480", "tool": "hostile", "reads_removed": 750367906, "reads_removed_percent": 99.61771632845944}, {"sample": "ERR3239491", "tool": "hostile", "reads_removed": 715015946, "reads_removed_percent": 99.29340307398299}, {"sample": "ERR3239492", "tool": "hostile", "reads_removed": 756695636, "reads_removed_percent": 99.58481276811477}, {"sample": "ERR3239557", "tool": "hostile", "reads_removed": 718214618, "reads_removed_percent": 99.40901230129239}, {"sample": "ERR3239683", "tool": "hostile", "reads_removed": 828823318, "reads_removed_percent": 99.2999457330521}, {"sample": "ERR3239832", "tool": "hostile", "reads_removed": 725022950, "reads_removed_percent": 99.4718672033503}, {"sample": "ERR3239893", "tool": "hostile", "reads_removed": 751943056, "reads_removed_percent": 99.58491084290311}, {"sample": "ERR3239894", "tool": "hostile", "reads_removed": 750267990, "reads_removed_percent": 99.26119530485707}, {"sample": "ERR3239997", "tool": "hostile", "reads_removed": 738545428, "reads_removed_percent": 99.42346981035406}, {"sample": "ERR3240184", "tool": "hostile", "reads_removed": 808831578, "reads_removed_percent": 99.5803514421767}, {"sample": "ERR3240193", "tool": "hostile", "reads_removed": 797905018, "reads_removed_percent": 99.57971434672152}, {"sample": "ERR3240223", "tool": "hostile", "reads_removed": 729430480, "reads_removed_percent": 99.54259699806484}, {"sample": "ERR3241843", "tool": "hostile", "reads_removed": 771429750, "reads_removed_percent": 99.38639935688529}, {"sample": "ERR3241862", "tool": "hostile", "reads_removed": 831344906, "reads_removed_percent": 99.62462133004841}, {"sample": "ERR3242189", "tool": "hostile", "reads_removed": 744478528, "reads_removed_percent": 99.48000801186588}, {"sample": "ERR3242199", "tool": "hostile", "reads_removed": 720906398, "reads_removed_percent": 99.40287462344102}, {"sample": "ERR3242202", "tool": "hostile", "reads_removed": 732772932, "reads_removed_percent": 98.22971345827456}, {"sample": "ERR3242208", "tool": "hostile", "reads_removed": 738834728, "reads_removed_percent": 99.55675529730837}, {"sample": "ERR3242214", "tool": "hostile", "reads_removed": 1078862336, "reads_removed_percent": 99.37075530032767}, {"sample": "ERR3242540", "tool": "hostile", "reads_removed": 763584738, "reads_removed_percent": 99.39601654665907}, {"sample": "ERR3242585", "tool": "hostile", "reads_removed": 729406790, "reads_removed_percent": 99.30291175699753}, {"sample": "ERR3242627", "tool": "hostile", "reads_removed": 712853482, "reads_removed_percent": 99.29220886724278}, {"sample": "ERR3242642", "tool": "hostile", "reads_removed": 866559714, "reads_removed_percent": 99.36878189660001}, {"sample": "ERR3242892", "tool": "hostile", "reads_removed": 741249772, "reads_removed_percent": 99.48209357866905}, {"sample": "ERR3242910", "tool": "hostile", "reads_removed": 681965582, "reads_removed_percent": 99.51423802650726}, {"sample": "ERR3242940", "tool": "hostile", "reads_removed": 710539070, "reads_removed_percent": 99.53998831569862}, {"sample": "ERR3239334", "tool": "ncbi-scrubber", "reads_removed": 752913218, "reads_removed_percent": 99.31120650083939}, {"sample": "ERR3239480", "tool": "ncbi-scrubber", "reads_removed": 749867868, "reads_removed_percent": 99.55133203451624}, {"sample": "ERR3239491", "tool": "ncbi-scrubber", "reads_removed": 714840862, "reads_removed_percent": 99.26908936981869}, {"sample": "ERR3239492", "tool": "ncbi-scrubber", "reads_removed": 755098288, "reads_removed_percent": 99.37459403030574}, {"sample": "ERR3239557", "tool": "ncbi-scrubber", "reads_removed": 717874496, "reads_removed_percent": 99.36193557626541}, {"sample": "ERR3239683", "tool": "ncbi-scrubber", "reads_removed": 828167844, "reads_removed_percent": 99.22141448131741}, {"sample": "ERR3239832", "tool": "ncbi-scrubber", "reads_removed": 723632498, "reads_removed_percent": 99.28109964668657}, {"sample": "ERR3239893", "tool": "ncbi-scrubber", "reads_removed": 751524978, "reads_removed_percent": 99.52954194226207}, {"sample": "ERR3239894", "tool": "ncbi-scrubber", "reads_removed": 749860020, "reads_removed_percent": 99.20722047134656}, {"sample": "ERR3239997", "tool": "ncbi-scrubber", "reads_removed": 736504034, "reads_removed_percent": 99.14865601145253}, {"sample": "ERR3240184", "tool": "ncbi-scrubber", "reads_removed": 808470550, "reads_removed_percent": 99.53590301051513}, {"sample": "ERR3240193", "tool": "ncbi-scrubber", "reads_removed": 797389476, "reads_removed_percent": 99.5153739503891}, {"sample": "ERR3240223", "tool": "ncbi-scrubber", "reads_removed": 729262050, "reads_removed_percent": 99.51961199802427}, {"sample": "ERR3241843", "tool": "ncbi-scrubber", "reads_removed": 771187220, "reads_removed_percent": 99.35515324090905}, {"sample": "ERR3241862", "tool": "ncbi-scrubber", "reads_removed": 829261180, "reads_removed_percent": 99.37491701092965}, {"sample": "ERR3242189", "tool": "ncbi-scrubber", "reads_removed": 742915118, "reads_removed_percent": 99.27109931473575}, {"sample": "ERR3242199", "tool": "ncbi-scrubber", "reads_removed": 719716518, "reads_removed_percent": 99.23880687097679}, {"sample": "ERR3242202", "tool": "ncbi-scrubber", "reads_removed": 730659098, "reads_removed_percent": 97.94634967796729}, {"sample": "ERR3242208", "tool": "ncbi-scrubber", "reads_removed": 738662882, "reads_removed_percent": 99.53359933356919}, {"sample": "ERR3242214", "tool": "ncbi-scrubber", "reads_removed": 1076439360, "reads_removed_percent": 99.14758229005534}, {"sample": "ERR3242540", "tool": "ncbi-scrubber", "reads_removed": 763161104, "reads_removed_percent": 99.34087200280135}, {"sample": "ERR3242585", "tool": "ncbi-scrubber", "reads_removed": 727841232, "reads_removed_percent": 99.08977353281885}, {"sample": "ERR3242627", "tool": "ncbi-scrubber", "reads_removed": 709949698, "reads_removed_percent": 98.88774548912413}, {"sample": "ERR3242642", "tool": "ncbi-scrubber", "reads_removed": 863554388, "reads_removed_percent": 99.02415984805855}, {"sample": "ERR3242892", "tool": "ncbi-scrubber", "reads_removed": 740212336, "reads_removed_percent": 99.34286074631969}, {"sample": "ERR3242910", "tool": "ncbi-scrubber", "reads_removed": 681832694, "reads_removed_percent": 99.4948466548429}, {"sample": "ERR3242940", "tool": "ncbi-scrubber", "reads_removed": 710141826, "reads_removed_percent": 99.48433808506671}, {"sample": "ERR3242214", "tool": "kraken2-standard", "reads_removed": 1082047876, "reads_removed_percent": 99.66416578030889}, {"sample": "ERR3242214", "tool": "kraken2-standard8", "reads_removed": 1076340012, "reads_removed_percent": 99.13843164546599}, {"sample": "ERR3242642", "tool": "kraken2-standard", "reads_removed": 869388574, "reads_removed_percent": 99.69316851164177}, {"sample": "ERR3242642", "tool": "kraken2-standard8", "reads_removed": 862160180, "reads_removed_percent": 98.86428540613349}, {"sample": "ERR3241862", "tool": "kraken2-standard", "reads_removed": 833937496, "reads_removed_percent": 99.9353056142125}, {"sample": "ERR3239683", "tool": "kraken2-standard", "reads_removed": 832323188, "reads_removed_percent": 99.71925934733525}, {"sample": "ERR3241862", "tool": "kraken2-standard8", "reads_removed": 828964890, "reads_removed_percent": 99.33941095460953}, {"sample": "ERR3239683", "tool": "kraken2-standard8", "reads_removed": 825235910, "reads_removed_percent": 98.87014433631785}, {"sample": "ERR3240184", "tool": "kraken2-standard", "reads_removed": 811439068, "reads_removed_percent": 99.90137596402342}, {"sample": "ERR3240184", "tool": "kraken2-standard8", "reads_removed": 806218974, "reads_removed_percent": 99.25869730357032}, {"sample": "ERR3240193", "tool": "kraken2-standard", "reads_removed": 800529652, "reads_removed_percent": 99.90727251227838}, {"sample": "ERR3240193", "tool": "kraken2-standard8", "reads_removed": 795085496, "reads_removed_percent": 99.22783387345659}, {"sample": "ERR3241843", "tool": "kraken2-standard", "reads_removed": 774286860, "reads_removed_percent": 99.75449233679247}, {"sample": "ERR3241843", "tool": "kraken2-standard8", "reads_removed": 768695818, "reads_removed_percent": 99.03417589445522}, {"sample": "ERR3242540", "tool": "kraken2-standard", "reads_removed": 766479156, "reads_removed_percent": 99.77278366247975}, {"sample": "ERR3242540", "tool": "kraken2-standard8", "reads_removed": 761080866, "reads_removed_percent": 99.07008690145089}, {"sample": "ERR3239492", "tool": "kraken2-standard", "reads_removed": 759274428, "reads_removed_percent": 99.92419429256157}, {"sample": "ERR3239334", "tool": "kraken2-standard", "reads_removed": 756039462, "reads_removed_percent": 99.72356619387378}, {"sample": "ERR3239893", "tool": "kraken2-standard", "reads_removed": 754393574, "reads_removed_percent": 99.90944953582897}, {"sample": "ERR3239492", "tool": "kraken2-standard8", "reads_removed": 753922758, "reads_removed_percent": 99.21988858549555}, {"sample": "ERR3239480", "tool": "kraken2-standard", "reads_removed": 752630350, "reads_removed_percent": 99.91807499625276}, {"sample": "ERR3239894", "tool": "kraken2-standard", "reads_removed": 752577886, "reads_removed_percent": 99.5667968246419}, {"sample": "ERR3239334", "tool": "kraken2-standard8", "reads_removed": 750917672, "reads_removed_percent": 99.04798880702023}, {"sample": "ERR3239893", "tool": "kraken2-standard8", "reads_removed": 749373382, "reads_removed_percent": 99.24459151400791}, {"sample": "ERR3239480", "tool": "kraken2-standard8", "reads_removed": 748311960, "reads_removed_percent": 99.3447720250358}, {"sample": "ERR3239894", "tool": "kraken2-standard8", "reads_removed": 747756762, "reads_removed_percent": 98.92895736283448}, {"sample": "ERR3242189", "tool": "kraken2-standard", "reads_removed": 746877990, "reads_removed_percent": 99.80063310715964}, {"sample": "ERR3242892", "tool": "kraken2-standard", "reads_removed": 744023788, "reads_removed_percent": 99.85439038026689}, {"sample": "ERR3242189", "tool": "kraken2-standard8", "reads_removed": 741958056, "reads_removed_percent": 99.14321310734758}, {"sample": "ERR3242208", "tool": "kraken2-standard", "reads_removed": 741285228, "reads_removed_percent": 99.8869561116589}, {"sample": "ERR3239997", "tool": "kraken2-standard", "reads_removed": 741241918, "reads_removed_percent": 99.78647306234755}, {"sample": "ERR3242892", "tool": "kraken2-standard8", "reads_removed": 738806996, "reads_removed_percent": 99.15425203347971}, {"sample": "ERR3242208", "tool": "kraken2-standard8", "reads_removed": 736677502, "reads_removed_percent": 99.26607266848237}, {"sample": "ERR3242202", "tool": "kraken2-standard", "reads_removed": 735731372, "reads_removed_percent": 98.62629840402349}, {"sample": "ERR3239997", "tool": "kraken2-standard8", "reads_removed": 735656680, "reads_removed_percent": 99.03458466032953}, {"sample": "ERR3242585", "tool": "kraken2-standard", "reads_removed": 732530354, "reads_removed_percent": 99.72816006084089}, {"sample": "ERR3240223", "tool": "kraken2-standard", "reads_removed": 731732706, "reads_removed_percent": 99.85677300414079}, {"sample": "ERR3242202", "tool": "kraken2-standard8", "reads_removed": 729871834, "reads_removed_percent": 97.84081532515633}, {"sample": "ERR3239832", "tool": "kraken2-standard", "reads_removed": 727602708, "reads_removed_percent": 99.82580544101958}, {"sample": "ERR3240223", "tool": "kraken2-standard8", "reads_removed": 727554816, "reads_removed_percent": 99.28663228206369}, {"sample": "ERR3242585", "tool": "kraken2-standard8", "reads_removed": 727132240, "reads_removed_percent": 98.99324993175337}, {"sample": "ERR3242199", "tool": "kraken2-standard", "reads_removed": 723553724, "reads_removed_percent": 99.76790372457735}, {"sample": "ERR3239832", "tool": "kraken2-standard8", "reads_removed": 722617232, "reads_removed_percent": 99.14180694605126}, {"sample": "ERR3239557", "tool": "kraken2-standard", "reads_removed": 720658608, "reads_removed_percent": 99.74728811173298}, {"sample": "ERR3242199", "tool": "kraken2-standard8", "reads_removed": 718778822, "reads_removed_percent": 99.10951175280125}, {"sample": "ERR3239491", "tool": "kraken2-standard", "reads_removed": 718073476, "reads_removed_percent": 99.71799858181632}, {"sample": "ERR3242627", "tool": "kraken2-standard", "reads_removed": 715952686, "reads_removed_percent": 99.72389198117921}, {"sample": "ERR3239557", "tool": "kraken2-standard8", "reads_removed": 715590212, "reads_removed_percent": 99.04576487942276}, {"sample": "ERR3242940", "tool": "kraken2-standard", "reads_removed": 713108808, "reads_removed_percent": 99.89998497358037}, {"sample": "ERR3239491", "tool": "kraken2-standard8", "reads_removed": 712428202, "reads_removed_percent": 98.93404618203994}, {"sample": "ERR3242940", "tool": "kraken2-standard8", "reads_removed": 708422512, "reads_removed_percent": 99.24347800756102}, {"sample": "ERR3242627", "tool": "kraken2-standard8", "reads_removed": 707824650, "reads_removed_percent": 98.59175098927693}, {"sample": "ERR3242910", "tool": "kraken2-standard", "reads_removed": 684246300, "reads_removed_percent": 99.84704648475486}, {"sample": "ERR3242910", "tool": "kraken2-standard8", "reads_removed": 680223564, "reads_removed_percent": 99.26003811015656}]}};
      var embedOpt = {"mode": "vega-lite"};

      function showError(el, error){
          el.innerHTML = ('<div style="color:red;">'
                          + '<p>JavaScript Error: ' + error.message + '</p>'
                          + "<p>This usually means there's a typo in your chart specification. "
                          + "See the javascript console for the full traceback.</p>"
                          + '</div>');
          throw error;
      }
      const el = document.getElementById('vis');
      vegaEmbed("#vis", spec, embedOpt)
        .catch(error => showError(el, error));
    })(vegaEmbed);

  </script>
